{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "03c0be05-b884-4263-8332-21671f9db5c4",
   "metadata": {
    "jp-MarkdownHeadingCollapsed": true,
    "tags": []
   },
   "source": [
    "## Set up\n",
    "First you need to install GraphDB locally on you machine, create a repo where you load the needed data\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "abf0b4b2-9f5e-411f-8723-2b55acbf9d0b",
   "metadata": {},
   "outputs": [],
   "source": [
    "from rdflib import ConjunctiveGraph\n",
    "from SPARQLWrapper import SPARQLWrapper, JSON\n",
    "\n",
    "import pandas as pd\n",
    "\n",
    "def remoteQuery(query, endpoint):\n",
    "    endpoint.setQuery(query)\n",
    "    try:\n",
    "        result = endpoint.queryAndConvert()\n",
    "        pd.set_option(\"display.max_rows\",None,\"display.max_colwidth\",5000,\"display.width\",5000,)\n",
    "        df = pd.DataFrame(result['results']['bindings'])\n",
    "        df = df.applymap(lambda x: x['value'])\n",
    "        return df\n",
    "        #return (result['results']['bindings'])\n",
    "    except Exception as e:\n",
    "        print(e)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c3d8bf47-d210-41d7-841f-67fc42549799",
   "metadata": {},
   "source": [
    "\n",
    "### With a GraphDB SPARQL endpoint\n",
    "\n",
    "ep_no_inference points to a SPARQL end point repository in Graphdb where the automatic inferences were deactivated\n",
    "\n",
    "The input for SPARQLWrapper is the link to the GraphDB repository where you loaded the appropriate data set, in this example the dev version of EDAM (https://raw.githubusercontent.com/edamontology/edamontology/main/EDAM_dev.owl) was loaded in the GrapphDB repository. \n",
    "\n",
    "You can find the url of you repository in GraphDB by going into the \"Setup\" tab on the left menu, then click on \"Repositories\". You can get the link by clicking on the \"link logo\"."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "30303cd1-605a-4b2b-89ec-56ce70435175",
   "metadata": {},
   "outputs": [],
   "source": [
    "ep_edam = SPARQLWrapper(\"http://llamothe-HP-EliteBook-x360-1040-G8-Notebook-PC:7200/repositories/EDAM\")\n",
    "#ep_edam = SPARQLWrapper(\"http://link/to/your/local/graphdb/repo\") \n",
    "ep_edam.setReturnFormat(JSON)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "55e5449c-55a4-4326-a31d-1ea20b21d1e5",
   "metadata": {},
   "outputs": [],
   "source": [
    "ep_no_inference = SPARQLWrapper(\"http://llamothe-HP-EliteBook-x360-1040-G8-Notebook-PC:7200/repositories/EDAM_no_inference\")\n",
    "ep_no_inference.setReturnFormat(JSON)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "718c58c4-0a0e-4d89-9fdd-abfb136653a4",
   "metadata": {
    "jp-MarkdownHeadingCollapsed": true,
    "tags": []
   },
   "source": [
    "## query to get format without the *is_format_of* property"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "e48c4ca9-664a-4a71-905d-f8e65514b360",
   "metadata": {},
   "outputs": [],
   "source": [
    "q = \"\"\"\n",
    "PREFIX obo: <http://purl.obolibrary.org/obo/>\n",
    "PREFIX owl: <http://www.w3.org/2002/07/owl#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>\n",
    "PREFIX edam:<http://edamontology.org/>\n",
    "PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>\n",
    "\n",
    "SELECT ?entity ?label ?property WHERE\n",
    "{\n",
    "  \n",
    "    ?entity rdfs:subClassOf+ edam:format_1915 .\n",
    "    ?entity rdfs:label ?label .\n",
    "\n",
    "     VALUES ?property { edam:is_format_of               \n",
    "                        }\n",
    "    FILTER NOT EXISTS {    \n",
    "        ?entity rdfs:subClassOf ?restriction . \n",
    "        ?restriction rdf:type owl:Restriction ; \n",
    "                owl:onProperty ?property  ; \n",
    "                owl:someValuesFrom ?data.}\n",
    "\n",
    "}ORDER BY ?entity\n",
    "    \n",
    "    \n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5f644dc5-70e8-4fe9-821f-cb44768d7a82",
   "metadata": {},
   "source": [
    "- With inferences:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "40831bf4-8832-4ea2-9ea1-6a5e91b52dc4",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "nb formats missing is_format_of property: 85\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>entity</th>\n",
       "      <th>label</th>\n",
       "      <th>property</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/format_1295</td>\n",
       "      <td>quicktandem</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>http://edamontology.org/format_1296</td>\n",
       "      <td>Sanger inverted repeats</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>http://edamontology.org/format_1297</td>\n",
       "      <td>EMBOSS repeat</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>http://edamontology.org/format_1318</td>\n",
       "      <td>restrict format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>http://edamontology.org/format_1319</td>\n",
       "      <td>restover format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>http://edamontology.org/format_1320</td>\n",
       "      <td>REBASE restriction sites</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>http://edamontology.org/format_1454</td>\n",
       "      <td>dssp</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>http://edamontology.org/format_1455</td>\n",
       "      <td>hssp</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>http://edamontology.org/format_1627</td>\n",
       "      <td>Primer3 primer</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>http://edamontology.org/format_1665</td>\n",
       "      <td>Taverna workflow format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>http://edamontology.org/format_1948</td>\n",
       "      <td>nbrf/pir</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>http://edamontology.org/format_1964</td>\n",
       "      <td>plain text format (unformatted)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>http://edamontology.org/format_2032</td>\n",
       "      <td>Workflow format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>http://edamontology.org/format_2033</td>\n",
       "      <td>Tertiary structure format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>http://edamontology.org/format_2061</td>\n",
       "      <td>Nucleic acid features (primers) format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>http://edamontology.org/format_2077</td>\n",
       "      <td>Protein secondary structure format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>http://edamontology.org/format_2155</td>\n",
       "      <td>Sequence features (repeats) format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>http://edamontology.org/format_2158</td>\n",
       "      <td>Nucleic acid features (restriction sites) format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>http://edamontology.org/format_2330</td>\n",
       "      <td>Textual format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>http://edamontology.org/format_2332</td>\n",
       "      <td>XML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>http://edamontology.org/format_2333</td>\n",
       "      <td>Binary format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>http://edamontology.org/format_2350</td>\n",
       "      <td>Format (by type of data)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>http://edamontology.org/format_3285</td>\n",
       "      <td>MAP</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>http://edamontology.org/format_3286</td>\n",
       "      <td>PED</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>http://edamontology.org/format_3287</td>\n",
       "      <td>Individual genetic data format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>http://edamontology.org/format_3288</td>\n",
       "      <td>PED/MAP</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>http://edamontology.org/format_3464</td>\n",
       "      <td>JSON</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>http://edamontology.org/format_3466</td>\n",
       "      <td>EPS</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>http://edamontology.org/format_3467</td>\n",
       "      <td>GIF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>http://edamontology.org/format_3468</td>\n",
       "      <td>xls</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>http://edamontology.org/format_3475</td>\n",
       "      <td>TSV</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>http://edamontology.org/format_3506</td>\n",
       "      <td>docx</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>http://edamontology.org/format_3507</td>\n",
       "      <td>Document format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>http://edamontology.org/format_3508</td>\n",
       "      <td>PDF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>http://edamontology.org/format_3554</td>\n",
       "      <td>R file format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>http://edamontology.org/format_3555</td>\n",
       "      <td>SPSS</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>http://edamontology.org/format_3581</td>\n",
       "      <td>arff</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>http://edamontology.org/format_3617</td>\n",
       "      <td>Graph format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>http://edamontology.org/format_3618</td>\n",
       "      <td>xgmml</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>http://edamontology.org/format_3620</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>http://edamontology.org/format_3621</td>\n",
       "      <td>SQLite format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>http://edamontology.org/format_3665</td>\n",
       "      <td>K-mer countgraph</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>http://edamontology.org/format_3690</td>\n",
       "      <td>BDML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>http://edamontology.org/format_3691</td>\n",
       "      <td>BEL</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>http://edamontology.org/format_3696</td>\n",
       "      <td>PS</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>http://edamontology.org/format_3698</td>\n",
       "      <td>SRA format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>http://edamontology.org/format_3699</td>\n",
       "      <td>VDB</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>http://edamontology.org/format_3725</td>\n",
       "      <td>SBOL</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>http://edamontology.org/format_3726</td>\n",
       "      <td>PMML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>http://edamontology.org/format_3728</td>\n",
       "      <td>LocARNA PP</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>http://edamontology.org/format_3729</td>\n",
       "      <td>dbGaP format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51</th>\n",
       "      <td>http://edamontology.org/format_3748</td>\n",
       "      <td>Linked data format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>http://edamontology.org/format_3750</td>\n",
       "      <td>YAML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>53</th>\n",
       "      <td>http://edamontology.org/format_3751</td>\n",
       "      <td>DSV</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>http://edamontology.org/format_3752</td>\n",
       "      <td>CSV</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>55</th>\n",
       "      <td>http://edamontology.org/format_3765</td>\n",
       "      <td>KNIME datatable format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>56</th>\n",
       "      <td>http://edamontology.org/format_3788</td>\n",
       "      <td>SQL</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>57</th>\n",
       "      <td>http://edamontology.org/format_3789</td>\n",
       "      <td>XQuery</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>58</th>\n",
       "      <td>http://edamontology.org/format_3790</td>\n",
       "      <td>SPARQL</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>59</th>\n",
       "      <td>http://edamontology.org/format_3804</td>\n",
       "      <td>xsd</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>60</th>\n",
       "      <td>http://edamontology.org/format_3813</td>\n",
       "      <td>SAMPLE file format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>61</th>\n",
       "      <td>http://edamontology.org/format_3817</td>\n",
       "      <td>latex</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>62</th>\n",
       "      <td>http://edamontology.org/format_3841</td>\n",
       "      <td>NLP format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>63</th>\n",
       "      <td>http://edamontology.org/format_3857</td>\n",
       "      <td>CWL</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>64</th>\n",
       "      <td>http://edamontology.org/format_3862</td>\n",
       "      <td>NLP annotation format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>65</th>\n",
       "      <td>http://edamontology.org/format_3863</td>\n",
       "      <td>NLP corpus format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66</th>\n",
       "      <td>http://edamontology.org/format_3865</td>\n",
       "      <td>RNA annotation format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>67</th>\n",
       "      <td>http://edamontology.org/format_3973</td>\n",
       "      <td>Docker image</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>68</th>\n",
       "      <td>http://edamontology.org/format_3977</td>\n",
       "      <td>ObjTables</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69</th>\n",
       "      <td>http://edamontology.org/format_3981</td>\n",
       "      <td>TAR format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>70</th>\n",
       "      <td>http://edamontology.org/format_3985</td>\n",
       "      <td>gxformat2</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71</th>\n",
       "      <td>http://edamontology.org/format_3987</td>\n",
       "      <td>ZIP format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72</th>\n",
       "      <td>http://edamontology.org/format_3989</td>\n",
       "      <td>GZIP format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>73</th>\n",
       "      <td>http://edamontology.org/format_3996</td>\n",
       "      <td>Python script</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>74</th>\n",
       "      <td>http://edamontology.org/format_3998</td>\n",
       "      <td>Perl script</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75</th>\n",
       "      <td>http://edamontology.org/format_3999</td>\n",
       "      <td>R script</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76</th>\n",
       "      <td>http://edamontology.org/format_4000</td>\n",
       "      <td>R markdown</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>77</th>\n",
       "      <td>http://edamontology.org/format_4002</td>\n",
       "      <td>pickle</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78</th>\n",
       "      <td>http://edamontology.org/format_4003</td>\n",
       "      <td>NumPy format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>79</th>\n",
       "      <td>http://edamontology.org/format_4004</td>\n",
       "      <td>SimTools repertoire file format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80</th>\n",
       "      <td>http://edamontology.org/format_4005</td>\n",
       "      <td>Configuration file format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>81</th>\n",
       "      <td>http://edamontology.org/format_4006</td>\n",
       "      <td>Zstandard format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>82</th>\n",
       "      <td>http://edamontology.org/format_4007</td>\n",
       "      <td>MATLAB script</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>83</th>\n",
       "      <td>http://edamontology.org/format_4025</td>\n",
       "      <td>BioSimulators format for the specifications of biosimulation tools</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>84</th>\n",
       "      <td>http://edamontology.org/format_4026</td>\n",
       "      <td>BioSimulators standard for command-line interfaces for biosimulation tools</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                 entity                                                                       label                              property\n",
       "0   http://edamontology.org/format_1295                                                                 quicktandem  http://edamontology.org/is_format_of\n",
       "1   http://edamontology.org/format_1296                                                     Sanger inverted repeats  http://edamontology.org/is_format_of\n",
       "2   http://edamontology.org/format_1297                                                               EMBOSS repeat  http://edamontology.org/is_format_of\n",
       "3   http://edamontology.org/format_1318                                                             restrict format  http://edamontology.org/is_format_of\n",
       "4   http://edamontology.org/format_1319                                                             restover format  http://edamontology.org/is_format_of\n",
       "5   http://edamontology.org/format_1320                                                    REBASE restriction sites  http://edamontology.org/is_format_of\n",
       "6   http://edamontology.org/format_1454                                                                        dssp  http://edamontology.org/is_format_of\n",
       "7   http://edamontology.org/format_1455                                                                        hssp  http://edamontology.org/is_format_of\n",
       "8   http://edamontology.org/format_1627                                                              Primer3 primer  http://edamontology.org/is_format_of\n",
       "9   http://edamontology.org/format_1665                                                     Taverna workflow format  http://edamontology.org/is_format_of\n",
       "10  http://edamontology.org/format_1948                                                                    nbrf/pir  http://edamontology.org/is_format_of\n",
       "11  http://edamontology.org/format_1964                                             plain text format (unformatted)  http://edamontology.org/is_format_of\n",
       "12  http://edamontology.org/format_2032                                                             Workflow format  http://edamontology.org/is_format_of\n",
       "13  http://edamontology.org/format_2033                                                   Tertiary structure format  http://edamontology.org/is_format_of\n",
       "14  http://edamontology.org/format_2061                                      Nucleic acid features (primers) format  http://edamontology.org/is_format_of\n",
       "15  http://edamontology.org/format_2077                                          Protein secondary structure format  http://edamontology.org/is_format_of\n",
       "16  http://edamontology.org/format_2155                                          Sequence features (repeats) format  http://edamontology.org/is_format_of\n",
       "17  http://edamontology.org/format_2158                            Nucleic acid features (restriction sites) format  http://edamontology.org/is_format_of\n",
       "18  http://edamontology.org/format_2330                                                              Textual format  http://edamontology.org/is_format_of\n",
       "19  http://edamontology.org/format_2332                                                                         XML  http://edamontology.org/is_format_of\n",
       "20  http://edamontology.org/format_2333                                                               Binary format  http://edamontology.org/is_format_of\n",
       "21  http://edamontology.org/format_2350                                                    Format (by type of data)  http://edamontology.org/is_format_of\n",
       "22  http://edamontology.org/format_3285                                                                         MAP  http://edamontology.org/is_format_of\n",
       "23  http://edamontology.org/format_3286                                                                         PED  http://edamontology.org/is_format_of\n",
       "24  http://edamontology.org/format_3287                                              Individual genetic data format  http://edamontology.org/is_format_of\n",
       "25  http://edamontology.org/format_3288                                                                     PED/MAP  http://edamontology.org/is_format_of\n",
       "26  http://edamontology.org/format_3464                                                                        JSON  http://edamontology.org/is_format_of\n",
       "27  http://edamontology.org/format_3466                                                                         EPS  http://edamontology.org/is_format_of\n",
       "28  http://edamontology.org/format_3467                                                                         GIF  http://edamontology.org/is_format_of\n",
       "29  http://edamontology.org/format_3468                                                                         xls  http://edamontology.org/is_format_of\n",
       "30  http://edamontology.org/format_3475                                                                         TSV  http://edamontology.org/is_format_of\n",
       "31  http://edamontology.org/format_3506                                                                        docx  http://edamontology.org/is_format_of\n",
       "32  http://edamontology.org/format_3507                                                             Document format  http://edamontology.org/is_format_of\n",
       "33  http://edamontology.org/format_3508                                                                         PDF  http://edamontology.org/is_format_of\n",
       "34  http://edamontology.org/format_3554                                                               R file format  http://edamontology.org/is_format_of\n",
       "35  http://edamontology.org/format_3555                                                                        SPSS  http://edamontology.org/is_format_of\n",
       "36  http://edamontology.org/format_3581                                                                        arff  http://edamontology.org/is_format_of\n",
       "37  http://edamontology.org/format_3617                                                                Graph format  http://edamontology.org/is_format_of\n",
       "38  http://edamontology.org/format_3618                                                                       xgmml  http://edamontology.org/is_format_of\n",
       "39  http://edamontology.org/format_3620                                                                        xlsx  http://edamontology.org/is_format_of\n",
       "40  http://edamontology.org/format_3621                                                               SQLite format  http://edamontology.org/is_format_of\n",
       "41  http://edamontology.org/format_3665                                                            K-mer countgraph  http://edamontology.org/is_format_of\n",
       "42  http://edamontology.org/format_3690                                                                        BDML  http://edamontology.org/is_format_of\n",
       "43  http://edamontology.org/format_3691                                                                         BEL  http://edamontology.org/is_format_of\n",
       "44  http://edamontology.org/format_3696                                                                          PS  http://edamontology.org/is_format_of\n",
       "45  http://edamontology.org/format_3698                                                                  SRA format  http://edamontology.org/is_format_of\n",
       "46  http://edamontology.org/format_3699                                                                         VDB  http://edamontology.org/is_format_of\n",
       "47  http://edamontology.org/format_3725                                                                        SBOL  http://edamontology.org/is_format_of\n",
       "48  http://edamontology.org/format_3726                                                                        PMML  http://edamontology.org/is_format_of\n",
       "49  http://edamontology.org/format_3728                                                                  LocARNA PP  http://edamontology.org/is_format_of\n",
       "50  http://edamontology.org/format_3729                                                                dbGaP format  http://edamontology.org/is_format_of\n",
       "51  http://edamontology.org/format_3748                                                          Linked data format  http://edamontology.org/is_format_of\n",
       "52  http://edamontology.org/format_3750                                                                        YAML  http://edamontology.org/is_format_of\n",
       "53  http://edamontology.org/format_3751                                                                         DSV  http://edamontology.org/is_format_of\n",
       "54  http://edamontology.org/format_3752                                                                         CSV  http://edamontology.org/is_format_of\n",
       "55  http://edamontology.org/format_3765                                                      KNIME datatable format  http://edamontology.org/is_format_of\n",
       "56  http://edamontology.org/format_3788                                                                         SQL  http://edamontology.org/is_format_of\n",
       "57  http://edamontology.org/format_3789                                                                      XQuery  http://edamontology.org/is_format_of\n",
       "58  http://edamontology.org/format_3790                                                                      SPARQL  http://edamontology.org/is_format_of\n",
       "59  http://edamontology.org/format_3804                                                                         xsd  http://edamontology.org/is_format_of\n",
       "60  http://edamontology.org/format_3813                                                          SAMPLE file format  http://edamontology.org/is_format_of\n",
       "61  http://edamontology.org/format_3817                                                                       latex  http://edamontology.org/is_format_of\n",
       "62  http://edamontology.org/format_3841                                                                  NLP format  http://edamontology.org/is_format_of\n",
       "63  http://edamontology.org/format_3857                                                                         CWL  http://edamontology.org/is_format_of\n",
       "64  http://edamontology.org/format_3862                                                       NLP annotation format  http://edamontology.org/is_format_of\n",
       "65  http://edamontology.org/format_3863                                                           NLP corpus format  http://edamontology.org/is_format_of\n",
       "66  http://edamontology.org/format_3865                                                       RNA annotation format  http://edamontology.org/is_format_of\n",
       "67  http://edamontology.org/format_3973                                                                Docker image  http://edamontology.org/is_format_of\n",
       "68  http://edamontology.org/format_3977                                                                   ObjTables  http://edamontology.org/is_format_of\n",
       "69  http://edamontology.org/format_3981                                                                  TAR format  http://edamontology.org/is_format_of\n",
       "70  http://edamontology.org/format_3985                                                                   gxformat2  http://edamontology.org/is_format_of\n",
       "71  http://edamontology.org/format_3987                                                                  ZIP format  http://edamontology.org/is_format_of\n",
       "72  http://edamontology.org/format_3989                                                                 GZIP format  http://edamontology.org/is_format_of\n",
       "73  http://edamontology.org/format_3996                                                               Python script  http://edamontology.org/is_format_of\n",
       "74  http://edamontology.org/format_3998                                                                 Perl script  http://edamontology.org/is_format_of\n",
       "75  http://edamontology.org/format_3999                                                                    R script  http://edamontology.org/is_format_of\n",
       "76  http://edamontology.org/format_4000                                                                  R markdown  http://edamontology.org/is_format_of\n",
       "77  http://edamontology.org/format_4002                                                                      pickle  http://edamontology.org/is_format_of\n",
       "78  http://edamontology.org/format_4003                                                                NumPy format  http://edamontology.org/is_format_of\n",
       "79  http://edamontology.org/format_4004                                             SimTools repertoire file format  http://edamontology.org/is_format_of\n",
       "80  http://edamontology.org/format_4005                                                   Configuration file format  http://edamontology.org/is_format_of\n",
       "81  http://edamontology.org/format_4006                                                            Zstandard format  http://edamontology.org/is_format_of\n",
       "82  http://edamontology.org/format_4007                                                               MATLAB script  http://edamontology.org/is_format_of\n",
       "83  http://edamontology.org/format_4025          BioSimulators format for the specifications of biosimulation tools  http://edamontology.org/is_format_of\n",
       "84  http://edamontology.org/format_4026  BioSimulators standard for command-line interfaces for biosimulation tools  http://edamontology.org/is_format_of"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "print(f\"nb formats missing is_format_of property: {len(remoteQuery(query=q, endpoint=ep_edam))}\")\n",
    "remoteQuery(query=q, endpoint=ep_edam)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "dc17f968-b745-4359-88be-25a35e864729",
   "metadata": {},
   "source": [
    "- Without inferences:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "9ec3cf61-b232-4e4e-ac73-8d369e81edc9",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "nb formats missing is_format_of property: 528\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>entity</th>\n",
       "      <th>label</th>\n",
       "      <th>property</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/format_1196</td>\n",
       "      <td>SMILES</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>http://edamontology.org/format_1197</td>\n",
       "      <td>InChI</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>http://edamontology.org/format_1198</td>\n",
       "      <td>mf</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>http://edamontology.org/format_1199</td>\n",
       "      <td>InChIKey</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>http://edamontology.org/format_1200</td>\n",
       "      <td>smarts</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>http://edamontology.org/format_1206</td>\n",
       "      <td>unambiguous pure</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>http://edamontology.org/format_1207</td>\n",
       "      <td>nucleotide</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>http://edamontology.org/format_1208</td>\n",
       "      <td>protein</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>http://edamontology.org/format_1209</td>\n",
       "      <td>consensus</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>http://edamontology.org/format_1210</td>\n",
       "      <td>pure nucleotide</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>http://edamontology.org/format_1211</td>\n",
       "      <td>unambiguous pure nucleotide</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>http://edamontology.org/format_1212</td>\n",
       "      <td>dna</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>http://edamontology.org/format_1213</td>\n",
       "      <td>rna</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>http://edamontology.org/format_1214</td>\n",
       "      <td>unambiguous pure dna</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>http://edamontology.org/format_1215</td>\n",
       "      <td>pure dna</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>http://edamontology.org/format_1216</td>\n",
       "      <td>unambiguous pure rna sequence</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>http://edamontology.org/format_1217</td>\n",
       "      <td>pure rna</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>http://edamontology.org/format_1218</td>\n",
       "      <td>unambiguous pure protein</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>http://edamontology.org/format_1219</td>\n",
       "      <td>pure protein</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>http://edamontology.org/format_1248</td>\n",
       "      <td>EMBL feature location</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>http://edamontology.org/format_1295</td>\n",
       "      <td>quicktandem</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>http://edamontology.org/format_1296</td>\n",
       "      <td>Sanger inverted repeats</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>http://edamontology.org/format_1297</td>\n",
       "      <td>EMBOSS repeat</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>http://edamontology.org/format_1316</td>\n",
       "      <td>est2genome format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>http://edamontology.org/format_1318</td>\n",
       "      <td>restrict format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>http://edamontology.org/format_1319</td>\n",
       "      <td>restover format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>http://edamontology.org/format_1320</td>\n",
       "      <td>REBASE restriction sites</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>http://edamontology.org/format_1332</td>\n",
       "      <td>FASTA search results format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>http://edamontology.org/format_1333</td>\n",
       "      <td>BLAST results</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>http://edamontology.org/format_1334</td>\n",
       "      <td>mspcrunch</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>http://edamontology.org/format_1335</td>\n",
       "      <td>Smith-Waterman format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>http://edamontology.org/format_1336</td>\n",
       "      <td>dhf</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>http://edamontology.org/format_1337</td>\n",
       "      <td>lhf</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>http://edamontology.org/format_1341</td>\n",
       "      <td>InterPro hits format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>http://edamontology.org/format_1342</td>\n",
       "      <td>InterPro protein view report format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>http://edamontology.org/format_1343</td>\n",
       "      <td>InterPro match table format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>http://edamontology.org/format_1349</td>\n",
       "      <td>HMMER Dirichlet prior</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>http://edamontology.org/format_1350</td>\n",
       "      <td>MEME Dirichlet prior</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>http://edamontology.org/format_1351</td>\n",
       "      <td>HMMER emission and transition</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>http://edamontology.org/format_1356</td>\n",
       "      <td>prosite-pattern</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>http://edamontology.org/format_1357</td>\n",
       "      <td>EMBOSS sequence pattern</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>http://edamontology.org/format_1360</td>\n",
       "      <td>meme-motif</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>http://edamontology.org/format_1366</td>\n",
       "      <td>prosite-profile</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>http://edamontology.org/format_1367</td>\n",
       "      <td>JASPAR format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>http://edamontology.org/format_1369</td>\n",
       "      <td>MEME background Markov model</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>http://edamontology.org/format_1370</td>\n",
       "      <td>HMMER format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>http://edamontology.org/format_1391</td>\n",
       "      <td>HMMER-aln</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>http://edamontology.org/format_1392</td>\n",
       "      <td>DIALIGN format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>http://edamontology.org/format_1393</td>\n",
       "      <td>daf</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>http://edamontology.org/format_1419</td>\n",
       "      <td>Sequence-MEME profile alignment</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>http://edamontology.org/format_1421</td>\n",
       "      <td>HMMER profile alignment (sequences versus HMMs)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51</th>\n",
       "      <td>http://edamontology.org/format_1422</td>\n",
       "      <td>HMMER profile alignment (HMM versus sequences)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>http://edamontology.org/format_1423</td>\n",
       "      <td>Phylip distance matrix</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>53</th>\n",
       "      <td>http://edamontology.org/format_1424</td>\n",
       "      <td>ClustalW dendrogram</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>http://edamontology.org/format_1425</td>\n",
       "      <td>Phylip tree raw</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>55</th>\n",
       "      <td>http://edamontology.org/format_1430</td>\n",
       "      <td>Phylip continuous quantitative characters</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>56</th>\n",
       "      <td>http://edamontology.org/format_1432</td>\n",
       "      <td>Phylip character frequencies format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>57</th>\n",
       "      <td>http://edamontology.org/format_1433</td>\n",
       "      <td>Phylip discrete states format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>58</th>\n",
       "      <td>http://edamontology.org/format_1434</td>\n",
       "      <td>Phylip cliques format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>59</th>\n",
       "      <td>http://edamontology.org/format_1435</td>\n",
       "      <td>Phylip tree format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>60</th>\n",
       "      <td>http://edamontology.org/format_1436</td>\n",
       "      <td>TreeBASE format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>61</th>\n",
       "      <td>http://edamontology.org/format_1437</td>\n",
       "      <td>TreeFam format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>62</th>\n",
       "      <td>http://edamontology.org/format_1445</td>\n",
       "      <td>Phylip tree distance format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>63</th>\n",
       "      <td>http://edamontology.org/format_1454</td>\n",
       "      <td>dssp</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>64</th>\n",
       "      <td>http://edamontology.org/format_1455</td>\n",
       "      <td>hssp</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>65</th>\n",
       "      <td>http://edamontology.org/format_1457</td>\n",
       "      <td>Dot-bracket format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66</th>\n",
       "      <td>http://edamontology.org/format_1458</td>\n",
       "      <td>Vienna local RNA secondary structure format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>67</th>\n",
       "      <td>http://edamontology.org/format_1476</td>\n",
       "      <td>PDB</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>68</th>\n",
       "      <td>http://edamontology.org/format_1477</td>\n",
       "      <td>mmCIF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69</th>\n",
       "      <td>http://edamontology.org/format_1478</td>\n",
       "      <td>PDBML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>70</th>\n",
       "      <td>http://edamontology.org/format_1504</td>\n",
       "      <td>aaindex</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71</th>\n",
       "      <td>http://edamontology.org/format_1551</td>\n",
       "      <td>Pcons report format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72</th>\n",
       "      <td>http://edamontology.org/format_1552</td>\n",
       "      <td>ProQ report format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>73</th>\n",
       "      <td>http://edamontology.org/format_1582</td>\n",
       "      <td>findkm</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>74</th>\n",
       "      <td>http://edamontology.org/format_1627</td>\n",
       "      <td>Primer3 primer</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75</th>\n",
       "      <td>http://edamontology.org/format_1628</td>\n",
       "      <td>ABI</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76</th>\n",
       "      <td>http://edamontology.org/format_1629</td>\n",
       "      <td>mira</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>77</th>\n",
       "      <td>http://edamontology.org/format_1630</td>\n",
       "      <td>CAF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78</th>\n",
       "      <td>http://edamontology.org/format_1631</td>\n",
       "      <td>EXP</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>79</th>\n",
       "      <td>http://edamontology.org/format_1632</td>\n",
       "      <td>SCF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80</th>\n",
       "      <td>http://edamontology.org/format_1633</td>\n",
       "      <td>PHD</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>81</th>\n",
       "      <td>http://edamontology.org/format_1639</td>\n",
       "      <td>affymetrix</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>82</th>\n",
       "      <td>http://edamontology.org/format_1641</td>\n",
       "      <td>affymetrix-exp</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>83</th>\n",
       "      <td>http://edamontology.org/format_1665</td>\n",
       "      <td>Taverna workflow format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>84</th>\n",
       "      <td>http://edamontology.org/format_1705</td>\n",
       "      <td>HET group dictionary entry format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>85</th>\n",
       "      <td>http://edamontology.org/format_1734</td>\n",
       "      <td>PubMed citation</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>86</th>\n",
       "      <td>http://edamontology.org/format_1735</td>\n",
       "      <td>Medline Display Format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>87</th>\n",
       "      <td>http://edamontology.org/format_1736</td>\n",
       "      <td>CiteXplore-core</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>88</th>\n",
       "      <td>http://edamontology.org/format_1737</td>\n",
       "      <td>CiteXplore-all</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>89</th>\n",
       "      <td>http://edamontology.org/format_1739</td>\n",
       "      <td>pmc</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>90</th>\n",
       "      <td>http://edamontology.org/format_1740</td>\n",
       "      <td>iHOP format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>91</th>\n",
       "      <td>http://edamontology.org/format_1741</td>\n",
       "      <td>OSCAR format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>92</th>\n",
       "      <td>http://edamontology.org/format_1861</td>\n",
       "      <td>PlasMapper TextMap</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>93</th>\n",
       "      <td>http://edamontology.org/format_1910</td>\n",
       "      <td>newick</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>94</th>\n",
       "      <td>http://edamontology.org/format_1911</td>\n",
       "      <td>TreeCon format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>95</th>\n",
       "      <td>http://edamontology.org/format_1912</td>\n",
       "      <td>Nexus format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>http://edamontology.org/format_1923</td>\n",
       "      <td>acedb</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>97</th>\n",
       "      <td>http://edamontology.org/format_1925</td>\n",
       "      <td>codata</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>98</th>\n",
       "      <td>http://edamontology.org/format_1926</td>\n",
       "      <td>dbid</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>99</th>\n",
       "      <td>http://edamontology.org/format_1927</td>\n",
       "      <td>EMBL format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>100</th>\n",
       "      <td>http://edamontology.org/format_1928</td>\n",
       "      <td>Staden experiment format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>101</th>\n",
       "      <td>http://edamontology.org/format_1929</td>\n",
       "      <td>FASTA</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>102</th>\n",
       "      <td>http://edamontology.org/format_1930</td>\n",
       "      <td>FASTQ</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>103</th>\n",
       "      <td>http://edamontology.org/format_1931</td>\n",
       "      <td>FASTQ-illumina</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>104</th>\n",
       "      <td>http://edamontology.org/format_1932</td>\n",
       "      <td>FASTQ-sanger</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>105</th>\n",
       "      <td>http://edamontology.org/format_1933</td>\n",
       "      <td>FASTQ-solexa</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>106</th>\n",
       "      <td>http://edamontology.org/format_1934</td>\n",
       "      <td>fitch program</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>107</th>\n",
       "      <td>http://edamontology.org/format_1935</td>\n",
       "      <td>GCG</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>108</th>\n",
       "      <td>http://edamontology.org/format_1936</td>\n",
       "      <td>GenBank format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>109</th>\n",
       "      <td>http://edamontology.org/format_1937</td>\n",
       "      <td>genpept</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>110</th>\n",
       "      <td>http://edamontology.org/format_1938</td>\n",
       "      <td>GFF2-seq</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>111</th>\n",
       "      <td>http://edamontology.org/format_1939</td>\n",
       "      <td>GFF3-seq</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>112</th>\n",
       "      <td>http://edamontology.org/format_1940</td>\n",
       "      <td>giFASTA format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>113</th>\n",
       "      <td>http://edamontology.org/format_1941</td>\n",
       "      <td>hennig86</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>114</th>\n",
       "      <td>http://edamontology.org/format_1942</td>\n",
       "      <td>ig</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>115</th>\n",
       "      <td>http://edamontology.org/format_1943</td>\n",
       "      <td>igstrict</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>116</th>\n",
       "      <td>http://edamontology.org/format_1944</td>\n",
       "      <td>jackknifer</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>117</th>\n",
       "      <td>http://edamontology.org/format_1945</td>\n",
       "      <td>mase format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>118</th>\n",
       "      <td>http://edamontology.org/format_1946</td>\n",
       "      <td>mega-seq</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>119</th>\n",
       "      <td>http://edamontology.org/format_1947</td>\n",
       "      <td>GCG MSF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>120</th>\n",
       "      <td>http://edamontology.org/format_1948</td>\n",
       "      <td>nbrf/pir</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>121</th>\n",
       "      <td>http://edamontology.org/format_1949</td>\n",
       "      <td>nexus-seq</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>122</th>\n",
       "      <td>http://edamontology.org/format_1950</td>\n",
       "      <td>pdbatom</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>123</th>\n",
       "      <td>http://edamontology.org/format_1951</td>\n",
       "      <td>pdbatomnuc</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>124</th>\n",
       "      <td>http://edamontology.org/format_1952</td>\n",
       "      <td>pdbseqresnuc</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>125</th>\n",
       "      <td>http://edamontology.org/format_1953</td>\n",
       "      <td>pdbseqres</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>126</th>\n",
       "      <td>http://edamontology.org/format_1954</td>\n",
       "      <td>Pearson format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>127</th>\n",
       "      <td>http://edamontology.org/format_1957</td>\n",
       "      <td>raw</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>128</th>\n",
       "      <td>http://edamontology.org/format_1958</td>\n",
       "      <td>refseqp</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>129</th>\n",
       "      <td>http://edamontology.org/format_1960</td>\n",
       "      <td>Staden format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>130</th>\n",
       "      <td>http://edamontology.org/format_1961</td>\n",
       "      <td>Stockholm format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>131</th>\n",
       "      <td>http://edamontology.org/format_1962</td>\n",
       "      <td>strider format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>132</th>\n",
       "      <td>http://edamontology.org/format_1963</td>\n",
       "      <td>UniProtKB format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>133</th>\n",
       "      <td>http://edamontology.org/format_1964</td>\n",
       "      <td>plain text format (unformatted)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>134</th>\n",
       "      <td>http://edamontology.org/format_1966</td>\n",
       "      <td>ASN.1 sequence format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>135</th>\n",
       "      <td>http://edamontology.org/format_1967</td>\n",
       "      <td>DAS format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>136</th>\n",
       "      <td>http://edamontology.org/format_1968</td>\n",
       "      <td>dasdna</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>137</th>\n",
       "      <td>http://edamontology.org/format_1969</td>\n",
       "      <td>debug-seq</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>138</th>\n",
       "      <td>http://edamontology.org/format_1970</td>\n",
       "      <td>jackknifernon</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>139</th>\n",
       "      <td>http://edamontology.org/format_1972</td>\n",
       "      <td>NCBI format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>140</th>\n",
       "      <td>http://edamontology.org/format_1973</td>\n",
       "      <td>nexusnon</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>141</th>\n",
       "      <td>http://edamontology.org/format_1974</td>\n",
       "      <td>GFF2</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>142</th>\n",
       "      <td>http://edamontology.org/format_1975</td>\n",
       "      <td>GFF3</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>143</th>\n",
       "      <td>http://edamontology.org/format_1978</td>\n",
       "      <td>DASGFF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>144</th>\n",
       "      <td>http://edamontology.org/format_1979</td>\n",
       "      <td>debug-feat</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>145</th>\n",
       "      <td>http://edamontology.org/format_1982</td>\n",
       "      <td>ClustalW format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>146</th>\n",
       "      <td>http://edamontology.org/format_1983</td>\n",
       "      <td>debug</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>147</th>\n",
       "      <td>http://edamontology.org/format_1984</td>\n",
       "      <td>FASTA-aln</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>148</th>\n",
       "      <td>http://edamontology.org/format_1985</td>\n",
       "      <td>markx0</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>149</th>\n",
       "      <td>http://edamontology.org/format_1986</td>\n",
       "      <td>markx1</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>150</th>\n",
       "      <td>http://edamontology.org/format_1987</td>\n",
       "      <td>markx10</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>151</th>\n",
       "      <td>http://edamontology.org/format_1988</td>\n",
       "      <td>markx2</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>152</th>\n",
       "      <td>http://edamontology.org/format_1989</td>\n",
       "      <td>markx3</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>153</th>\n",
       "      <td>http://edamontology.org/format_1990</td>\n",
       "      <td>match</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>154</th>\n",
       "      <td>http://edamontology.org/format_1991</td>\n",
       "      <td>mega</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>155</th>\n",
       "      <td>http://edamontology.org/format_1992</td>\n",
       "      <td>meganon</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>156</th>\n",
       "      <td>http://edamontology.org/format_1996</td>\n",
       "      <td>pair</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>157</th>\n",
       "      <td>http://edamontology.org/format_1997</td>\n",
       "      <td>PHYLIP format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>158</th>\n",
       "      <td>http://edamontology.org/format_1998</td>\n",
       "      <td>PHYLIP sequential</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>159</th>\n",
       "      <td>http://edamontology.org/format_1999</td>\n",
       "      <td>scores format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>160</th>\n",
       "      <td>http://edamontology.org/format_2000</td>\n",
       "      <td>selex</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>161</th>\n",
       "      <td>http://edamontology.org/format_2001</td>\n",
       "      <td>EMBOSS simple format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>162</th>\n",
       "      <td>http://edamontology.org/format_2002</td>\n",
       "      <td>srs format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>163</th>\n",
       "      <td>http://edamontology.org/format_2003</td>\n",
       "      <td>srspair</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>164</th>\n",
       "      <td>http://edamontology.org/format_2004</td>\n",
       "      <td>T-Coffee format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>165</th>\n",
       "      <td>http://edamontology.org/format_2005</td>\n",
       "      <td>TreeCon-seq</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>166</th>\n",
       "      <td>http://edamontology.org/format_2032</td>\n",
       "      <td>Workflow format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>167</th>\n",
       "      <td>http://edamontology.org/format_2033</td>\n",
       "      <td>Tertiary structure format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>168</th>\n",
       "      <td>http://edamontology.org/format_2056</td>\n",
       "      <td>Microarray experiment data format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>169</th>\n",
       "      <td>http://edamontology.org/format_2061</td>\n",
       "      <td>Nucleic acid features (primers) format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>170</th>\n",
       "      <td>http://edamontology.org/format_2077</td>\n",
       "      <td>Protein secondary structure format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171</th>\n",
       "      <td>http://edamontology.org/format_2094</td>\n",
       "      <td>pure</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>172</th>\n",
       "      <td>http://edamontology.org/format_2095</td>\n",
       "      <td>unpure</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>173</th>\n",
       "      <td>http://edamontology.org/format_2096</td>\n",
       "      <td>unambiguous sequence</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>174</th>\n",
       "      <td>http://edamontology.org/format_2097</td>\n",
       "      <td>ambiguous</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>175</th>\n",
       "      <td>http://edamontology.org/format_2155</td>\n",
       "      <td>Sequence features (repeats) format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>176</th>\n",
       "      <td>http://edamontology.org/format_2158</td>\n",
       "      <td>Nucleic acid features (restriction sites) format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>177</th>\n",
       "      <td>http://edamontology.org/format_2171</td>\n",
       "      <td>Sequence cluster format (protein)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>178</th>\n",
       "      <td>http://edamontology.org/format_2172</td>\n",
       "      <td>Sequence cluster format (nucleic acid)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>179</th>\n",
       "      <td>http://edamontology.org/format_2181</td>\n",
       "      <td>EMBL-like (text)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>180</th>\n",
       "      <td>http://edamontology.org/format_2182</td>\n",
       "      <td>FASTQ-like format (text)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>181</th>\n",
       "      <td>http://edamontology.org/format_2183</td>\n",
       "      <td>EMBLXML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>182</th>\n",
       "      <td>http://edamontology.org/format_2184</td>\n",
       "      <td>cdsxml</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>183</th>\n",
       "      <td>http://edamontology.org/format_2185</td>\n",
       "      <td>INSDSeq</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>184</th>\n",
       "      <td>http://edamontology.org/format_2186</td>\n",
       "      <td>geneseq</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>185</th>\n",
       "      <td>http://edamontology.org/format_2187</td>\n",
       "      <td>UniProt-like (text)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>186</th>\n",
       "      <td>http://edamontology.org/format_2194</td>\n",
       "      <td>medline</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>187</th>\n",
       "      <td>http://edamontology.org/format_2196</td>\n",
       "      <td>OBO format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>188</th>\n",
       "      <td>http://edamontology.org/format_2197</td>\n",
       "      <td>OWL format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>189</th>\n",
       "      <td>http://edamontology.org/format_2200</td>\n",
       "      <td>FASTA-like (text)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>190</th>\n",
       "      <td>http://edamontology.org/format_2204</td>\n",
       "      <td>EMBL format (XML)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>191</th>\n",
       "      <td>http://edamontology.org/format_2205</td>\n",
       "      <td>GenBank-like format (text)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>192</th>\n",
       "      <td>http://edamontology.org/format_2206</td>\n",
       "      <td>Sequence feature table format (text)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>193</th>\n",
       "      <td>http://edamontology.org/format_2304</td>\n",
       "      <td>STRING entry format (XML)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>194</th>\n",
       "      <td>http://edamontology.org/format_2305</td>\n",
       "      <td>GFF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>195</th>\n",
       "      <td>http://edamontology.org/format_2306</td>\n",
       "      <td>GTF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>196</th>\n",
       "      <td>http://edamontology.org/format_2310</td>\n",
       "      <td>FASTA-HTML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>197</th>\n",
       "      <td>http://edamontology.org/format_2311</td>\n",
       "      <td>EMBL-HTML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>198</th>\n",
       "      <td>http://edamontology.org/format_2330</td>\n",
       "      <td>Textual format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>199</th>\n",
       "      <td>http://edamontology.org/format_2332</td>\n",
       "      <td>XML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>200</th>\n",
       "      <td>http://edamontology.org/format_2333</td>\n",
       "      <td>Binary format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>201</th>\n",
       "      <td>http://edamontology.org/format_2350</td>\n",
       "      <td>Format (by type of data)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>202</th>\n",
       "      <td>http://edamontology.org/format_2376</td>\n",
       "      <td>RDF format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>203</th>\n",
       "      <td>http://edamontology.org/format_2532</td>\n",
       "      <td>GenBank-HTML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>204</th>\n",
       "      <td>http://edamontology.org/format_2543</td>\n",
       "      <td>EMBL-like format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>205</th>\n",
       "      <td>http://edamontology.org/format_2545</td>\n",
       "      <td>FASTQ-like format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>206</th>\n",
       "      <td>http://edamontology.org/format_2546</td>\n",
       "      <td>FASTA-like</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>207</th>\n",
       "      <td>http://edamontology.org/format_2547</td>\n",
       "      <td>uniprotkb-like format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>208</th>\n",
       "      <td>http://edamontology.org/format_2549</td>\n",
       "      <td>OBO</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>209</th>\n",
       "      <td>http://edamontology.org/format_2550</td>\n",
       "      <td>OBO-XML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>210</th>\n",
       "      <td>http://edamontology.org/format_2551</td>\n",
       "      <td>Sequence record format (text)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>211</th>\n",
       "      <td>http://edamontology.org/format_2552</td>\n",
       "      <td>Sequence record format (XML)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>212</th>\n",
       "      <td>http://edamontology.org/format_2553</td>\n",
       "      <td>Sequence feature table format (XML)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>213</th>\n",
       "      <td>http://edamontology.org/format_2554</td>\n",
       "      <td>Alignment format (text)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>214</th>\n",
       "      <td>http://edamontology.org/format_2555</td>\n",
       "      <td>Alignment format (XML)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>215</th>\n",
       "      <td>http://edamontology.org/format_2556</td>\n",
       "      <td>Phylogenetic tree format (text)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>216</th>\n",
       "      <td>http://edamontology.org/format_2557</td>\n",
       "      <td>Phylogenetic tree format (XML)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>217</th>\n",
       "      <td>http://edamontology.org/format_2558</td>\n",
       "      <td>EMBL-like (XML)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>218</th>\n",
       "      <td>http://edamontology.org/format_2559</td>\n",
       "      <td>GenBank-like format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>219</th>\n",
       "      <td>http://edamontology.org/format_2561</td>\n",
       "      <td>Sequence assembly format (text)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>220</th>\n",
       "      <td>http://edamontology.org/format_2566</td>\n",
       "      <td>completely unambiguous</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>221</th>\n",
       "      <td>http://edamontology.org/format_2567</td>\n",
       "      <td>completely unambiguous pure</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>222</th>\n",
       "      <td>http://edamontology.org/format_2568</td>\n",
       "      <td>completely unambiguous pure nucleotide</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>223</th>\n",
       "      <td>http://edamontology.org/format_2569</td>\n",
       "      <td>completely unambiguous pure dna</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>224</th>\n",
       "      <td>http://edamontology.org/format_2570</td>\n",
       "      <td>completely unambiguous pure rna sequence</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>225</th>\n",
       "      <td>http://edamontology.org/format_2572</td>\n",
       "      <td>BAM</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>226</th>\n",
       "      <td>http://edamontology.org/format_2573</td>\n",
       "      <td>SAM</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>227</th>\n",
       "      <td>http://edamontology.org/format_2585</td>\n",
       "      <td>SBML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>228</th>\n",
       "      <td>http://edamontology.org/format_2607</td>\n",
       "      <td>completely unambiguous pure protein</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>229</th>\n",
       "      <td>http://edamontology.org/format_2922</td>\n",
       "      <td>markx0 variant</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>230</th>\n",
       "      <td>http://edamontology.org/format_2923</td>\n",
       "      <td>mega variant</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>231</th>\n",
       "      <td>http://edamontology.org/format_2924</td>\n",
       "      <td>Phylip format variant</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>232</th>\n",
       "      <td>http://edamontology.org/format_3000</td>\n",
       "      <td>AB1</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>233</th>\n",
       "      <td>http://edamontology.org/format_3001</td>\n",
       "      <td>ACE</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>234</th>\n",
       "      <td>http://edamontology.org/format_3003</td>\n",
       "      <td>BED</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>235</th>\n",
       "      <td>http://edamontology.org/format_3004</td>\n",
       "      <td>bigBed</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>236</th>\n",
       "      <td>http://edamontology.org/format_3005</td>\n",
       "      <td>WIG</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>237</th>\n",
       "      <td>http://edamontology.org/format_3006</td>\n",
       "      <td>bigWig</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>238</th>\n",
       "      <td>http://edamontology.org/format_3007</td>\n",
       "      <td>PSL</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>239</th>\n",
       "      <td>http://edamontology.org/format_3008</td>\n",
       "      <td>MAF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>240</th>\n",
       "      <td>http://edamontology.org/format_3009</td>\n",
       "      <td>2bit</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>241</th>\n",
       "      <td>http://edamontology.org/format_3010</td>\n",
       "      <td>.nib</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>242</th>\n",
       "      <td>http://edamontology.org/format_3011</td>\n",
       "      <td>genePred</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>243</th>\n",
       "      <td>http://edamontology.org/format_3012</td>\n",
       "      <td>pgSnp</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>244</th>\n",
       "      <td>http://edamontology.org/format_3013</td>\n",
       "      <td>axt</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>245</th>\n",
       "      <td>http://edamontology.org/format_3014</td>\n",
       "      <td>LAV</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>246</th>\n",
       "      <td>http://edamontology.org/format_3015</td>\n",
       "      <td>Pileup</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>247</th>\n",
       "      <td>http://edamontology.org/format_3016</td>\n",
       "      <td>VCF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>248</th>\n",
       "      <td>http://edamontology.org/format_3017</td>\n",
       "      <td>SRF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>249</th>\n",
       "      <td>http://edamontology.org/format_3018</td>\n",
       "      <td>ZTR</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>250</th>\n",
       "      <td>http://edamontology.org/format_3019</td>\n",
       "      <td>GVF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>251</th>\n",
       "      <td>http://edamontology.org/format_3020</td>\n",
       "      <td>BCF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>252</th>\n",
       "      <td>http://edamontology.org/format_3098</td>\n",
       "      <td>Raw SCOP domain classification format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>253</th>\n",
       "      <td>http://edamontology.org/format_3099</td>\n",
       "      <td>Raw CATH domain classification format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>254</th>\n",
       "      <td>http://edamontology.org/format_3100</td>\n",
       "      <td>CATH domain report format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>255</th>\n",
       "      <td>http://edamontology.org/format_3155</td>\n",
       "      <td>SBRML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>256</th>\n",
       "      <td>http://edamontology.org/format_3156</td>\n",
       "      <td>BioPAX</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>257</th>\n",
       "      <td>http://edamontology.org/format_3157</td>\n",
       "      <td>EBI Application Result XML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>258</th>\n",
       "      <td>http://edamontology.org/format_3158</td>\n",
       "      <td>PSI MI XML (MIF)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>259</th>\n",
       "      <td>http://edamontology.org/format_3159</td>\n",
       "      <td>phyloXML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>260</th>\n",
       "      <td>http://edamontology.org/format_3160</td>\n",
       "      <td>NeXML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>261</th>\n",
       "      <td>http://edamontology.org/format_3163</td>\n",
       "      <td>GCDML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>262</th>\n",
       "      <td>http://edamontology.org/format_3164</td>\n",
       "      <td>GTrack</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>263</th>\n",
       "      <td>http://edamontology.org/format_3239</td>\n",
       "      <td>CopasiML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>264</th>\n",
       "      <td>http://edamontology.org/format_3240</td>\n",
       "      <td>CellML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>265</th>\n",
       "      <td>http://edamontology.org/format_3242</td>\n",
       "      <td>PSI MI TAB (MITAB)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>266</th>\n",
       "      <td>http://edamontology.org/format_3243</td>\n",
       "      <td>PSI-PAR</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>267</th>\n",
       "      <td>http://edamontology.org/format_3244</td>\n",
       "      <td>mzML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>268</th>\n",
       "      <td>http://edamontology.org/format_3246</td>\n",
       "      <td>TraML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>269</th>\n",
       "      <td>http://edamontology.org/format_3247</td>\n",
       "      <td>mzIdentML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>270</th>\n",
       "      <td>http://edamontology.org/format_3248</td>\n",
       "      <td>mzQuantML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>271</th>\n",
       "      <td>http://edamontology.org/format_3249</td>\n",
       "      <td>GelML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>272</th>\n",
       "      <td>http://edamontology.org/format_3250</td>\n",
       "      <td>spML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>273</th>\n",
       "      <td>http://edamontology.org/format_3252</td>\n",
       "      <td>OWL Functional Syntax</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>274</th>\n",
       "      <td>http://edamontology.org/format_3253</td>\n",
       "      <td>Manchester OWL Syntax</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>275</th>\n",
       "      <td>http://edamontology.org/format_3254</td>\n",
       "      <td>KRSS2 Syntax</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>276</th>\n",
       "      <td>http://edamontology.org/format_3255</td>\n",
       "      <td>Turtle</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>277</th>\n",
       "      <td>http://edamontology.org/format_3256</td>\n",
       "      <td>N-Triples</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>278</th>\n",
       "      <td>http://edamontology.org/format_3257</td>\n",
       "      <td>Notation3</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>279</th>\n",
       "      <td>http://edamontology.org/format_3261</td>\n",
       "      <td>RDF/XML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>280</th>\n",
       "      <td>http://edamontology.org/format_3262</td>\n",
       "      <td>OWL/XML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>281</th>\n",
       "      <td>http://edamontology.org/format_3281</td>\n",
       "      <td>A2M</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>282</th>\n",
       "      <td>http://edamontology.org/format_3284</td>\n",
       "      <td>SFF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>283</th>\n",
       "      <td>http://edamontology.org/format_3285</td>\n",
       "      <td>MAP</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>284</th>\n",
       "      <td>http://edamontology.org/format_3286</td>\n",
       "      <td>PED</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>285</th>\n",
       "      <td>http://edamontology.org/format_3287</td>\n",
       "      <td>Individual genetic data format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>286</th>\n",
       "      <td>http://edamontology.org/format_3288</td>\n",
       "      <td>PED/MAP</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>287</th>\n",
       "      <td>http://edamontology.org/format_3309</td>\n",
       "      <td>CT</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>288</th>\n",
       "      <td>http://edamontology.org/format_3310</td>\n",
       "      <td>SS</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>289</th>\n",
       "      <td>http://edamontology.org/format_3311</td>\n",
       "      <td>RNAML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>290</th>\n",
       "      <td>http://edamontology.org/format_3312</td>\n",
       "      <td>GDE</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>291</th>\n",
       "      <td>http://edamontology.org/format_3313</td>\n",
       "      <td>BLC</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>292</th>\n",
       "      <td>http://edamontology.org/format_3328</td>\n",
       "      <td>HMMER2</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>293</th>\n",
       "      <td>http://edamontology.org/format_3329</td>\n",
       "      <td>HMMER3</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>294</th>\n",
       "      <td>http://edamontology.org/format_3330</td>\n",
       "      <td>PO</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>295</th>\n",
       "      <td>http://edamontology.org/format_3331</td>\n",
       "      <td>BLAST XML results format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>296</th>\n",
       "      <td>http://edamontology.org/format_3462</td>\n",
       "      <td>CRAM</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>297</th>\n",
       "      <td>http://edamontology.org/format_3464</td>\n",
       "      <td>JSON</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>298</th>\n",
       "      <td>http://edamontology.org/format_3466</td>\n",
       "      <td>EPS</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>299</th>\n",
       "      <td>http://edamontology.org/format_3467</td>\n",
       "      <td>GIF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>300</th>\n",
       "      <td>http://edamontology.org/format_3468</td>\n",
       "      <td>xls</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>301</th>\n",
       "      <td>http://edamontology.org/format_3475</td>\n",
       "      <td>TSV</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>302</th>\n",
       "      <td>http://edamontology.org/format_3477</td>\n",
       "      <td>Cytoscape input file format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>303</th>\n",
       "      <td>http://edamontology.org/format_3485</td>\n",
       "      <td>RSF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>304</th>\n",
       "      <td>http://edamontology.org/format_3486</td>\n",
       "      <td>GCG format variant</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>305</th>\n",
       "      <td>http://edamontology.org/format_3487</td>\n",
       "      <td>BSML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>306</th>\n",
       "      <td>http://edamontology.org/format_3499</td>\n",
       "      <td>Ensembl variation file format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>307</th>\n",
       "      <td>http://edamontology.org/format_3506</td>\n",
       "      <td>docx</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>308</th>\n",
       "      <td>http://edamontology.org/format_3507</td>\n",
       "      <td>Document format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>309</th>\n",
       "      <td>http://edamontology.org/format_3508</td>\n",
       "      <td>PDF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>310</th>\n",
       "      <td>http://edamontology.org/format_3548</td>\n",
       "      <td>DICOM format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>311</th>\n",
       "      <td>http://edamontology.org/format_3549</td>\n",
       "      <td>nii</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>312</th>\n",
       "      <td>http://edamontology.org/format_3550</td>\n",
       "      <td>mhd</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>313</th>\n",
       "      <td>http://edamontology.org/format_3551</td>\n",
       "      <td>nrrd</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>314</th>\n",
       "      <td>http://edamontology.org/format_3554</td>\n",
       "      <td>R file format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>315</th>\n",
       "      <td>http://edamontology.org/format_3555</td>\n",
       "      <td>SPSS</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>316</th>\n",
       "      <td>http://edamontology.org/format_3556</td>\n",
       "      <td>MHTML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>317</th>\n",
       "      <td>http://edamontology.org/format_3579</td>\n",
       "      <td>JPG</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>318</th>\n",
       "      <td>http://edamontology.org/format_3580</td>\n",
       "      <td>rcc</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>319</th>\n",
       "      <td>http://edamontology.org/format_3581</td>\n",
       "      <td>arff</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>320</th>\n",
       "      <td>http://edamontology.org/format_3582</td>\n",
       "      <td>afg</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>321</th>\n",
       "      <td>http://edamontology.org/format_3583</td>\n",
       "      <td>bedgraph</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>322</th>\n",
       "      <td>http://edamontology.org/format_3584</td>\n",
       "      <td>bedstrict</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>323</th>\n",
       "      <td>http://edamontology.org/format_3585</td>\n",
       "      <td>bed6</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>324</th>\n",
       "      <td>http://edamontology.org/format_3586</td>\n",
       "      <td>bed12</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>325</th>\n",
       "      <td>http://edamontology.org/format_3587</td>\n",
       "      <td>chrominfo</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>326</th>\n",
       "      <td>http://edamontology.org/format_3588</td>\n",
       "      <td>customtrack</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>327</th>\n",
       "      <td>http://edamontology.org/format_3589</td>\n",
       "      <td>csfasta</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>328</th>\n",
       "      <td>http://edamontology.org/format_3590</td>\n",
       "      <td>HDF5</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>329</th>\n",
       "      <td>http://edamontology.org/format_3591</td>\n",
       "      <td>TIFF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>330</th>\n",
       "      <td>http://edamontology.org/format_3592</td>\n",
       "      <td>BMP</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>331</th>\n",
       "      <td>http://edamontology.org/format_3593</td>\n",
       "      <td>im</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>332</th>\n",
       "      <td>http://edamontology.org/format_3594</td>\n",
       "      <td>pcd</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>333</th>\n",
       "      <td>http://edamontology.org/format_3595</td>\n",
       "      <td>pcx</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>334</th>\n",
       "      <td>http://edamontology.org/format_3596</td>\n",
       "      <td>ppm</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>335</th>\n",
       "      <td>http://edamontology.org/format_3597</td>\n",
       "      <td>psd</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>336</th>\n",
       "      <td>http://edamontology.org/format_3598</td>\n",
       "      <td>xbm</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>337</th>\n",
       "      <td>http://edamontology.org/format_3599</td>\n",
       "      <td>xpm</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>338</th>\n",
       "      <td>http://edamontology.org/format_3600</td>\n",
       "      <td>rgb</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>339</th>\n",
       "      <td>http://edamontology.org/format_3601</td>\n",
       "      <td>pbm</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>340</th>\n",
       "      <td>http://edamontology.org/format_3602</td>\n",
       "      <td>pgm</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>341</th>\n",
       "      <td>http://edamontology.org/format_3603</td>\n",
       "      <td>PNG</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>342</th>\n",
       "      <td>http://edamontology.org/format_3604</td>\n",
       "      <td>SVG</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>343</th>\n",
       "      <td>http://edamontology.org/format_3605</td>\n",
       "      <td>rast</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>344</th>\n",
       "      <td>http://edamontology.org/format_3607</td>\n",
       "      <td>qual</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>345</th>\n",
       "      <td>http://edamontology.org/format_3608</td>\n",
       "      <td>qualsolexa</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>346</th>\n",
       "      <td>http://edamontology.org/format_3609</td>\n",
       "      <td>qualillumina</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>347</th>\n",
       "      <td>http://edamontology.org/format_3610</td>\n",
       "      <td>qualsolid</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>348</th>\n",
       "      <td>http://edamontology.org/format_3611</td>\n",
       "      <td>qual454</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>349</th>\n",
       "      <td>http://edamontology.org/format_3612</td>\n",
       "      <td>ENCODE peak format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>350</th>\n",
       "      <td>http://edamontology.org/format_3613</td>\n",
       "      <td>ENCODE narrow peak format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>351</th>\n",
       "      <td>http://edamontology.org/format_3614</td>\n",
       "      <td>ENCODE broad peak format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>352</th>\n",
       "      <td>http://edamontology.org/format_3615</td>\n",
       "      <td>bgzip</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>353</th>\n",
       "      <td>http://edamontology.org/format_3616</td>\n",
       "      <td>tabix</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>354</th>\n",
       "      <td>http://edamontology.org/format_3617</td>\n",
       "      <td>Graph format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>355</th>\n",
       "      <td>http://edamontology.org/format_3618</td>\n",
       "      <td>xgmml</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>356</th>\n",
       "      <td>http://edamontology.org/format_3619</td>\n",
       "      <td>sif</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>357</th>\n",
       "      <td>http://edamontology.org/format_3620</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>358</th>\n",
       "      <td>http://edamontology.org/format_3621</td>\n",
       "      <td>SQLite format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>359</th>\n",
       "      <td>http://edamontology.org/format_3622</td>\n",
       "      <td>Gemini SQLite format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>360</th>\n",
       "      <td>http://edamontology.org/format_3624</td>\n",
       "      <td>snpeffdb</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>361</th>\n",
       "      <td>http://edamontology.org/format_3650</td>\n",
       "      <td>netCDF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>362</th>\n",
       "      <td>http://edamontology.org/format_3651</td>\n",
       "      <td>MGF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>363</th>\n",
       "      <td>http://edamontology.org/format_3652</td>\n",
       "      <td>dta</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>364</th>\n",
       "      <td>http://edamontology.org/format_3653</td>\n",
       "      <td>pkl</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>365</th>\n",
       "      <td>http://edamontology.org/format_3654</td>\n",
       "      <td>mzXML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>366</th>\n",
       "      <td>http://edamontology.org/format_3655</td>\n",
       "      <td>pepXML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>367</th>\n",
       "      <td>http://edamontology.org/format_3657</td>\n",
       "      <td>GPML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>368</th>\n",
       "      <td>http://edamontology.org/format_3665</td>\n",
       "      <td>K-mer countgraph</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>369</th>\n",
       "      <td>http://edamontology.org/format_3681</td>\n",
       "      <td>mzTab</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>370</th>\n",
       "      <td>http://edamontology.org/format_3682</td>\n",
       "      <td>imzML metadata file</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>371</th>\n",
       "      <td>http://edamontology.org/format_3683</td>\n",
       "      <td>qcML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>372</th>\n",
       "      <td>http://edamontology.org/format_3684</td>\n",
       "      <td>PRIDE XML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>373</th>\n",
       "      <td>http://edamontology.org/format_3685</td>\n",
       "      <td>SED-ML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>374</th>\n",
       "      <td>http://edamontology.org/format_3686</td>\n",
       "      <td>COMBINE OMEX</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>375</th>\n",
       "      <td>http://edamontology.org/format_3687</td>\n",
       "      <td>ISA-TAB</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>376</th>\n",
       "      <td>http://edamontology.org/format_3688</td>\n",
       "      <td>SBtab</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>377</th>\n",
       "      <td>http://edamontology.org/format_3689</td>\n",
       "      <td>BCML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>378</th>\n",
       "      <td>http://edamontology.org/format_3690</td>\n",
       "      <td>BDML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>379</th>\n",
       "      <td>http://edamontology.org/format_3691</td>\n",
       "      <td>BEL</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>380</th>\n",
       "      <td>http://edamontology.org/format_3692</td>\n",
       "      <td>SBGN-ML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>381</th>\n",
       "      <td>http://edamontology.org/format_3693</td>\n",
       "      <td>AGP</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>382</th>\n",
       "      <td>http://edamontology.org/format_3696</td>\n",
       "      <td>PS</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>383</th>\n",
       "      <td>http://edamontology.org/format_3698</td>\n",
       "      <td>SRA format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>384</th>\n",
       "      <td>http://edamontology.org/format_3699</td>\n",
       "      <td>VDB</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>385</th>\n",
       "      <td>http://edamontology.org/format_3701</td>\n",
       "      <td>Sequin format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>386</th>\n",
       "      <td>http://edamontology.org/format_3702</td>\n",
       "      <td>MSF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>387</th>\n",
       "      <td>http://edamontology.org/format_3709</td>\n",
       "      <td>GCT/Res format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>388</th>\n",
       "      <td>http://edamontology.org/format_3710</td>\n",
       "      <td>WIFF format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>389</th>\n",
       "      <td>http://edamontology.org/format_3711</td>\n",
       "      <td>X!Tandem XML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>390</th>\n",
       "      <td>http://edamontology.org/format_3712</td>\n",
       "      <td>Thermo RAW</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>391</th>\n",
       "      <td>http://edamontology.org/format_3713</td>\n",
       "      <td>Mascot .dat file</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>392</th>\n",
       "      <td>http://edamontology.org/format_3714</td>\n",
       "      <td>MaxQuant APL peaklist format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>393</th>\n",
       "      <td>http://edamontology.org/format_3725</td>\n",
       "      <td>SBOL</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>394</th>\n",
       "      <td>http://edamontology.org/format_3726</td>\n",
       "      <td>PMML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>395</th>\n",
       "      <td>http://edamontology.org/format_3727</td>\n",
       "      <td>OME-TIFF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>396</th>\n",
       "      <td>http://edamontology.org/format_3728</td>\n",
       "      <td>LocARNA PP</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>397</th>\n",
       "      <td>http://edamontology.org/format_3729</td>\n",
       "      <td>dbGaP format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>398</th>\n",
       "      <td>http://edamontology.org/format_3746</td>\n",
       "      <td>BIOM format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>399</th>\n",
       "      <td>http://edamontology.org/format_3747</td>\n",
       "      <td>protXML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>400</th>\n",
       "      <td>http://edamontology.org/format_3748</td>\n",
       "      <td>Linked data format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>401</th>\n",
       "      <td>http://edamontology.org/format_3749</td>\n",
       "      <td>JSON-LD</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>402</th>\n",
       "      <td>http://edamontology.org/format_3750</td>\n",
       "      <td>YAML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>403</th>\n",
       "      <td>http://edamontology.org/format_3751</td>\n",
       "      <td>DSV</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>404</th>\n",
       "      <td>http://edamontology.org/format_3752</td>\n",
       "      <td>CSV</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>405</th>\n",
       "      <td>http://edamontology.org/format_3758</td>\n",
       "      <td>SEQUEST .out file</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>406</th>\n",
       "      <td>http://edamontology.org/format_3764</td>\n",
       "      <td>idXML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>407</th>\n",
       "      <td>http://edamontology.org/format_3765</td>\n",
       "      <td>KNIME datatable format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>408</th>\n",
       "      <td>http://edamontology.org/format_3770</td>\n",
       "      <td>UniProtKB XML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>409</th>\n",
       "      <td>http://edamontology.org/format_3771</td>\n",
       "      <td>UniProtKB RDF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>410</th>\n",
       "      <td>http://edamontology.org/format_3775</td>\n",
       "      <td>GSuite</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>411</th>\n",
       "      <td>http://edamontology.org/format_3776</td>\n",
       "      <td>BTrack</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>412</th>\n",
       "      <td>http://edamontology.org/format_3781</td>\n",
       "      <td>PubAnnotation format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>413</th>\n",
       "      <td>http://edamontology.org/format_3782</td>\n",
       "      <td>BioC</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>414</th>\n",
       "      <td>http://edamontology.org/format_3783</td>\n",
       "      <td>PubTator format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>415</th>\n",
       "      <td>http://edamontology.org/format_3784</td>\n",
       "      <td>Open Annotation format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>416</th>\n",
       "      <td>http://edamontology.org/format_3785</td>\n",
       "      <td>BioNLP Shared Task format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>417</th>\n",
       "      <td>http://edamontology.org/format_3788</td>\n",
       "      <td>SQL</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>418</th>\n",
       "      <td>http://edamontology.org/format_3789</td>\n",
       "      <td>XQuery</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>419</th>\n",
       "      <td>http://edamontology.org/format_3790</td>\n",
       "      <td>SPARQL</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>420</th>\n",
       "      <td>http://edamontology.org/format_3804</td>\n",
       "      <td>xsd</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>421</th>\n",
       "      <td>http://edamontology.org/format_3811</td>\n",
       "      <td>XMFA</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>422</th>\n",
       "      <td>http://edamontology.org/format_3812</td>\n",
       "      <td>GEN</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>423</th>\n",
       "      <td>http://edamontology.org/format_3813</td>\n",
       "      <td>SAMPLE file format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>424</th>\n",
       "      <td>http://edamontology.org/format_3814</td>\n",
       "      <td>SDF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>425</th>\n",
       "      <td>http://edamontology.org/format_3815</td>\n",
       "      <td>Molfile</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>426</th>\n",
       "      <td>http://edamontology.org/format_3816</td>\n",
       "      <td>Mol2</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>427</th>\n",
       "      <td>http://edamontology.org/format_3817</td>\n",
       "      <td>latex</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>428</th>\n",
       "      <td>http://edamontology.org/format_3818</td>\n",
       "      <td>ELAND format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>429</th>\n",
       "      <td>http://edamontology.org/format_3819</td>\n",
       "      <td>Relaxed PHYLIP Interleaved</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>430</th>\n",
       "      <td>http://edamontology.org/format_3820</td>\n",
       "      <td>Relaxed PHYLIP Sequential</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>431</th>\n",
       "      <td>http://edamontology.org/format_3821</td>\n",
       "      <td>VisML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>432</th>\n",
       "      <td>http://edamontology.org/format_3822</td>\n",
       "      <td>GML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>433</th>\n",
       "      <td>http://edamontology.org/format_3823</td>\n",
       "      <td>FASTG</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>434</th>\n",
       "      <td>http://edamontology.org/format_3825</td>\n",
       "      <td>nmrML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>435</th>\n",
       "      <td>http://edamontology.org/format_3826</td>\n",
       "      <td>proBAM</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>436</th>\n",
       "      <td>http://edamontology.org/format_3827</td>\n",
       "      <td>proBED</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>437</th>\n",
       "      <td>http://edamontology.org/format_3829</td>\n",
       "      <td>GPR</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>438</th>\n",
       "      <td>http://edamontology.org/format_3830</td>\n",
       "      <td>ARB</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>439</th>\n",
       "      <td>http://edamontology.org/format_3832</td>\n",
       "      <td>consensusXML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>440</th>\n",
       "      <td>http://edamontology.org/format_3833</td>\n",
       "      <td>featureXML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>441</th>\n",
       "      <td>http://edamontology.org/format_3834</td>\n",
       "      <td>mzData</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>442</th>\n",
       "      <td>http://edamontology.org/format_3835</td>\n",
       "      <td>TIDE TXT</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>443</th>\n",
       "      <td>http://edamontology.org/format_3836</td>\n",
       "      <td>BLAST XML v2 results format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>444</th>\n",
       "      <td>http://edamontology.org/format_3838</td>\n",
       "      <td>pptx</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>445</th>\n",
       "      <td>http://edamontology.org/format_3839</td>\n",
       "      <td>ibd</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>446</th>\n",
       "      <td>http://edamontology.org/format_3841</td>\n",
       "      <td>NLP format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>447</th>\n",
       "      <td>http://edamontology.org/format_3843</td>\n",
       "      <td>BEAST</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>448</th>\n",
       "      <td>http://edamontology.org/format_3844</td>\n",
       "      <td>Chado-XML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>449</th>\n",
       "      <td>http://edamontology.org/format_3845</td>\n",
       "      <td>HSAML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>450</th>\n",
       "      <td>http://edamontology.org/format_3846</td>\n",
       "      <td>InterProScan XML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>451</th>\n",
       "      <td>http://edamontology.org/format_3847</td>\n",
       "      <td>KGML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>452</th>\n",
       "      <td>http://edamontology.org/format_3848</td>\n",
       "      <td>PubMed XML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>453</th>\n",
       "      <td>http://edamontology.org/format_3849</td>\n",
       "      <td>MSAML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>454</th>\n",
       "      <td>http://edamontology.org/format_3850</td>\n",
       "      <td>OrthoXML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>455</th>\n",
       "      <td>http://edamontology.org/format_3851</td>\n",
       "      <td>PSDML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>456</th>\n",
       "      <td>http://edamontology.org/format_3852</td>\n",
       "      <td>SeqXML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>457</th>\n",
       "      <td>http://edamontology.org/format_3853</td>\n",
       "      <td>UniParc XML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>458</th>\n",
       "      <td>http://edamontology.org/format_3854</td>\n",
       "      <td>UniRef XML</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>459</th>\n",
       "      <td>http://edamontology.org/format_3857</td>\n",
       "      <td>CWL</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>460</th>\n",
       "      <td>http://edamontology.org/format_3858</td>\n",
       "      <td>Waters RAW</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>461</th>\n",
       "      <td>http://edamontology.org/format_3859</td>\n",
       "      <td>JCAMP-DX</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>462</th>\n",
       "      <td>http://edamontology.org/format_3862</td>\n",
       "      <td>NLP annotation format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>463</th>\n",
       "      <td>http://edamontology.org/format_3863</td>\n",
       "      <td>NLP corpus format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>464</th>\n",
       "      <td>http://edamontology.org/format_3864</td>\n",
       "      <td>mirGFF3</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>465</th>\n",
       "      <td>http://edamontology.org/format_3865</td>\n",
       "      <td>RNA annotation format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>466</th>\n",
       "      <td>http://edamontology.org/format_3867</td>\n",
       "      <td>Trajectory format (binary)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>467</th>\n",
       "      <td>http://edamontology.org/format_3868</td>\n",
       "      <td>Trajectory format (text)</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>468</th>\n",
       "      <td>http://edamontology.org/format_3873</td>\n",
       "      <td>HDF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>469</th>\n",
       "      <td>http://edamontology.org/format_3874</td>\n",
       "      <td>PCAzip</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>470</th>\n",
       "      <td>http://edamontology.org/format_3875</td>\n",
       "      <td>XTC</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>471</th>\n",
       "      <td>http://edamontology.org/format_3876</td>\n",
       "      <td>TNG</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>472</th>\n",
       "      <td>http://edamontology.org/format_3877</td>\n",
       "      <td>XYZ</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>473</th>\n",
       "      <td>http://edamontology.org/format_3878</td>\n",
       "      <td>mdcrd</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>474</th>\n",
       "      <td>http://edamontology.org/format_3880</td>\n",
       "      <td>GROMACS top</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>475</th>\n",
       "      <td>http://edamontology.org/format_3881</td>\n",
       "      <td>AMBER top</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>476</th>\n",
       "      <td>http://edamontology.org/format_3882</td>\n",
       "      <td>PSF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>477</th>\n",
       "      <td>http://edamontology.org/format_3883</td>\n",
       "      <td>GROMACS itp</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>478</th>\n",
       "      <td>http://edamontology.org/format_3885</td>\n",
       "      <td>BinPos</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>479</th>\n",
       "      <td>http://edamontology.org/format_3886</td>\n",
       "      <td>RST</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>480</th>\n",
       "      <td>http://edamontology.org/format_3887</td>\n",
       "      <td>CHARMM rtf</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>481</th>\n",
       "      <td>http://edamontology.org/format_3888</td>\n",
       "      <td>AMBER frcmod</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>482</th>\n",
       "      <td>http://edamontology.org/format_3889</td>\n",
       "      <td>AMBER off</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>483</th>\n",
       "      <td>http://edamontology.org/format_3906</td>\n",
       "      <td>NMReDATA</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>484</th>\n",
       "      <td>http://edamontology.org/format_3910</td>\n",
       "      <td>trr</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>485</th>\n",
       "      <td>http://edamontology.org/format_3951</td>\n",
       "      <td>BcForms</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>486</th>\n",
       "      <td>http://edamontology.org/format_3956</td>\n",
       "      <td>N-Quads</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>487</th>\n",
       "      <td>http://edamontology.org/format_3969</td>\n",
       "      <td>Vega</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>488</th>\n",
       "      <td>http://edamontology.org/format_3970</td>\n",
       "      <td>Vega-lite</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>489</th>\n",
       "      <td>http://edamontology.org/format_3973</td>\n",
       "      <td>Docker image</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>490</th>\n",
       "      <td>http://edamontology.org/format_3975</td>\n",
       "      <td>GFA 1</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>491</th>\n",
       "      <td>http://edamontology.org/format_3976</td>\n",
       "      <td>GFA 2</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>492</th>\n",
       "      <td>http://edamontology.org/format_3977</td>\n",
       "      <td>ObjTables</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>493</th>\n",
       "      <td>http://edamontology.org/format_3978</td>\n",
       "      <td>CONTIG</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>494</th>\n",
       "      <td>http://edamontology.org/format_3979</td>\n",
       "      <td>WEGO</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>495</th>\n",
       "      <td>http://edamontology.org/format_3980</td>\n",
       "      <td>RPKM</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>496</th>\n",
       "      <td>http://edamontology.org/format_3981</td>\n",
       "      <td>TAR format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>497</th>\n",
       "      <td>http://edamontology.org/format_3982</td>\n",
       "      <td>CHAIN</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>498</th>\n",
       "      <td>http://edamontology.org/format_3983</td>\n",
       "      <td>NET</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>499</th>\n",
       "      <td>http://edamontology.org/format_3984</td>\n",
       "      <td>QMAP</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>500</th>\n",
       "      <td>http://edamontology.org/format_3985</td>\n",
       "      <td>gxformat2</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>501</th>\n",
       "      <td>http://edamontology.org/format_3986</td>\n",
       "      <td>WMV</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>502</th>\n",
       "      <td>http://edamontology.org/format_3987</td>\n",
       "      <td>ZIP format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>503</th>\n",
       "      <td>http://edamontology.org/format_3988</td>\n",
       "      <td>LSM</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>504</th>\n",
       "      <td>http://edamontology.org/format_3989</td>\n",
       "      <td>GZIP format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>505</th>\n",
       "      <td>http://edamontology.org/format_3990</td>\n",
       "      <td>AVI</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>506</th>\n",
       "      <td>http://edamontology.org/format_3991</td>\n",
       "      <td>TrackDB</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>507</th>\n",
       "      <td>http://edamontology.org/format_3992</td>\n",
       "      <td>CIGAR format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>508</th>\n",
       "      <td>http://edamontology.org/format_3993</td>\n",
       "      <td>Stereolithography format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>509</th>\n",
       "      <td>http://edamontology.org/format_3994</td>\n",
       "      <td>U3D</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>510</th>\n",
       "      <td>http://edamontology.org/format_3995</td>\n",
       "      <td>Texture file format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>511</th>\n",
       "      <td>http://edamontology.org/format_3996</td>\n",
       "      <td>Python script</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>512</th>\n",
       "      <td>http://edamontology.org/format_3997</td>\n",
       "      <td>MPEG-4</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>513</th>\n",
       "      <td>http://edamontology.org/format_3998</td>\n",
       "      <td>Perl script</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>514</th>\n",
       "      <td>http://edamontology.org/format_3999</td>\n",
       "      <td>R script</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>515</th>\n",
       "      <td>http://edamontology.org/format_4000</td>\n",
       "      <td>R markdown</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>516</th>\n",
       "      <td>http://edamontology.org/format_4002</td>\n",
       "      <td>pickle</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>517</th>\n",
       "      <td>http://edamontology.org/format_4003</td>\n",
       "      <td>NumPy format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>518</th>\n",
       "      <td>http://edamontology.org/format_4004</td>\n",
       "      <td>SimTools repertoire file format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>519</th>\n",
       "      <td>http://edamontology.org/format_4005</td>\n",
       "      <td>Configuration file format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>520</th>\n",
       "      <td>http://edamontology.org/format_4006</td>\n",
       "      <td>Zstandard format</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>521</th>\n",
       "      <td>http://edamontology.org/format_4007</td>\n",
       "      <td>MATLAB script</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>522</th>\n",
       "      <td>http://edamontology.org/format_4018</td>\n",
       "      <td>gVCF</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>523</th>\n",
       "      <td>http://edamontology.org/format_4023</td>\n",
       "      <td>cml</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>524</th>\n",
       "      <td>http://edamontology.org/format_4025</td>\n",
       "      <td>BioSimulators format for the specifications of biosimulation tools</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>525</th>\n",
       "      <td>http://edamontology.org/format_4026</td>\n",
       "      <td>BioSimulators standard for command-line interfaces for biosimulation tools</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>526</th>\n",
       "      <td>http://edamontology.org/format_4035</td>\n",
       "      <td>PQR</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>527</th>\n",
       "      <td>http://edamontology.org/format_4036</td>\n",
       "      <td>PDBQT</td>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                  entity                                                                       label                              property\n",
       "0    http://edamontology.org/format_1196                                                                      SMILES  http://edamontology.org/is_format_of\n",
       "1    http://edamontology.org/format_1197                                                                       InChI  http://edamontology.org/is_format_of\n",
       "2    http://edamontology.org/format_1198                                                                          mf  http://edamontology.org/is_format_of\n",
       "3    http://edamontology.org/format_1199                                                                    InChIKey  http://edamontology.org/is_format_of\n",
       "4    http://edamontology.org/format_1200                                                                      smarts  http://edamontology.org/is_format_of\n",
       "5    http://edamontology.org/format_1206                                                            unambiguous pure  http://edamontology.org/is_format_of\n",
       "6    http://edamontology.org/format_1207                                                                  nucleotide  http://edamontology.org/is_format_of\n",
       "7    http://edamontology.org/format_1208                                                                     protein  http://edamontology.org/is_format_of\n",
       "8    http://edamontology.org/format_1209                                                                   consensus  http://edamontology.org/is_format_of\n",
       "9    http://edamontology.org/format_1210                                                             pure nucleotide  http://edamontology.org/is_format_of\n",
       "10   http://edamontology.org/format_1211                                                 unambiguous pure nucleotide  http://edamontology.org/is_format_of\n",
       "11   http://edamontology.org/format_1212                                                                         dna  http://edamontology.org/is_format_of\n",
       "12   http://edamontology.org/format_1213                                                                         rna  http://edamontology.org/is_format_of\n",
       "13   http://edamontology.org/format_1214                                                        unambiguous pure dna  http://edamontology.org/is_format_of\n",
       "14   http://edamontology.org/format_1215                                                                    pure dna  http://edamontology.org/is_format_of\n",
       "15   http://edamontology.org/format_1216                                               unambiguous pure rna sequence  http://edamontology.org/is_format_of\n",
       "16   http://edamontology.org/format_1217                                                                    pure rna  http://edamontology.org/is_format_of\n",
       "17   http://edamontology.org/format_1218                                                    unambiguous pure protein  http://edamontology.org/is_format_of\n",
       "18   http://edamontology.org/format_1219                                                                pure protein  http://edamontology.org/is_format_of\n",
       "19   http://edamontology.org/format_1248                                                       EMBL feature location  http://edamontology.org/is_format_of\n",
       "20   http://edamontology.org/format_1295                                                                 quicktandem  http://edamontology.org/is_format_of\n",
       "21   http://edamontology.org/format_1296                                                     Sanger inverted repeats  http://edamontology.org/is_format_of\n",
       "22   http://edamontology.org/format_1297                                                               EMBOSS repeat  http://edamontology.org/is_format_of\n",
       "23   http://edamontology.org/format_1316                                                           est2genome format  http://edamontology.org/is_format_of\n",
       "24   http://edamontology.org/format_1318                                                             restrict format  http://edamontology.org/is_format_of\n",
       "25   http://edamontology.org/format_1319                                                             restover format  http://edamontology.org/is_format_of\n",
       "26   http://edamontology.org/format_1320                                                    REBASE restriction sites  http://edamontology.org/is_format_of\n",
       "27   http://edamontology.org/format_1332                                                 FASTA search results format  http://edamontology.org/is_format_of\n",
       "28   http://edamontology.org/format_1333                                                               BLAST results  http://edamontology.org/is_format_of\n",
       "29   http://edamontology.org/format_1334                                                                   mspcrunch  http://edamontology.org/is_format_of\n",
       "30   http://edamontology.org/format_1335                                                       Smith-Waterman format  http://edamontology.org/is_format_of\n",
       "31   http://edamontology.org/format_1336                                                                         dhf  http://edamontology.org/is_format_of\n",
       "32   http://edamontology.org/format_1337                                                                         lhf  http://edamontology.org/is_format_of\n",
       "33   http://edamontology.org/format_1341                                                        InterPro hits format  http://edamontology.org/is_format_of\n",
       "34   http://edamontology.org/format_1342                                         InterPro protein view report format  http://edamontology.org/is_format_of\n",
       "35   http://edamontology.org/format_1343                                                 InterPro match table format  http://edamontology.org/is_format_of\n",
       "36   http://edamontology.org/format_1349                                                       HMMER Dirichlet prior  http://edamontology.org/is_format_of\n",
       "37   http://edamontology.org/format_1350                                                        MEME Dirichlet prior  http://edamontology.org/is_format_of\n",
       "38   http://edamontology.org/format_1351                                               HMMER emission and transition  http://edamontology.org/is_format_of\n",
       "39   http://edamontology.org/format_1356                                                             prosite-pattern  http://edamontology.org/is_format_of\n",
       "40   http://edamontology.org/format_1357                                                     EMBOSS sequence pattern  http://edamontology.org/is_format_of\n",
       "41   http://edamontology.org/format_1360                                                                  meme-motif  http://edamontology.org/is_format_of\n",
       "42   http://edamontology.org/format_1366                                                             prosite-profile  http://edamontology.org/is_format_of\n",
       "43   http://edamontology.org/format_1367                                                               JASPAR format  http://edamontology.org/is_format_of\n",
       "44   http://edamontology.org/format_1369                                                MEME background Markov model  http://edamontology.org/is_format_of\n",
       "45   http://edamontology.org/format_1370                                                                HMMER format  http://edamontology.org/is_format_of\n",
       "46   http://edamontology.org/format_1391                                                                   HMMER-aln  http://edamontology.org/is_format_of\n",
       "47   http://edamontology.org/format_1392                                                              DIALIGN format  http://edamontology.org/is_format_of\n",
       "48   http://edamontology.org/format_1393                                                                         daf  http://edamontology.org/is_format_of\n",
       "49   http://edamontology.org/format_1419                                             Sequence-MEME profile alignment  http://edamontology.org/is_format_of\n",
       "50   http://edamontology.org/format_1421                             HMMER profile alignment (sequences versus HMMs)  http://edamontology.org/is_format_of\n",
       "51   http://edamontology.org/format_1422                              HMMER profile alignment (HMM versus sequences)  http://edamontology.org/is_format_of\n",
       "52   http://edamontology.org/format_1423                                                      Phylip distance matrix  http://edamontology.org/is_format_of\n",
       "53   http://edamontology.org/format_1424                                                         ClustalW dendrogram  http://edamontology.org/is_format_of\n",
       "54   http://edamontology.org/format_1425                                                             Phylip tree raw  http://edamontology.org/is_format_of\n",
       "55   http://edamontology.org/format_1430                                   Phylip continuous quantitative characters  http://edamontology.org/is_format_of\n",
       "56   http://edamontology.org/format_1432                                         Phylip character frequencies format  http://edamontology.org/is_format_of\n",
       "57   http://edamontology.org/format_1433                                               Phylip discrete states format  http://edamontology.org/is_format_of\n",
       "58   http://edamontology.org/format_1434                                                       Phylip cliques format  http://edamontology.org/is_format_of\n",
       "59   http://edamontology.org/format_1435                                                          Phylip tree format  http://edamontology.org/is_format_of\n",
       "60   http://edamontology.org/format_1436                                                             TreeBASE format  http://edamontology.org/is_format_of\n",
       "61   http://edamontology.org/format_1437                                                              TreeFam format  http://edamontology.org/is_format_of\n",
       "62   http://edamontology.org/format_1445                                                 Phylip tree distance format  http://edamontology.org/is_format_of\n",
       "63   http://edamontology.org/format_1454                                                                        dssp  http://edamontology.org/is_format_of\n",
       "64   http://edamontology.org/format_1455                                                                        hssp  http://edamontology.org/is_format_of\n",
       "65   http://edamontology.org/format_1457                                                          Dot-bracket format  http://edamontology.org/is_format_of\n",
       "66   http://edamontology.org/format_1458                                 Vienna local RNA secondary structure format  http://edamontology.org/is_format_of\n",
       "67   http://edamontology.org/format_1476                                                                         PDB  http://edamontology.org/is_format_of\n",
       "68   http://edamontology.org/format_1477                                                                       mmCIF  http://edamontology.org/is_format_of\n",
       "69   http://edamontology.org/format_1478                                                                       PDBML  http://edamontology.org/is_format_of\n",
       "70   http://edamontology.org/format_1504                                                                     aaindex  http://edamontology.org/is_format_of\n",
       "71   http://edamontology.org/format_1551                                                         Pcons report format  http://edamontology.org/is_format_of\n",
       "72   http://edamontology.org/format_1552                                                          ProQ report format  http://edamontology.org/is_format_of\n",
       "73   http://edamontology.org/format_1582                                                                      findkm  http://edamontology.org/is_format_of\n",
       "74   http://edamontology.org/format_1627                                                              Primer3 primer  http://edamontology.org/is_format_of\n",
       "75   http://edamontology.org/format_1628                                                                         ABI  http://edamontology.org/is_format_of\n",
       "76   http://edamontology.org/format_1629                                                                        mira  http://edamontology.org/is_format_of\n",
       "77   http://edamontology.org/format_1630                                                                         CAF  http://edamontology.org/is_format_of\n",
       "78   http://edamontology.org/format_1631                                                                         EXP  http://edamontology.org/is_format_of\n",
       "79   http://edamontology.org/format_1632                                                                         SCF  http://edamontology.org/is_format_of\n",
       "80   http://edamontology.org/format_1633                                                                         PHD  http://edamontology.org/is_format_of\n",
       "81   http://edamontology.org/format_1639                                                                  affymetrix  http://edamontology.org/is_format_of\n",
       "82   http://edamontology.org/format_1641                                                              affymetrix-exp  http://edamontology.org/is_format_of\n",
       "83   http://edamontology.org/format_1665                                                     Taverna workflow format  http://edamontology.org/is_format_of\n",
       "84   http://edamontology.org/format_1705                                           HET group dictionary entry format  http://edamontology.org/is_format_of\n",
       "85   http://edamontology.org/format_1734                                                             PubMed citation  http://edamontology.org/is_format_of\n",
       "86   http://edamontology.org/format_1735                                                      Medline Display Format  http://edamontology.org/is_format_of\n",
       "87   http://edamontology.org/format_1736                                                             CiteXplore-core  http://edamontology.org/is_format_of\n",
       "88   http://edamontology.org/format_1737                                                              CiteXplore-all  http://edamontology.org/is_format_of\n",
       "89   http://edamontology.org/format_1739                                                                         pmc  http://edamontology.org/is_format_of\n",
       "90   http://edamontology.org/format_1740                                                                 iHOP format  http://edamontology.org/is_format_of\n",
       "91   http://edamontology.org/format_1741                                                                OSCAR format  http://edamontology.org/is_format_of\n",
       "92   http://edamontology.org/format_1861                                                          PlasMapper TextMap  http://edamontology.org/is_format_of\n",
       "93   http://edamontology.org/format_1910                                                                      newick  http://edamontology.org/is_format_of\n",
       "94   http://edamontology.org/format_1911                                                              TreeCon format  http://edamontology.org/is_format_of\n",
       "95   http://edamontology.org/format_1912                                                                Nexus format  http://edamontology.org/is_format_of\n",
       "96   http://edamontology.org/format_1923                                                                       acedb  http://edamontology.org/is_format_of\n",
       "97   http://edamontology.org/format_1925                                                                      codata  http://edamontology.org/is_format_of\n",
       "98   http://edamontology.org/format_1926                                                                        dbid  http://edamontology.org/is_format_of\n",
       "99   http://edamontology.org/format_1927                                                                 EMBL format  http://edamontology.org/is_format_of\n",
       "100  http://edamontology.org/format_1928                                                    Staden experiment format  http://edamontology.org/is_format_of\n",
       "101  http://edamontology.org/format_1929                                                                       FASTA  http://edamontology.org/is_format_of\n",
       "102  http://edamontology.org/format_1930                                                                       FASTQ  http://edamontology.org/is_format_of\n",
       "103  http://edamontology.org/format_1931                                                              FASTQ-illumina  http://edamontology.org/is_format_of\n",
       "104  http://edamontology.org/format_1932                                                                FASTQ-sanger  http://edamontology.org/is_format_of\n",
       "105  http://edamontology.org/format_1933                                                                FASTQ-solexa  http://edamontology.org/is_format_of\n",
       "106  http://edamontology.org/format_1934                                                               fitch program  http://edamontology.org/is_format_of\n",
       "107  http://edamontology.org/format_1935                                                                         GCG  http://edamontology.org/is_format_of\n",
       "108  http://edamontology.org/format_1936                                                              GenBank format  http://edamontology.org/is_format_of\n",
       "109  http://edamontology.org/format_1937                                                                     genpept  http://edamontology.org/is_format_of\n",
       "110  http://edamontology.org/format_1938                                                                    GFF2-seq  http://edamontology.org/is_format_of\n",
       "111  http://edamontology.org/format_1939                                                                    GFF3-seq  http://edamontology.org/is_format_of\n",
       "112  http://edamontology.org/format_1940                                                              giFASTA format  http://edamontology.org/is_format_of\n",
       "113  http://edamontology.org/format_1941                                                                    hennig86  http://edamontology.org/is_format_of\n",
       "114  http://edamontology.org/format_1942                                                                          ig  http://edamontology.org/is_format_of\n",
       "115  http://edamontology.org/format_1943                                                                    igstrict  http://edamontology.org/is_format_of\n",
       "116  http://edamontology.org/format_1944                                                                  jackknifer  http://edamontology.org/is_format_of\n",
       "117  http://edamontology.org/format_1945                                                                 mase format  http://edamontology.org/is_format_of\n",
       "118  http://edamontology.org/format_1946                                                                    mega-seq  http://edamontology.org/is_format_of\n",
       "119  http://edamontology.org/format_1947                                                                     GCG MSF  http://edamontology.org/is_format_of\n",
       "120  http://edamontology.org/format_1948                                                                    nbrf/pir  http://edamontology.org/is_format_of\n",
       "121  http://edamontology.org/format_1949                                                                   nexus-seq  http://edamontology.org/is_format_of\n",
       "122  http://edamontology.org/format_1950                                                                     pdbatom  http://edamontology.org/is_format_of\n",
       "123  http://edamontology.org/format_1951                                                                  pdbatomnuc  http://edamontology.org/is_format_of\n",
       "124  http://edamontology.org/format_1952                                                                pdbseqresnuc  http://edamontology.org/is_format_of\n",
       "125  http://edamontology.org/format_1953                                                                   pdbseqres  http://edamontology.org/is_format_of\n",
       "126  http://edamontology.org/format_1954                                                              Pearson format  http://edamontology.org/is_format_of\n",
       "127  http://edamontology.org/format_1957                                                                         raw  http://edamontology.org/is_format_of\n",
       "128  http://edamontology.org/format_1958                                                                     refseqp  http://edamontology.org/is_format_of\n",
       "129  http://edamontology.org/format_1960                                                               Staden format  http://edamontology.org/is_format_of\n",
       "130  http://edamontology.org/format_1961                                                            Stockholm format  http://edamontology.org/is_format_of\n",
       "131  http://edamontology.org/format_1962                                                              strider format  http://edamontology.org/is_format_of\n",
       "132  http://edamontology.org/format_1963                                                            UniProtKB format  http://edamontology.org/is_format_of\n",
       "133  http://edamontology.org/format_1964                                             plain text format (unformatted)  http://edamontology.org/is_format_of\n",
       "134  http://edamontology.org/format_1966                                                       ASN.1 sequence format  http://edamontology.org/is_format_of\n",
       "135  http://edamontology.org/format_1967                                                                  DAS format  http://edamontology.org/is_format_of\n",
       "136  http://edamontology.org/format_1968                                                                      dasdna  http://edamontology.org/is_format_of\n",
       "137  http://edamontology.org/format_1969                                                                   debug-seq  http://edamontology.org/is_format_of\n",
       "138  http://edamontology.org/format_1970                                                               jackknifernon  http://edamontology.org/is_format_of\n",
       "139  http://edamontology.org/format_1972                                                                 NCBI format  http://edamontology.org/is_format_of\n",
       "140  http://edamontology.org/format_1973                                                                    nexusnon  http://edamontology.org/is_format_of\n",
       "141  http://edamontology.org/format_1974                                                                        GFF2  http://edamontology.org/is_format_of\n",
       "142  http://edamontology.org/format_1975                                                                        GFF3  http://edamontology.org/is_format_of\n",
       "143  http://edamontology.org/format_1978                                                                      DASGFF  http://edamontology.org/is_format_of\n",
       "144  http://edamontology.org/format_1979                                                                  debug-feat  http://edamontology.org/is_format_of\n",
       "145  http://edamontology.org/format_1982                                                             ClustalW format  http://edamontology.org/is_format_of\n",
       "146  http://edamontology.org/format_1983                                                                       debug  http://edamontology.org/is_format_of\n",
       "147  http://edamontology.org/format_1984                                                                   FASTA-aln  http://edamontology.org/is_format_of\n",
       "148  http://edamontology.org/format_1985                                                                      markx0  http://edamontology.org/is_format_of\n",
       "149  http://edamontology.org/format_1986                                                                      markx1  http://edamontology.org/is_format_of\n",
       "150  http://edamontology.org/format_1987                                                                     markx10  http://edamontology.org/is_format_of\n",
       "151  http://edamontology.org/format_1988                                                                      markx2  http://edamontology.org/is_format_of\n",
       "152  http://edamontology.org/format_1989                                                                      markx3  http://edamontology.org/is_format_of\n",
       "153  http://edamontology.org/format_1990                                                                       match  http://edamontology.org/is_format_of\n",
       "154  http://edamontology.org/format_1991                                                                        mega  http://edamontology.org/is_format_of\n",
       "155  http://edamontology.org/format_1992                                                                     meganon  http://edamontology.org/is_format_of\n",
       "156  http://edamontology.org/format_1996                                                                        pair  http://edamontology.org/is_format_of\n",
       "157  http://edamontology.org/format_1997                                                               PHYLIP format  http://edamontology.org/is_format_of\n",
       "158  http://edamontology.org/format_1998                                                           PHYLIP sequential  http://edamontology.org/is_format_of\n",
       "159  http://edamontology.org/format_1999                                                               scores format  http://edamontology.org/is_format_of\n",
       "160  http://edamontology.org/format_2000                                                                       selex  http://edamontology.org/is_format_of\n",
       "161  http://edamontology.org/format_2001                                                        EMBOSS simple format  http://edamontology.org/is_format_of\n",
       "162  http://edamontology.org/format_2002                                                                  srs format  http://edamontology.org/is_format_of\n",
       "163  http://edamontology.org/format_2003                                                                     srspair  http://edamontology.org/is_format_of\n",
       "164  http://edamontology.org/format_2004                                                             T-Coffee format  http://edamontology.org/is_format_of\n",
       "165  http://edamontology.org/format_2005                                                                 TreeCon-seq  http://edamontology.org/is_format_of\n",
       "166  http://edamontology.org/format_2032                                                             Workflow format  http://edamontology.org/is_format_of\n",
       "167  http://edamontology.org/format_2033                                                   Tertiary structure format  http://edamontology.org/is_format_of\n",
       "168  http://edamontology.org/format_2056                                           Microarray experiment data format  http://edamontology.org/is_format_of\n",
       "169  http://edamontology.org/format_2061                                      Nucleic acid features (primers) format  http://edamontology.org/is_format_of\n",
       "170  http://edamontology.org/format_2077                                          Protein secondary structure format  http://edamontology.org/is_format_of\n",
       "171  http://edamontology.org/format_2094                                                                        pure  http://edamontology.org/is_format_of\n",
       "172  http://edamontology.org/format_2095                                                                      unpure  http://edamontology.org/is_format_of\n",
       "173  http://edamontology.org/format_2096                                                        unambiguous sequence  http://edamontology.org/is_format_of\n",
       "174  http://edamontology.org/format_2097                                                                   ambiguous  http://edamontology.org/is_format_of\n",
       "175  http://edamontology.org/format_2155                                          Sequence features (repeats) format  http://edamontology.org/is_format_of\n",
       "176  http://edamontology.org/format_2158                            Nucleic acid features (restriction sites) format  http://edamontology.org/is_format_of\n",
       "177  http://edamontology.org/format_2171                                           Sequence cluster format (protein)  http://edamontology.org/is_format_of\n",
       "178  http://edamontology.org/format_2172                                      Sequence cluster format (nucleic acid)  http://edamontology.org/is_format_of\n",
       "179  http://edamontology.org/format_2181                                                            EMBL-like (text)  http://edamontology.org/is_format_of\n",
       "180  http://edamontology.org/format_2182                                                    FASTQ-like format (text)  http://edamontology.org/is_format_of\n",
       "181  http://edamontology.org/format_2183                                                                     EMBLXML  http://edamontology.org/is_format_of\n",
       "182  http://edamontology.org/format_2184                                                                      cdsxml  http://edamontology.org/is_format_of\n",
       "183  http://edamontology.org/format_2185                                                                     INSDSeq  http://edamontology.org/is_format_of\n",
       "184  http://edamontology.org/format_2186                                                                     geneseq  http://edamontology.org/is_format_of\n",
       "185  http://edamontology.org/format_2187                                                         UniProt-like (text)  http://edamontology.org/is_format_of\n",
       "186  http://edamontology.org/format_2194                                                                     medline  http://edamontology.org/is_format_of\n",
       "187  http://edamontology.org/format_2196                                                                  OBO format  http://edamontology.org/is_format_of\n",
       "188  http://edamontology.org/format_2197                                                                  OWL format  http://edamontology.org/is_format_of\n",
       "189  http://edamontology.org/format_2200                                                           FASTA-like (text)  http://edamontology.org/is_format_of\n",
       "190  http://edamontology.org/format_2204                                                           EMBL format (XML)  http://edamontology.org/is_format_of\n",
       "191  http://edamontology.org/format_2205                                                  GenBank-like format (text)  http://edamontology.org/is_format_of\n",
       "192  http://edamontology.org/format_2206                                        Sequence feature table format (text)  http://edamontology.org/is_format_of\n",
       "193  http://edamontology.org/format_2304                                                   STRING entry format (XML)  http://edamontology.org/is_format_of\n",
       "194  http://edamontology.org/format_2305                                                                         GFF  http://edamontology.org/is_format_of\n",
       "195  http://edamontology.org/format_2306                                                                         GTF  http://edamontology.org/is_format_of\n",
       "196  http://edamontology.org/format_2310                                                                  FASTA-HTML  http://edamontology.org/is_format_of\n",
       "197  http://edamontology.org/format_2311                                                                   EMBL-HTML  http://edamontology.org/is_format_of\n",
       "198  http://edamontology.org/format_2330                                                              Textual format  http://edamontology.org/is_format_of\n",
       "199  http://edamontology.org/format_2332                                                                         XML  http://edamontology.org/is_format_of\n",
       "200  http://edamontology.org/format_2333                                                               Binary format  http://edamontology.org/is_format_of\n",
       "201  http://edamontology.org/format_2350                                                    Format (by type of data)  http://edamontology.org/is_format_of\n",
       "202  http://edamontology.org/format_2376                                                                  RDF format  http://edamontology.org/is_format_of\n",
       "203  http://edamontology.org/format_2532                                                                GenBank-HTML  http://edamontology.org/is_format_of\n",
       "204  http://edamontology.org/format_2543                                                            EMBL-like format  http://edamontology.org/is_format_of\n",
       "205  http://edamontology.org/format_2545                                                           FASTQ-like format  http://edamontology.org/is_format_of\n",
       "206  http://edamontology.org/format_2546                                                                  FASTA-like  http://edamontology.org/is_format_of\n",
       "207  http://edamontology.org/format_2547                                                       uniprotkb-like format  http://edamontology.org/is_format_of\n",
       "208  http://edamontology.org/format_2549                                                                         OBO  http://edamontology.org/is_format_of\n",
       "209  http://edamontology.org/format_2550                                                                     OBO-XML  http://edamontology.org/is_format_of\n",
       "210  http://edamontology.org/format_2551                                               Sequence record format (text)  http://edamontology.org/is_format_of\n",
       "211  http://edamontology.org/format_2552                                                Sequence record format (XML)  http://edamontology.org/is_format_of\n",
       "212  http://edamontology.org/format_2553                                         Sequence feature table format (XML)  http://edamontology.org/is_format_of\n",
       "213  http://edamontology.org/format_2554                                                     Alignment format (text)  http://edamontology.org/is_format_of\n",
       "214  http://edamontology.org/format_2555                                                      Alignment format (XML)  http://edamontology.org/is_format_of\n",
       "215  http://edamontology.org/format_2556                                             Phylogenetic tree format (text)  http://edamontology.org/is_format_of\n",
       "216  http://edamontology.org/format_2557                                              Phylogenetic tree format (XML)  http://edamontology.org/is_format_of\n",
       "217  http://edamontology.org/format_2558                                                             EMBL-like (XML)  http://edamontology.org/is_format_of\n",
       "218  http://edamontology.org/format_2559                                                         GenBank-like format  http://edamontology.org/is_format_of\n",
       "219  http://edamontology.org/format_2561                                             Sequence assembly format (text)  http://edamontology.org/is_format_of\n",
       "220  http://edamontology.org/format_2566                                                      completely unambiguous  http://edamontology.org/is_format_of\n",
       "221  http://edamontology.org/format_2567                                                 completely unambiguous pure  http://edamontology.org/is_format_of\n",
       "222  http://edamontology.org/format_2568                                      completely unambiguous pure nucleotide  http://edamontology.org/is_format_of\n",
       "223  http://edamontology.org/format_2569                                             completely unambiguous pure dna  http://edamontology.org/is_format_of\n",
       "224  http://edamontology.org/format_2570                                    completely unambiguous pure rna sequence  http://edamontology.org/is_format_of\n",
       "225  http://edamontology.org/format_2572                                                                         BAM  http://edamontology.org/is_format_of\n",
       "226  http://edamontology.org/format_2573                                                                         SAM  http://edamontology.org/is_format_of\n",
       "227  http://edamontology.org/format_2585                                                                        SBML  http://edamontology.org/is_format_of\n",
       "228  http://edamontology.org/format_2607                                         completely unambiguous pure protein  http://edamontology.org/is_format_of\n",
       "229  http://edamontology.org/format_2922                                                              markx0 variant  http://edamontology.org/is_format_of\n",
       "230  http://edamontology.org/format_2923                                                                mega variant  http://edamontology.org/is_format_of\n",
       "231  http://edamontology.org/format_2924                                                       Phylip format variant  http://edamontology.org/is_format_of\n",
       "232  http://edamontology.org/format_3000                                                                         AB1  http://edamontology.org/is_format_of\n",
       "233  http://edamontology.org/format_3001                                                                         ACE  http://edamontology.org/is_format_of\n",
       "234  http://edamontology.org/format_3003                                                                         BED  http://edamontology.org/is_format_of\n",
       "235  http://edamontology.org/format_3004                                                                      bigBed  http://edamontology.org/is_format_of\n",
       "236  http://edamontology.org/format_3005                                                                         WIG  http://edamontology.org/is_format_of\n",
       "237  http://edamontology.org/format_3006                                                                      bigWig  http://edamontology.org/is_format_of\n",
       "238  http://edamontology.org/format_3007                                                                         PSL  http://edamontology.org/is_format_of\n",
       "239  http://edamontology.org/format_3008                                                                         MAF  http://edamontology.org/is_format_of\n",
       "240  http://edamontology.org/format_3009                                                                        2bit  http://edamontology.org/is_format_of\n",
       "241  http://edamontology.org/format_3010                                                                        .nib  http://edamontology.org/is_format_of\n",
       "242  http://edamontology.org/format_3011                                                                    genePred  http://edamontology.org/is_format_of\n",
       "243  http://edamontology.org/format_3012                                                                       pgSnp  http://edamontology.org/is_format_of\n",
       "244  http://edamontology.org/format_3013                                                                         axt  http://edamontology.org/is_format_of\n",
       "245  http://edamontology.org/format_3014                                                                         LAV  http://edamontology.org/is_format_of\n",
       "246  http://edamontology.org/format_3015                                                                      Pileup  http://edamontology.org/is_format_of\n",
       "247  http://edamontology.org/format_3016                                                                         VCF  http://edamontology.org/is_format_of\n",
       "248  http://edamontology.org/format_3017                                                                         SRF  http://edamontology.org/is_format_of\n",
       "249  http://edamontology.org/format_3018                                                                         ZTR  http://edamontology.org/is_format_of\n",
       "250  http://edamontology.org/format_3019                                                                         GVF  http://edamontology.org/is_format_of\n",
       "251  http://edamontology.org/format_3020                                                                         BCF  http://edamontology.org/is_format_of\n",
       "252  http://edamontology.org/format_3098                                       Raw SCOP domain classification format  http://edamontology.org/is_format_of\n",
       "253  http://edamontology.org/format_3099                                       Raw CATH domain classification format  http://edamontology.org/is_format_of\n",
       "254  http://edamontology.org/format_3100                                                   CATH domain report format  http://edamontology.org/is_format_of\n",
       "255  http://edamontology.org/format_3155                                                                       SBRML  http://edamontology.org/is_format_of\n",
       "256  http://edamontology.org/format_3156                                                                      BioPAX  http://edamontology.org/is_format_of\n",
       "257  http://edamontology.org/format_3157                                                  EBI Application Result XML  http://edamontology.org/is_format_of\n",
       "258  http://edamontology.org/format_3158                                                            PSI MI XML (MIF)  http://edamontology.org/is_format_of\n",
       "259  http://edamontology.org/format_3159                                                                    phyloXML  http://edamontology.org/is_format_of\n",
       "260  http://edamontology.org/format_3160                                                                       NeXML  http://edamontology.org/is_format_of\n",
       "261  http://edamontology.org/format_3163                                                                       GCDML  http://edamontology.org/is_format_of\n",
       "262  http://edamontology.org/format_3164                                                                      GTrack  http://edamontology.org/is_format_of\n",
       "263  http://edamontology.org/format_3239                                                                    CopasiML  http://edamontology.org/is_format_of\n",
       "264  http://edamontology.org/format_3240                                                                      CellML  http://edamontology.org/is_format_of\n",
       "265  http://edamontology.org/format_3242                                                          PSI MI TAB (MITAB)  http://edamontology.org/is_format_of\n",
       "266  http://edamontology.org/format_3243                                                                     PSI-PAR  http://edamontology.org/is_format_of\n",
       "267  http://edamontology.org/format_3244                                                                        mzML  http://edamontology.org/is_format_of\n",
       "268  http://edamontology.org/format_3246                                                                       TraML  http://edamontology.org/is_format_of\n",
       "269  http://edamontology.org/format_3247                                                                   mzIdentML  http://edamontology.org/is_format_of\n",
       "270  http://edamontology.org/format_3248                                                                   mzQuantML  http://edamontology.org/is_format_of\n",
       "271  http://edamontology.org/format_3249                                                                       GelML  http://edamontology.org/is_format_of\n",
       "272  http://edamontology.org/format_3250                                                                        spML  http://edamontology.org/is_format_of\n",
       "273  http://edamontology.org/format_3252                                                       OWL Functional Syntax  http://edamontology.org/is_format_of\n",
       "274  http://edamontology.org/format_3253                                                       Manchester OWL Syntax  http://edamontology.org/is_format_of\n",
       "275  http://edamontology.org/format_3254                                                                KRSS2 Syntax  http://edamontology.org/is_format_of\n",
       "276  http://edamontology.org/format_3255                                                                      Turtle  http://edamontology.org/is_format_of\n",
       "277  http://edamontology.org/format_3256                                                                   N-Triples  http://edamontology.org/is_format_of\n",
       "278  http://edamontology.org/format_3257                                                                   Notation3  http://edamontology.org/is_format_of\n",
       "279  http://edamontology.org/format_3261                                                                     RDF/XML  http://edamontology.org/is_format_of\n",
       "280  http://edamontology.org/format_3262                                                                     OWL/XML  http://edamontology.org/is_format_of\n",
       "281  http://edamontology.org/format_3281                                                                         A2M  http://edamontology.org/is_format_of\n",
       "282  http://edamontology.org/format_3284                                                                         SFF  http://edamontology.org/is_format_of\n",
       "283  http://edamontology.org/format_3285                                                                         MAP  http://edamontology.org/is_format_of\n",
       "284  http://edamontology.org/format_3286                                                                         PED  http://edamontology.org/is_format_of\n",
       "285  http://edamontology.org/format_3287                                              Individual genetic data format  http://edamontology.org/is_format_of\n",
       "286  http://edamontology.org/format_3288                                                                     PED/MAP  http://edamontology.org/is_format_of\n",
       "287  http://edamontology.org/format_3309                                                                          CT  http://edamontology.org/is_format_of\n",
       "288  http://edamontology.org/format_3310                                                                          SS  http://edamontology.org/is_format_of\n",
       "289  http://edamontology.org/format_3311                                                                       RNAML  http://edamontology.org/is_format_of\n",
       "290  http://edamontology.org/format_3312                                                                         GDE  http://edamontology.org/is_format_of\n",
       "291  http://edamontology.org/format_3313                                                                         BLC  http://edamontology.org/is_format_of\n",
       "292  http://edamontology.org/format_3328                                                                      HMMER2  http://edamontology.org/is_format_of\n",
       "293  http://edamontology.org/format_3329                                                                      HMMER3  http://edamontology.org/is_format_of\n",
       "294  http://edamontology.org/format_3330                                                                          PO  http://edamontology.org/is_format_of\n",
       "295  http://edamontology.org/format_3331                                                    BLAST XML results format  http://edamontology.org/is_format_of\n",
       "296  http://edamontology.org/format_3462                                                                        CRAM  http://edamontology.org/is_format_of\n",
       "297  http://edamontology.org/format_3464                                                                        JSON  http://edamontology.org/is_format_of\n",
       "298  http://edamontology.org/format_3466                                                                         EPS  http://edamontology.org/is_format_of\n",
       "299  http://edamontology.org/format_3467                                                                         GIF  http://edamontology.org/is_format_of\n",
       "300  http://edamontology.org/format_3468                                                                         xls  http://edamontology.org/is_format_of\n",
       "301  http://edamontology.org/format_3475                                                                         TSV  http://edamontology.org/is_format_of\n",
       "302  http://edamontology.org/format_3477                                                 Cytoscape input file format  http://edamontology.org/is_format_of\n",
       "303  http://edamontology.org/format_3485                                                                         RSF  http://edamontology.org/is_format_of\n",
       "304  http://edamontology.org/format_3486                                                          GCG format variant  http://edamontology.org/is_format_of\n",
       "305  http://edamontology.org/format_3487                                                                        BSML  http://edamontology.org/is_format_of\n",
       "306  http://edamontology.org/format_3499                                               Ensembl variation file format  http://edamontology.org/is_format_of\n",
       "307  http://edamontology.org/format_3506                                                                        docx  http://edamontology.org/is_format_of\n",
       "308  http://edamontology.org/format_3507                                                             Document format  http://edamontology.org/is_format_of\n",
       "309  http://edamontology.org/format_3508                                                                         PDF  http://edamontology.org/is_format_of\n",
       "310  http://edamontology.org/format_3548                                                                DICOM format  http://edamontology.org/is_format_of\n",
       "311  http://edamontology.org/format_3549                                                                         nii  http://edamontology.org/is_format_of\n",
       "312  http://edamontology.org/format_3550                                                                         mhd  http://edamontology.org/is_format_of\n",
       "313  http://edamontology.org/format_3551                                                                        nrrd  http://edamontology.org/is_format_of\n",
       "314  http://edamontology.org/format_3554                                                               R file format  http://edamontology.org/is_format_of\n",
       "315  http://edamontology.org/format_3555                                                                        SPSS  http://edamontology.org/is_format_of\n",
       "316  http://edamontology.org/format_3556                                                                       MHTML  http://edamontology.org/is_format_of\n",
       "317  http://edamontology.org/format_3579                                                                         JPG  http://edamontology.org/is_format_of\n",
       "318  http://edamontology.org/format_3580                                                                         rcc  http://edamontology.org/is_format_of\n",
       "319  http://edamontology.org/format_3581                                                                        arff  http://edamontology.org/is_format_of\n",
       "320  http://edamontology.org/format_3582                                                                         afg  http://edamontology.org/is_format_of\n",
       "321  http://edamontology.org/format_3583                                                                    bedgraph  http://edamontology.org/is_format_of\n",
       "322  http://edamontology.org/format_3584                                                                   bedstrict  http://edamontology.org/is_format_of\n",
       "323  http://edamontology.org/format_3585                                                                        bed6  http://edamontology.org/is_format_of\n",
       "324  http://edamontology.org/format_3586                                                                       bed12  http://edamontology.org/is_format_of\n",
       "325  http://edamontology.org/format_3587                                                                   chrominfo  http://edamontology.org/is_format_of\n",
       "326  http://edamontology.org/format_3588                                                                 customtrack  http://edamontology.org/is_format_of\n",
       "327  http://edamontology.org/format_3589                                                                     csfasta  http://edamontology.org/is_format_of\n",
       "328  http://edamontology.org/format_3590                                                                        HDF5  http://edamontology.org/is_format_of\n",
       "329  http://edamontology.org/format_3591                                                                        TIFF  http://edamontology.org/is_format_of\n",
       "330  http://edamontology.org/format_3592                                                                         BMP  http://edamontology.org/is_format_of\n",
       "331  http://edamontology.org/format_3593                                                                          im  http://edamontology.org/is_format_of\n",
       "332  http://edamontology.org/format_3594                                                                         pcd  http://edamontology.org/is_format_of\n",
       "333  http://edamontology.org/format_3595                                                                         pcx  http://edamontology.org/is_format_of\n",
       "334  http://edamontology.org/format_3596                                                                         ppm  http://edamontology.org/is_format_of\n",
       "335  http://edamontology.org/format_3597                                                                         psd  http://edamontology.org/is_format_of\n",
       "336  http://edamontology.org/format_3598                                                                         xbm  http://edamontology.org/is_format_of\n",
       "337  http://edamontology.org/format_3599                                                                         xpm  http://edamontology.org/is_format_of\n",
       "338  http://edamontology.org/format_3600                                                                         rgb  http://edamontology.org/is_format_of\n",
       "339  http://edamontology.org/format_3601                                                                         pbm  http://edamontology.org/is_format_of\n",
       "340  http://edamontology.org/format_3602                                                                         pgm  http://edamontology.org/is_format_of\n",
       "341  http://edamontology.org/format_3603                                                                         PNG  http://edamontology.org/is_format_of\n",
       "342  http://edamontology.org/format_3604                                                                         SVG  http://edamontology.org/is_format_of\n",
       "343  http://edamontology.org/format_3605                                                                        rast  http://edamontology.org/is_format_of\n",
       "344  http://edamontology.org/format_3607                                                                        qual  http://edamontology.org/is_format_of\n",
       "345  http://edamontology.org/format_3608                                                                  qualsolexa  http://edamontology.org/is_format_of\n",
       "346  http://edamontology.org/format_3609                                                                qualillumina  http://edamontology.org/is_format_of\n",
       "347  http://edamontology.org/format_3610                                                                   qualsolid  http://edamontology.org/is_format_of\n",
       "348  http://edamontology.org/format_3611                                                                     qual454  http://edamontology.org/is_format_of\n",
       "349  http://edamontology.org/format_3612                                                          ENCODE peak format  http://edamontology.org/is_format_of\n",
       "350  http://edamontology.org/format_3613                                                   ENCODE narrow peak format  http://edamontology.org/is_format_of\n",
       "351  http://edamontology.org/format_3614                                                    ENCODE broad peak format  http://edamontology.org/is_format_of\n",
       "352  http://edamontology.org/format_3615                                                                       bgzip  http://edamontology.org/is_format_of\n",
       "353  http://edamontology.org/format_3616                                                                       tabix  http://edamontology.org/is_format_of\n",
       "354  http://edamontology.org/format_3617                                                                Graph format  http://edamontology.org/is_format_of\n",
       "355  http://edamontology.org/format_3618                                                                       xgmml  http://edamontology.org/is_format_of\n",
       "356  http://edamontology.org/format_3619                                                                         sif  http://edamontology.org/is_format_of\n",
       "357  http://edamontology.org/format_3620                                                                        xlsx  http://edamontology.org/is_format_of\n",
       "358  http://edamontology.org/format_3621                                                               SQLite format  http://edamontology.org/is_format_of\n",
       "359  http://edamontology.org/format_3622                                                        Gemini SQLite format  http://edamontology.org/is_format_of\n",
       "360  http://edamontology.org/format_3624                                                                    snpeffdb  http://edamontology.org/is_format_of\n",
       "361  http://edamontology.org/format_3650                                                                      netCDF  http://edamontology.org/is_format_of\n",
       "362  http://edamontology.org/format_3651                                                                         MGF  http://edamontology.org/is_format_of\n",
       "363  http://edamontology.org/format_3652                                                                         dta  http://edamontology.org/is_format_of\n",
       "364  http://edamontology.org/format_3653                                                                         pkl  http://edamontology.org/is_format_of\n",
       "365  http://edamontology.org/format_3654                                                                       mzXML  http://edamontology.org/is_format_of\n",
       "366  http://edamontology.org/format_3655                                                                      pepXML  http://edamontology.org/is_format_of\n",
       "367  http://edamontology.org/format_3657                                                                        GPML  http://edamontology.org/is_format_of\n",
       "368  http://edamontology.org/format_3665                                                            K-mer countgraph  http://edamontology.org/is_format_of\n",
       "369  http://edamontology.org/format_3681                                                                       mzTab  http://edamontology.org/is_format_of\n",
       "370  http://edamontology.org/format_3682                                                         imzML metadata file  http://edamontology.org/is_format_of\n",
       "371  http://edamontology.org/format_3683                                                                        qcML  http://edamontology.org/is_format_of\n",
       "372  http://edamontology.org/format_3684                                                                   PRIDE XML  http://edamontology.org/is_format_of\n",
       "373  http://edamontology.org/format_3685                                                                      SED-ML  http://edamontology.org/is_format_of\n",
       "374  http://edamontology.org/format_3686                                                                COMBINE OMEX  http://edamontology.org/is_format_of\n",
       "375  http://edamontology.org/format_3687                                                                     ISA-TAB  http://edamontology.org/is_format_of\n",
       "376  http://edamontology.org/format_3688                                                                       SBtab  http://edamontology.org/is_format_of\n",
       "377  http://edamontology.org/format_3689                                                                        BCML  http://edamontology.org/is_format_of\n",
       "378  http://edamontology.org/format_3690                                                                        BDML  http://edamontology.org/is_format_of\n",
       "379  http://edamontology.org/format_3691                                                                         BEL  http://edamontology.org/is_format_of\n",
       "380  http://edamontology.org/format_3692                                                                     SBGN-ML  http://edamontology.org/is_format_of\n",
       "381  http://edamontology.org/format_3693                                                                         AGP  http://edamontology.org/is_format_of\n",
       "382  http://edamontology.org/format_3696                                                                          PS  http://edamontology.org/is_format_of\n",
       "383  http://edamontology.org/format_3698                                                                  SRA format  http://edamontology.org/is_format_of\n",
       "384  http://edamontology.org/format_3699                                                                         VDB  http://edamontology.org/is_format_of\n",
       "385  http://edamontology.org/format_3701                                                               Sequin format  http://edamontology.org/is_format_of\n",
       "386  http://edamontology.org/format_3702                                                                         MSF  http://edamontology.org/is_format_of\n",
       "387  http://edamontology.org/format_3709                                                              GCT/Res format  http://edamontology.org/is_format_of\n",
       "388  http://edamontology.org/format_3710                                                                 WIFF format  http://edamontology.org/is_format_of\n",
       "389  http://edamontology.org/format_3711                                                                X!Tandem XML  http://edamontology.org/is_format_of\n",
       "390  http://edamontology.org/format_3712                                                                  Thermo RAW  http://edamontology.org/is_format_of\n",
       "391  http://edamontology.org/format_3713                                                            Mascot .dat file  http://edamontology.org/is_format_of\n",
       "392  http://edamontology.org/format_3714                                                MaxQuant APL peaklist format  http://edamontology.org/is_format_of\n",
       "393  http://edamontology.org/format_3725                                                                        SBOL  http://edamontology.org/is_format_of\n",
       "394  http://edamontology.org/format_3726                                                                        PMML  http://edamontology.org/is_format_of\n",
       "395  http://edamontology.org/format_3727                                                                    OME-TIFF  http://edamontology.org/is_format_of\n",
       "396  http://edamontology.org/format_3728                                                                  LocARNA PP  http://edamontology.org/is_format_of\n",
       "397  http://edamontology.org/format_3729                                                                dbGaP format  http://edamontology.org/is_format_of\n",
       "398  http://edamontology.org/format_3746                                                                 BIOM format  http://edamontology.org/is_format_of\n",
       "399  http://edamontology.org/format_3747                                                                     protXML  http://edamontology.org/is_format_of\n",
       "400  http://edamontology.org/format_3748                                                          Linked data format  http://edamontology.org/is_format_of\n",
       "401  http://edamontology.org/format_3749                                                                     JSON-LD  http://edamontology.org/is_format_of\n",
       "402  http://edamontology.org/format_3750                                                                        YAML  http://edamontology.org/is_format_of\n",
       "403  http://edamontology.org/format_3751                                                                         DSV  http://edamontology.org/is_format_of\n",
       "404  http://edamontology.org/format_3752                                                                         CSV  http://edamontology.org/is_format_of\n",
       "405  http://edamontology.org/format_3758                                                           SEQUEST .out file  http://edamontology.org/is_format_of\n",
       "406  http://edamontology.org/format_3764                                                                       idXML  http://edamontology.org/is_format_of\n",
       "407  http://edamontology.org/format_3765                                                      KNIME datatable format  http://edamontology.org/is_format_of\n",
       "408  http://edamontology.org/format_3770                                                               UniProtKB XML  http://edamontology.org/is_format_of\n",
       "409  http://edamontology.org/format_3771                                                               UniProtKB RDF  http://edamontology.org/is_format_of\n",
       "410  http://edamontology.org/format_3775                                                                      GSuite  http://edamontology.org/is_format_of\n",
       "411  http://edamontology.org/format_3776                                                                      BTrack  http://edamontology.org/is_format_of\n",
       "412  http://edamontology.org/format_3781                                                        PubAnnotation format  http://edamontology.org/is_format_of\n",
       "413  http://edamontology.org/format_3782                                                                        BioC  http://edamontology.org/is_format_of\n",
       "414  http://edamontology.org/format_3783                                                             PubTator format  http://edamontology.org/is_format_of\n",
       "415  http://edamontology.org/format_3784                                                      Open Annotation format  http://edamontology.org/is_format_of\n",
       "416  http://edamontology.org/format_3785                                                   BioNLP Shared Task format  http://edamontology.org/is_format_of\n",
       "417  http://edamontology.org/format_3788                                                                         SQL  http://edamontology.org/is_format_of\n",
       "418  http://edamontology.org/format_3789                                                                      XQuery  http://edamontology.org/is_format_of\n",
       "419  http://edamontology.org/format_3790                                                                      SPARQL  http://edamontology.org/is_format_of\n",
       "420  http://edamontology.org/format_3804                                                                         xsd  http://edamontology.org/is_format_of\n",
       "421  http://edamontology.org/format_3811                                                                        XMFA  http://edamontology.org/is_format_of\n",
       "422  http://edamontology.org/format_3812                                                                         GEN  http://edamontology.org/is_format_of\n",
       "423  http://edamontology.org/format_3813                                                          SAMPLE file format  http://edamontology.org/is_format_of\n",
       "424  http://edamontology.org/format_3814                                                                         SDF  http://edamontology.org/is_format_of\n",
       "425  http://edamontology.org/format_3815                                                                     Molfile  http://edamontology.org/is_format_of\n",
       "426  http://edamontology.org/format_3816                                                                        Mol2  http://edamontology.org/is_format_of\n",
       "427  http://edamontology.org/format_3817                                                                       latex  http://edamontology.org/is_format_of\n",
       "428  http://edamontology.org/format_3818                                                                ELAND format  http://edamontology.org/is_format_of\n",
       "429  http://edamontology.org/format_3819                                                  Relaxed PHYLIP Interleaved  http://edamontology.org/is_format_of\n",
       "430  http://edamontology.org/format_3820                                                   Relaxed PHYLIP Sequential  http://edamontology.org/is_format_of\n",
       "431  http://edamontology.org/format_3821                                                                       VisML  http://edamontology.org/is_format_of\n",
       "432  http://edamontology.org/format_3822                                                                         GML  http://edamontology.org/is_format_of\n",
       "433  http://edamontology.org/format_3823                                                                       FASTG  http://edamontology.org/is_format_of\n",
       "434  http://edamontology.org/format_3825                                                                       nmrML  http://edamontology.org/is_format_of\n",
       "435  http://edamontology.org/format_3826                                                                      proBAM  http://edamontology.org/is_format_of\n",
       "436  http://edamontology.org/format_3827                                                                      proBED  http://edamontology.org/is_format_of\n",
       "437  http://edamontology.org/format_3829                                                                         GPR  http://edamontology.org/is_format_of\n",
       "438  http://edamontology.org/format_3830                                                                         ARB  http://edamontology.org/is_format_of\n",
       "439  http://edamontology.org/format_3832                                                                consensusXML  http://edamontology.org/is_format_of\n",
       "440  http://edamontology.org/format_3833                                                                  featureXML  http://edamontology.org/is_format_of\n",
       "441  http://edamontology.org/format_3834                                                                      mzData  http://edamontology.org/is_format_of\n",
       "442  http://edamontology.org/format_3835                                                                    TIDE TXT  http://edamontology.org/is_format_of\n",
       "443  http://edamontology.org/format_3836                                                 BLAST XML v2 results format  http://edamontology.org/is_format_of\n",
       "444  http://edamontology.org/format_3838                                                                        pptx  http://edamontology.org/is_format_of\n",
       "445  http://edamontology.org/format_3839                                                                         ibd  http://edamontology.org/is_format_of\n",
       "446  http://edamontology.org/format_3841                                                                  NLP format  http://edamontology.org/is_format_of\n",
       "447  http://edamontology.org/format_3843                                                                       BEAST  http://edamontology.org/is_format_of\n",
       "448  http://edamontology.org/format_3844                                                                   Chado-XML  http://edamontology.org/is_format_of\n",
       "449  http://edamontology.org/format_3845                                                                       HSAML  http://edamontology.org/is_format_of\n",
       "450  http://edamontology.org/format_3846                                                            InterProScan XML  http://edamontology.org/is_format_of\n",
       "451  http://edamontology.org/format_3847                                                                        KGML  http://edamontology.org/is_format_of\n",
       "452  http://edamontology.org/format_3848                                                                  PubMed XML  http://edamontology.org/is_format_of\n",
       "453  http://edamontology.org/format_3849                                                                       MSAML  http://edamontology.org/is_format_of\n",
       "454  http://edamontology.org/format_3850                                                                    OrthoXML  http://edamontology.org/is_format_of\n",
       "455  http://edamontology.org/format_3851                                                                       PSDML  http://edamontology.org/is_format_of\n",
       "456  http://edamontology.org/format_3852                                                                      SeqXML  http://edamontology.org/is_format_of\n",
       "457  http://edamontology.org/format_3853                                                                 UniParc XML  http://edamontology.org/is_format_of\n",
       "458  http://edamontology.org/format_3854                                                                  UniRef XML  http://edamontology.org/is_format_of\n",
       "459  http://edamontology.org/format_3857                                                                         CWL  http://edamontology.org/is_format_of\n",
       "460  http://edamontology.org/format_3858                                                                  Waters RAW  http://edamontology.org/is_format_of\n",
       "461  http://edamontology.org/format_3859                                                                    JCAMP-DX  http://edamontology.org/is_format_of\n",
       "462  http://edamontology.org/format_3862                                                       NLP annotation format  http://edamontology.org/is_format_of\n",
       "463  http://edamontology.org/format_3863                                                           NLP corpus format  http://edamontology.org/is_format_of\n",
       "464  http://edamontology.org/format_3864                                                                     mirGFF3  http://edamontology.org/is_format_of\n",
       "465  http://edamontology.org/format_3865                                                       RNA annotation format  http://edamontology.org/is_format_of\n",
       "466  http://edamontology.org/format_3867                                                  Trajectory format (binary)  http://edamontology.org/is_format_of\n",
       "467  http://edamontology.org/format_3868                                                    Trajectory format (text)  http://edamontology.org/is_format_of\n",
       "468  http://edamontology.org/format_3873                                                                         HDF  http://edamontology.org/is_format_of\n",
       "469  http://edamontology.org/format_3874                                                                      PCAzip  http://edamontology.org/is_format_of\n",
       "470  http://edamontology.org/format_3875                                                                         XTC  http://edamontology.org/is_format_of\n",
       "471  http://edamontology.org/format_3876                                                                         TNG  http://edamontology.org/is_format_of\n",
       "472  http://edamontology.org/format_3877                                                                         XYZ  http://edamontology.org/is_format_of\n",
       "473  http://edamontology.org/format_3878                                                                       mdcrd  http://edamontology.org/is_format_of\n",
       "474  http://edamontology.org/format_3880                                                                 GROMACS top  http://edamontology.org/is_format_of\n",
       "475  http://edamontology.org/format_3881                                                                   AMBER top  http://edamontology.org/is_format_of\n",
       "476  http://edamontology.org/format_3882                                                                         PSF  http://edamontology.org/is_format_of\n",
       "477  http://edamontology.org/format_3883                                                                 GROMACS itp  http://edamontology.org/is_format_of\n",
       "478  http://edamontology.org/format_3885                                                                      BinPos  http://edamontology.org/is_format_of\n",
       "479  http://edamontology.org/format_3886                                                                         RST  http://edamontology.org/is_format_of\n",
       "480  http://edamontology.org/format_3887                                                                  CHARMM rtf  http://edamontology.org/is_format_of\n",
       "481  http://edamontology.org/format_3888                                                                AMBER frcmod  http://edamontology.org/is_format_of\n",
       "482  http://edamontology.org/format_3889                                                                   AMBER off  http://edamontology.org/is_format_of\n",
       "483  http://edamontology.org/format_3906                                                                    NMReDATA  http://edamontology.org/is_format_of\n",
       "484  http://edamontology.org/format_3910                                                                         trr  http://edamontology.org/is_format_of\n",
       "485  http://edamontology.org/format_3951                                                                     BcForms  http://edamontology.org/is_format_of\n",
       "486  http://edamontology.org/format_3956                                                                     N-Quads  http://edamontology.org/is_format_of\n",
       "487  http://edamontology.org/format_3969                                                                        Vega  http://edamontology.org/is_format_of\n",
       "488  http://edamontology.org/format_3970                                                                   Vega-lite  http://edamontology.org/is_format_of\n",
       "489  http://edamontology.org/format_3973                                                                Docker image  http://edamontology.org/is_format_of\n",
       "490  http://edamontology.org/format_3975                                                                       GFA 1  http://edamontology.org/is_format_of\n",
       "491  http://edamontology.org/format_3976                                                                       GFA 2  http://edamontology.org/is_format_of\n",
       "492  http://edamontology.org/format_3977                                                                   ObjTables  http://edamontology.org/is_format_of\n",
       "493  http://edamontology.org/format_3978                                                                      CONTIG  http://edamontology.org/is_format_of\n",
       "494  http://edamontology.org/format_3979                                                                        WEGO  http://edamontology.org/is_format_of\n",
       "495  http://edamontology.org/format_3980                                                                        RPKM  http://edamontology.org/is_format_of\n",
       "496  http://edamontology.org/format_3981                                                                  TAR format  http://edamontology.org/is_format_of\n",
       "497  http://edamontology.org/format_3982                                                                       CHAIN  http://edamontology.org/is_format_of\n",
       "498  http://edamontology.org/format_3983                                                                         NET  http://edamontology.org/is_format_of\n",
       "499  http://edamontology.org/format_3984                                                                        QMAP  http://edamontology.org/is_format_of\n",
       "500  http://edamontology.org/format_3985                                                                   gxformat2  http://edamontology.org/is_format_of\n",
       "501  http://edamontology.org/format_3986                                                                         WMV  http://edamontology.org/is_format_of\n",
       "502  http://edamontology.org/format_3987                                                                  ZIP format  http://edamontology.org/is_format_of\n",
       "503  http://edamontology.org/format_3988                                                                         LSM  http://edamontology.org/is_format_of\n",
       "504  http://edamontology.org/format_3989                                                                 GZIP format  http://edamontology.org/is_format_of\n",
       "505  http://edamontology.org/format_3990                                                                         AVI  http://edamontology.org/is_format_of\n",
       "506  http://edamontology.org/format_3991                                                                     TrackDB  http://edamontology.org/is_format_of\n",
       "507  http://edamontology.org/format_3992                                                                CIGAR format  http://edamontology.org/is_format_of\n",
       "508  http://edamontology.org/format_3993                                                    Stereolithography format  http://edamontology.org/is_format_of\n",
       "509  http://edamontology.org/format_3994                                                                         U3D  http://edamontology.org/is_format_of\n",
       "510  http://edamontology.org/format_3995                                                         Texture file format  http://edamontology.org/is_format_of\n",
       "511  http://edamontology.org/format_3996                                                               Python script  http://edamontology.org/is_format_of\n",
       "512  http://edamontology.org/format_3997                                                                      MPEG-4  http://edamontology.org/is_format_of\n",
       "513  http://edamontology.org/format_3998                                                                 Perl script  http://edamontology.org/is_format_of\n",
       "514  http://edamontology.org/format_3999                                                                    R script  http://edamontology.org/is_format_of\n",
       "515  http://edamontology.org/format_4000                                                                  R markdown  http://edamontology.org/is_format_of\n",
       "516  http://edamontology.org/format_4002                                                                      pickle  http://edamontology.org/is_format_of\n",
       "517  http://edamontology.org/format_4003                                                                NumPy format  http://edamontology.org/is_format_of\n",
       "518  http://edamontology.org/format_4004                                             SimTools repertoire file format  http://edamontology.org/is_format_of\n",
       "519  http://edamontology.org/format_4005                                                   Configuration file format  http://edamontology.org/is_format_of\n",
       "520  http://edamontology.org/format_4006                                                            Zstandard format  http://edamontology.org/is_format_of\n",
       "521  http://edamontology.org/format_4007                                                               MATLAB script  http://edamontology.org/is_format_of\n",
       "522  http://edamontology.org/format_4018                                                                        gVCF  http://edamontology.org/is_format_of\n",
       "523  http://edamontology.org/format_4023                                                                         cml  http://edamontology.org/is_format_of\n",
       "524  http://edamontology.org/format_4025          BioSimulators format for the specifications of biosimulation tools  http://edamontology.org/is_format_of\n",
       "525  http://edamontology.org/format_4026  BioSimulators standard for command-line interfaces for biosimulation tools  http://edamontology.org/is_format_of\n",
       "526  http://edamontology.org/format_4035                                                                         PQR  http://edamontology.org/is_format_of\n",
       "527  http://edamontology.org/format_4036                                                                       PDBQT  http://edamontology.org/is_format_of"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "print(f\"nb formats missing is_format_of property: {len(remoteQuery(query=q, endpoint=ep_no_inference))}\")\n",
    "remoteQuery(query=q, endpoint=ep_no_inference)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "cb88aa41-2073-49c8-8631-25e26b713d4c",
   "metadata": {
    "jp-MarkdownHeadingCollapsed": true,
    "tags": []
   },
   "source": [
    "##  query to get data and operantion without *has_topic* property"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "dd69a4bc-aaa9-4708-bac8-03973fe38336",
   "metadata": {},
   "outputs": [],
   "source": [
    "q = \"\"\"\n",
    "PREFIX obo: <http://purl.obolibrary.org/obo/>\n",
    "PREFIX owl: <http://www.w3.org/2002/07/owl#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>\n",
    "PREFIX edam:<http://edamontology.org/>\n",
    "PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>\n",
    "\n",
    "SELECT ?entity ?label ?property WHERE\n",
    "{\n",
    "  \n",
    "    {?entity rdfs:subClassOf+ edam:data_0006 .}\n",
    "    UNION\n",
    "    {?entity rdfs:subClassOf+ edam:operation_0004 .}\n",
    "\n",
    "    ?entity rdfs:label ?label .\n",
    "\n",
    "     VALUES ?property { edam:has_topic               \n",
    "                        }\n",
    "    FILTER NOT EXISTS {    \n",
    "        ?entity rdfs:subClassOf ?restriction . \n",
    "        ?restriction rdf:type owl:Restriction ; \n",
    "                owl:onProperty ?property  ; \n",
    "                owl:someValuesFrom ?topic.}\n",
    "\n",
    "}ORDER BY ?entity\n",
    "    \n",
    "    \n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "5985750a-abcb-4775-ad24-038aa00b460b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "nb formats missing is_format_of property: 866\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>entity</th>\n",
       "      <th>label</th>\n",
       "      <th>property</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/data_0842</td>\n",
       "      <td>Identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>http://edamontology.org/data_0844</td>\n",
       "      <td>Molecular mass</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>http://edamontology.org/data_0845</td>\n",
       "      <td>Molecular charge</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>http://edamontology.org/data_0846</td>\n",
       "      <td>Chemical formula</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>http://edamontology.org/data_0847</td>\n",
       "      <td>QSAR descriptor</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>http://edamontology.org/data_0850</td>\n",
       "      <td>Sequence set</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>http://edamontology.org/data_0856</td>\n",
       "      <td>Sequence feature source</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>http://edamontology.org/data_0857</td>\n",
       "      <td>Sequence search results</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>http://edamontology.org/data_0862</td>\n",
       "      <td>Dotplot</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>http://edamontology.org/data_0865</td>\n",
       "      <td>Sequence similarity score</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>http://edamontology.org/data_0867</td>\n",
       "      <td>Sequence alignment report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>http://edamontology.org/data_0870</td>\n",
       "      <td>Sequence distance matrix</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>http://edamontology.org/data_0871</td>\n",
       "      <td>Phylogenetic character data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>http://edamontology.org/data_0874</td>\n",
       "      <td>Comparison matrix</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>http://edamontology.org/data_0878</td>\n",
       "      <td>Protein secondary structure alignment</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>http://edamontology.org/data_0881</td>\n",
       "      <td>RNA secondary structure alignment</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>http://edamontology.org/data_0887</td>\n",
       "      <td>Structure alignment report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>http://edamontology.org/data_0888</td>\n",
       "      <td>Structure similarity score</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>http://edamontology.org/data_0890</td>\n",
       "      <td>Structural (3D) profile alignment</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>http://edamontology.org/data_0892</td>\n",
       "      <td>Protein sequence-structure scoring matrix</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>http://edamontology.org/data_0893</td>\n",
       "      <td>Sequence-structure alignment</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>http://edamontology.org/data_0896</td>\n",
       "      <td>Protein report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>http://edamontology.org/data_0897</td>\n",
       "      <td>Protein property</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>http://edamontology.org/data_0905</td>\n",
       "      <td>Protein interaction raw data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>http://edamontology.org/data_0909</td>\n",
       "      <td>Vmax</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>http://edamontology.org/data_0910</td>\n",
       "      <td>Km</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>http://edamontology.org/data_0912</td>\n",
       "      <td>Nucleic acid property</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>http://edamontology.org/data_0916</td>\n",
       "      <td>Gene report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>http://edamontology.org/data_0919</td>\n",
       "      <td>Chromosome report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>http://edamontology.org/data_0920</td>\n",
       "      <td>Genotype/phenotype report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>http://edamontology.org/data_0924</td>\n",
       "      <td>Sequence trace</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>http://edamontology.org/data_0925</td>\n",
       "      <td>Sequence assembly</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>http://edamontology.org/data_0926</td>\n",
       "      <td>RH scores</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>http://edamontology.org/data_0927</td>\n",
       "      <td>Genetic linkage report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>http://edamontology.org/data_0937</td>\n",
       "      <td>Electron density map</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>http://edamontology.org/data_0938</td>\n",
       "      <td>Raw NMR data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>http://edamontology.org/data_0939</td>\n",
       "      <td>CD spectra</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>http://edamontology.org/data_0949</td>\n",
       "      <td>Workflow metadata</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>http://edamontology.org/data_0951</td>\n",
       "      <td>Statistical estimate score</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>http://edamontology.org/data_0954</td>\n",
       "      <td>Database cross-mapping</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>http://edamontology.org/data_0957</td>\n",
       "      <td>Database metadata</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>http://edamontology.org/data_0958</td>\n",
       "      <td>Tool metadata</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>http://edamontology.org/data_0960</td>\n",
       "      <td>User metadata</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>http://edamontology.org/data_0963</td>\n",
       "      <td>Cell line report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>http://edamontology.org/data_0968</td>\n",
       "      <td>Keyword</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>http://edamontology.org/data_0976</td>\n",
       "      <td>Identifier (by type of entity)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>http://edamontology.org/data_0977</td>\n",
       "      <td>Tool identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>http://edamontology.org/data_0982</td>\n",
       "      <td>Molecule identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>http://edamontology.org/data_0983</td>\n",
       "      <td>Atom ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>http://edamontology.org/data_0984</td>\n",
       "      <td>Molecule name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>http://edamontology.org/data_0987</td>\n",
       "      <td>Chromosome name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51</th>\n",
       "      <td>http://edamontology.org/data_0988</td>\n",
       "      <td>Peptide identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>http://edamontology.org/data_0989</td>\n",
       "      <td>Protein identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>53</th>\n",
       "      <td>http://edamontology.org/data_0990</td>\n",
       "      <td>Compound name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>http://edamontology.org/data_0991</td>\n",
       "      <td>Chemical registry number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>55</th>\n",
       "      <td>http://edamontology.org/data_0993</td>\n",
       "      <td>Drug identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>56</th>\n",
       "      <td>http://edamontology.org/data_0994</td>\n",
       "      <td>Amino acid identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>57</th>\n",
       "      <td>http://edamontology.org/data_0995</td>\n",
       "      <td>Nucleotide identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>58</th>\n",
       "      <td>http://edamontology.org/data_0996</td>\n",
       "      <td>Monosaccharide identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>59</th>\n",
       "      <td>http://edamontology.org/data_0997</td>\n",
       "      <td>Chemical name (ChEBI)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>60</th>\n",
       "      <td>http://edamontology.org/data_0998</td>\n",
       "      <td>Chemical name (IUPAC)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>61</th>\n",
       "      <td>http://edamontology.org/data_0999</td>\n",
       "      <td>Chemical name (INN)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>62</th>\n",
       "      <td>http://edamontology.org/data_1000</td>\n",
       "      <td>Chemical name (brand)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>63</th>\n",
       "      <td>http://edamontology.org/data_1001</td>\n",
       "      <td>Chemical name (synonymous)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>64</th>\n",
       "      <td>http://edamontology.org/data_1002</td>\n",
       "      <td>CAS number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>65</th>\n",
       "      <td>http://edamontology.org/data_1003</td>\n",
       "      <td>Chemical registry number (Beilstein)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66</th>\n",
       "      <td>http://edamontology.org/data_1004</td>\n",
       "      <td>Chemical registry number (Gmelin)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>67</th>\n",
       "      <td>http://edamontology.org/data_1005</td>\n",
       "      <td>HET group name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>68</th>\n",
       "      <td>http://edamontology.org/data_1006</td>\n",
       "      <td>Amino acid name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69</th>\n",
       "      <td>http://edamontology.org/data_1007</td>\n",
       "      <td>Nucleotide code</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>70</th>\n",
       "      <td>http://edamontology.org/data_1008</td>\n",
       "      <td>Polypeptide chain ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71</th>\n",
       "      <td>http://edamontology.org/data_1009</td>\n",
       "      <td>Protein name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72</th>\n",
       "      <td>http://edamontology.org/data_1010</td>\n",
       "      <td>Enzyme identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>73</th>\n",
       "      <td>http://edamontology.org/data_1011</td>\n",
       "      <td>EC number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>74</th>\n",
       "      <td>http://edamontology.org/data_1012</td>\n",
       "      <td>Enzyme name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75</th>\n",
       "      <td>http://edamontology.org/data_1013</td>\n",
       "      <td>Restriction enzyme name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76</th>\n",
       "      <td>http://edamontology.org/data_1015</td>\n",
       "      <td>Sequence feature ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>77</th>\n",
       "      <td>http://edamontology.org/data_1016</td>\n",
       "      <td>Sequence position</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78</th>\n",
       "      <td>http://edamontology.org/data_1017</td>\n",
       "      <td>Sequence range</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>79</th>\n",
       "      <td>http://edamontology.org/data_1020</td>\n",
       "      <td>Sequence feature key</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80</th>\n",
       "      <td>http://edamontology.org/data_1021</td>\n",
       "      <td>Sequence feature qualifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>81</th>\n",
       "      <td>http://edamontology.org/data_1022</td>\n",
       "      <td>Sequence feature label</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>82</th>\n",
       "      <td>http://edamontology.org/data_1023</td>\n",
       "      <td>EMBOSS Uniform Feature Object</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>83</th>\n",
       "      <td>http://edamontology.org/data_1025</td>\n",
       "      <td>Gene identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>84</th>\n",
       "      <td>http://edamontology.org/data_1026</td>\n",
       "      <td>Gene symbol</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>85</th>\n",
       "      <td>http://edamontology.org/data_1027</td>\n",
       "      <td>Gene ID (NCBI)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>86</th>\n",
       "      <td>http://edamontology.org/data_1031</td>\n",
       "      <td>Gene ID (CGD)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>87</th>\n",
       "      <td>http://edamontology.org/data_1032</td>\n",
       "      <td>Gene ID (DictyBase)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>88</th>\n",
       "      <td>http://edamontology.org/data_1033</td>\n",
       "      <td>Ensembl gene ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>89</th>\n",
       "      <td>http://edamontology.org/data_1034</td>\n",
       "      <td>Gene ID (SGD)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>90</th>\n",
       "      <td>http://edamontology.org/data_1035</td>\n",
       "      <td>Gene ID (GeneDB)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>91</th>\n",
       "      <td>http://edamontology.org/data_1036</td>\n",
       "      <td>TIGR identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>92</th>\n",
       "      <td>http://edamontology.org/data_1037</td>\n",
       "      <td>TAIR accession (gene)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>93</th>\n",
       "      <td>http://edamontology.org/data_1038</td>\n",
       "      <td>Protein domain ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>94</th>\n",
       "      <td>http://edamontology.org/data_1039</td>\n",
       "      <td>SCOP domain identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>95</th>\n",
       "      <td>http://edamontology.org/data_1040</td>\n",
       "      <td>CATH domain ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>http://edamontology.org/data_1041</td>\n",
       "      <td>SCOP concise classification string (sccs)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>97</th>\n",
       "      <td>http://edamontology.org/data_1042</td>\n",
       "      <td>SCOP sunid</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>98</th>\n",
       "      <td>http://edamontology.org/data_1043</td>\n",
       "      <td>CATH node ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>99</th>\n",
       "      <td>http://edamontology.org/data_1044</td>\n",
       "      <td>Kingdom name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>100</th>\n",
       "      <td>http://edamontology.org/data_1045</td>\n",
       "      <td>Species name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>101</th>\n",
       "      <td>http://edamontology.org/data_1046</td>\n",
       "      <td>Strain name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>102</th>\n",
       "      <td>http://edamontology.org/data_1047</td>\n",
       "      <td>URI</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>103</th>\n",
       "      <td>http://edamontology.org/data_1048</td>\n",
       "      <td>Database ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>104</th>\n",
       "      <td>http://edamontology.org/data_1049</td>\n",
       "      <td>Directory name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>105</th>\n",
       "      <td>http://edamontology.org/data_1050</td>\n",
       "      <td>File name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>106</th>\n",
       "      <td>http://edamontology.org/data_1051</td>\n",
       "      <td>Ontology name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>107</th>\n",
       "      <td>http://edamontology.org/data_1052</td>\n",
       "      <td>URL</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>108</th>\n",
       "      <td>http://edamontology.org/data_1053</td>\n",
       "      <td>URN</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>109</th>\n",
       "      <td>http://edamontology.org/data_1055</td>\n",
       "      <td>LSID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>110</th>\n",
       "      <td>http://edamontology.org/data_1056</td>\n",
       "      <td>Database name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>111</th>\n",
       "      <td>http://edamontology.org/data_1058</td>\n",
       "      <td>Enumerated file name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>112</th>\n",
       "      <td>http://edamontology.org/data_1059</td>\n",
       "      <td>File name extension</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>113</th>\n",
       "      <td>http://edamontology.org/data_1060</td>\n",
       "      <td>File base name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>114</th>\n",
       "      <td>http://edamontology.org/data_1061</td>\n",
       "      <td>QSAR descriptor name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>115</th>\n",
       "      <td>http://edamontology.org/data_1063</td>\n",
       "      <td>Sequence identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>116</th>\n",
       "      <td>http://edamontology.org/data_1064</td>\n",
       "      <td>Sequence set ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>117</th>\n",
       "      <td>http://edamontology.org/data_1066</td>\n",
       "      <td>Sequence alignment ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>118</th>\n",
       "      <td>http://edamontology.org/data_1068</td>\n",
       "      <td>Phylogenetic tree ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>119</th>\n",
       "      <td>http://edamontology.org/data_1069</td>\n",
       "      <td>Comparison matrix identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>120</th>\n",
       "      <td>http://edamontology.org/data_1070</td>\n",
       "      <td>Structure ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>121</th>\n",
       "      <td>http://edamontology.org/data_1071</td>\n",
       "      <td>Structural (3D) profile ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>122</th>\n",
       "      <td>http://edamontology.org/data_1072</td>\n",
       "      <td>Structure alignment ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>123</th>\n",
       "      <td>http://edamontology.org/data_1073</td>\n",
       "      <td>Amino acid index ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>124</th>\n",
       "      <td>http://edamontology.org/data_1074</td>\n",
       "      <td>Protein interaction ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>125</th>\n",
       "      <td>http://edamontology.org/data_1075</td>\n",
       "      <td>Protein family identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>126</th>\n",
       "      <td>http://edamontology.org/data_1076</td>\n",
       "      <td>Codon usage table name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>127</th>\n",
       "      <td>http://edamontology.org/data_1077</td>\n",
       "      <td>Transcription factor identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>128</th>\n",
       "      <td>http://edamontology.org/data_1078</td>\n",
       "      <td>Experiment annotation ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>129</th>\n",
       "      <td>http://edamontology.org/data_1079</td>\n",
       "      <td>Electron microscopy model ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>130</th>\n",
       "      <td>http://edamontology.org/data_1080</td>\n",
       "      <td>Gene expression report ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>131</th>\n",
       "      <td>http://edamontology.org/data_1081</td>\n",
       "      <td>Genotype and phenotype annotation ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>132</th>\n",
       "      <td>http://edamontology.org/data_1082</td>\n",
       "      <td>Pathway or network identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>133</th>\n",
       "      <td>http://edamontology.org/data_1083</td>\n",
       "      <td>Workflow ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>134</th>\n",
       "      <td>http://edamontology.org/data_1084</td>\n",
       "      <td>Data resource definition ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>135</th>\n",
       "      <td>http://edamontology.org/data_1085</td>\n",
       "      <td>Biological model ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>136</th>\n",
       "      <td>http://edamontology.org/data_1086</td>\n",
       "      <td>Compound identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>137</th>\n",
       "      <td>http://edamontology.org/data_1087</td>\n",
       "      <td>Ontology concept ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>138</th>\n",
       "      <td>http://edamontology.org/data_1088</td>\n",
       "      <td>Article ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>139</th>\n",
       "      <td>http://edamontology.org/data_1089</td>\n",
       "      <td>FlyBase ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>140</th>\n",
       "      <td>http://edamontology.org/data_1091</td>\n",
       "      <td>WormBase name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>141</th>\n",
       "      <td>http://edamontology.org/data_1092</td>\n",
       "      <td>WormBase class</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>142</th>\n",
       "      <td>http://edamontology.org/data_1093</td>\n",
       "      <td>Sequence accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>143</th>\n",
       "      <td>http://edamontology.org/data_1095</td>\n",
       "      <td>EMBOSS Uniform Sequence Address</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>144</th>\n",
       "      <td>http://edamontology.org/data_1096</td>\n",
       "      <td>Sequence accession (protein)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>145</th>\n",
       "      <td>http://edamontology.org/data_1097</td>\n",
       "      <td>Sequence accession (nucleic acid)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>146</th>\n",
       "      <td>http://edamontology.org/data_1098</td>\n",
       "      <td>RefSeq accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>147</th>\n",
       "      <td>http://edamontology.org/data_1100</td>\n",
       "      <td>PIR identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>148</th>\n",
       "      <td>http://edamontology.org/data_1102</td>\n",
       "      <td>Gramene primary identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>149</th>\n",
       "      <td>http://edamontology.org/data_1103</td>\n",
       "      <td>EMBL/GenBank/DDBJ ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>150</th>\n",
       "      <td>http://edamontology.org/data_1104</td>\n",
       "      <td>Sequence cluster ID (UniGene)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>151</th>\n",
       "      <td>http://edamontology.org/data_1105</td>\n",
       "      <td>dbEST accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>152</th>\n",
       "      <td>http://edamontology.org/data_1106</td>\n",
       "      <td>dbSNP ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>153</th>\n",
       "      <td>http://edamontology.org/data_1112</td>\n",
       "      <td>Sequence cluster ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>154</th>\n",
       "      <td>http://edamontology.org/data_1113</td>\n",
       "      <td>Sequence cluster ID (COG)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>155</th>\n",
       "      <td>http://edamontology.org/data_1114</td>\n",
       "      <td>Sequence motif identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>156</th>\n",
       "      <td>http://edamontology.org/data_1115</td>\n",
       "      <td>Sequence profile ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>157</th>\n",
       "      <td>http://edamontology.org/data_1116</td>\n",
       "      <td>ELM ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>158</th>\n",
       "      <td>http://edamontology.org/data_1117</td>\n",
       "      <td>Prosite accession number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>159</th>\n",
       "      <td>http://edamontology.org/data_1118</td>\n",
       "      <td>HMMER hidden Markov model ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>160</th>\n",
       "      <td>http://edamontology.org/data_1119</td>\n",
       "      <td>JASPAR profile ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>161</th>\n",
       "      <td>http://edamontology.org/data_1123</td>\n",
       "      <td>TreeBASE study accession number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>162</th>\n",
       "      <td>http://edamontology.org/data_1124</td>\n",
       "      <td>TreeFam accession number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>163</th>\n",
       "      <td>http://edamontology.org/data_1126</td>\n",
       "      <td>Comparison matrix name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>164</th>\n",
       "      <td>http://edamontology.org/data_1127</td>\n",
       "      <td>PDB ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>165</th>\n",
       "      <td>http://edamontology.org/data_1128</td>\n",
       "      <td>AAindex ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>166</th>\n",
       "      <td>http://edamontology.org/data_1129</td>\n",
       "      <td>BIND accession number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>167</th>\n",
       "      <td>http://edamontology.org/data_1130</td>\n",
       "      <td>IntAct accession number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>168</th>\n",
       "      <td>http://edamontology.org/data_1131</td>\n",
       "      <td>Protein family name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>169</th>\n",
       "      <td>http://edamontology.org/data_1132</td>\n",
       "      <td>InterPro entry name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>170</th>\n",
       "      <td>http://edamontology.org/data_1133</td>\n",
       "      <td>InterPro accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171</th>\n",
       "      <td>http://edamontology.org/data_1134</td>\n",
       "      <td>InterPro secondary accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>172</th>\n",
       "      <td>http://edamontology.org/data_1135</td>\n",
       "      <td>Gene3D ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>173</th>\n",
       "      <td>http://edamontology.org/data_1136</td>\n",
       "      <td>PIRSF ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>174</th>\n",
       "      <td>http://edamontology.org/data_1137</td>\n",
       "      <td>PRINTS code</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>175</th>\n",
       "      <td>http://edamontology.org/data_1138</td>\n",
       "      <td>Pfam accession number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>176</th>\n",
       "      <td>http://edamontology.org/data_1139</td>\n",
       "      <td>SMART accession number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>177</th>\n",
       "      <td>http://edamontology.org/data_1140</td>\n",
       "      <td>Superfamily hidden Markov model number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>178</th>\n",
       "      <td>http://edamontology.org/data_1141</td>\n",
       "      <td>TIGRFam ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>179</th>\n",
       "      <td>http://edamontology.org/data_1142</td>\n",
       "      <td>ProDom accession number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>180</th>\n",
       "      <td>http://edamontology.org/data_1143</td>\n",
       "      <td>TRANSFAC accession number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>181</th>\n",
       "      <td>http://edamontology.org/data_1144</td>\n",
       "      <td>ArrayExpress accession number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>182</th>\n",
       "      <td>http://edamontology.org/data_1145</td>\n",
       "      <td>PRIDE experiment accession number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>183</th>\n",
       "      <td>http://edamontology.org/data_1146</td>\n",
       "      <td>EMDB ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>184</th>\n",
       "      <td>http://edamontology.org/data_1147</td>\n",
       "      <td>GEO accession number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>185</th>\n",
       "      <td>http://edamontology.org/data_1148</td>\n",
       "      <td>GermOnline ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>186</th>\n",
       "      <td>http://edamontology.org/data_1149</td>\n",
       "      <td>EMAGE ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>187</th>\n",
       "      <td>http://edamontology.org/data_1150</td>\n",
       "      <td>Disease ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>188</th>\n",
       "      <td>http://edamontology.org/data_1151</td>\n",
       "      <td>HGVbase ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>189</th>\n",
       "      <td>http://edamontology.org/data_1153</td>\n",
       "      <td>OMIM ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>190</th>\n",
       "      <td>http://edamontology.org/data_1154</td>\n",
       "      <td>KEGG object identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>191</th>\n",
       "      <td>http://edamontology.org/data_1155</td>\n",
       "      <td>Pathway ID (reactome)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>192</th>\n",
       "      <td>http://edamontology.org/data_1157</td>\n",
       "      <td>Pathway ID (BioCyc)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>193</th>\n",
       "      <td>http://edamontology.org/data_1158</td>\n",
       "      <td>Pathway ID (INOH)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>194</th>\n",
       "      <td>http://edamontology.org/data_1159</td>\n",
       "      <td>Pathway ID (PATIKA)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>195</th>\n",
       "      <td>http://edamontology.org/data_1160</td>\n",
       "      <td>Pathway ID (CPDB)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>196</th>\n",
       "      <td>http://edamontology.org/data_1161</td>\n",
       "      <td>Pathway ID (Panther)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>197</th>\n",
       "      <td>http://edamontology.org/data_1162</td>\n",
       "      <td>MIRIAM identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>198</th>\n",
       "      <td>http://edamontology.org/data_1163</td>\n",
       "      <td>MIRIAM data type name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>199</th>\n",
       "      <td>http://edamontology.org/data_1164</td>\n",
       "      <td>MIRIAM URI</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>200</th>\n",
       "      <td>http://edamontology.org/data_1165</td>\n",
       "      <td>MIRIAM data type primary name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>201</th>\n",
       "      <td>http://edamontology.org/data_1166</td>\n",
       "      <td>MIRIAM data type synonymous name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>202</th>\n",
       "      <td>http://edamontology.org/data_1167</td>\n",
       "      <td>Taverna workflow ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>203</th>\n",
       "      <td>http://edamontology.org/data_1170</td>\n",
       "      <td>Biological model name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>204</th>\n",
       "      <td>http://edamontology.org/data_1171</td>\n",
       "      <td>BioModel ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>205</th>\n",
       "      <td>http://edamontology.org/data_1172</td>\n",
       "      <td>PubChem CID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>206</th>\n",
       "      <td>http://edamontology.org/data_1173</td>\n",
       "      <td>ChemSpider ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>207</th>\n",
       "      <td>http://edamontology.org/data_1174</td>\n",
       "      <td>ChEBI ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>208</th>\n",
       "      <td>http://edamontology.org/data_1175</td>\n",
       "      <td>BioPax concept ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>209</th>\n",
       "      <td>http://edamontology.org/data_1176</td>\n",
       "      <td>GO concept ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>210</th>\n",
       "      <td>http://edamontology.org/data_1177</td>\n",
       "      <td>MeSH concept ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>211</th>\n",
       "      <td>http://edamontology.org/data_1178</td>\n",
       "      <td>HGNC concept ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>212</th>\n",
       "      <td>http://edamontology.org/data_1179</td>\n",
       "      <td>NCBI taxonomy ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>213</th>\n",
       "      <td>http://edamontology.org/data_1180</td>\n",
       "      <td>Plant Ontology concept ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>214</th>\n",
       "      <td>http://edamontology.org/data_1181</td>\n",
       "      <td>UMLS concept ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>215</th>\n",
       "      <td>http://edamontology.org/data_1182</td>\n",
       "      <td>FMA concept ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>216</th>\n",
       "      <td>http://edamontology.org/data_1183</td>\n",
       "      <td>EMAP concept ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>217</th>\n",
       "      <td>http://edamontology.org/data_1184</td>\n",
       "      <td>ChEBI concept ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>218</th>\n",
       "      <td>http://edamontology.org/data_1185</td>\n",
       "      <td>MGED concept ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>219</th>\n",
       "      <td>http://edamontology.org/data_1186</td>\n",
       "      <td>myGrid concept ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>220</th>\n",
       "      <td>http://edamontology.org/data_1187</td>\n",
       "      <td>PubMed ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>221</th>\n",
       "      <td>http://edamontology.org/data_1188</td>\n",
       "      <td>DOI</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>222</th>\n",
       "      <td>http://edamontology.org/data_1189</td>\n",
       "      <td>Medline UI</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>223</th>\n",
       "      <td>http://edamontology.org/data_1190</td>\n",
       "      <td>Tool name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>224</th>\n",
       "      <td>http://edamontology.org/data_1191</td>\n",
       "      <td>Tool name (signature)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>225</th>\n",
       "      <td>http://edamontology.org/data_1192</td>\n",
       "      <td>Tool name (BLAST)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>226</th>\n",
       "      <td>http://edamontology.org/data_1193</td>\n",
       "      <td>Tool name (FASTA)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>227</th>\n",
       "      <td>http://edamontology.org/data_1194</td>\n",
       "      <td>Tool name (EMBOSS)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>228</th>\n",
       "      <td>http://edamontology.org/data_1195</td>\n",
       "      <td>Tool name (EMBASSY package)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>229</th>\n",
       "      <td>http://edamontology.org/data_1201</td>\n",
       "      <td>QSAR descriptor (constitutional)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>230</th>\n",
       "      <td>http://edamontology.org/data_1202</td>\n",
       "      <td>QSAR descriptor (electronic)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>231</th>\n",
       "      <td>http://edamontology.org/data_1203</td>\n",
       "      <td>QSAR descriptor (geometrical)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>232</th>\n",
       "      <td>http://edamontology.org/data_1204</td>\n",
       "      <td>QSAR descriptor (topological)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>233</th>\n",
       "      <td>http://edamontology.org/data_1205</td>\n",
       "      <td>QSAR descriptor (molecular)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>234</th>\n",
       "      <td>http://edamontology.org/data_1233</td>\n",
       "      <td>Sequence set (protein)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>235</th>\n",
       "      <td>http://edamontology.org/data_1234</td>\n",
       "      <td>Sequence set (nucleic acid)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>236</th>\n",
       "      <td>http://edamontology.org/data_1239</td>\n",
       "      <td>Restriction digest</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>237</th>\n",
       "      <td>http://edamontology.org/data_1240</td>\n",
       "      <td>PCR primers</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>238</th>\n",
       "      <td>http://edamontology.org/data_1249</td>\n",
       "      <td>Sequence length</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>239</th>\n",
       "      <td>http://edamontology.org/data_1254</td>\n",
       "      <td>Sequence property</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>240</th>\n",
       "      <td>http://edamontology.org/data_1255</td>\n",
       "      <td>Sequence features</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>241</th>\n",
       "      <td>http://edamontology.org/data_1259</td>\n",
       "      <td>Sequence complexity report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>242</th>\n",
       "      <td>http://edamontology.org/data_1260</td>\n",
       "      <td>Sequence ambiguity report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>243</th>\n",
       "      <td>http://edamontology.org/data_1261</td>\n",
       "      <td>Sequence composition report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>244</th>\n",
       "      <td>http://edamontology.org/data_1262</td>\n",
       "      <td>Peptide molecular weight hits</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>245</th>\n",
       "      <td>http://edamontology.org/data_1263</td>\n",
       "      <td>Base position variability plot</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>246</th>\n",
       "      <td>http://edamontology.org/data_1265</td>\n",
       "      <td>Base frequencies table</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>247</th>\n",
       "      <td>http://edamontology.org/data_1266</td>\n",
       "      <td>Base word frequencies table</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>248</th>\n",
       "      <td>http://edamontology.org/data_1267</td>\n",
       "      <td>Amino acid frequencies table</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>249</th>\n",
       "      <td>http://edamontology.org/data_1268</td>\n",
       "      <td>Amino acid word frequencies table</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>250</th>\n",
       "      <td>http://edamontology.org/data_1270</td>\n",
       "      <td>Feature table</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>251</th>\n",
       "      <td>http://edamontology.org/data_1276</td>\n",
       "      <td>Nucleic acid features</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>252</th>\n",
       "      <td>http://edamontology.org/data_1277</td>\n",
       "      <td>Protein features</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>253</th>\n",
       "      <td>http://edamontology.org/data_1352</td>\n",
       "      <td>Regular expression</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>254</th>\n",
       "      <td>http://edamontology.org/data_1394</td>\n",
       "      <td>Alignment score or penalty</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>255</th>\n",
       "      <td>http://edamontology.org/data_1397</td>\n",
       "      <td>Gap opening penalty</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>256</th>\n",
       "      <td>http://edamontology.org/data_1398</td>\n",
       "      <td>Gap extension penalty</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>257</th>\n",
       "      <td>http://edamontology.org/data_1399</td>\n",
       "      <td>Gap separation penalty</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>258</th>\n",
       "      <td>http://edamontology.org/data_1401</td>\n",
       "      <td>Match reward score</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>259</th>\n",
       "      <td>http://edamontology.org/data_1402</td>\n",
       "      <td>Mismatch penalty score</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>260</th>\n",
       "      <td>http://edamontology.org/data_1403</td>\n",
       "      <td>Drop off score</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>261</th>\n",
       "      <td>http://edamontology.org/data_1410</td>\n",
       "      <td>Terminal gap opening penalty</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>262</th>\n",
       "      <td>http://edamontology.org/data_1411</td>\n",
       "      <td>Terminal gap extension penalty</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>263</th>\n",
       "      <td>http://edamontology.org/data_1412</td>\n",
       "      <td>Sequence identity</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>264</th>\n",
       "      <td>http://edamontology.org/data_1413</td>\n",
       "      <td>Sequence similarity</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>265</th>\n",
       "      <td>http://edamontology.org/data_1426</td>\n",
       "      <td>Phylogenetic continuous quantitative data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>266</th>\n",
       "      <td>http://edamontology.org/data_1427</td>\n",
       "      <td>Phylogenetic discrete data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>267</th>\n",
       "      <td>http://edamontology.org/data_1428</td>\n",
       "      <td>Phylogenetic character cliques</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>268</th>\n",
       "      <td>http://edamontology.org/data_1442</td>\n",
       "      <td>Phylogenetic tree distances</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>269</th>\n",
       "      <td>http://edamontology.org/data_1444</td>\n",
       "      <td>Phylogenetic character contrasts</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>270</th>\n",
       "      <td>http://edamontology.org/data_1448</td>\n",
       "      <td>Comparison matrix (nucleotide)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>271</th>\n",
       "      <td>http://edamontology.org/data_1449</td>\n",
       "      <td>Comparison matrix (amino acid)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>272</th>\n",
       "      <td>http://edamontology.org/data_1494</td>\n",
       "      <td>Structural transformation matrix</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>273</th>\n",
       "      <td>http://edamontology.org/data_1497</td>\n",
       "      <td>Root-mean-square deviation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>274</th>\n",
       "      <td>http://edamontology.org/data_1498</td>\n",
       "      <td>Tanimoto similarity score</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>275</th>\n",
       "      <td>http://edamontology.org/data_1499</td>\n",
       "      <td>3D-1D scoring matrix</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>276</th>\n",
       "      <td>http://edamontology.org/data_1501</td>\n",
       "      <td>Amino acid index</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>277</th>\n",
       "      <td>http://edamontology.org/data_1502</td>\n",
       "      <td>Amino acid index (chemical classes)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>278</th>\n",
       "      <td>http://edamontology.org/data_1503</td>\n",
       "      <td>Amino acid pair-wise contact potentials</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>279</th>\n",
       "      <td>http://edamontology.org/data_1505</td>\n",
       "      <td>Amino acid index (molecular weight)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>280</th>\n",
       "      <td>http://edamontology.org/data_1506</td>\n",
       "      <td>Amino acid index (hydropathy)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>281</th>\n",
       "      <td>http://edamontology.org/data_1507</td>\n",
       "      <td>Amino acid index (White-Wimley data)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>282</th>\n",
       "      <td>http://edamontology.org/data_1508</td>\n",
       "      <td>Amino acid index (van der Waals radii)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>283</th>\n",
       "      <td>http://edamontology.org/data_1519</td>\n",
       "      <td>Peptide molecular weights</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>284</th>\n",
       "      <td>http://edamontology.org/data_1520</td>\n",
       "      <td>Peptide hydrophobic moment</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>285</th>\n",
       "      <td>http://edamontology.org/data_1521</td>\n",
       "      <td>Protein aliphatic index</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>286</th>\n",
       "      <td>http://edamontology.org/data_1522</td>\n",
       "      <td>Protein sequence hydropathy plot</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>287</th>\n",
       "      <td>http://edamontology.org/data_1523</td>\n",
       "      <td>Protein charge plot</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>288</th>\n",
       "      <td>http://edamontology.org/data_1524</td>\n",
       "      <td>Protein solubility</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>289</th>\n",
       "      <td>http://edamontology.org/data_1525</td>\n",
       "      <td>Protein crystallizability</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>290</th>\n",
       "      <td>http://edamontology.org/data_1526</td>\n",
       "      <td>Protein globularity</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>291</th>\n",
       "      <td>http://edamontology.org/data_1527</td>\n",
       "      <td>Protein titration curve</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>292</th>\n",
       "      <td>http://edamontology.org/data_1528</td>\n",
       "      <td>Protein isoelectric point</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>293</th>\n",
       "      <td>http://edamontology.org/data_1529</td>\n",
       "      <td>Protein pKa value</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>294</th>\n",
       "      <td>http://edamontology.org/data_1530</td>\n",
       "      <td>Protein hydrogen exchange rate</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>295</th>\n",
       "      <td>http://edamontology.org/data_1531</td>\n",
       "      <td>Protein extinction coefficient</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>296</th>\n",
       "      <td>http://edamontology.org/data_1532</td>\n",
       "      <td>Protein optical density</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>297</th>\n",
       "      <td>http://edamontology.org/data_1534</td>\n",
       "      <td>Peptide immunogenicity data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>298</th>\n",
       "      <td>http://edamontology.org/data_1537</td>\n",
       "      <td>Protein structure report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>299</th>\n",
       "      <td>http://edamontology.org/data_1539</td>\n",
       "      <td>Protein structural quality report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>300</th>\n",
       "      <td>http://edamontology.org/data_1542</td>\n",
       "      <td>Protein solvent accessibility</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>301</th>\n",
       "      <td>http://edamontology.org/data_1544</td>\n",
       "      <td>Ramachandran plot</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>302</th>\n",
       "      <td>http://edamontology.org/data_1545</td>\n",
       "      <td>Protein dipole moment</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>303</th>\n",
       "      <td>http://edamontology.org/data_1583</td>\n",
       "      <td>Nucleic acid melting profile</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>304</th>\n",
       "      <td>http://edamontology.org/data_1584</td>\n",
       "      <td>Nucleic acid enthalpy</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>305</th>\n",
       "      <td>http://edamontology.org/data_1585</td>\n",
       "      <td>Nucleic acid entropy</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>306</th>\n",
       "      <td>http://edamontology.org/data_1588</td>\n",
       "      <td>DNA base pair stacking energies data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>307</th>\n",
       "      <td>http://edamontology.org/data_1589</td>\n",
       "      <td>DNA base pair twist angle data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>308</th>\n",
       "      <td>http://edamontology.org/data_1590</td>\n",
       "      <td>DNA base trimer roll angles data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>309</th>\n",
       "      <td>http://edamontology.org/data_1595</td>\n",
       "      <td>Base pairing probability matrix dotplot</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>310</th>\n",
       "      <td>http://edamontology.org/data_1596</td>\n",
       "      <td>Nucleic acid folding report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>311</th>\n",
       "      <td>http://edamontology.org/data_1621</td>\n",
       "      <td>Pharmacogenomic test report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>312</th>\n",
       "      <td>http://edamontology.org/data_1667</td>\n",
       "      <td>E-value</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>313</th>\n",
       "      <td>http://edamontology.org/data_1668</td>\n",
       "      <td>Z-value</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>314</th>\n",
       "      <td>http://edamontology.org/data_1669</td>\n",
       "      <td>P-value</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>315</th>\n",
       "      <td>http://edamontology.org/data_1689</td>\n",
       "      <td>Username</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>316</th>\n",
       "      <td>http://edamontology.org/data_1690</td>\n",
       "      <td>Password</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>317</th>\n",
       "      <td>http://edamontology.org/data_1691</td>\n",
       "      <td>Email address</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>318</th>\n",
       "      <td>http://edamontology.org/data_1692</td>\n",
       "      <td>Person name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>319</th>\n",
       "      <td>http://edamontology.org/data_1707</td>\n",
       "      <td>Phylogenetic tree image</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>320</th>\n",
       "      <td>http://edamontology.org/data_1708</td>\n",
       "      <td>RNA secondary structure image</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>321</th>\n",
       "      <td>http://edamontology.org/data_1709</td>\n",
       "      <td>Protein secondary structure image</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>322</th>\n",
       "      <td>http://edamontology.org/data_1710</td>\n",
       "      <td>Structure image</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>323</th>\n",
       "      <td>http://edamontology.org/data_1711</td>\n",
       "      <td>Sequence alignment image</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>324</th>\n",
       "      <td>http://edamontology.org/data_1712</td>\n",
       "      <td>Chemical structure image</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>325</th>\n",
       "      <td>http://edamontology.org/data_1742</td>\n",
       "      <td>PDB residue number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>326</th>\n",
       "      <td>http://edamontology.org/data_1743</td>\n",
       "      <td>Atomic coordinate</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>327</th>\n",
       "      <td>http://edamontology.org/data_1748</td>\n",
       "      <td>PDB atom name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>328</th>\n",
       "      <td>http://edamontology.org/data_1757</td>\n",
       "      <td>Atom name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>329</th>\n",
       "      <td>http://edamontology.org/data_1758</td>\n",
       "      <td>PDB residue name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>330</th>\n",
       "      <td>http://edamontology.org/data_1759</td>\n",
       "      <td>PDB model number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>331</th>\n",
       "      <td>http://edamontology.org/data_1771</td>\n",
       "      <td>Sequence version</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>332</th>\n",
       "      <td>http://edamontology.org/data_1772</td>\n",
       "      <td>Score</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>333</th>\n",
       "      <td>http://edamontology.org/data_1794</td>\n",
       "      <td>Gene ID (PlasmoDB)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>334</th>\n",
       "      <td>http://edamontology.org/data_1795</td>\n",
       "      <td>Gene ID (EcoGene)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>335</th>\n",
       "      <td>http://edamontology.org/data_1796</td>\n",
       "      <td>Gene ID (FlyBase)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>336</th>\n",
       "      <td>http://edamontology.org/data_1802</td>\n",
       "      <td>Gene ID (Gramene)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>337</th>\n",
       "      <td>http://edamontology.org/data_1803</td>\n",
       "      <td>Gene ID (Virginia microbial)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>338</th>\n",
       "      <td>http://edamontology.org/data_1804</td>\n",
       "      <td>Gene ID (SGN)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>339</th>\n",
       "      <td>http://edamontology.org/data_1805</td>\n",
       "      <td>Gene ID (WormBase)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>340</th>\n",
       "      <td>http://edamontology.org/data_1807</td>\n",
       "      <td>ORF name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>341</th>\n",
       "      <td>http://edamontology.org/data_1855</td>\n",
       "      <td>Clone ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>342</th>\n",
       "      <td>http://edamontology.org/data_1856</td>\n",
       "      <td>PDB insertion code</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>343</th>\n",
       "      <td>http://edamontology.org/data_1857</td>\n",
       "      <td>Atomic occupancy</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>344</th>\n",
       "      <td>http://edamontology.org/data_1858</td>\n",
       "      <td>Isotropic B factor</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>345</th>\n",
       "      <td>http://edamontology.org/data_1867</td>\n",
       "      <td>Protein fold name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>346</th>\n",
       "      <td>http://edamontology.org/data_1868</td>\n",
       "      <td>Taxon</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>347</th>\n",
       "      <td>http://edamontology.org/data_1869</td>\n",
       "      <td>Organism identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>348</th>\n",
       "      <td>http://edamontology.org/data_1870</td>\n",
       "      <td>Genus name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>349</th>\n",
       "      <td>http://edamontology.org/data_1872</td>\n",
       "      <td>Taxonomic classification</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>350</th>\n",
       "      <td>http://edamontology.org/data_1873</td>\n",
       "      <td>iHOP organism ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>351</th>\n",
       "      <td>http://edamontology.org/data_1874</td>\n",
       "      <td>Genbank common name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>352</th>\n",
       "      <td>http://edamontology.org/data_1875</td>\n",
       "      <td>NCBI taxon</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>353</th>\n",
       "      <td>http://edamontology.org/data_1881</td>\n",
       "      <td>Author ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>354</th>\n",
       "      <td>http://edamontology.org/data_1882</td>\n",
       "      <td>DragonDB author identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>355</th>\n",
       "      <td>http://edamontology.org/data_1883</td>\n",
       "      <td>Annotated URI</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>356</th>\n",
       "      <td>http://edamontology.org/data_1885</td>\n",
       "      <td>Gene ID (GeneFarm)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>357</th>\n",
       "      <td>http://edamontology.org/data_1886</td>\n",
       "      <td>Blattner number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>358</th>\n",
       "      <td>http://edamontology.org/data_1891</td>\n",
       "      <td>iHOP symbol</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>359</th>\n",
       "      <td>http://edamontology.org/data_1893</td>\n",
       "      <td>Locus ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>360</th>\n",
       "      <td>http://edamontology.org/data_1895</td>\n",
       "      <td>Locus ID (AGI)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>361</th>\n",
       "      <td>http://edamontology.org/data_1896</td>\n",
       "      <td>Locus ID (ASPGD)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>362</th>\n",
       "      <td>http://edamontology.org/data_1897</td>\n",
       "      <td>Locus ID (MGG)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>363</th>\n",
       "      <td>http://edamontology.org/data_1898</td>\n",
       "      <td>Locus ID (CGD)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>364</th>\n",
       "      <td>http://edamontology.org/data_1899</td>\n",
       "      <td>Locus ID (CMR)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>365</th>\n",
       "      <td>http://edamontology.org/data_1900</td>\n",
       "      <td>NCBI locus tag</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>366</th>\n",
       "      <td>http://edamontology.org/data_1901</td>\n",
       "      <td>Locus ID (SGD)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>367</th>\n",
       "      <td>http://edamontology.org/data_1902</td>\n",
       "      <td>Locus ID (MMP)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>368</th>\n",
       "      <td>http://edamontology.org/data_1903</td>\n",
       "      <td>Locus ID (DictyBase)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>369</th>\n",
       "      <td>http://edamontology.org/data_1904</td>\n",
       "      <td>Locus ID (EntrezGene)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>370</th>\n",
       "      <td>http://edamontology.org/data_1905</td>\n",
       "      <td>Locus ID (MaizeGDB)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>371</th>\n",
       "      <td>http://edamontology.org/data_1907</td>\n",
       "      <td>Gene ID (KOME)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>372</th>\n",
       "      <td>http://edamontology.org/data_1908</td>\n",
       "      <td>Locus ID (Tropgene)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>373</th>\n",
       "      <td>http://edamontology.org/data_1916</td>\n",
       "      <td>Alignment</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>374</th>\n",
       "      <td>http://edamontology.org/data_1917</td>\n",
       "      <td>Atomic property</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>375</th>\n",
       "      <td>http://edamontology.org/data_2007</td>\n",
       "      <td>UniProt keyword</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>376</th>\n",
       "      <td>http://edamontology.org/data_2012</td>\n",
       "      <td>Sequence coordinates</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>377</th>\n",
       "      <td>http://edamontology.org/data_2016</td>\n",
       "      <td>Amino acid property</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>378</th>\n",
       "      <td>http://edamontology.org/data_2024</td>\n",
       "      <td>Enzyme kinetics data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>379</th>\n",
       "      <td>http://edamontology.org/data_2025</td>\n",
       "      <td>Michaelis Menten plot</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>380</th>\n",
       "      <td>http://edamontology.org/data_2026</td>\n",
       "      <td>Hanes Woolf plot</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>381</th>\n",
       "      <td>http://edamontology.org/data_2042</td>\n",
       "      <td>Evidence</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>382</th>\n",
       "      <td>http://edamontology.org/data_2048</td>\n",
       "      <td>Report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>383</th>\n",
       "      <td>http://edamontology.org/data_2050</td>\n",
       "      <td>Molecular property (general)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>384</th>\n",
       "      <td>http://edamontology.org/data_2080</td>\n",
       "      <td>Database search results</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>385</th>\n",
       "      <td>http://edamontology.org/data_2082</td>\n",
       "      <td>Matrix</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>386</th>\n",
       "      <td>http://edamontology.org/data_2084</td>\n",
       "      <td>Nucleic acid report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>387</th>\n",
       "      <td>http://edamontology.org/data_2085</td>\n",
       "      <td>Structure report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>388</th>\n",
       "      <td>http://edamontology.org/data_2087</td>\n",
       "      <td>Molecular property</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>389</th>\n",
       "      <td>http://edamontology.org/data_2088</td>\n",
       "      <td>DNA base structural data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>390</th>\n",
       "      <td>http://edamontology.org/data_2091</td>\n",
       "      <td>Accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>391</th>\n",
       "      <td>http://edamontology.org/data_2093</td>\n",
       "      <td>Data reference</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>392</th>\n",
       "      <td>http://edamontology.org/data_2098</td>\n",
       "      <td>Job identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>393</th>\n",
       "      <td>http://edamontology.org/data_2099</td>\n",
       "      <td>Name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>394</th>\n",
       "      <td>http://edamontology.org/data_2101</td>\n",
       "      <td>Account authentication</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>395</th>\n",
       "      <td>http://edamontology.org/data_2102</td>\n",
       "      <td>KEGG organism code</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>396</th>\n",
       "      <td>http://edamontology.org/data_2104</td>\n",
       "      <td>BioCyc ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>397</th>\n",
       "      <td>http://edamontology.org/data_2105</td>\n",
       "      <td>Compound ID (BioCyc)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>398</th>\n",
       "      <td>http://edamontology.org/data_2106</td>\n",
       "      <td>Reaction ID (BioCyc)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>399</th>\n",
       "      <td>http://edamontology.org/data_2107</td>\n",
       "      <td>Enzyme ID (BioCyc)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>400</th>\n",
       "      <td>http://edamontology.org/data_2108</td>\n",
       "      <td>Reaction ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>401</th>\n",
       "      <td>http://edamontology.org/data_2109</td>\n",
       "      <td>Identifier (hybrid)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>402</th>\n",
       "      <td>http://edamontology.org/data_2110</td>\n",
       "      <td>Molecular property identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>403</th>\n",
       "      <td>http://edamontology.org/data_2111</td>\n",
       "      <td>Codon usage table ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>404</th>\n",
       "      <td>http://edamontology.org/data_2112</td>\n",
       "      <td>FlyBase primary identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>405</th>\n",
       "      <td>http://edamontology.org/data_2113</td>\n",
       "      <td>WormBase identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>406</th>\n",
       "      <td>http://edamontology.org/data_2114</td>\n",
       "      <td>WormBase wormpep ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>407</th>\n",
       "      <td>http://edamontology.org/data_2117</td>\n",
       "      <td>Map identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>408</th>\n",
       "      <td>http://edamontology.org/data_2118</td>\n",
       "      <td>Person identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>409</th>\n",
       "      <td>http://edamontology.org/data_2119</td>\n",
       "      <td>Nucleic acid identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>410</th>\n",
       "      <td>http://edamontology.org/data_2127</td>\n",
       "      <td>Genetic code identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>411</th>\n",
       "      <td>http://edamontology.org/data_2128</td>\n",
       "      <td>Genetic code name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>412</th>\n",
       "      <td>http://edamontology.org/data_2129</td>\n",
       "      <td>File format name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>413</th>\n",
       "      <td>http://edamontology.org/data_2131</td>\n",
       "      <td>Operating system name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>414</th>\n",
       "      <td>http://edamontology.org/data_2133</td>\n",
       "      <td>Logical operator</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>415</th>\n",
       "      <td>http://edamontology.org/data_2137</td>\n",
       "      <td>Gap penalty</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>416</th>\n",
       "      <td>http://edamontology.org/data_2139</td>\n",
       "      <td>Nucleic acid melting temperature</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>417</th>\n",
       "      <td>http://edamontology.org/data_2140</td>\n",
       "      <td>Concentration</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>418</th>\n",
       "      <td>http://edamontology.org/data_2154</td>\n",
       "      <td>Sequence name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>419</th>\n",
       "      <td>http://edamontology.org/data_2160</td>\n",
       "      <td>Fickett testcode plot</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>420</th>\n",
       "      <td>http://edamontology.org/data_2161</td>\n",
       "      <td>Sequence similarity plot</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>421</th>\n",
       "      <td>http://edamontology.org/data_2162</td>\n",
       "      <td>Helical wheel</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>422</th>\n",
       "      <td>http://edamontology.org/data_2163</td>\n",
       "      <td>Helical net</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>423</th>\n",
       "      <td>http://edamontology.org/data_2165</td>\n",
       "      <td>Protein ionisation curve</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>424</th>\n",
       "      <td>http://edamontology.org/data_2166</td>\n",
       "      <td>Sequence composition plot</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>425</th>\n",
       "      <td>http://edamontology.org/data_2167</td>\n",
       "      <td>Nucleic acid density plot</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>426</th>\n",
       "      <td>http://edamontology.org/data_2168</td>\n",
       "      <td>Sequence trace image</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>427</th>\n",
       "      <td>http://edamontology.org/data_2174</td>\n",
       "      <td>FlyBase secondary identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>428</th>\n",
       "      <td>http://edamontology.org/data_2190</td>\n",
       "      <td>Sequence checksum</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>429</th>\n",
       "      <td>http://edamontology.org/data_2193</td>\n",
       "      <td>Database entry metadata</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>430</th>\n",
       "      <td>http://edamontology.org/data_2208</td>\n",
       "      <td>Plasmid identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>431</th>\n",
       "      <td>http://edamontology.org/data_2209</td>\n",
       "      <td>Mutation ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>432</th>\n",
       "      <td>http://edamontology.org/data_2216</td>\n",
       "      <td>Codon number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>433</th>\n",
       "      <td>http://edamontology.org/data_2219</td>\n",
       "      <td>Database field name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>434</th>\n",
       "      <td>http://edamontology.org/data_2220</td>\n",
       "      <td>Sequence cluster ID (SYSTERS)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>435</th>\n",
       "      <td>http://edamontology.org/data_2253</td>\n",
       "      <td>Data resource definition name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>436</th>\n",
       "      <td>http://edamontology.org/data_2254</td>\n",
       "      <td>OBO file format name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>437</th>\n",
       "      <td>http://edamontology.org/data_2285</td>\n",
       "      <td>Gene ID (MIPS)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>438</th>\n",
       "      <td>http://edamontology.org/data_2290</td>\n",
       "      <td>EMBL accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>439</th>\n",
       "      <td>http://edamontology.org/data_2291</td>\n",
       "      <td>UniProt ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>440</th>\n",
       "      <td>http://edamontology.org/data_2292</td>\n",
       "      <td>GenBank accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>441</th>\n",
       "      <td>http://edamontology.org/data_2293</td>\n",
       "      <td>Gramene secondary identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>442</th>\n",
       "      <td>http://edamontology.org/data_2294</td>\n",
       "      <td>Sequence variation ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>443</th>\n",
       "      <td>http://edamontology.org/data_2295</td>\n",
       "      <td>Gene ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>444</th>\n",
       "      <td>http://edamontology.org/data_2297</td>\n",
       "      <td>Gene ID (ECK)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>445</th>\n",
       "      <td>http://edamontology.org/data_2298</td>\n",
       "      <td>Gene ID (HGNC)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>446</th>\n",
       "      <td>http://edamontology.org/data_2299</td>\n",
       "      <td>Gene name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>447</th>\n",
       "      <td>http://edamontology.org/data_2301</td>\n",
       "      <td>SMILES string</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>448</th>\n",
       "      <td>http://edamontology.org/data_2302</td>\n",
       "      <td>STRING ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>449</th>\n",
       "      <td>http://edamontology.org/data_2309</td>\n",
       "      <td>Reaction ID (SABIO-RK)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>450</th>\n",
       "      <td>http://edamontology.org/data_2313</td>\n",
       "      <td>Carbohydrate report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>451</th>\n",
       "      <td>http://edamontology.org/data_2314</td>\n",
       "      <td>GI number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>452</th>\n",
       "      <td>http://edamontology.org/data_2315</td>\n",
       "      <td>NCBI version</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>453</th>\n",
       "      <td>http://edamontology.org/data_2316</td>\n",
       "      <td>Cell line name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>454</th>\n",
       "      <td>http://edamontology.org/data_2317</td>\n",
       "      <td>Cell line name (exact)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>455</th>\n",
       "      <td>http://edamontology.org/data_2318</td>\n",
       "      <td>Cell line name (truncated)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>456</th>\n",
       "      <td>http://edamontology.org/data_2319</td>\n",
       "      <td>Cell line name (no punctuation)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>457</th>\n",
       "      <td>http://edamontology.org/data_2320</td>\n",
       "      <td>Cell line name (assonant)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>458</th>\n",
       "      <td>http://edamontology.org/data_2321</td>\n",
       "      <td>Enzyme ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>459</th>\n",
       "      <td>http://edamontology.org/data_2325</td>\n",
       "      <td>REBASE enzyme number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>460</th>\n",
       "      <td>http://edamontology.org/data_2326</td>\n",
       "      <td>DrugBank ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>461</th>\n",
       "      <td>http://edamontology.org/data_2327</td>\n",
       "      <td>GI number (protein)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>462</th>\n",
       "      <td>http://edamontology.org/data_2335</td>\n",
       "      <td>Bit score</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>463</th>\n",
       "      <td>http://edamontology.org/data_2337</td>\n",
       "      <td>Resource metadata</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>464</th>\n",
       "      <td>http://edamontology.org/data_2338</td>\n",
       "      <td>Ontology identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>465</th>\n",
       "      <td>http://edamontology.org/data_2339</td>\n",
       "      <td>Ontology concept name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>466</th>\n",
       "      <td>http://edamontology.org/data_2340</td>\n",
       "      <td>Genome build identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>467</th>\n",
       "      <td>http://edamontology.org/data_2342</td>\n",
       "      <td>Pathway or network name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>468</th>\n",
       "      <td>http://edamontology.org/data_2343</td>\n",
       "      <td>Pathway ID (KEGG)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>469</th>\n",
       "      <td>http://edamontology.org/data_2344</td>\n",
       "      <td>Pathway ID (NCI-Nature)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>470</th>\n",
       "      <td>http://edamontology.org/data_2345</td>\n",
       "      <td>Pathway ID (ConsensusPathDB)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>471</th>\n",
       "      <td>http://edamontology.org/data_2346</td>\n",
       "      <td>Sequence cluster ID (UniRef)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>472</th>\n",
       "      <td>http://edamontology.org/data_2347</td>\n",
       "      <td>Sequence cluster ID (UniRef100)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>473</th>\n",
       "      <td>http://edamontology.org/data_2348</td>\n",
       "      <td>Sequence cluster ID (UniRef90)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>474</th>\n",
       "      <td>http://edamontology.org/data_2349</td>\n",
       "      <td>Sequence cluster ID (UniRef50)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>475</th>\n",
       "      <td>http://edamontology.org/data_2354</td>\n",
       "      <td>RNA family report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>476</th>\n",
       "      <td>http://edamontology.org/data_2355</td>\n",
       "      <td>RNA family identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>477</th>\n",
       "      <td>http://edamontology.org/data_2356</td>\n",
       "      <td>RFAM accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>478</th>\n",
       "      <td>http://edamontology.org/data_2362</td>\n",
       "      <td>Sequence accession (hybrid)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>479</th>\n",
       "      <td>http://edamontology.org/data_2365</td>\n",
       "      <td>Pathway or network accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>480</th>\n",
       "      <td>http://edamontology.org/data_2366</td>\n",
       "      <td>Secondary structure alignment</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>481</th>\n",
       "      <td>http://edamontology.org/data_2367</td>\n",
       "      <td>ASTD ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>482</th>\n",
       "      <td>http://edamontology.org/data_2368</td>\n",
       "      <td>ASTD ID (exon)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>483</th>\n",
       "      <td>http://edamontology.org/data_2369</td>\n",
       "      <td>ASTD ID (intron)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>484</th>\n",
       "      <td>http://edamontology.org/data_2370</td>\n",
       "      <td>ASTD ID (polya)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>485</th>\n",
       "      <td>http://edamontology.org/data_2371</td>\n",
       "      <td>ASTD ID (tss)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>486</th>\n",
       "      <td>http://edamontology.org/data_2373</td>\n",
       "      <td>Spot ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>487</th>\n",
       "      <td>http://edamontology.org/data_2374</td>\n",
       "      <td>Spot serial number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>488</th>\n",
       "      <td>http://edamontology.org/data_2375</td>\n",
       "      <td>Spot ID (HSC-2DPAGE)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>489</th>\n",
       "      <td>http://edamontology.org/data_2379</td>\n",
       "      <td>Strain identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>490</th>\n",
       "      <td>http://edamontology.org/data_2380</td>\n",
       "      <td>CABRI accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>491</th>\n",
       "      <td>http://edamontology.org/data_2382</td>\n",
       "      <td>Genotype experiment ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>492</th>\n",
       "      <td>http://edamontology.org/data_2383</td>\n",
       "      <td>EGA accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>493</th>\n",
       "      <td>http://edamontology.org/data_2384</td>\n",
       "      <td>IPI protein ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>494</th>\n",
       "      <td>http://edamontology.org/data_2385</td>\n",
       "      <td>RefSeq accession (protein)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>495</th>\n",
       "      <td>http://edamontology.org/data_2386</td>\n",
       "      <td>EPD ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>496</th>\n",
       "      <td>http://edamontology.org/data_2387</td>\n",
       "      <td>TAIR accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>497</th>\n",
       "      <td>http://edamontology.org/data_2388</td>\n",
       "      <td>TAIR accession (At gene)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>498</th>\n",
       "      <td>http://edamontology.org/data_2389</td>\n",
       "      <td>UniSTS accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>499</th>\n",
       "      <td>http://edamontology.org/data_2390</td>\n",
       "      <td>UNITE accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>500</th>\n",
       "      <td>http://edamontology.org/data_2391</td>\n",
       "      <td>UTR accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>501</th>\n",
       "      <td>http://edamontology.org/data_2392</td>\n",
       "      <td>UniParc accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>502</th>\n",
       "      <td>http://edamontology.org/data_2393</td>\n",
       "      <td>mFLJ/mKIAA number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>503</th>\n",
       "      <td>http://edamontology.org/data_2398</td>\n",
       "      <td>Ensembl protein ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>504</th>\n",
       "      <td>http://edamontology.org/data_2523</td>\n",
       "      <td>Phylogenetic data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>505</th>\n",
       "      <td>http://edamontology.org/data_2530</td>\n",
       "      <td>Organism report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>506</th>\n",
       "      <td>http://edamontology.org/data_2531</td>\n",
       "      <td>Protocol</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>507</th>\n",
       "      <td>http://edamontology.org/data_2534</td>\n",
       "      <td>Sequence attribute</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>508</th>\n",
       "      <td>http://edamontology.org/data_2536</td>\n",
       "      <td>Mass spectrometry data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>509</th>\n",
       "      <td>http://edamontology.org/data_2537</td>\n",
       "      <td>Protein structure raw data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>510</th>\n",
       "      <td>http://edamontology.org/data_2538</td>\n",
       "      <td>Mutation identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>511</th>\n",
       "      <td>http://edamontology.org/data_2563</td>\n",
       "      <td>Amino acid name (single letter)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>512</th>\n",
       "      <td>http://edamontology.org/data_2564</td>\n",
       "      <td>Amino acid name (three letter)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>513</th>\n",
       "      <td>http://edamontology.org/data_2565</td>\n",
       "      <td>Amino acid name (full name)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>514</th>\n",
       "      <td>http://edamontology.org/data_2576</td>\n",
       "      <td>Toxin identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>515</th>\n",
       "      <td>http://edamontology.org/data_2578</td>\n",
       "      <td>ArachnoServer ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>516</th>\n",
       "      <td>http://edamontology.org/data_2580</td>\n",
       "      <td>BindingDB Monomer ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>517</th>\n",
       "      <td>http://edamontology.org/data_2582</td>\n",
       "      <td>GO concept ID (biological process)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>518</th>\n",
       "      <td>http://edamontology.org/data_2583</td>\n",
       "      <td>GO concept ID (molecular function)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>519</th>\n",
       "      <td>http://edamontology.org/data_2586</td>\n",
       "      <td>Northern blot image</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>520</th>\n",
       "      <td>http://edamontology.org/data_2587</td>\n",
       "      <td>Blot ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>521</th>\n",
       "      <td>http://edamontology.org/data_2588</td>\n",
       "      <td>BlotBase blot ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>522</th>\n",
       "      <td>http://edamontology.org/data_2589</td>\n",
       "      <td>Hierarchy</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>523</th>\n",
       "      <td>http://edamontology.org/data_2591</td>\n",
       "      <td>Brite hierarchy ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>524</th>\n",
       "      <td>http://edamontology.org/data_2593</td>\n",
       "      <td>BRENDA organism ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>525</th>\n",
       "      <td>http://edamontology.org/data_2594</td>\n",
       "      <td>UniGene taxon</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>526</th>\n",
       "      <td>http://edamontology.org/data_2595</td>\n",
       "      <td>UTRdb taxon</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>527</th>\n",
       "      <td>http://edamontology.org/data_2596</td>\n",
       "      <td>Catalogue ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>528</th>\n",
       "      <td>http://edamontology.org/data_2597</td>\n",
       "      <td>CABRI catalogue name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>529</th>\n",
       "      <td>http://edamontology.org/data_2605</td>\n",
       "      <td>Compound ID (KEGG)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>530</th>\n",
       "      <td>http://edamontology.org/data_2606</td>\n",
       "      <td>RFAM name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>531</th>\n",
       "      <td>http://edamontology.org/data_2608</td>\n",
       "      <td>Reaction ID (KEGG)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>532</th>\n",
       "      <td>http://edamontology.org/data_2609</td>\n",
       "      <td>Drug ID (KEGG)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>533</th>\n",
       "      <td>http://edamontology.org/data_2610</td>\n",
       "      <td>Ensembl ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>534</th>\n",
       "      <td>http://edamontology.org/data_2611</td>\n",
       "      <td>ICD identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>535</th>\n",
       "      <td>http://edamontology.org/data_2612</td>\n",
       "      <td>Sequence cluster ID (CluSTr)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>536</th>\n",
       "      <td>http://edamontology.org/data_2613</td>\n",
       "      <td>KEGG Glycan ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>537</th>\n",
       "      <td>http://edamontology.org/data_2614</td>\n",
       "      <td>TCDB ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>538</th>\n",
       "      <td>http://edamontology.org/data_2615</td>\n",
       "      <td>MINT ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>539</th>\n",
       "      <td>http://edamontology.org/data_2616</td>\n",
       "      <td>DIP ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>540</th>\n",
       "      <td>http://edamontology.org/data_2617</td>\n",
       "      <td>Signaling Gateway protein ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>541</th>\n",
       "      <td>http://edamontology.org/data_2618</td>\n",
       "      <td>Protein modification ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>542</th>\n",
       "      <td>http://edamontology.org/data_2619</td>\n",
       "      <td>RESID ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>543</th>\n",
       "      <td>http://edamontology.org/data_2620</td>\n",
       "      <td>RGD ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>544</th>\n",
       "      <td>http://edamontology.org/data_2621</td>\n",
       "      <td>TAIR accession (protein)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>545</th>\n",
       "      <td>http://edamontology.org/data_2622</td>\n",
       "      <td>Compound ID (HMDB)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>546</th>\n",
       "      <td>http://edamontology.org/data_2625</td>\n",
       "      <td>LIPID MAPS ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>547</th>\n",
       "      <td>http://edamontology.org/data_2626</td>\n",
       "      <td>PeptideAtlas ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>548</th>\n",
       "      <td>http://edamontology.org/data_2628</td>\n",
       "      <td>BioGRID interaction ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>549</th>\n",
       "      <td>http://edamontology.org/data_2629</td>\n",
       "      <td>Enzyme ID (MEROPS)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>550</th>\n",
       "      <td>http://edamontology.org/data_2630</td>\n",
       "      <td>Mobile genetic element ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>551</th>\n",
       "      <td>http://edamontology.org/data_2631</td>\n",
       "      <td>ACLAME ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>552</th>\n",
       "      <td>http://edamontology.org/data_2632</td>\n",
       "      <td>SGD ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>553</th>\n",
       "      <td>http://edamontology.org/data_2633</td>\n",
       "      <td>Book ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>554</th>\n",
       "      <td>http://edamontology.org/data_2634</td>\n",
       "      <td>ISBN</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>555</th>\n",
       "      <td>http://edamontology.org/data_2635</td>\n",
       "      <td>Compound ID (3DMET)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>556</th>\n",
       "      <td>http://edamontology.org/data_2636</td>\n",
       "      <td>MatrixDB interaction ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>557</th>\n",
       "      <td>http://edamontology.org/data_2637</td>\n",
       "      <td>cPath ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>558</th>\n",
       "      <td>http://edamontology.org/data_2638</td>\n",
       "      <td>PubChem bioassay ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>559</th>\n",
       "      <td>http://edamontology.org/data_2639</td>\n",
       "      <td>PubChem ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>560</th>\n",
       "      <td>http://edamontology.org/data_2641</td>\n",
       "      <td>Reaction ID (MACie)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>561</th>\n",
       "      <td>http://edamontology.org/data_2642</td>\n",
       "      <td>Gene ID (miRBase)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>562</th>\n",
       "      <td>http://edamontology.org/data_2643</td>\n",
       "      <td>Gene ID (ZFIN)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>563</th>\n",
       "      <td>http://edamontology.org/data_2644</td>\n",
       "      <td>Reaction ID (Rhea)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>564</th>\n",
       "      <td>http://edamontology.org/data_2645</td>\n",
       "      <td>Pathway ID (Unipathway)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>565</th>\n",
       "      <td>http://edamontology.org/data_2646</td>\n",
       "      <td>Compound ID (ChEMBL)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>566</th>\n",
       "      <td>http://edamontology.org/data_2647</td>\n",
       "      <td>LGICdb identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>567</th>\n",
       "      <td>http://edamontology.org/data_2648</td>\n",
       "      <td>Reaction kinetics ID (SABIO-RK)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>568</th>\n",
       "      <td>http://edamontology.org/data_2649</td>\n",
       "      <td>PharmGKB ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>569</th>\n",
       "      <td>http://edamontology.org/data_2650</td>\n",
       "      <td>Pathway ID (PharmGKB)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>570</th>\n",
       "      <td>http://edamontology.org/data_2651</td>\n",
       "      <td>Disease ID (PharmGKB)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>571</th>\n",
       "      <td>http://edamontology.org/data_2652</td>\n",
       "      <td>Drug ID (PharmGKB)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>572</th>\n",
       "      <td>http://edamontology.org/data_2653</td>\n",
       "      <td>Drug ID (TTD)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>573</th>\n",
       "      <td>http://edamontology.org/data_2654</td>\n",
       "      <td>Target ID (TTD)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>574</th>\n",
       "      <td>http://edamontology.org/data_2655</td>\n",
       "      <td>Cell type identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>575</th>\n",
       "      <td>http://edamontology.org/data_2656</td>\n",
       "      <td>NeuronDB ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>576</th>\n",
       "      <td>http://edamontology.org/data_2657</td>\n",
       "      <td>NeuroMorpho ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>577</th>\n",
       "      <td>http://edamontology.org/data_2658</td>\n",
       "      <td>Compound ID (ChemIDplus)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>578</th>\n",
       "      <td>http://edamontology.org/data_2659</td>\n",
       "      <td>Pathway ID (SMPDB)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>579</th>\n",
       "      <td>http://edamontology.org/data_2660</td>\n",
       "      <td>BioNumbers ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>580</th>\n",
       "      <td>http://edamontology.org/data_2662</td>\n",
       "      <td>T3DB ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>581</th>\n",
       "      <td>http://edamontology.org/data_2663</td>\n",
       "      <td>Carbohydrate identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>582</th>\n",
       "      <td>http://edamontology.org/data_2664</td>\n",
       "      <td>GlycomeDB ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>583</th>\n",
       "      <td>http://edamontology.org/data_2665</td>\n",
       "      <td>LipidBank ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>584</th>\n",
       "      <td>http://edamontology.org/data_2666</td>\n",
       "      <td>CDD ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>585</th>\n",
       "      <td>http://edamontology.org/data_2667</td>\n",
       "      <td>MMDB ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>586</th>\n",
       "      <td>http://edamontology.org/data_2668</td>\n",
       "      <td>iRefIndex ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>587</th>\n",
       "      <td>http://edamontology.org/data_2669</td>\n",
       "      <td>ModelDB ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>588</th>\n",
       "      <td>http://edamontology.org/data_2670</td>\n",
       "      <td>Pathway ID (DQCS)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>589</th>\n",
       "      <td>http://edamontology.org/data_2700</td>\n",
       "      <td>CATH identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>590</th>\n",
       "      <td>http://edamontology.org/data_2701</td>\n",
       "      <td>CATH node ID (family)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>591</th>\n",
       "      <td>http://edamontology.org/data_2702</td>\n",
       "      <td>Enzyme ID (CAZy)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>592</th>\n",
       "      <td>http://edamontology.org/data_2704</td>\n",
       "      <td>Clone ID (IMAGE)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>593</th>\n",
       "      <td>http://edamontology.org/data_2705</td>\n",
       "      <td>GO concept ID (cellular component)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>594</th>\n",
       "      <td>http://edamontology.org/data_2706</td>\n",
       "      <td>Chromosome name (BioCyc)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>595</th>\n",
       "      <td>http://edamontology.org/data_2709</td>\n",
       "      <td>CleanEx entry name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>596</th>\n",
       "      <td>http://edamontology.org/data_2710</td>\n",
       "      <td>CleanEx dataset code</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>597</th>\n",
       "      <td>http://edamontology.org/data_2711</td>\n",
       "      <td>Genome report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>598</th>\n",
       "      <td>http://edamontology.org/data_2713</td>\n",
       "      <td>Protein ID (CORUM)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>599</th>\n",
       "      <td>http://edamontology.org/data_2714</td>\n",
       "      <td>CDD PSSM-ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>600</th>\n",
       "      <td>http://edamontology.org/data_2715</td>\n",
       "      <td>Protein ID (CuticleDB)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>601</th>\n",
       "      <td>http://edamontology.org/data_2716</td>\n",
       "      <td>DBD ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>602</th>\n",
       "      <td>http://edamontology.org/data_2718</td>\n",
       "      <td>Oligonucleotide ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>603</th>\n",
       "      <td>http://edamontology.org/data_2719</td>\n",
       "      <td>dbProbe ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>604</th>\n",
       "      <td>http://edamontology.org/data_2720</td>\n",
       "      <td>Dinucleotide property</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>605</th>\n",
       "      <td>http://edamontology.org/data_2721</td>\n",
       "      <td>DiProDB ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>606</th>\n",
       "      <td>http://edamontology.org/data_2723</td>\n",
       "      <td>Protein ID (DisProt)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>607</th>\n",
       "      <td>http://edamontology.org/data_2725</td>\n",
       "      <td>Ensembl transcript ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>608</th>\n",
       "      <td>http://edamontology.org/data_2727</td>\n",
       "      <td>Promoter ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>609</th>\n",
       "      <td>http://edamontology.org/data_2728</td>\n",
       "      <td>EST accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>610</th>\n",
       "      <td>http://edamontology.org/data_2729</td>\n",
       "      <td>COGEME EST ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>611</th>\n",
       "      <td>http://edamontology.org/data_2730</td>\n",
       "      <td>COGEME unisequence ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>612</th>\n",
       "      <td>http://edamontology.org/data_2731</td>\n",
       "      <td>Protein family ID (GeneFarm)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>613</th>\n",
       "      <td>http://edamontology.org/data_2732</td>\n",
       "      <td>Family name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>614</th>\n",
       "      <td>http://edamontology.org/data_2736</td>\n",
       "      <td>Sequence feature ID (SwissRegulon)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>615</th>\n",
       "      <td>http://edamontology.org/data_2737</td>\n",
       "      <td>FIG ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>616</th>\n",
       "      <td>http://edamontology.org/data_2738</td>\n",
       "      <td>Gene ID (Xenbase)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>617</th>\n",
       "      <td>http://edamontology.org/data_2739</td>\n",
       "      <td>Gene ID (Genolist)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>618</th>\n",
       "      <td>http://edamontology.org/data_2741</td>\n",
       "      <td>ABS ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>619</th>\n",
       "      <td>http://edamontology.org/data_2742</td>\n",
       "      <td>AraC-XylS ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>620</th>\n",
       "      <td>http://edamontology.org/data_2744</td>\n",
       "      <td>Locus ID (PseudoCAP)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>621</th>\n",
       "      <td>http://edamontology.org/data_2745</td>\n",
       "      <td>Locus ID (UTR)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>622</th>\n",
       "      <td>http://edamontology.org/data_2746</td>\n",
       "      <td>MonosaccharideDB ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>623</th>\n",
       "      <td>http://edamontology.org/data_2749</td>\n",
       "      <td>Genome identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>624</th>\n",
       "      <td>http://edamontology.org/data_2752</td>\n",
       "      <td>GlycoMap ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>625</th>\n",
       "      <td>http://edamontology.org/data_2753</td>\n",
       "      <td>Carbohydrate conformational map</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>626</th>\n",
       "      <td>http://edamontology.org/data_2755</td>\n",
       "      <td>Transcription factor name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>627</th>\n",
       "      <td>http://edamontology.org/data_2756</td>\n",
       "      <td>TCID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>628</th>\n",
       "      <td>http://edamontology.org/data_2757</td>\n",
       "      <td>Pfam domain name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>629</th>\n",
       "      <td>http://edamontology.org/data_2758</td>\n",
       "      <td>Pfam clan ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>630</th>\n",
       "      <td>http://edamontology.org/data_2759</td>\n",
       "      <td>Gene ID (VectorBase)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>631</th>\n",
       "      <td>http://edamontology.org/data_2761</td>\n",
       "      <td>UTRSite ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>632</th>\n",
       "      <td>http://edamontology.org/data_2764</td>\n",
       "      <td>Protein name (UniProt)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>633</th>\n",
       "      <td>http://edamontology.org/data_2766</td>\n",
       "      <td>HAMAP ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>634</th>\n",
       "      <td>http://edamontology.org/data_2769</td>\n",
       "      <td>Transcript ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>635</th>\n",
       "      <td>http://edamontology.org/data_2770</td>\n",
       "      <td>HIT ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>636</th>\n",
       "      <td>http://edamontology.org/data_2771</td>\n",
       "      <td>HIX ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>637</th>\n",
       "      <td>http://edamontology.org/data_2772</td>\n",
       "      <td>HPA antibody id</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>638</th>\n",
       "      <td>http://edamontology.org/data_2773</td>\n",
       "      <td>IMGT/HLA ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>639</th>\n",
       "      <td>http://edamontology.org/data_2774</td>\n",
       "      <td>Gene ID (JCVI)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>640</th>\n",
       "      <td>http://edamontology.org/data_2775</td>\n",
       "      <td>Kinase name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>641</th>\n",
       "      <td>http://edamontology.org/data_2776</td>\n",
       "      <td>ConsensusPathDB entity ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>642</th>\n",
       "      <td>http://edamontology.org/data_2777</td>\n",
       "      <td>ConsensusPathDB entity name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>643</th>\n",
       "      <td>http://edamontology.org/data_2778</td>\n",
       "      <td>CCAP strain number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>644</th>\n",
       "      <td>http://edamontology.org/data_2779</td>\n",
       "      <td>Stock number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>645</th>\n",
       "      <td>http://edamontology.org/data_2780</td>\n",
       "      <td>Stock number (TAIR)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>646</th>\n",
       "      <td>http://edamontology.org/data_2781</td>\n",
       "      <td>REDIdb ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>647</th>\n",
       "      <td>http://edamontology.org/data_2782</td>\n",
       "      <td>SMART domain name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>648</th>\n",
       "      <td>http://edamontology.org/data_2783</td>\n",
       "      <td>Protein family ID (PANTHER)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>649</th>\n",
       "      <td>http://edamontology.org/data_2784</td>\n",
       "      <td>RNAVirusDB ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>650</th>\n",
       "      <td>http://edamontology.org/data_2785</td>\n",
       "      <td>Virus identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>651</th>\n",
       "      <td>http://edamontology.org/data_2786</td>\n",
       "      <td>NCBI Genome Project ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>652</th>\n",
       "      <td>http://edamontology.org/data_2787</td>\n",
       "      <td>NCBI genome accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>653</th>\n",
       "      <td>http://edamontology.org/data_2789</td>\n",
       "      <td>Protein ID (TopDB)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>654</th>\n",
       "      <td>http://edamontology.org/data_2790</td>\n",
       "      <td>Gel ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>655</th>\n",
       "      <td>http://edamontology.org/data_2791</td>\n",
       "      <td>Reference map name (SWISS-2DPAGE)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>656</th>\n",
       "      <td>http://edamontology.org/data_2792</td>\n",
       "      <td>Protein ID (PeroxiBase)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>657</th>\n",
       "      <td>http://edamontology.org/data_2793</td>\n",
       "      <td>SISYPHUS ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>658</th>\n",
       "      <td>http://edamontology.org/data_2794</td>\n",
       "      <td>ORF ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>659</th>\n",
       "      <td>http://edamontology.org/data_2795</td>\n",
       "      <td>ORF identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>660</th>\n",
       "      <td>http://edamontology.org/data_2796</td>\n",
       "      <td>LINUCS ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>661</th>\n",
       "      <td>http://edamontology.org/data_2797</td>\n",
       "      <td>Protein ID (LGICdb)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>662</th>\n",
       "      <td>http://edamontology.org/data_2798</td>\n",
       "      <td>MaizeDB ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>663</th>\n",
       "      <td>http://edamontology.org/data_2799</td>\n",
       "      <td>Gene ID (MfunGD)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>664</th>\n",
       "      <td>http://edamontology.org/data_2800</td>\n",
       "      <td>Orpha number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>665</th>\n",
       "      <td>http://edamontology.org/data_2802</td>\n",
       "      <td>Protein ID (EcID)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>666</th>\n",
       "      <td>http://edamontology.org/data_2803</td>\n",
       "      <td>Clone ID (RefSeq)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>667</th>\n",
       "      <td>http://edamontology.org/data_2804</td>\n",
       "      <td>Protein ID (ConoServer)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>668</th>\n",
       "      <td>http://edamontology.org/data_2805</td>\n",
       "      <td>GeneSNP ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>669</th>\n",
       "      <td>http://edamontology.org/data_2812</td>\n",
       "      <td>Lipid identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>670</th>\n",
       "      <td>http://edamontology.org/data_2835</td>\n",
       "      <td>Gene ID (VBASE2)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>671</th>\n",
       "      <td>http://edamontology.org/data_2836</td>\n",
       "      <td>DPVweb ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>672</th>\n",
       "      <td>http://edamontology.org/data_2837</td>\n",
       "      <td>Pathway ID (BioSystems)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>673</th>\n",
       "      <td>http://edamontology.org/data_2855</td>\n",
       "      <td>Distance matrix</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>674</th>\n",
       "      <td>http://edamontology.org/data_2856</td>\n",
       "      <td>Structural distance matrix</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>675</th>\n",
       "      <td>http://edamontology.org/data_2872</td>\n",
       "      <td>ID list</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>676</th>\n",
       "      <td>http://edamontology.org/data_2873</td>\n",
       "      <td>Phylogenetic gene frequencies data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>677</th>\n",
       "      <td>http://edamontology.org/data_2879</td>\n",
       "      <td>Lipid report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>678</th>\n",
       "      <td>http://edamontology.org/data_2884</td>\n",
       "      <td>Plot</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>679</th>\n",
       "      <td>http://edamontology.org/data_2891</td>\n",
       "      <td>Biological model accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>680</th>\n",
       "      <td>http://edamontology.org/data_2892</td>\n",
       "      <td>Cell type name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>681</th>\n",
       "      <td>http://edamontology.org/data_2893</td>\n",
       "      <td>Cell type accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>682</th>\n",
       "      <td>http://edamontology.org/data_2894</td>\n",
       "      <td>Compound accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>683</th>\n",
       "      <td>http://edamontology.org/data_2895</td>\n",
       "      <td>Drug accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>684</th>\n",
       "      <td>http://edamontology.org/data_2896</td>\n",
       "      <td>Toxin name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>685</th>\n",
       "      <td>http://edamontology.org/data_2897</td>\n",
       "      <td>Toxin accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>686</th>\n",
       "      <td>http://edamontology.org/data_2898</td>\n",
       "      <td>Monosaccharide accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>687</th>\n",
       "      <td>http://edamontology.org/data_2899</td>\n",
       "      <td>Drug name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>688</th>\n",
       "      <td>http://edamontology.org/data_2900</td>\n",
       "      <td>Carbohydrate accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>689</th>\n",
       "      <td>http://edamontology.org/data_2901</td>\n",
       "      <td>Molecule accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>690</th>\n",
       "      <td>http://edamontology.org/data_2902</td>\n",
       "      <td>Data resource definition accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>691</th>\n",
       "      <td>http://edamontology.org/data_2903</td>\n",
       "      <td>Genome accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>692</th>\n",
       "      <td>http://edamontology.org/data_2904</td>\n",
       "      <td>Map accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>693</th>\n",
       "      <td>http://edamontology.org/data_2905</td>\n",
       "      <td>Lipid accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>694</th>\n",
       "      <td>http://edamontology.org/data_2906</td>\n",
       "      <td>Peptide ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>695</th>\n",
       "      <td>http://edamontology.org/data_2907</td>\n",
       "      <td>Protein accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>696</th>\n",
       "      <td>http://edamontology.org/data_2908</td>\n",
       "      <td>Organism accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>697</th>\n",
       "      <td>http://edamontology.org/data_2909</td>\n",
       "      <td>Organism name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>698</th>\n",
       "      <td>http://edamontology.org/data_2910</td>\n",
       "      <td>Protein family accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>699</th>\n",
       "      <td>http://edamontology.org/data_2911</td>\n",
       "      <td>Transcription factor accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>700</th>\n",
       "      <td>http://edamontology.org/data_2912</td>\n",
       "      <td>Strain accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>701</th>\n",
       "      <td>http://edamontology.org/data_2914</td>\n",
       "      <td>Sequence features metadata</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>702</th>\n",
       "      <td>http://edamontology.org/data_2915</td>\n",
       "      <td>Gramene identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>703</th>\n",
       "      <td>http://edamontology.org/data_2916</td>\n",
       "      <td>DDBJ accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>704</th>\n",
       "      <td>http://edamontology.org/data_2917</td>\n",
       "      <td>ConsensusPathDB identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>705</th>\n",
       "      <td>http://edamontology.org/data_2955</td>\n",
       "      <td>Sequence report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>706</th>\n",
       "      <td>http://edamontology.org/data_2956</td>\n",
       "      <td>Protein secondary structure</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>707</th>\n",
       "      <td>http://edamontology.org/data_2957</td>\n",
       "      <td>Hopp and Woods plot</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>708</th>\n",
       "      <td>http://edamontology.org/data_2968</td>\n",
       "      <td>Image</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>709</th>\n",
       "      <td>http://edamontology.org/data_2969</td>\n",
       "      <td>Sequence image</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>710</th>\n",
       "      <td>http://edamontology.org/data_2970</td>\n",
       "      <td>Protein hydropathy data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>711</th>\n",
       "      <td>http://edamontology.org/data_2978</td>\n",
       "      <td>Reaction data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>712</th>\n",
       "      <td>http://edamontology.org/data_2979</td>\n",
       "      <td>Peptide property</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>713</th>\n",
       "      <td>http://edamontology.org/data_2985</td>\n",
       "      <td>Nucleic acid thermodynamic data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>714</th>\n",
       "      <td>http://edamontology.org/data_2991</td>\n",
       "      <td>Protein geometry data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>715</th>\n",
       "      <td>http://edamontology.org/data_2992</td>\n",
       "      <td>Protein structure image</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>716</th>\n",
       "      <td>http://edamontology.org/data_2994</td>\n",
       "      <td>Phylogenetic character weights</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>717</th>\n",
       "      <td>http://edamontology.org/data_3002</td>\n",
       "      <td>Annotation track</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>718</th>\n",
       "      <td>http://edamontology.org/data_3021</td>\n",
       "      <td>UniProt accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>719</th>\n",
       "      <td>http://edamontology.org/data_3022</td>\n",
       "      <td>NCBI genetic code ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>720</th>\n",
       "      <td>http://edamontology.org/data_3025</td>\n",
       "      <td>Ontology concept identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>721</th>\n",
       "      <td>http://edamontology.org/data_3029</td>\n",
       "      <td>Protein ID (EMBL/GenBank/DDBJ)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>722</th>\n",
       "      <td>http://edamontology.org/data_3034</td>\n",
       "      <td>Sequence feature identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>723</th>\n",
       "      <td>http://edamontology.org/data_3035</td>\n",
       "      <td>Structure identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>724</th>\n",
       "      <td>http://edamontology.org/data_3036</td>\n",
       "      <td>Matrix identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>725</th>\n",
       "      <td>http://edamontology.org/data_3103</td>\n",
       "      <td>ATC code</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>726</th>\n",
       "      <td>http://edamontology.org/data_3104</td>\n",
       "      <td>UNII</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>727</th>\n",
       "      <td>http://edamontology.org/data_3106</td>\n",
       "      <td>System metadata</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>728</th>\n",
       "      <td>http://edamontology.org/data_3108</td>\n",
       "      <td>Experimental measurement</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>729</th>\n",
       "      <td>http://edamontology.org/data_3113</td>\n",
       "      <td>Sample annotation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>730</th>\n",
       "      <td>http://edamontology.org/data_3128</td>\n",
       "      <td>Nucleic acid structure report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>731</th>\n",
       "      <td>http://edamontology.org/data_3134</td>\n",
       "      <td>Gene transcript report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>732</th>\n",
       "      <td>http://edamontology.org/data_3148</td>\n",
       "      <td>Gene family report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>733</th>\n",
       "      <td>http://edamontology.org/data_3153</td>\n",
       "      <td>Protein image</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>734</th>\n",
       "      <td>http://edamontology.org/data_3181</td>\n",
       "      <td>Sequence assembly report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>735</th>\n",
       "      <td>http://edamontology.org/data_3236</td>\n",
       "      <td>Cytoband position</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>736</th>\n",
       "      <td>http://edamontology.org/data_3238</td>\n",
       "      <td>Cell type ontology ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>737</th>\n",
       "      <td>http://edamontology.org/data_3264</td>\n",
       "      <td>COSMIC ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>738</th>\n",
       "      <td>http://edamontology.org/data_3265</td>\n",
       "      <td>HGMD ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>739</th>\n",
       "      <td>http://edamontology.org/data_3266</td>\n",
       "      <td>Sequence assembly ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>740</th>\n",
       "      <td>http://edamontology.org/data_3270</td>\n",
       "      <td>Ensembl gene tree ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>741</th>\n",
       "      <td>http://edamontology.org/data_3273</td>\n",
       "      <td>Sample ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>742</th>\n",
       "      <td>http://edamontology.org/data_3274</td>\n",
       "      <td>MGI accession</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>743</th>\n",
       "      <td>http://edamontology.org/data_3275</td>\n",
       "      <td>Phenotype name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>744</th>\n",
       "      <td>http://edamontology.org/data_3354</td>\n",
       "      <td>Transition matrix</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>745</th>\n",
       "      <td>http://edamontology.org/data_3355</td>\n",
       "      <td>Emission matrix</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>746</th>\n",
       "      <td>http://edamontology.org/data_3358</td>\n",
       "      <td>Format identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>747</th>\n",
       "      <td>http://edamontology.org/data_3424</td>\n",
       "      <td>Raw image</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>748</th>\n",
       "      <td>http://edamontology.org/data_3425</td>\n",
       "      <td>Carbohydrate property</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>749</th>\n",
       "      <td>http://edamontology.org/data_3451</td>\n",
       "      <td>Rate of association</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>750</th>\n",
       "      <td>http://edamontology.org/data_3479</td>\n",
       "      <td>Gene order</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>751</th>\n",
       "      <td>http://edamontology.org/data_3483</td>\n",
       "      <td>Spectrum</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>752</th>\n",
       "      <td>http://edamontology.org/data_3509</td>\n",
       "      <td>Ontology mapping</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>753</th>\n",
       "      <td>http://edamontology.org/data_3546</td>\n",
       "      <td>Image metadata</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>754</th>\n",
       "      <td>http://edamontology.org/data_3558</td>\n",
       "      <td>Clinical trial report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>755</th>\n",
       "      <td>http://edamontology.org/data_3567</td>\n",
       "      <td>Reference sample report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>756</th>\n",
       "      <td>http://edamontology.org/data_3568</td>\n",
       "      <td>Gene Expression Atlas Experiment ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>757</th>\n",
       "      <td>http://edamontology.org/data_3667</td>\n",
       "      <td>Disease identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>758</th>\n",
       "      <td>http://edamontology.org/data_3668</td>\n",
       "      <td>Disease name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>759</th>\n",
       "      <td>http://edamontology.org/data_3669</td>\n",
       "      <td>Training material</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>760</th>\n",
       "      <td>http://edamontology.org/data_3670</td>\n",
       "      <td>Online course</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>761</th>\n",
       "      <td>http://edamontology.org/data_3707</td>\n",
       "      <td>Biodiversity data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>762</th>\n",
       "      <td>http://edamontology.org/data_3716</td>\n",
       "      <td>Biosafety report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>763</th>\n",
       "      <td>http://edamontology.org/data_3717</td>\n",
       "      <td>Isolation report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>764</th>\n",
       "      <td>http://edamontology.org/data_3718</td>\n",
       "      <td>Pathogenicity report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>765</th>\n",
       "      <td>http://edamontology.org/data_3719</td>\n",
       "      <td>Biosafety classification</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>766</th>\n",
       "      <td>http://edamontology.org/data_3720</td>\n",
       "      <td>Geographic location</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>767</th>\n",
       "      <td>http://edamontology.org/data_3721</td>\n",
       "      <td>Isolation source</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>768</th>\n",
       "      <td>http://edamontology.org/data_3722</td>\n",
       "      <td>Physiology parameter</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>769</th>\n",
       "      <td>http://edamontology.org/data_3723</td>\n",
       "      <td>Morphology parameter</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>770</th>\n",
       "      <td>http://edamontology.org/data_3724</td>\n",
       "      <td>Cultivation parameter</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>771</th>\n",
       "      <td>http://edamontology.org/data_3732</td>\n",
       "      <td>Sequencing metadata name</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>772</th>\n",
       "      <td>http://edamontology.org/data_3733</td>\n",
       "      <td>Flow cell identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>773</th>\n",
       "      <td>http://edamontology.org/data_3734</td>\n",
       "      <td>Lane identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>774</th>\n",
       "      <td>http://edamontology.org/data_3735</td>\n",
       "      <td>Run number</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>775</th>\n",
       "      <td>http://edamontology.org/data_3736</td>\n",
       "      <td>Ecological data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>776</th>\n",
       "      <td>http://edamontology.org/data_3737</td>\n",
       "      <td>Alpha diversity data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>777</th>\n",
       "      <td>http://edamontology.org/data_3738</td>\n",
       "      <td>Beta diversity data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>778</th>\n",
       "      <td>http://edamontology.org/data_3739</td>\n",
       "      <td>Gamma diversity data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>779</th>\n",
       "      <td>http://edamontology.org/data_3743</td>\n",
       "      <td>Ordination plot</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>780</th>\n",
       "      <td>http://edamontology.org/data_3753</td>\n",
       "      <td>Over-representation data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>781</th>\n",
       "      <td>http://edamontology.org/data_3756</td>\n",
       "      <td>Localisation score</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>782</th>\n",
       "      <td>http://edamontology.org/data_3757</td>\n",
       "      <td>Unimod ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>783</th>\n",
       "      <td>http://edamontology.org/data_3759</td>\n",
       "      <td>ProteomeXchange ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>784</th>\n",
       "      <td>http://edamontology.org/data_3769</td>\n",
       "      <td>BRENDA ontology concept ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>785</th>\n",
       "      <td>http://edamontology.org/data_3786</td>\n",
       "      <td>Query script</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>786</th>\n",
       "      <td>http://edamontology.org/data_3856</td>\n",
       "      <td>RNA central ID</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>787</th>\n",
       "      <td>http://edamontology.org/data_3861</td>\n",
       "      <td>Electronic health record</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>788</th>\n",
       "      <td>http://edamontology.org/data_3869</td>\n",
       "      <td>Simulation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>789</th>\n",
       "      <td>http://edamontology.org/data_3870</td>\n",
       "      <td>Trajectory data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>790</th>\n",
       "      <td>http://edamontology.org/data_3871</td>\n",
       "      <td>Forcefield parameters</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>791</th>\n",
       "      <td>http://edamontology.org/data_3872</td>\n",
       "      <td>Topology data</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>792</th>\n",
       "      <td>http://edamontology.org/data_3905</td>\n",
       "      <td>Histogram</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>793</th>\n",
       "      <td>http://edamontology.org/data_3914</td>\n",
       "      <td>Quality control report</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>794</th>\n",
       "      <td>http://edamontology.org/data_3917</td>\n",
       "      <td>Count matrix</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>795</th>\n",
       "      <td>http://edamontology.org/data_3932</td>\n",
       "      <td>Q-value</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>796</th>\n",
       "      <td>http://edamontology.org/data_3952</td>\n",
       "      <td>Pathway ID (WikiPathways)</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>797</th>\n",
       "      <td>http://edamontology.org/data_4022</td>\n",
       "      <td>ORCID Identifier</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>798</th>\n",
       "      <td>http://edamontology.org/operation_0227</td>\n",
       "      <td>Indexing</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>799</th>\n",
       "      <td>http://edamontology.org/operation_0244</td>\n",
       "      <td>Simulation analysis</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>800</th>\n",
       "      <td>http://edamontology.org/operation_0249</td>\n",
       "      <td>Protein geometry calculation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>801</th>\n",
       "      <td>http://edamontology.org/operation_0250</td>\n",
       "      <td>Protein property calculation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>802</th>\n",
       "      <td>http://edamontology.org/operation_0262</td>\n",
       "      <td>Nucleic acid property calculation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>803</th>\n",
       "      <td>http://edamontology.org/operation_0336</td>\n",
       "      <td>Format validation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>804</th>\n",
       "      <td>http://edamontology.org/operation_0386</td>\n",
       "      <td>Protein dipole moment calculation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>805</th>\n",
       "      <td>http://edamontology.org/operation_0396</td>\n",
       "      <td>Ramachandran plot calculation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>806</th>\n",
       "      <td>http://edamontology.org/operation_0399</td>\n",
       "      <td>Protein extinction coefficient calculation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>807</th>\n",
       "      <td>http://edamontology.org/operation_0455</td>\n",
       "      <td>Nucleic acid thermodynamic property calculation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>808</th>\n",
       "      <td>http://edamontology.org/operation_0461</td>\n",
       "      <td>Nucleic acid curvature calculation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>809</th>\n",
       "      <td>http://edamontology.org/operation_2238</td>\n",
       "      <td>Statistical calculation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>810</th>\n",
       "      <td>http://edamontology.org/operation_2423</td>\n",
       "      <td>Prediction and recognition</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>811</th>\n",
       "      <td>http://edamontology.org/operation_2424</td>\n",
       "      <td>Comparison</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>812</th>\n",
       "      <td>http://edamontology.org/operation_2425</td>\n",
       "      <td>Optimisation and refinement</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>813</th>\n",
       "      <td>http://edamontology.org/operation_2428</td>\n",
       "      <td>Validation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>814</th>\n",
       "      <td>http://edamontology.org/operation_2429</td>\n",
       "      <td>Mapping</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>815</th>\n",
       "      <td>http://edamontology.org/operation_2430</td>\n",
       "      <td>Design</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>816</th>\n",
       "      <td>http://edamontology.org/operation_2844</td>\n",
       "      <td>Structure clustering</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>817</th>\n",
       "      <td>http://edamontology.org/operation_2928</td>\n",
       "      <td>Alignment</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>818</th>\n",
       "      <td>http://edamontology.org/operation_2930</td>\n",
       "      <td>Protein property comparison</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>819</th>\n",
       "      <td>http://edamontology.org/operation_2945</td>\n",
       "      <td>Analysis</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>820</th>\n",
       "      <td>http://edamontology.org/operation_2990</td>\n",
       "      <td>Classification</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>821</th>\n",
       "      <td>http://edamontology.org/operation_2997</td>\n",
       "      <td>Protein comparison</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>822</th>\n",
       "      <td>http://edamontology.org/operation_2998</td>\n",
       "      <td>Nucleic acid comparison</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>823</th>\n",
       "      <td>http://edamontology.org/operation_3196</td>\n",
       "      <td>Genotyping</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>824</th>\n",
       "      <td>http://edamontology.org/operation_3197</td>\n",
       "      <td>Genetic variation analysis</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>825</th>\n",
       "      <td>http://edamontology.org/operation_3226</td>\n",
       "      <td>Variant prioritisation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>826</th>\n",
       "      <td>http://edamontology.org/operation_3228</td>\n",
       "      <td>Structural variation detection</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>827</th>\n",
       "      <td>http://edamontology.org/operation_3233</td>\n",
       "      <td>Copy number estimation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>828</th>\n",
       "      <td>http://edamontology.org/operation_3267</td>\n",
       "      <td>Sequence coordinate conversion</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>829</th>\n",
       "      <td>http://edamontology.org/operation_3282</td>\n",
       "      <td>ID mapping</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>830</th>\n",
       "      <td>http://edamontology.org/operation_3349</td>\n",
       "      <td>Bibliography generation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>831</th>\n",
       "      <td>http://edamontology.org/operation_3350</td>\n",
       "      <td>Protein quaternary structure prediction</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>832</th>\n",
       "      <td>http://edamontology.org/operation_3352</td>\n",
       "      <td>Ontology comparison</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>833</th>\n",
       "      <td>http://edamontology.org/operation_3429</td>\n",
       "      <td>Generation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>834</th>\n",
       "      <td>http://edamontology.org/operation_3432</td>\n",
       "      <td>Clustering</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>835</th>\n",
       "      <td>http://edamontology.org/operation_3434</td>\n",
       "      <td>Conversion</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>836</th>\n",
       "      <td>http://edamontology.org/operation_3435</td>\n",
       "      <td>Standardisation and normalisation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>837</th>\n",
       "      <td>http://edamontology.org/operation_3438</td>\n",
       "      <td>Calculation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>838</th>\n",
       "      <td>http://edamontology.org/operation_3480</td>\n",
       "      <td>Probabilistic data generation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>839</th>\n",
       "      <td>http://edamontology.org/operation_3501</td>\n",
       "      <td>Enrichment analysis</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>840</th>\n",
       "      <td>http://edamontology.org/operation_3504</td>\n",
       "      <td>Variant pattern analysis</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>841</th>\n",
       "      <td>http://edamontology.org/operation_3557</td>\n",
       "      <td>Imputation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>842</th>\n",
       "      <td>http://edamontology.org/operation_3561</td>\n",
       "      <td>Database comparison</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>843</th>\n",
       "      <td>http://edamontology.org/operation_3633</td>\n",
       "      <td>Retention time prediction</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>844</th>\n",
       "      <td>http://edamontology.org/operation_3658</td>\n",
       "      <td>Statistical inference</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>845</th>\n",
       "      <td>http://edamontology.org/operation_3659</td>\n",
       "      <td>Regression analysis</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>846</th>\n",
       "      <td>http://edamontology.org/operation_3664</td>\n",
       "      <td>Statistical modelling</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>847</th>\n",
       "      <td>http://edamontology.org/operation_3703</td>\n",
       "      <td>Reference identification</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>848</th>\n",
       "      <td>http://edamontology.org/operation_3731</td>\n",
       "      <td>Sample comparison</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>849</th>\n",
       "      <td>http://edamontology.org/operation_3760</td>\n",
       "      <td>Service management</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>850</th>\n",
       "      <td>http://edamontology.org/operation_3761</td>\n",
       "      <td>Service discovery</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>851</th>\n",
       "      <td>http://edamontology.org/operation_3762</td>\n",
       "      <td>Service composition</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>852</th>\n",
       "      <td>http://edamontology.org/operation_3763</td>\n",
       "      <td>Service invocation</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>853</th>\n",
       "      <td>http://edamontology.org/operation_3791</td>\n",
       "      <td>Collapsing methods</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>854</th>\n",
       "      <td>http://edamontology.org/operation_3797</td>\n",
       "      <td>Rarefaction</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>855</th>\n",
       "      <td>http://edamontology.org/operation_3799</td>\n",
       "      <td>Quantification</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>856</th>\n",
       "      <td>http://edamontology.org/operation_3800</td>\n",
       "      <td>RNA-Seq quantification</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>857</th>\n",
       "      <td>http://edamontology.org/operation_3840</td>\n",
       "      <td>Multilocus sequence typing</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>858</th>\n",
       "      <td>http://edamontology.org/operation_3960</td>\n",
       "      <td>Principal component analysis</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>859</th>\n",
       "      <td>http://edamontology.org/operation_3961</td>\n",
       "      <td>Copy number variation detection</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>860</th>\n",
       "      <td>http://edamontology.org/operation_3962</td>\n",
       "      <td>Deletion detection</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>861</th>\n",
       "      <td>http://edamontology.org/operation_3963</td>\n",
       "      <td>Duplication detection</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>862</th>\n",
       "      <td>http://edamontology.org/operation_3964</td>\n",
       "      <td>Complex CNV detection</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>863</th>\n",
       "      <td>http://edamontology.org/operation_3965</td>\n",
       "      <td>Amplification detection</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>864</th>\n",
       "      <td>http://edamontology.org/operation_4008</td>\n",
       "      <td>Protein design</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>865</th>\n",
       "      <td>http://edamontology.org/operation_4009</td>\n",
       "      <td>Small molecule design</td>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                     entity                                            label                           property\n",
       "0         http://edamontology.org/data_0842                                       Identifier  http://edamontology.org/has_topic\n",
       "1         http://edamontology.org/data_0844                                   Molecular mass  http://edamontology.org/has_topic\n",
       "2         http://edamontology.org/data_0845                                 Molecular charge  http://edamontology.org/has_topic\n",
       "3         http://edamontology.org/data_0846                                 Chemical formula  http://edamontology.org/has_topic\n",
       "4         http://edamontology.org/data_0847                                  QSAR descriptor  http://edamontology.org/has_topic\n",
       "5         http://edamontology.org/data_0850                                     Sequence set  http://edamontology.org/has_topic\n",
       "6         http://edamontology.org/data_0856                          Sequence feature source  http://edamontology.org/has_topic\n",
       "7         http://edamontology.org/data_0857                          Sequence search results  http://edamontology.org/has_topic\n",
       "8         http://edamontology.org/data_0862                                          Dotplot  http://edamontology.org/has_topic\n",
       "9         http://edamontology.org/data_0865                        Sequence similarity score  http://edamontology.org/has_topic\n",
       "10        http://edamontology.org/data_0867                        Sequence alignment report  http://edamontology.org/has_topic\n",
       "11        http://edamontology.org/data_0870                         Sequence distance matrix  http://edamontology.org/has_topic\n",
       "12        http://edamontology.org/data_0871                      Phylogenetic character data  http://edamontology.org/has_topic\n",
       "13        http://edamontology.org/data_0874                                Comparison matrix  http://edamontology.org/has_topic\n",
       "14        http://edamontology.org/data_0878            Protein secondary structure alignment  http://edamontology.org/has_topic\n",
       "15        http://edamontology.org/data_0881                RNA secondary structure alignment  http://edamontology.org/has_topic\n",
       "16        http://edamontology.org/data_0887                       Structure alignment report  http://edamontology.org/has_topic\n",
       "17        http://edamontology.org/data_0888                       Structure similarity score  http://edamontology.org/has_topic\n",
       "18        http://edamontology.org/data_0890                Structural (3D) profile alignment  http://edamontology.org/has_topic\n",
       "19        http://edamontology.org/data_0892        Protein sequence-structure scoring matrix  http://edamontology.org/has_topic\n",
       "20        http://edamontology.org/data_0893                     Sequence-structure alignment  http://edamontology.org/has_topic\n",
       "21        http://edamontology.org/data_0896                                   Protein report  http://edamontology.org/has_topic\n",
       "22        http://edamontology.org/data_0897                                 Protein property  http://edamontology.org/has_topic\n",
       "23        http://edamontology.org/data_0905                     Protein interaction raw data  http://edamontology.org/has_topic\n",
       "24        http://edamontology.org/data_0909                                             Vmax  http://edamontology.org/has_topic\n",
       "25        http://edamontology.org/data_0910                                               Km  http://edamontology.org/has_topic\n",
       "26        http://edamontology.org/data_0912                            Nucleic acid property  http://edamontology.org/has_topic\n",
       "27        http://edamontology.org/data_0916                                      Gene report  http://edamontology.org/has_topic\n",
       "28        http://edamontology.org/data_0919                                Chromosome report  http://edamontology.org/has_topic\n",
       "29        http://edamontology.org/data_0920                        Genotype/phenotype report  http://edamontology.org/has_topic\n",
       "30        http://edamontology.org/data_0924                                   Sequence trace  http://edamontology.org/has_topic\n",
       "31        http://edamontology.org/data_0925                                Sequence assembly  http://edamontology.org/has_topic\n",
       "32        http://edamontology.org/data_0926                                        RH scores  http://edamontology.org/has_topic\n",
       "33        http://edamontology.org/data_0927                           Genetic linkage report  http://edamontology.org/has_topic\n",
       "34        http://edamontology.org/data_0937                             Electron density map  http://edamontology.org/has_topic\n",
       "35        http://edamontology.org/data_0938                                     Raw NMR data  http://edamontology.org/has_topic\n",
       "36        http://edamontology.org/data_0939                                       CD spectra  http://edamontology.org/has_topic\n",
       "37        http://edamontology.org/data_0949                                Workflow metadata  http://edamontology.org/has_topic\n",
       "38        http://edamontology.org/data_0951                       Statistical estimate score  http://edamontology.org/has_topic\n",
       "39        http://edamontology.org/data_0954                           Database cross-mapping  http://edamontology.org/has_topic\n",
       "40        http://edamontology.org/data_0957                                Database metadata  http://edamontology.org/has_topic\n",
       "41        http://edamontology.org/data_0958                                    Tool metadata  http://edamontology.org/has_topic\n",
       "42        http://edamontology.org/data_0960                                    User metadata  http://edamontology.org/has_topic\n",
       "43        http://edamontology.org/data_0963                                 Cell line report  http://edamontology.org/has_topic\n",
       "44        http://edamontology.org/data_0968                                          Keyword  http://edamontology.org/has_topic\n",
       "45        http://edamontology.org/data_0976                   Identifier (by type of entity)  http://edamontology.org/has_topic\n",
       "46        http://edamontology.org/data_0977                                  Tool identifier  http://edamontology.org/has_topic\n",
       "47        http://edamontology.org/data_0982                              Molecule identifier  http://edamontology.org/has_topic\n",
       "48        http://edamontology.org/data_0983                                          Atom ID  http://edamontology.org/has_topic\n",
       "49        http://edamontology.org/data_0984                                    Molecule name  http://edamontology.org/has_topic\n",
       "50        http://edamontology.org/data_0987                                  Chromosome name  http://edamontology.org/has_topic\n",
       "51        http://edamontology.org/data_0988                               Peptide identifier  http://edamontology.org/has_topic\n",
       "52        http://edamontology.org/data_0989                               Protein identifier  http://edamontology.org/has_topic\n",
       "53        http://edamontology.org/data_0990                                    Compound name  http://edamontology.org/has_topic\n",
       "54        http://edamontology.org/data_0991                         Chemical registry number  http://edamontology.org/has_topic\n",
       "55        http://edamontology.org/data_0993                                  Drug identifier  http://edamontology.org/has_topic\n",
       "56        http://edamontology.org/data_0994                            Amino acid identifier  http://edamontology.org/has_topic\n",
       "57        http://edamontology.org/data_0995                            Nucleotide identifier  http://edamontology.org/has_topic\n",
       "58        http://edamontology.org/data_0996                        Monosaccharide identifier  http://edamontology.org/has_topic\n",
       "59        http://edamontology.org/data_0997                            Chemical name (ChEBI)  http://edamontology.org/has_topic\n",
       "60        http://edamontology.org/data_0998                            Chemical name (IUPAC)  http://edamontology.org/has_topic\n",
       "61        http://edamontology.org/data_0999                              Chemical name (INN)  http://edamontology.org/has_topic\n",
       "62        http://edamontology.org/data_1000                            Chemical name (brand)  http://edamontology.org/has_topic\n",
       "63        http://edamontology.org/data_1001                       Chemical name (synonymous)  http://edamontology.org/has_topic\n",
       "64        http://edamontology.org/data_1002                                       CAS number  http://edamontology.org/has_topic\n",
       "65        http://edamontology.org/data_1003             Chemical registry number (Beilstein)  http://edamontology.org/has_topic\n",
       "66        http://edamontology.org/data_1004                Chemical registry number (Gmelin)  http://edamontology.org/has_topic\n",
       "67        http://edamontology.org/data_1005                                   HET group name  http://edamontology.org/has_topic\n",
       "68        http://edamontology.org/data_1006                                  Amino acid name  http://edamontology.org/has_topic\n",
       "69        http://edamontology.org/data_1007                                  Nucleotide code  http://edamontology.org/has_topic\n",
       "70        http://edamontology.org/data_1008                             Polypeptide chain ID  http://edamontology.org/has_topic\n",
       "71        http://edamontology.org/data_1009                                     Protein name  http://edamontology.org/has_topic\n",
       "72        http://edamontology.org/data_1010                                Enzyme identifier  http://edamontology.org/has_topic\n",
       "73        http://edamontology.org/data_1011                                        EC number  http://edamontology.org/has_topic\n",
       "74        http://edamontology.org/data_1012                                      Enzyme name  http://edamontology.org/has_topic\n",
       "75        http://edamontology.org/data_1013                          Restriction enzyme name  http://edamontology.org/has_topic\n",
       "76        http://edamontology.org/data_1015                              Sequence feature ID  http://edamontology.org/has_topic\n",
       "77        http://edamontology.org/data_1016                                Sequence position  http://edamontology.org/has_topic\n",
       "78        http://edamontology.org/data_1017                                   Sequence range  http://edamontology.org/has_topic\n",
       "79        http://edamontology.org/data_1020                             Sequence feature key  http://edamontology.org/has_topic\n",
       "80        http://edamontology.org/data_1021                       Sequence feature qualifier  http://edamontology.org/has_topic\n",
       "81        http://edamontology.org/data_1022                           Sequence feature label  http://edamontology.org/has_topic\n",
       "82        http://edamontology.org/data_1023                    EMBOSS Uniform Feature Object  http://edamontology.org/has_topic\n",
       "83        http://edamontology.org/data_1025                                  Gene identifier  http://edamontology.org/has_topic\n",
       "84        http://edamontology.org/data_1026                                      Gene symbol  http://edamontology.org/has_topic\n",
       "85        http://edamontology.org/data_1027                                   Gene ID (NCBI)  http://edamontology.org/has_topic\n",
       "86        http://edamontology.org/data_1031                                    Gene ID (CGD)  http://edamontology.org/has_topic\n",
       "87        http://edamontology.org/data_1032                              Gene ID (DictyBase)  http://edamontology.org/has_topic\n",
       "88        http://edamontology.org/data_1033                                  Ensembl gene ID  http://edamontology.org/has_topic\n",
       "89        http://edamontology.org/data_1034                                    Gene ID (SGD)  http://edamontology.org/has_topic\n",
       "90        http://edamontology.org/data_1035                                 Gene ID (GeneDB)  http://edamontology.org/has_topic\n",
       "91        http://edamontology.org/data_1036                                  TIGR identifier  http://edamontology.org/has_topic\n",
       "92        http://edamontology.org/data_1037                            TAIR accession (gene)  http://edamontology.org/has_topic\n",
       "93        http://edamontology.org/data_1038                                Protein domain ID  http://edamontology.org/has_topic\n",
       "94        http://edamontology.org/data_1039                           SCOP domain identifier  http://edamontology.org/has_topic\n",
       "95        http://edamontology.org/data_1040                                   CATH domain ID  http://edamontology.org/has_topic\n",
       "96        http://edamontology.org/data_1041        SCOP concise classification string (sccs)  http://edamontology.org/has_topic\n",
       "97        http://edamontology.org/data_1042                                       SCOP sunid  http://edamontology.org/has_topic\n",
       "98        http://edamontology.org/data_1043                                     CATH node ID  http://edamontology.org/has_topic\n",
       "99        http://edamontology.org/data_1044                                     Kingdom name  http://edamontology.org/has_topic\n",
       "100       http://edamontology.org/data_1045                                     Species name  http://edamontology.org/has_topic\n",
       "101       http://edamontology.org/data_1046                                      Strain name  http://edamontology.org/has_topic\n",
       "102       http://edamontology.org/data_1047                                              URI  http://edamontology.org/has_topic\n",
       "103       http://edamontology.org/data_1048                                      Database ID  http://edamontology.org/has_topic\n",
       "104       http://edamontology.org/data_1049                                   Directory name  http://edamontology.org/has_topic\n",
       "105       http://edamontology.org/data_1050                                        File name  http://edamontology.org/has_topic\n",
       "106       http://edamontology.org/data_1051                                    Ontology name  http://edamontology.org/has_topic\n",
       "107       http://edamontology.org/data_1052                                              URL  http://edamontology.org/has_topic\n",
       "108       http://edamontology.org/data_1053                                              URN  http://edamontology.org/has_topic\n",
       "109       http://edamontology.org/data_1055                                             LSID  http://edamontology.org/has_topic\n",
       "110       http://edamontology.org/data_1056                                    Database name  http://edamontology.org/has_topic\n",
       "111       http://edamontology.org/data_1058                             Enumerated file name  http://edamontology.org/has_topic\n",
       "112       http://edamontology.org/data_1059                              File name extension  http://edamontology.org/has_topic\n",
       "113       http://edamontology.org/data_1060                                   File base name  http://edamontology.org/has_topic\n",
       "114       http://edamontology.org/data_1061                             QSAR descriptor name  http://edamontology.org/has_topic\n",
       "115       http://edamontology.org/data_1063                              Sequence identifier  http://edamontology.org/has_topic\n",
       "116       http://edamontology.org/data_1064                                  Sequence set ID  http://edamontology.org/has_topic\n",
       "117       http://edamontology.org/data_1066                            Sequence alignment ID  http://edamontology.org/has_topic\n",
       "118       http://edamontology.org/data_1068                             Phylogenetic tree ID  http://edamontology.org/has_topic\n",
       "119       http://edamontology.org/data_1069                     Comparison matrix identifier  http://edamontology.org/has_topic\n",
       "120       http://edamontology.org/data_1070                                     Structure ID  http://edamontology.org/has_topic\n",
       "121       http://edamontology.org/data_1071                       Structural (3D) profile ID  http://edamontology.org/has_topic\n",
       "122       http://edamontology.org/data_1072                           Structure alignment ID  http://edamontology.org/has_topic\n",
       "123       http://edamontology.org/data_1073                              Amino acid index ID  http://edamontology.org/has_topic\n",
       "124       http://edamontology.org/data_1074                           Protein interaction ID  http://edamontology.org/has_topic\n",
       "125       http://edamontology.org/data_1075                        Protein family identifier  http://edamontology.org/has_topic\n",
       "126       http://edamontology.org/data_1076                           Codon usage table name  http://edamontology.org/has_topic\n",
       "127       http://edamontology.org/data_1077                  Transcription factor identifier  http://edamontology.org/has_topic\n",
       "128       http://edamontology.org/data_1078                         Experiment annotation ID  http://edamontology.org/has_topic\n",
       "129       http://edamontology.org/data_1079                     Electron microscopy model ID  http://edamontology.org/has_topic\n",
       "130       http://edamontology.org/data_1080                        Gene expression report ID  http://edamontology.org/has_topic\n",
       "131       http://edamontology.org/data_1081             Genotype and phenotype annotation ID  http://edamontology.org/has_topic\n",
       "132       http://edamontology.org/data_1082                    Pathway or network identifier  http://edamontology.org/has_topic\n",
       "133       http://edamontology.org/data_1083                                      Workflow ID  http://edamontology.org/has_topic\n",
       "134       http://edamontology.org/data_1084                      Data resource definition ID  http://edamontology.org/has_topic\n",
       "135       http://edamontology.org/data_1085                              Biological model ID  http://edamontology.org/has_topic\n",
       "136       http://edamontology.org/data_1086                              Compound identifier  http://edamontology.org/has_topic\n",
       "137       http://edamontology.org/data_1087                              Ontology concept ID  http://edamontology.org/has_topic\n",
       "138       http://edamontology.org/data_1088                                       Article ID  http://edamontology.org/has_topic\n",
       "139       http://edamontology.org/data_1089                                       FlyBase ID  http://edamontology.org/has_topic\n",
       "140       http://edamontology.org/data_1091                                    WormBase name  http://edamontology.org/has_topic\n",
       "141       http://edamontology.org/data_1092                                   WormBase class  http://edamontology.org/has_topic\n",
       "142       http://edamontology.org/data_1093                               Sequence accession  http://edamontology.org/has_topic\n",
       "143       http://edamontology.org/data_1095                  EMBOSS Uniform Sequence Address  http://edamontology.org/has_topic\n",
       "144       http://edamontology.org/data_1096                     Sequence accession (protein)  http://edamontology.org/has_topic\n",
       "145       http://edamontology.org/data_1097                Sequence accession (nucleic acid)  http://edamontology.org/has_topic\n",
       "146       http://edamontology.org/data_1098                                 RefSeq accession  http://edamontology.org/has_topic\n",
       "147       http://edamontology.org/data_1100                                   PIR identifier  http://edamontology.org/has_topic\n",
       "148       http://edamontology.org/data_1102                       Gramene primary identifier  http://edamontology.org/has_topic\n",
       "149       http://edamontology.org/data_1103                             EMBL/GenBank/DDBJ ID  http://edamontology.org/has_topic\n",
       "150       http://edamontology.org/data_1104                    Sequence cluster ID (UniGene)  http://edamontology.org/has_topic\n",
       "151       http://edamontology.org/data_1105                                  dbEST accession  http://edamontology.org/has_topic\n",
       "152       http://edamontology.org/data_1106                                         dbSNP ID  http://edamontology.org/has_topic\n",
       "153       http://edamontology.org/data_1112                              Sequence cluster ID  http://edamontology.org/has_topic\n",
       "154       http://edamontology.org/data_1113                        Sequence cluster ID (COG)  http://edamontology.org/has_topic\n",
       "155       http://edamontology.org/data_1114                        Sequence motif identifier  http://edamontology.org/has_topic\n",
       "156       http://edamontology.org/data_1115                              Sequence profile ID  http://edamontology.org/has_topic\n",
       "157       http://edamontology.org/data_1116                                           ELM ID  http://edamontology.org/has_topic\n",
       "158       http://edamontology.org/data_1117                         Prosite accession number  http://edamontology.org/has_topic\n",
       "159       http://edamontology.org/data_1118                     HMMER hidden Markov model ID  http://edamontology.org/has_topic\n",
       "160       http://edamontology.org/data_1119                                JASPAR profile ID  http://edamontology.org/has_topic\n",
       "161       http://edamontology.org/data_1123                  TreeBASE study accession number  http://edamontology.org/has_topic\n",
       "162       http://edamontology.org/data_1124                         TreeFam accession number  http://edamontology.org/has_topic\n",
       "163       http://edamontology.org/data_1126                           Comparison matrix name  http://edamontology.org/has_topic\n",
       "164       http://edamontology.org/data_1127                                           PDB ID  http://edamontology.org/has_topic\n",
       "165       http://edamontology.org/data_1128                                       AAindex ID  http://edamontology.org/has_topic\n",
       "166       http://edamontology.org/data_1129                            BIND accession number  http://edamontology.org/has_topic\n",
       "167       http://edamontology.org/data_1130                          IntAct accession number  http://edamontology.org/has_topic\n",
       "168       http://edamontology.org/data_1131                              Protein family name  http://edamontology.org/has_topic\n",
       "169       http://edamontology.org/data_1132                              InterPro entry name  http://edamontology.org/has_topic\n",
       "170       http://edamontology.org/data_1133                               InterPro accession  http://edamontology.org/has_topic\n",
       "171       http://edamontology.org/data_1134                     InterPro secondary accession  http://edamontology.org/has_topic\n",
       "172       http://edamontology.org/data_1135                                        Gene3D ID  http://edamontology.org/has_topic\n",
       "173       http://edamontology.org/data_1136                                         PIRSF ID  http://edamontology.org/has_topic\n",
       "174       http://edamontology.org/data_1137                                      PRINTS code  http://edamontology.org/has_topic\n",
       "175       http://edamontology.org/data_1138                            Pfam accession number  http://edamontology.org/has_topic\n",
       "176       http://edamontology.org/data_1139                           SMART accession number  http://edamontology.org/has_topic\n",
       "177       http://edamontology.org/data_1140           Superfamily hidden Markov model number  http://edamontology.org/has_topic\n",
       "178       http://edamontology.org/data_1141                                       TIGRFam ID  http://edamontology.org/has_topic\n",
       "179       http://edamontology.org/data_1142                          ProDom accession number  http://edamontology.org/has_topic\n",
       "180       http://edamontology.org/data_1143                        TRANSFAC accession number  http://edamontology.org/has_topic\n",
       "181       http://edamontology.org/data_1144                    ArrayExpress accession number  http://edamontology.org/has_topic\n",
       "182       http://edamontology.org/data_1145                PRIDE experiment accession number  http://edamontology.org/has_topic\n",
       "183       http://edamontology.org/data_1146                                          EMDB ID  http://edamontology.org/has_topic\n",
       "184       http://edamontology.org/data_1147                             GEO accession number  http://edamontology.org/has_topic\n",
       "185       http://edamontology.org/data_1148                                    GermOnline ID  http://edamontology.org/has_topic\n",
       "186       http://edamontology.org/data_1149                                         EMAGE ID  http://edamontology.org/has_topic\n",
       "187       http://edamontology.org/data_1150                                       Disease ID  http://edamontology.org/has_topic\n",
       "188       http://edamontology.org/data_1151                                       HGVbase ID  http://edamontology.org/has_topic\n",
       "189       http://edamontology.org/data_1153                                          OMIM ID  http://edamontology.org/has_topic\n",
       "190       http://edamontology.org/data_1154                           KEGG object identifier  http://edamontology.org/has_topic\n",
       "191       http://edamontology.org/data_1155                            Pathway ID (reactome)  http://edamontology.org/has_topic\n",
       "192       http://edamontology.org/data_1157                              Pathway ID (BioCyc)  http://edamontology.org/has_topic\n",
       "193       http://edamontology.org/data_1158                                Pathway ID (INOH)  http://edamontology.org/has_topic\n",
       "194       http://edamontology.org/data_1159                              Pathway ID (PATIKA)  http://edamontology.org/has_topic\n",
       "195       http://edamontology.org/data_1160                                Pathway ID (CPDB)  http://edamontology.org/has_topic\n",
       "196       http://edamontology.org/data_1161                             Pathway ID (Panther)  http://edamontology.org/has_topic\n",
       "197       http://edamontology.org/data_1162                                MIRIAM identifier  http://edamontology.org/has_topic\n",
       "198       http://edamontology.org/data_1163                            MIRIAM data type name  http://edamontology.org/has_topic\n",
       "199       http://edamontology.org/data_1164                                       MIRIAM URI  http://edamontology.org/has_topic\n",
       "200       http://edamontology.org/data_1165                    MIRIAM data type primary name  http://edamontology.org/has_topic\n",
       "201       http://edamontology.org/data_1166                 MIRIAM data type synonymous name  http://edamontology.org/has_topic\n",
       "202       http://edamontology.org/data_1167                              Taverna workflow ID  http://edamontology.org/has_topic\n",
       "203       http://edamontology.org/data_1170                            Biological model name  http://edamontology.org/has_topic\n",
       "204       http://edamontology.org/data_1171                                      BioModel ID  http://edamontology.org/has_topic\n",
       "205       http://edamontology.org/data_1172                                      PubChem CID  http://edamontology.org/has_topic\n",
       "206       http://edamontology.org/data_1173                                    ChemSpider ID  http://edamontology.org/has_topic\n",
       "207       http://edamontology.org/data_1174                                         ChEBI ID  http://edamontology.org/has_topic\n",
       "208       http://edamontology.org/data_1175                                BioPax concept ID  http://edamontology.org/has_topic\n",
       "209       http://edamontology.org/data_1176                                    GO concept ID  http://edamontology.org/has_topic\n",
       "210       http://edamontology.org/data_1177                                  MeSH concept ID  http://edamontology.org/has_topic\n",
       "211       http://edamontology.org/data_1178                                  HGNC concept ID  http://edamontology.org/has_topic\n",
       "212       http://edamontology.org/data_1179                                 NCBI taxonomy ID  http://edamontology.org/has_topic\n",
       "213       http://edamontology.org/data_1180                        Plant Ontology concept ID  http://edamontology.org/has_topic\n",
       "214       http://edamontology.org/data_1181                                  UMLS concept ID  http://edamontology.org/has_topic\n",
       "215       http://edamontology.org/data_1182                                   FMA concept ID  http://edamontology.org/has_topic\n",
       "216       http://edamontology.org/data_1183                                  EMAP concept ID  http://edamontology.org/has_topic\n",
       "217       http://edamontology.org/data_1184                                 ChEBI concept ID  http://edamontology.org/has_topic\n",
       "218       http://edamontology.org/data_1185                                  MGED concept ID  http://edamontology.org/has_topic\n",
       "219       http://edamontology.org/data_1186                                myGrid concept ID  http://edamontology.org/has_topic\n",
       "220       http://edamontology.org/data_1187                                        PubMed ID  http://edamontology.org/has_topic\n",
       "221       http://edamontology.org/data_1188                                              DOI  http://edamontology.org/has_topic\n",
       "222       http://edamontology.org/data_1189                                       Medline UI  http://edamontology.org/has_topic\n",
       "223       http://edamontology.org/data_1190                                        Tool name  http://edamontology.org/has_topic\n",
       "224       http://edamontology.org/data_1191                            Tool name (signature)  http://edamontology.org/has_topic\n",
       "225       http://edamontology.org/data_1192                                Tool name (BLAST)  http://edamontology.org/has_topic\n",
       "226       http://edamontology.org/data_1193                                Tool name (FASTA)  http://edamontology.org/has_topic\n",
       "227       http://edamontology.org/data_1194                               Tool name (EMBOSS)  http://edamontology.org/has_topic\n",
       "228       http://edamontology.org/data_1195                      Tool name (EMBASSY package)  http://edamontology.org/has_topic\n",
       "229       http://edamontology.org/data_1201                 QSAR descriptor (constitutional)  http://edamontology.org/has_topic\n",
       "230       http://edamontology.org/data_1202                     QSAR descriptor (electronic)  http://edamontology.org/has_topic\n",
       "231       http://edamontology.org/data_1203                    QSAR descriptor (geometrical)  http://edamontology.org/has_topic\n",
       "232       http://edamontology.org/data_1204                    QSAR descriptor (topological)  http://edamontology.org/has_topic\n",
       "233       http://edamontology.org/data_1205                      QSAR descriptor (molecular)  http://edamontology.org/has_topic\n",
       "234       http://edamontology.org/data_1233                           Sequence set (protein)  http://edamontology.org/has_topic\n",
       "235       http://edamontology.org/data_1234                      Sequence set (nucleic acid)  http://edamontology.org/has_topic\n",
       "236       http://edamontology.org/data_1239                               Restriction digest  http://edamontology.org/has_topic\n",
       "237       http://edamontology.org/data_1240                                      PCR primers  http://edamontology.org/has_topic\n",
       "238       http://edamontology.org/data_1249                                  Sequence length  http://edamontology.org/has_topic\n",
       "239       http://edamontology.org/data_1254                                Sequence property  http://edamontology.org/has_topic\n",
       "240       http://edamontology.org/data_1255                                Sequence features  http://edamontology.org/has_topic\n",
       "241       http://edamontology.org/data_1259                       Sequence complexity report  http://edamontology.org/has_topic\n",
       "242       http://edamontology.org/data_1260                        Sequence ambiguity report  http://edamontology.org/has_topic\n",
       "243       http://edamontology.org/data_1261                      Sequence composition report  http://edamontology.org/has_topic\n",
       "244       http://edamontology.org/data_1262                    Peptide molecular weight hits  http://edamontology.org/has_topic\n",
       "245       http://edamontology.org/data_1263                   Base position variability plot  http://edamontology.org/has_topic\n",
       "246       http://edamontology.org/data_1265                           Base frequencies table  http://edamontology.org/has_topic\n",
       "247       http://edamontology.org/data_1266                      Base word frequencies table  http://edamontology.org/has_topic\n",
       "248       http://edamontology.org/data_1267                     Amino acid frequencies table  http://edamontology.org/has_topic\n",
       "249       http://edamontology.org/data_1268                Amino acid word frequencies table  http://edamontology.org/has_topic\n",
       "250       http://edamontology.org/data_1270                                    Feature table  http://edamontology.org/has_topic\n",
       "251       http://edamontology.org/data_1276                            Nucleic acid features  http://edamontology.org/has_topic\n",
       "252       http://edamontology.org/data_1277                                 Protein features  http://edamontology.org/has_topic\n",
       "253       http://edamontology.org/data_1352                               Regular expression  http://edamontology.org/has_topic\n",
       "254       http://edamontology.org/data_1394                       Alignment score or penalty  http://edamontology.org/has_topic\n",
       "255       http://edamontology.org/data_1397                              Gap opening penalty  http://edamontology.org/has_topic\n",
       "256       http://edamontology.org/data_1398                            Gap extension penalty  http://edamontology.org/has_topic\n",
       "257       http://edamontology.org/data_1399                           Gap separation penalty  http://edamontology.org/has_topic\n",
       "258       http://edamontology.org/data_1401                               Match reward score  http://edamontology.org/has_topic\n",
       "259       http://edamontology.org/data_1402                           Mismatch penalty score  http://edamontology.org/has_topic\n",
       "260       http://edamontology.org/data_1403                                   Drop off score  http://edamontology.org/has_topic\n",
       "261       http://edamontology.org/data_1410                     Terminal gap opening penalty  http://edamontology.org/has_topic\n",
       "262       http://edamontology.org/data_1411                   Terminal gap extension penalty  http://edamontology.org/has_topic\n",
       "263       http://edamontology.org/data_1412                                Sequence identity  http://edamontology.org/has_topic\n",
       "264       http://edamontology.org/data_1413                              Sequence similarity  http://edamontology.org/has_topic\n",
       "265       http://edamontology.org/data_1426        Phylogenetic continuous quantitative data  http://edamontology.org/has_topic\n",
       "266       http://edamontology.org/data_1427                       Phylogenetic discrete data  http://edamontology.org/has_topic\n",
       "267       http://edamontology.org/data_1428                   Phylogenetic character cliques  http://edamontology.org/has_topic\n",
       "268       http://edamontology.org/data_1442                      Phylogenetic tree distances  http://edamontology.org/has_topic\n",
       "269       http://edamontology.org/data_1444                 Phylogenetic character contrasts  http://edamontology.org/has_topic\n",
       "270       http://edamontology.org/data_1448                   Comparison matrix (nucleotide)  http://edamontology.org/has_topic\n",
       "271       http://edamontology.org/data_1449                   Comparison matrix (amino acid)  http://edamontology.org/has_topic\n",
       "272       http://edamontology.org/data_1494                 Structural transformation matrix  http://edamontology.org/has_topic\n",
       "273       http://edamontology.org/data_1497                       Root-mean-square deviation  http://edamontology.org/has_topic\n",
       "274       http://edamontology.org/data_1498                        Tanimoto similarity score  http://edamontology.org/has_topic\n",
       "275       http://edamontology.org/data_1499                             3D-1D scoring matrix  http://edamontology.org/has_topic\n",
       "276       http://edamontology.org/data_1501                                 Amino acid index  http://edamontology.org/has_topic\n",
       "277       http://edamontology.org/data_1502              Amino acid index (chemical classes)  http://edamontology.org/has_topic\n",
       "278       http://edamontology.org/data_1503          Amino acid pair-wise contact potentials  http://edamontology.org/has_topic\n",
       "279       http://edamontology.org/data_1505              Amino acid index (molecular weight)  http://edamontology.org/has_topic\n",
       "280       http://edamontology.org/data_1506                    Amino acid index (hydropathy)  http://edamontology.org/has_topic\n",
       "281       http://edamontology.org/data_1507             Amino acid index (White-Wimley data)  http://edamontology.org/has_topic\n",
       "282       http://edamontology.org/data_1508           Amino acid index (van der Waals radii)  http://edamontology.org/has_topic\n",
       "283       http://edamontology.org/data_1519                        Peptide molecular weights  http://edamontology.org/has_topic\n",
       "284       http://edamontology.org/data_1520                       Peptide hydrophobic moment  http://edamontology.org/has_topic\n",
       "285       http://edamontology.org/data_1521                          Protein aliphatic index  http://edamontology.org/has_topic\n",
       "286       http://edamontology.org/data_1522                 Protein sequence hydropathy plot  http://edamontology.org/has_topic\n",
       "287       http://edamontology.org/data_1523                              Protein charge plot  http://edamontology.org/has_topic\n",
       "288       http://edamontology.org/data_1524                               Protein solubility  http://edamontology.org/has_topic\n",
       "289       http://edamontology.org/data_1525                        Protein crystallizability  http://edamontology.org/has_topic\n",
       "290       http://edamontology.org/data_1526                              Protein globularity  http://edamontology.org/has_topic\n",
       "291       http://edamontology.org/data_1527                          Protein titration curve  http://edamontology.org/has_topic\n",
       "292       http://edamontology.org/data_1528                        Protein isoelectric point  http://edamontology.org/has_topic\n",
       "293       http://edamontology.org/data_1529                                Protein pKa value  http://edamontology.org/has_topic\n",
       "294       http://edamontology.org/data_1530                   Protein hydrogen exchange rate  http://edamontology.org/has_topic\n",
       "295       http://edamontology.org/data_1531                   Protein extinction coefficient  http://edamontology.org/has_topic\n",
       "296       http://edamontology.org/data_1532                          Protein optical density  http://edamontology.org/has_topic\n",
       "297       http://edamontology.org/data_1534                      Peptide immunogenicity data  http://edamontology.org/has_topic\n",
       "298       http://edamontology.org/data_1537                         Protein structure report  http://edamontology.org/has_topic\n",
       "299       http://edamontology.org/data_1539                Protein structural quality report  http://edamontology.org/has_topic\n",
       "300       http://edamontology.org/data_1542                    Protein solvent accessibility  http://edamontology.org/has_topic\n",
       "301       http://edamontology.org/data_1544                                Ramachandran plot  http://edamontology.org/has_topic\n",
       "302       http://edamontology.org/data_1545                            Protein dipole moment  http://edamontology.org/has_topic\n",
       "303       http://edamontology.org/data_1583                     Nucleic acid melting profile  http://edamontology.org/has_topic\n",
       "304       http://edamontology.org/data_1584                            Nucleic acid enthalpy  http://edamontology.org/has_topic\n",
       "305       http://edamontology.org/data_1585                             Nucleic acid entropy  http://edamontology.org/has_topic\n",
       "306       http://edamontology.org/data_1588             DNA base pair stacking energies data  http://edamontology.org/has_topic\n",
       "307       http://edamontology.org/data_1589                   DNA base pair twist angle data  http://edamontology.org/has_topic\n",
       "308       http://edamontology.org/data_1590                 DNA base trimer roll angles data  http://edamontology.org/has_topic\n",
       "309       http://edamontology.org/data_1595          Base pairing probability matrix dotplot  http://edamontology.org/has_topic\n",
       "310       http://edamontology.org/data_1596                      Nucleic acid folding report  http://edamontology.org/has_topic\n",
       "311       http://edamontology.org/data_1621                      Pharmacogenomic test report  http://edamontology.org/has_topic\n",
       "312       http://edamontology.org/data_1667                                          E-value  http://edamontology.org/has_topic\n",
       "313       http://edamontology.org/data_1668                                          Z-value  http://edamontology.org/has_topic\n",
       "314       http://edamontology.org/data_1669                                          P-value  http://edamontology.org/has_topic\n",
       "315       http://edamontology.org/data_1689                                         Username  http://edamontology.org/has_topic\n",
       "316       http://edamontology.org/data_1690                                         Password  http://edamontology.org/has_topic\n",
       "317       http://edamontology.org/data_1691                                    Email address  http://edamontology.org/has_topic\n",
       "318       http://edamontology.org/data_1692                                      Person name  http://edamontology.org/has_topic\n",
       "319       http://edamontology.org/data_1707                          Phylogenetic tree image  http://edamontology.org/has_topic\n",
       "320       http://edamontology.org/data_1708                    RNA secondary structure image  http://edamontology.org/has_topic\n",
       "321       http://edamontology.org/data_1709                Protein secondary structure image  http://edamontology.org/has_topic\n",
       "322       http://edamontology.org/data_1710                                  Structure image  http://edamontology.org/has_topic\n",
       "323       http://edamontology.org/data_1711                         Sequence alignment image  http://edamontology.org/has_topic\n",
       "324       http://edamontology.org/data_1712                         Chemical structure image  http://edamontology.org/has_topic\n",
       "325       http://edamontology.org/data_1742                               PDB residue number  http://edamontology.org/has_topic\n",
       "326       http://edamontology.org/data_1743                                Atomic coordinate  http://edamontology.org/has_topic\n",
       "327       http://edamontology.org/data_1748                                    PDB atom name  http://edamontology.org/has_topic\n",
       "328       http://edamontology.org/data_1757                                        Atom name  http://edamontology.org/has_topic\n",
       "329       http://edamontology.org/data_1758                                 PDB residue name  http://edamontology.org/has_topic\n",
       "330       http://edamontology.org/data_1759                                 PDB model number  http://edamontology.org/has_topic\n",
       "331       http://edamontology.org/data_1771                                 Sequence version  http://edamontology.org/has_topic\n",
       "332       http://edamontology.org/data_1772                                            Score  http://edamontology.org/has_topic\n",
       "333       http://edamontology.org/data_1794                               Gene ID (PlasmoDB)  http://edamontology.org/has_topic\n",
       "334       http://edamontology.org/data_1795                                Gene ID (EcoGene)  http://edamontology.org/has_topic\n",
       "335       http://edamontology.org/data_1796                                Gene ID (FlyBase)  http://edamontology.org/has_topic\n",
       "336       http://edamontology.org/data_1802                                Gene ID (Gramene)  http://edamontology.org/has_topic\n",
       "337       http://edamontology.org/data_1803                     Gene ID (Virginia microbial)  http://edamontology.org/has_topic\n",
       "338       http://edamontology.org/data_1804                                    Gene ID (SGN)  http://edamontology.org/has_topic\n",
       "339       http://edamontology.org/data_1805                               Gene ID (WormBase)  http://edamontology.org/has_topic\n",
       "340       http://edamontology.org/data_1807                                         ORF name  http://edamontology.org/has_topic\n",
       "341       http://edamontology.org/data_1855                                         Clone ID  http://edamontology.org/has_topic\n",
       "342       http://edamontology.org/data_1856                               PDB insertion code  http://edamontology.org/has_topic\n",
       "343       http://edamontology.org/data_1857                                 Atomic occupancy  http://edamontology.org/has_topic\n",
       "344       http://edamontology.org/data_1858                               Isotropic B factor  http://edamontology.org/has_topic\n",
       "345       http://edamontology.org/data_1867                                Protein fold name  http://edamontology.org/has_topic\n",
       "346       http://edamontology.org/data_1868                                            Taxon  http://edamontology.org/has_topic\n",
       "347       http://edamontology.org/data_1869                              Organism identifier  http://edamontology.org/has_topic\n",
       "348       http://edamontology.org/data_1870                                       Genus name  http://edamontology.org/has_topic\n",
       "349       http://edamontology.org/data_1872                         Taxonomic classification  http://edamontology.org/has_topic\n",
       "350       http://edamontology.org/data_1873                                 iHOP organism ID  http://edamontology.org/has_topic\n",
       "351       http://edamontology.org/data_1874                              Genbank common name  http://edamontology.org/has_topic\n",
       "352       http://edamontology.org/data_1875                                       NCBI taxon  http://edamontology.org/has_topic\n",
       "353       http://edamontology.org/data_1881                                        Author ID  http://edamontology.org/has_topic\n",
       "354       http://edamontology.org/data_1882                       DragonDB author identifier  http://edamontology.org/has_topic\n",
       "355       http://edamontology.org/data_1883                                    Annotated URI  http://edamontology.org/has_topic\n",
       "356       http://edamontology.org/data_1885                               Gene ID (GeneFarm)  http://edamontology.org/has_topic\n",
       "357       http://edamontology.org/data_1886                                  Blattner number  http://edamontology.org/has_topic\n",
       "358       http://edamontology.org/data_1891                                      iHOP symbol  http://edamontology.org/has_topic\n",
       "359       http://edamontology.org/data_1893                                         Locus ID  http://edamontology.org/has_topic\n",
       "360       http://edamontology.org/data_1895                                   Locus ID (AGI)  http://edamontology.org/has_topic\n",
       "361       http://edamontology.org/data_1896                                 Locus ID (ASPGD)  http://edamontology.org/has_topic\n",
       "362       http://edamontology.org/data_1897                                   Locus ID (MGG)  http://edamontology.org/has_topic\n",
       "363       http://edamontology.org/data_1898                                   Locus ID (CGD)  http://edamontology.org/has_topic\n",
       "364       http://edamontology.org/data_1899                                   Locus ID (CMR)  http://edamontology.org/has_topic\n",
       "365       http://edamontology.org/data_1900                                   NCBI locus tag  http://edamontology.org/has_topic\n",
       "366       http://edamontology.org/data_1901                                   Locus ID (SGD)  http://edamontology.org/has_topic\n",
       "367       http://edamontology.org/data_1902                                   Locus ID (MMP)  http://edamontology.org/has_topic\n",
       "368       http://edamontology.org/data_1903                             Locus ID (DictyBase)  http://edamontology.org/has_topic\n",
       "369       http://edamontology.org/data_1904                            Locus ID (EntrezGene)  http://edamontology.org/has_topic\n",
       "370       http://edamontology.org/data_1905                              Locus ID (MaizeGDB)  http://edamontology.org/has_topic\n",
       "371       http://edamontology.org/data_1907                                   Gene ID (KOME)  http://edamontology.org/has_topic\n",
       "372       http://edamontology.org/data_1908                              Locus ID (Tropgene)  http://edamontology.org/has_topic\n",
       "373       http://edamontology.org/data_1916                                        Alignment  http://edamontology.org/has_topic\n",
       "374       http://edamontology.org/data_1917                                  Atomic property  http://edamontology.org/has_topic\n",
       "375       http://edamontology.org/data_2007                                  UniProt keyword  http://edamontology.org/has_topic\n",
       "376       http://edamontology.org/data_2012                             Sequence coordinates  http://edamontology.org/has_topic\n",
       "377       http://edamontology.org/data_2016                              Amino acid property  http://edamontology.org/has_topic\n",
       "378       http://edamontology.org/data_2024                             Enzyme kinetics data  http://edamontology.org/has_topic\n",
       "379       http://edamontology.org/data_2025                            Michaelis Menten plot  http://edamontology.org/has_topic\n",
       "380       http://edamontology.org/data_2026                                 Hanes Woolf plot  http://edamontology.org/has_topic\n",
       "381       http://edamontology.org/data_2042                                         Evidence  http://edamontology.org/has_topic\n",
       "382       http://edamontology.org/data_2048                                           Report  http://edamontology.org/has_topic\n",
       "383       http://edamontology.org/data_2050                     Molecular property (general)  http://edamontology.org/has_topic\n",
       "384       http://edamontology.org/data_2080                          Database search results  http://edamontology.org/has_topic\n",
       "385       http://edamontology.org/data_2082                                           Matrix  http://edamontology.org/has_topic\n",
       "386       http://edamontology.org/data_2084                              Nucleic acid report  http://edamontology.org/has_topic\n",
       "387       http://edamontology.org/data_2085                                 Structure report  http://edamontology.org/has_topic\n",
       "388       http://edamontology.org/data_2087                               Molecular property  http://edamontology.org/has_topic\n",
       "389       http://edamontology.org/data_2088                         DNA base structural data  http://edamontology.org/has_topic\n",
       "390       http://edamontology.org/data_2091                                        Accession  http://edamontology.org/has_topic\n",
       "391       http://edamontology.org/data_2093                                   Data reference  http://edamontology.org/has_topic\n",
       "392       http://edamontology.org/data_2098                                   Job identifier  http://edamontology.org/has_topic\n",
       "393       http://edamontology.org/data_2099                                             Name  http://edamontology.org/has_topic\n",
       "394       http://edamontology.org/data_2101                           Account authentication  http://edamontology.org/has_topic\n",
       "395       http://edamontology.org/data_2102                               KEGG organism code  http://edamontology.org/has_topic\n",
       "396       http://edamontology.org/data_2104                                        BioCyc ID  http://edamontology.org/has_topic\n",
       "397       http://edamontology.org/data_2105                             Compound ID (BioCyc)  http://edamontology.org/has_topic\n",
       "398       http://edamontology.org/data_2106                             Reaction ID (BioCyc)  http://edamontology.org/has_topic\n",
       "399       http://edamontology.org/data_2107                               Enzyme ID (BioCyc)  http://edamontology.org/has_topic\n",
       "400       http://edamontology.org/data_2108                                      Reaction ID  http://edamontology.org/has_topic\n",
       "401       http://edamontology.org/data_2109                              Identifier (hybrid)  http://edamontology.org/has_topic\n",
       "402       http://edamontology.org/data_2110                    Molecular property identifier  http://edamontology.org/has_topic\n",
       "403       http://edamontology.org/data_2111                             Codon usage table ID  http://edamontology.org/has_topic\n",
       "404       http://edamontology.org/data_2112                       FlyBase primary identifier  http://edamontology.org/has_topic\n",
       "405       http://edamontology.org/data_2113                              WormBase identifier  http://edamontology.org/has_topic\n",
       "406       http://edamontology.org/data_2114                              WormBase wormpep ID  http://edamontology.org/has_topic\n",
       "407       http://edamontology.org/data_2117                                   Map identifier  http://edamontology.org/has_topic\n",
       "408       http://edamontology.org/data_2118                                Person identifier  http://edamontology.org/has_topic\n",
       "409       http://edamontology.org/data_2119                          Nucleic acid identifier  http://edamontology.org/has_topic\n",
       "410       http://edamontology.org/data_2127                          Genetic code identifier  http://edamontology.org/has_topic\n",
       "411       http://edamontology.org/data_2128                                Genetic code name  http://edamontology.org/has_topic\n",
       "412       http://edamontology.org/data_2129                                 File format name  http://edamontology.org/has_topic\n",
       "413       http://edamontology.org/data_2131                            Operating system name  http://edamontology.org/has_topic\n",
       "414       http://edamontology.org/data_2133                                 Logical operator  http://edamontology.org/has_topic\n",
       "415       http://edamontology.org/data_2137                                      Gap penalty  http://edamontology.org/has_topic\n",
       "416       http://edamontology.org/data_2139                 Nucleic acid melting temperature  http://edamontology.org/has_topic\n",
       "417       http://edamontology.org/data_2140                                    Concentration  http://edamontology.org/has_topic\n",
       "418       http://edamontology.org/data_2154                                    Sequence name  http://edamontology.org/has_topic\n",
       "419       http://edamontology.org/data_2160                            Fickett testcode plot  http://edamontology.org/has_topic\n",
       "420       http://edamontology.org/data_2161                         Sequence similarity plot  http://edamontology.org/has_topic\n",
       "421       http://edamontology.org/data_2162                                    Helical wheel  http://edamontology.org/has_topic\n",
       "422       http://edamontology.org/data_2163                                      Helical net  http://edamontology.org/has_topic\n",
       "423       http://edamontology.org/data_2165                         Protein ionisation curve  http://edamontology.org/has_topic\n",
       "424       http://edamontology.org/data_2166                        Sequence composition plot  http://edamontology.org/has_topic\n",
       "425       http://edamontology.org/data_2167                        Nucleic acid density plot  http://edamontology.org/has_topic\n",
       "426       http://edamontology.org/data_2168                             Sequence trace image  http://edamontology.org/has_topic\n",
       "427       http://edamontology.org/data_2174                     FlyBase secondary identifier  http://edamontology.org/has_topic\n",
       "428       http://edamontology.org/data_2190                                Sequence checksum  http://edamontology.org/has_topic\n",
       "429       http://edamontology.org/data_2193                          Database entry metadata  http://edamontology.org/has_topic\n",
       "430       http://edamontology.org/data_2208                               Plasmid identifier  http://edamontology.org/has_topic\n",
       "431       http://edamontology.org/data_2209                                      Mutation ID  http://edamontology.org/has_topic\n",
       "432       http://edamontology.org/data_2216                                     Codon number  http://edamontology.org/has_topic\n",
       "433       http://edamontology.org/data_2219                              Database field name  http://edamontology.org/has_topic\n",
       "434       http://edamontology.org/data_2220                    Sequence cluster ID (SYSTERS)  http://edamontology.org/has_topic\n",
       "435       http://edamontology.org/data_2253                    Data resource definition name  http://edamontology.org/has_topic\n",
       "436       http://edamontology.org/data_2254                             OBO file format name  http://edamontology.org/has_topic\n",
       "437       http://edamontology.org/data_2285                                   Gene ID (MIPS)  http://edamontology.org/has_topic\n",
       "438       http://edamontology.org/data_2290                                   EMBL accession  http://edamontology.org/has_topic\n",
       "439       http://edamontology.org/data_2291                                       UniProt ID  http://edamontology.org/has_topic\n",
       "440       http://edamontology.org/data_2292                                GenBank accession  http://edamontology.org/has_topic\n",
       "441       http://edamontology.org/data_2293                     Gramene secondary identifier  http://edamontology.org/has_topic\n",
       "442       http://edamontology.org/data_2294                            Sequence variation ID  http://edamontology.org/has_topic\n",
       "443       http://edamontology.org/data_2295                                          Gene ID  http://edamontology.org/has_topic\n",
       "444       http://edamontology.org/data_2297                                    Gene ID (ECK)  http://edamontology.org/has_topic\n",
       "445       http://edamontology.org/data_2298                                   Gene ID (HGNC)  http://edamontology.org/has_topic\n",
       "446       http://edamontology.org/data_2299                                        Gene name  http://edamontology.org/has_topic\n",
       "447       http://edamontology.org/data_2301                                    SMILES string  http://edamontology.org/has_topic\n",
       "448       http://edamontology.org/data_2302                                        STRING ID  http://edamontology.org/has_topic\n",
       "449       http://edamontology.org/data_2309                           Reaction ID (SABIO-RK)  http://edamontology.org/has_topic\n",
       "450       http://edamontology.org/data_2313                              Carbohydrate report  http://edamontology.org/has_topic\n",
       "451       http://edamontology.org/data_2314                                        GI number  http://edamontology.org/has_topic\n",
       "452       http://edamontology.org/data_2315                                     NCBI version  http://edamontology.org/has_topic\n",
       "453       http://edamontology.org/data_2316                                   Cell line name  http://edamontology.org/has_topic\n",
       "454       http://edamontology.org/data_2317                           Cell line name (exact)  http://edamontology.org/has_topic\n",
       "455       http://edamontology.org/data_2318                       Cell line name (truncated)  http://edamontology.org/has_topic\n",
       "456       http://edamontology.org/data_2319                  Cell line name (no punctuation)  http://edamontology.org/has_topic\n",
       "457       http://edamontology.org/data_2320                        Cell line name (assonant)  http://edamontology.org/has_topic\n",
       "458       http://edamontology.org/data_2321                                        Enzyme ID  http://edamontology.org/has_topic\n",
       "459       http://edamontology.org/data_2325                             REBASE enzyme number  http://edamontology.org/has_topic\n",
       "460       http://edamontology.org/data_2326                                      DrugBank ID  http://edamontology.org/has_topic\n",
       "461       http://edamontology.org/data_2327                              GI number (protein)  http://edamontology.org/has_topic\n",
       "462       http://edamontology.org/data_2335                                        Bit score  http://edamontology.org/has_topic\n",
       "463       http://edamontology.org/data_2337                                Resource metadata  http://edamontology.org/has_topic\n",
       "464       http://edamontology.org/data_2338                              Ontology identifier  http://edamontology.org/has_topic\n",
       "465       http://edamontology.org/data_2339                            Ontology concept name  http://edamontology.org/has_topic\n",
       "466       http://edamontology.org/data_2340                          Genome build identifier  http://edamontology.org/has_topic\n",
       "467       http://edamontology.org/data_2342                          Pathway or network name  http://edamontology.org/has_topic\n",
       "468       http://edamontology.org/data_2343                                Pathway ID (KEGG)  http://edamontology.org/has_topic\n",
       "469       http://edamontology.org/data_2344                          Pathway ID (NCI-Nature)  http://edamontology.org/has_topic\n",
       "470       http://edamontology.org/data_2345                     Pathway ID (ConsensusPathDB)  http://edamontology.org/has_topic\n",
       "471       http://edamontology.org/data_2346                     Sequence cluster ID (UniRef)  http://edamontology.org/has_topic\n",
       "472       http://edamontology.org/data_2347                  Sequence cluster ID (UniRef100)  http://edamontology.org/has_topic\n",
       "473       http://edamontology.org/data_2348                   Sequence cluster ID (UniRef90)  http://edamontology.org/has_topic\n",
       "474       http://edamontology.org/data_2349                   Sequence cluster ID (UniRef50)  http://edamontology.org/has_topic\n",
       "475       http://edamontology.org/data_2354                                RNA family report  http://edamontology.org/has_topic\n",
       "476       http://edamontology.org/data_2355                            RNA family identifier  http://edamontology.org/has_topic\n",
       "477       http://edamontology.org/data_2356                                   RFAM accession  http://edamontology.org/has_topic\n",
       "478       http://edamontology.org/data_2362                      Sequence accession (hybrid)  http://edamontology.org/has_topic\n",
       "479       http://edamontology.org/data_2365                     Pathway or network accession  http://edamontology.org/has_topic\n",
       "480       http://edamontology.org/data_2366                    Secondary structure alignment  http://edamontology.org/has_topic\n",
       "481       http://edamontology.org/data_2367                                          ASTD ID  http://edamontology.org/has_topic\n",
       "482       http://edamontology.org/data_2368                                   ASTD ID (exon)  http://edamontology.org/has_topic\n",
       "483       http://edamontology.org/data_2369                                 ASTD ID (intron)  http://edamontology.org/has_topic\n",
       "484       http://edamontology.org/data_2370                                  ASTD ID (polya)  http://edamontology.org/has_topic\n",
       "485       http://edamontology.org/data_2371                                    ASTD ID (tss)  http://edamontology.org/has_topic\n",
       "486       http://edamontology.org/data_2373                                          Spot ID  http://edamontology.org/has_topic\n",
       "487       http://edamontology.org/data_2374                               Spot serial number  http://edamontology.org/has_topic\n",
       "488       http://edamontology.org/data_2375                             Spot ID (HSC-2DPAGE)  http://edamontology.org/has_topic\n",
       "489       http://edamontology.org/data_2379                                Strain identifier  http://edamontology.org/has_topic\n",
       "490       http://edamontology.org/data_2380                                  CABRI accession  http://edamontology.org/has_topic\n",
       "491       http://edamontology.org/data_2382                           Genotype experiment ID  http://edamontology.org/has_topic\n",
       "492       http://edamontology.org/data_2383                                    EGA accession  http://edamontology.org/has_topic\n",
       "493       http://edamontology.org/data_2384                                   IPI protein ID  http://edamontology.org/has_topic\n",
       "494       http://edamontology.org/data_2385                       RefSeq accession (protein)  http://edamontology.org/has_topic\n",
       "495       http://edamontology.org/data_2386                                           EPD ID  http://edamontology.org/has_topic\n",
       "496       http://edamontology.org/data_2387                                   TAIR accession  http://edamontology.org/has_topic\n",
       "497       http://edamontology.org/data_2388                         TAIR accession (At gene)  http://edamontology.org/has_topic\n",
       "498       http://edamontology.org/data_2389                                 UniSTS accession  http://edamontology.org/has_topic\n",
       "499       http://edamontology.org/data_2390                                  UNITE accession  http://edamontology.org/has_topic\n",
       "500       http://edamontology.org/data_2391                                    UTR accession  http://edamontology.org/has_topic\n",
       "501       http://edamontology.org/data_2392                                UniParc accession  http://edamontology.org/has_topic\n",
       "502       http://edamontology.org/data_2393                                mFLJ/mKIAA number  http://edamontology.org/has_topic\n",
       "503       http://edamontology.org/data_2398                               Ensembl protein ID  http://edamontology.org/has_topic\n",
       "504       http://edamontology.org/data_2523                                Phylogenetic data  http://edamontology.org/has_topic\n",
       "505       http://edamontology.org/data_2530                                  Organism report  http://edamontology.org/has_topic\n",
       "506       http://edamontology.org/data_2531                                         Protocol  http://edamontology.org/has_topic\n",
       "507       http://edamontology.org/data_2534                               Sequence attribute  http://edamontology.org/has_topic\n",
       "508       http://edamontology.org/data_2536                           Mass spectrometry data  http://edamontology.org/has_topic\n",
       "509       http://edamontology.org/data_2537                       Protein structure raw data  http://edamontology.org/has_topic\n",
       "510       http://edamontology.org/data_2538                              Mutation identifier  http://edamontology.org/has_topic\n",
       "511       http://edamontology.org/data_2563                  Amino acid name (single letter)  http://edamontology.org/has_topic\n",
       "512       http://edamontology.org/data_2564                   Amino acid name (three letter)  http://edamontology.org/has_topic\n",
       "513       http://edamontology.org/data_2565                      Amino acid name (full name)  http://edamontology.org/has_topic\n",
       "514       http://edamontology.org/data_2576                                 Toxin identifier  http://edamontology.org/has_topic\n",
       "515       http://edamontology.org/data_2578                                 ArachnoServer ID  http://edamontology.org/has_topic\n",
       "516       http://edamontology.org/data_2580                             BindingDB Monomer ID  http://edamontology.org/has_topic\n",
       "517       http://edamontology.org/data_2582               GO concept ID (biological process)  http://edamontology.org/has_topic\n",
       "518       http://edamontology.org/data_2583               GO concept ID (molecular function)  http://edamontology.org/has_topic\n",
       "519       http://edamontology.org/data_2586                              Northern blot image  http://edamontology.org/has_topic\n",
       "520       http://edamontology.org/data_2587                                          Blot ID  http://edamontology.org/has_topic\n",
       "521       http://edamontology.org/data_2588                                 BlotBase blot ID  http://edamontology.org/has_topic\n",
       "522       http://edamontology.org/data_2589                                        Hierarchy  http://edamontology.org/has_topic\n",
       "523       http://edamontology.org/data_2591                               Brite hierarchy ID  http://edamontology.org/has_topic\n",
       "524       http://edamontology.org/data_2593                               BRENDA organism ID  http://edamontology.org/has_topic\n",
       "525       http://edamontology.org/data_2594                                    UniGene taxon  http://edamontology.org/has_topic\n",
       "526       http://edamontology.org/data_2595                                      UTRdb taxon  http://edamontology.org/has_topic\n",
       "527       http://edamontology.org/data_2596                                     Catalogue ID  http://edamontology.org/has_topic\n",
       "528       http://edamontology.org/data_2597                             CABRI catalogue name  http://edamontology.org/has_topic\n",
       "529       http://edamontology.org/data_2605                               Compound ID (KEGG)  http://edamontology.org/has_topic\n",
       "530       http://edamontology.org/data_2606                                        RFAM name  http://edamontology.org/has_topic\n",
       "531       http://edamontology.org/data_2608                               Reaction ID (KEGG)  http://edamontology.org/has_topic\n",
       "532       http://edamontology.org/data_2609                                   Drug ID (KEGG)  http://edamontology.org/has_topic\n",
       "533       http://edamontology.org/data_2610                                       Ensembl ID  http://edamontology.org/has_topic\n",
       "534       http://edamontology.org/data_2611                                   ICD identifier  http://edamontology.org/has_topic\n",
       "535       http://edamontology.org/data_2612                     Sequence cluster ID (CluSTr)  http://edamontology.org/has_topic\n",
       "536       http://edamontology.org/data_2613                                   KEGG Glycan ID  http://edamontology.org/has_topic\n",
       "537       http://edamontology.org/data_2614                                          TCDB ID  http://edamontology.org/has_topic\n",
       "538       http://edamontology.org/data_2615                                          MINT ID  http://edamontology.org/has_topic\n",
       "539       http://edamontology.org/data_2616                                           DIP ID  http://edamontology.org/has_topic\n",
       "540       http://edamontology.org/data_2617                     Signaling Gateway protein ID  http://edamontology.org/has_topic\n",
       "541       http://edamontology.org/data_2618                          Protein modification ID  http://edamontology.org/has_topic\n",
       "542       http://edamontology.org/data_2619                                         RESID ID  http://edamontology.org/has_topic\n",
       "543       http://edamontology.org/data_2620                                           RGD ID  http://edamontology.org/has_topic\n",
       "544       http://edamontology.org/data_2621                         TAIR accession (protein)  http://edamontology.org/has_topic\n",
       "545       http://edamontology.org/data_2622                               Compound ID (HMDB)  http://edamontology.org/has_topic\n",
       "546       http://edamontology.org/data_2625                                    LIPID MAPS ID  http://edamontology.org/has_topic\n",
       "547       http://edamontology.org/data_2626                                  PeptideAtlas ID  http://edamontology.org/has_topic\n",
       "548       http://edamontology.org/data_2628                           BioGRID interaction ID  http://edamontology.org/has_topic\n",
       "549       http://edamontology.org/data_2629                               Enzyme ID (MEROPS)  http://edamontology.org/has_topic\n",
       "550       http://edamontology.org/data_2630                        Mobile genetic element ID  http://edamontology.org/has_topic\n",
       "551       http://edamontology.org/data_2631                                        ACLAME ID  http://edamontology.org/has_topic\n",
       "552       http://edamontology.org/data_2632                                           SGD ID  http://edamontology.org/has_topic\n",
       "553       http://edamontology.org/data_2633                                          Book ID  http://edamontology.org/has_topic\n",
       "554       http://edamontology.org/data_2634                                             ISBN  http://edamontology.org/has_topic\n",
       "555       http://edamontology.org/data_2635                              Compound ID (3DMET)  http://edamontology.org/has_topic\n",
       "556       http://edamontology.org/data_2636                          MatrixDB interaction ID  http://edamontology.org/has_topic\n",
       "557       http://edamontology.org/data_2637                                         cPath ID  http://edamontology.org/has_topic\n",
       "558       http://edamontology.org/data_2638                              PubChem bioassay ID  http://edamontology.org/has_topic\n",
       "559       http://edamontology.org/data_2639                                       PubChem ID  http://edamontology.org/has_topic\n",
       "560       http://edamontology.org/data_2641                              Reaction ID (MACie)  http://edamontology.org/has_topic\n",
       "561       http://edamontology.org/data_2642                                Gene ID (miRBase)  http://edamontology.org/has_topic\n",
       "562       http://edamontology.org/data_2643                                   Gene ID (ZFIN)  http://edamontology.org/has_topic\n",
       "563       http://edamontology.org/data_2644                               Reaction ID (Rhea)  http://edamontology.org/has_topic\n",
       "564       http://edamontology.org/data_2645                          Pathway ID (Unipathway)  http://edamontology.org/has_topic\n",
       "565       http://edamontology.org/data_2646                             Compound ID (ChEMBL)  http://edamontology.org/has_topic\n",
       "566       http://edamontology.org/data_2647                                LGICdb identifier  http://edamontology.org/has_topic\n",
       "567       http://edamontology.org/data_2648                  Reaction kinetics ID (SABIO-RK)  http://edamontology.org/has_topic\n",
       "568       http://edamontology.org/data_2649                                      PharmGKB ID  http://edamontology.org/has_topic\n",
       "569       http://edamontology.org/data_2650                            Pathway ID (PharmGKB)  http://edamontology.org/has_topic\n",
       "570       http://edamontology.org/data_2651                            Disease ID (PharmGKB)  http://edamontology.org/has_topic\n",
       "571       http://edamontology.org/data_2652                               Drug ID (PharmGKB)  http://edamontology.org/has_topic\n",
       "572       http://edamontology.org/data_2653                                    Drug ID (TTD)  http://edamontology.org/has_topic\n",
       "573       http://edamontology.org/data_2654                                  Target ID (TTD)  http://edamontology.org/has_topic\n",
       "574       http://edamontology.org/data_2655                             Cell type identifier  http://edamontology.org/has_topic\n",
       "575       http://edamontology.org/data_2656                                      NeuronDB ID  http://edamontology.org/has_topic\n",
       "576       http://edamontology.org/data_2657                                   NeuroMorpho ID  http://edamontology.org/has_topic\n",
       "577       http://edamontology.org/data_2658                         Compound ID (ChemIDplus)  http://edamontology.org/has_topic\n",
       "578       http://edamontology.org/data_2659                               Pathway ID (SMPDB)  http://edamontology.org/has_topic\n",
       "579       http://edamontology.org/data_2660                                    BioNumbers ID  http://edamontology.org/has_topic\n",
       "580       http://edamontology.org/data_2662                                          T3DB ID  http://edamontology.org/has_topic\n",
       "581       http://edamontology.org/data_2663                          Carbohydrate identifier  http://edamontology.org/has_topic\n",
       "582       http://edamontology.org/data_2664                                     GlycomeDB ID  http://edamontology.org/has_topic\n",
       "583       http://edamontology.org/data_2665                                     LipidBank ID  http://edamontology.org/has_topic\n",
       "584       http://edamontology.org/data_2666                                           CDD ID  http://edamontology.org/has_topic\n",
       "585       http://edamontology.org/data_2667                                          MMDB ID  http://edamontology.org/has_topic\n",
       "586       http://edamontology.org/data_2668                                     iRefIndex ID  http://edamontology.org/has_topic\n",
       "587       http://edamontology.org/data_2669                                       ModelDB ID  http://edamontology.org/has_topic\n",
       "588       http://edamontology.org/data_2670                                Pathway ID (DQCS)  http://edamontology.org/has_topic\n",
       "589       http://edamontology.org/data_2700                                  CATH identifier  http://edamontology.org/has_topic\n",
       "590       http://edamontology.org/data_2701                            CATH node ID (family)  http://edamontology.org/has_topic\n",
       "591       http://edamontology.org/data_2702                                 Enzyme ID (CAZy)  http://edamontology.org/has_topic\n",
       "592       http://edamontology.org/data_2704                                 Clone ID (IMAGE)  http://edamontology.org/has_topic\n",
       "593       http://edamontology.org/data_2705               GO concept ID (cellular component)  http://edamontology.org/has_topic\n",
       "594       http://edamontology.org/data_2706                         Chromosome name (BioCyc)  http://edamontology.org/has_topic\n",
       "595       http://edamontology.org/data_2709                               CleanEx entry name  http://edamontology.org/has_topic\n",
       "596       http://edamontology.org/data_2710                             CleanEx dataset code  http://edamontology.org/has_topic\n",
       "597       http://edamontology.org/data_2711                                    Genome report  http://edamontology.org/has_topic\n",
       "598       http://edamontology.org/data_2713                               Protein ID (CORUM)  http://edamontology.org/has_topic\n",
       "599       http://edamontology.org/data_2714                                      CDD PSSM-ID  http://edamontology.org/has_topic\n",
       "600       http://edamontology.org/data_2715                           Protein ID (CuticleDB)  http://edamontology.org/has_topic\n",
       "601       http://edamontology.org/data_2716                                           DBD ID  http://edamontology.org/has_topic\n",
       "602       http://edamontology.org/data_2718                               Oligonucleotide ID  http://edamontology.org/has_topic\n",
       "603       http://edamontology.org/data_2719                                       dbProbe ID  http://edamontology.org/has_topic\n",
       "604       http://edamontology.org/data_2720                            Dinucleotide property  http://edamontology.org/has_topic\n",
       "605       http://edamontology.org/data_2721                                       DiProDB ID  http://edamontology.org/has_topic\n",
       "606       http://edamontology.org/data_2723                             Protein ID (DisProt)  http://edamontology.org/has_topic\n",
       "607       http://edamontology.org/data_2725                            Ensembl transcript ID  http://edamontology.org/has_topic\n",
       "608       http://edamontology.org/data_2727                                      Promoter ID  http://edamontology.org/has_topic\n",
       "609       http://edamontology.org/data_2728                                    EST accession  http://edamontology.org/has_topic\n",
       "610       http://edamontology.org/data_2729                                    COGEME EST ID  http://edamontology.org/has_topic\n",
       "611       http://edamontology.org/data_2730                            COGEME unisequence ID  http://edamontology.org/has_topic\n",
       "612       http://edamontology.org/data_2731                     Protein family ID (GeneFarm)  http://edamontology.org/has_topic\n",
       "613       http://edamontology.org/data_2732                                      Family name  http://edamontology.org/has_topic\n",
       "614       http://edamontology.org/data_2736               Sequence feature ID (SwissRegulon)  http://edamontology.org/has_topic\n",
       "615       http://edamontology.org/data_2737                                           FIG ID  http://edamontology.org/has_topic\n",
       "616       http://edamontology.org/data_2738                                Gene ID (Xenbase)  http://edamontology.org/has_topic\n",
       "617       http://edamontology.org/data_2739                               Gene ID (Genolist)  http://edamontology.org/has_topic\n",
       "618       http://edamontology.org/data_2741                                           ABS ID  http://edamontology.org/has_topic\n",
       "619       http://edamontology.org/data_2742                                     AraC-XylS ID  http://edamontology.org/has_topic\n",
       "620       http://edamontology.org/data_2744                             Locus ID (PseudoCAP)  http://edamontology.org/has_topic\n",
       "621       http://edamontology.org/data_2745                                   Locus ID (UTR)  http://edamontology.org/has_topic\n",
       "622       http://edamontology.org/data_2746                              MonosaccharideDB ID  http://edamontology.org/has_topic\n",
       "623       http://edamontology.org/data_2749                                Genome identifier  http://edamontology.org/has_topic\n",
       "624       http://edamontology.org/data_2752                                      GlycoMap ID  http://edamontology.org/has_topic\n",
       "625       http://edamontology.org/data_2753                  Carbohydrate conformational map  http://edamontology.org/has_topic\n",
       "626       http://edamontology.org/data_2755                        Transcription factor name  http://edamontology.org/has_topic\n",
       "627       http://edamontology.org/data_2756                                             TCID  http://edamontology.org/has_topic\n",
       "628       http://edamontology.org/data_2757                                 Pfam domain name  http://edamontology.org/has_topic\n",
       "629       http://edamontology.org/data_2758                                     Pfam clan ID  http://edamontology.org/has_topic\n",
       "630       http://edamontology.org/data_2759                             Gene ID (VectorBase)  http://edamontology.org/has_topic\n",
       "631       http://edamontology.org/data_2761                                       UTRSite ID  http://edamontology.org/has_topic\n",
       "632       http://edamontology.org/data_2764                           Protein name (UniProt)  http://edamontology.org/has_topic\n",
       "633       http://edamontology.org/data_2766                                         HAMAP ID  http://edamontology.org/has_topic\n",
       "634       http://edamontology.org/data_2769                                    Transcript ID  http://edamontology.org/has_topic\n",
       "635       http://edamontology.org/data_2770                                           HIT ID  http://edamontology.org/has_topic\n",
       "636       http://edamontology.org/data_2771                                           HIX ID  http://edamontology.org/has_topic\n",
       "637       http://edamontology.org/data_2772                                  HPA antibody id  http://edamontology.org/has_topic\n",
       "638       http://edamontology.org/data_2773                                      IMGT/HLA ID  http://edamontology.org/has_topic\n",
       "639       http://edamontology.org/data_2774                                   Gene ID (JCVI)  http://edamontology.org/has_topic\n",
       "640       http://edamontology.org/data_2775                                      Kinase name  http://edamontology.org/has_topic\n",
       "641       http://edamontology.org/data_2776                        ConsensusPathDB entity ID  http://edamontology.org/has_topic\n",
       "642       http://edamontology.org/data_2777                      ConsensusPathDB entity name  http://edamontology.org/has_topic\n",
       "643       http://edamontology.org/data_2778                               CCAP strain number  http://edamontology.org/has_topic\n",
       "644       http://edamontology.org/data_2779                                     Stock number  http://edamontology.org/has_topic\n",
       "645       http://edamontology.org/data_2780                              Stock number (TAIR)  http://edamontology.org/has_topic\n",
       "646       http://edamontology.org/data_2781                                        REDIdb ID  http://edamontology.org/has_topic\n",
       "647       http://edamontology.org/data_2782                                SMART domain name  http://edamontology.org/has_topic\n",
       "648       http://edamontology.org/data_2783                      Protein family ID (PANTHER)  http://edamontology.org/has_topic\n",
       "649       http://edamontology.org/data_2784                                    RNAVirusDB ID  http://edamontology.org/has_topic\n",
       "650       http://edamontology.org/data_2785                                 Virus identifier  http://edamontology.org/has_topic\n",
       "651       http://edamontology.org/data_2786                           NCBI Genome Project ID  http://edamontology.org/has_topic\n",
       "652       http://edamontology.org/data_2787                            NCBI genome accession  http://edamontology.org/has_topic\n",
       "653       http://edamontology.org/data_2789                               Protein ID (TopDB)  http://edamontology.org/has_topic\n",
       "654       http://edamontology.org/data_2790                                           Gel ID  http://edamontology.org/has_topic\n",
       "655       http://edamontology.org/data_2791                Reference map name (SWISS-2DPAGE)  http://edamontology.org/has_topic\n",
       "656       http://edamontology.org/data_2792                          Protein ID (PeroxiBase)  http://edamontology.org/has_topic\n",
       "657       http://edamontology.org/data_2793                                      SISYPHUS ID  http://edamontology.org/has_topic\n",
       "658       http://edamontology.org/data_2794                                           ORF ID  http://edamontology.org/has_topic\n",
       "659       http://edamontology.org/data_2795                                   ORF identifier  http://edamontology.org/has_topic\n",
       "660       http://edamontology.org/data_2796                                        LINUCS ID  http://edamontology.org/has_topic\n",
       "661       http://edamontology.org/data_2797                              Protein ID (LGICdb)  http://edamontology.org/has_topic\n",
       "662       http://edamontology.org/data_2798                                       MaizeDB ID  http://edamontology.org/has_topic\n",
       "663       http://edamontology.org/data_2799                                 Gene ID (MfunGD)  http://edamontology.org/has_topic\n",
       "664       http://edamontology.org/data_2800                                     Orpha number  http://edamontology.org/has_topic\n",
       "665       http://edamontology.org/data_2802                                Protein ID (EcID)  http://edamontology.org/has_topic\n",
       "666       http://edamontology.org/data_2803                                Clone ID (RefSeq)  http://edamontology.org/has_topic\n",
       "667       http://edamontology.org/data_2804                          Protein ID (ConoServer)  http://edamontology.org/has_topic\n",
       "668       http://edamontology.org/data_2805                                       GeneSNP ID  http://edamontology.org/has_topic\n",
       "669       http://edamontology.org/data_2812                                 Lipid identifier  http://edamontology.org/has_topic\n",
       "670       http://edamontology.org/data_2835                                 Gene ID (VBASE2)  http://edamontology.org/has_topic\n",
       "671       http://edamontology.org/data_2836                                        DPVweb ID  http://edamontology.org/has_topic\n",
       "672       http://edamontology.org/data_2837                          Pathway ID (BioSystems)  http://edamontology.org/has_topic\n",
       "673       http://edamontology.org/data_2855                                  Distance matrix  http://edamontology.org/has_topic\n",
       "674       http://edamontology.org/data_2856                       Structural distance matrix  http://edamontology.org/has_topic\n",
       "675       http://edamontology.org/data_2872                                          ID list  http://edamontology.org/has_topic\n",
       "676       http://edamontology.org/data_2873               Phylogenetic gene frequencies data  http://edamontology.org/has_topic\n",
       "677       http://edamontology.org/data_2879                                     Lipid report  http://edamontology.org/has_topic\n",
       "678       http://edamontology.org/data_2884                                             Plot  http://edamontology.org/has_topic\n",
       "679       http://edamontology.org/data_2891                       Biological model accession  http://edamontology.org/has_topic\n",
       "680       http://edamontology.org/data_2892                                   Cell type name  http://edamontology.org/has_topic\n",
       "681       http://edamontology.org/data_2893                              Cell type accession  http://edamontology.org/has_topic\n",
       "682       http://edamontology.org/data_2894                               Compound accession  http://edamontology.org/has_topic\n",
       "683       http://edamontology.org/data_2895                                   Drug accession  http://edamontology.org/has_topic\n",
       "684       http://edamontology.org/data_2896                                       Toxin name  http://edamontology.org/has_topic\n",
       "685       http://edamontology.org/data_2897                                  Toxin accession  http://edamontology.org/has_topic\n",
       "686       http://edamontology.org/data_2898                         Monosaccharide accession  http://edamontology.org/has_topic\n",
       "687       http://edamontology.org/data_2899                                        Drug name  http://edamontology.org/has_topic\n",
       "688       http://edamontology.org/data_2900                           Carbohydrate accession  http://edamontology.org/has_topic\n",
       "689       http://edamontology.org/data_2901                               Molecule accession  http://edamontology.org/has_topic\n",
       "690       http://edamontology.org/data_2902               Data resource definition accession  http://edamontology.org/has_topic\n",
       "691       http://edamontology.org/data_2903                                 Genome accession  http://edamontology.org/has_topic\n",
       "692       http://edamontology.org/data_2904                                    Map accession  http://edamontology.org/has_topic\n",
       "693       http://edamontology.org/data_2905                                  Lipid accession  http://edamontology.org/has_topic\n",
       "694       http://edamontology.org/data_2906                                       Peptide ID  http://edamontology.org/has_topic\n",
       "695       http://edamontology.org/data_2907                                Protein accession  http://edamontology.org/has_topic\n",
       "696       http://edamontology.org/data_2908                               Organism accession  http://edamontology.org/has_topic\n",
       "697       http://edamontology.org/data_2909                                    Organism name  http://edamontology.org/has_topic\n",
       "698       http://edamontology.org/data_2910                         Protein family accession  http://edamontology.org/has_topic\n",
       "699       http://edamontology.org/data_2911                   Transcription factor accession  http://edamontology.org/has_topic\n",
       "700       http://edamontology.org/data_2912                                 Strain accession  http://edamontology.org/has_topic\n",
       "701       http://edamontology.org/data_2914                       Sequence features metadata  http://edamontology.org/has_topic\n",
       "702       http://edamontology.org/data_2915                               Gramene identifier  http://edamontology.org/has_topic\n",
       "703       http://edamontology.org/data_2916                                   DDBJ accession  http://edamontology.org/has_topic\n",
       "704       http://edamontology.org/data_2917                       ConsensusPathDB identifier  http://edamontology.org/has_topic\n",
       "705       http://edamontology.org/data_2955                                  Sequence report  http://edamontology.org/has_topic\n",
       "706       http://edamontology.org/data_2956                      Protein secondary structure  http://edamontology.org/has_topic\n",
       "707       http://edamontology.org/data_2957                              Hopp and Woods plot  http://edamontology.org/has_topic\n",
       "708       http://edamontology.org/data_2968                                            Image  http://edamontology.org/has_topic\n",
       "709       http://edamontology.org/data_2969                                   Sequence image  http://edamontology.org/has_topic\n",
       "710       http://edamontology.org/data_2970                          Protein hydropathy data  http://edamontology.org/has_topic\n",
       "711       http://edamontology.org/data_2978                                    Reaction data  http://edamontology.org/has_topic\n",
       "712       http://edamontology.org/data_2979                                 Peptide property  http://edamontology.org/has_topic\n",
       "713       http://edamontology.org/data_2985                  Nucleic acid thermodynamic data  http://edamontology.org/has_topic\n",
       "714       http://edamontology.org/data_2991                            Protein geometry data  http://edamontology.org/has_topic\n",
       "715       http://edamontology.org/data_2992                          Protein structure image  http://edamontology.org/has_topic\n",
       "716       http://edamontology.org/data_2994                   Phylogenetic character weights  http://edamontology.org/has_topic\n",
       "717       http://edamontology.org/data_3002                                 Annotation track  http://edamontology.org/has_topic\n",
       "718       http://edamontology.org/data_3021                                UniProt accession  http://edamontology.org/has_topic\n",
       "719       http://edamontology.org/data_3022                             NCBI genetic code ID  http://edamontology.org/has_topic\n",
       "720       http://edamontology.org/data_3025                      Ontology concept identifier  http://edamontology.org/has_topic\n",
       "721       http://edamontology.org/data_3029                   Protein ID (EMBL/GenBank/DDBJ)  http://edamontology.org/has_topic\n",
       "722       http://edamontology.org/data_3034                      Sequence feature identifier  http://edamontology.org/has_topic\n",
       "723       http://edamontology.org/data_3035                             Structure identifier  http://edamontology.org/has_topic\n",
       "724       http://edamontology.org/data_3036                                Matrix identifier  http://edamontology.org/has_topic\n",
       "725       http://edamontology.org/data_3103                                         ATC code  http://edamontology.org/has_topic\n",
       "726       http://edamontology.org/data_3104                                             UNII  http://edamontology.org/has_topic\n",
       "727       http://edamontology.org/data_3106                                  System metadata  http://edamontology.org/has_topic\n",
       "728       http://edamontology.org/data_3108                         Experimental measurement  http://edamontology.org/has_topic\n",
       "729       http://edamontology.org/data_3113                                Sample annotation  http://edamontology.org/has_topic\n",
       "730       http://edamontology.org/data_3128                    Nucleic acid structure report  http://edamontology.org/has_topic\n",
       "731       http://edamontology.org/data_3134                           Gene transcript report  http://edamontology.org/has_topic\n",
       "732       http://edamontology.org/data_3148                               Gene family report  http://edamontology.org/has_topic\n",
       "733       http://edamontology.org/data_3153                                    Protein image  http://edamontology.org/has_topic\n",
       "734       http://edamontology.org/data_3181                         Sequence assembly report  http://edamontology.org/has_topic\n",
       "735       http://edamontology.org/data_3236                                Cytoband position  http://edamontology.org/has_topic\n",
       "736       http://edamontology.org/data_3238                            Cell type ontology ID  http://edamontology.org/has_topic\n",
       "737       http://edamontology.org/data_3264                                        COSMIC ID  http://edamontology.org/has_topic\n",
       "738       http://edamontology.org/data_3265                                          HGMD ID  http://edamontology.org/has_topic\n",
       "739       http://edamontology.org/data_3266                             Sequence assembly ID  http://edamontology.org/has_topic\n",
       "740       http://edamontology.org/data_3270                             Ensembl gene tree ID  http://edamontology.org/has_topic\n",
       "741       http://edamontology.org/data_3273                                        Sample ID  http://edamontology.org/has_topic\n",
       "742       http://edamontology.org/data_3274                                    MGI accession  http://edamontology.org/has_topic\n",
       "743       http://edamontology.org/data_3275                                   Phenotype name  http://edamontology.org/has_topic\n",
       "744       http://edamontology.org/data_3354                                Transition matrix  http://edamontology.org/has_topic\n",
       "745       http://edamontology.org/data_3355                                  Emission matrix  http://edamontology.org/has_topic\n",
       "746       http://edamontology.org/data_3358                                Format identifier  http://edamontology.org/has_topic\n",
       "747       http://edamontology.org/data_3424                                        Raw image  http://edamontology.org/has_topic\n",
       "748       http://edamontology.org/data_3425                            Carbohydrate property  http://edamontology.org/has_topic\n",
       "749       http://edamontology.org/data_3451                              Rate of association  http://edamontology.org/has_topic\n",
       "750       http://edamontology.org/data_3479                                       Gene order  http://edamontology.org/has_topic\n",
       "751       http://edamontology.org/data_3483                                         Spectrum  http://edamontology.org/has_topic\n",
       "752       http://edamontology.org/data_3509                                 Ontology mapping  http://edamontology.org/has_topic\n",
       "753       http://edamontology.org/data_3546                                   Image metadata  http://edamontology.org/has_topic\n",
       "754       http://edamontology.org/data_3558                            Clinical trial report  http://edamontology.org/has_topic\n",
       "755       http://edamontology.org/data_3567                          Reference sample report  http://edamontology.org/has_topic\n",
       "756       http://edamontology.org/data_3568              Gene Expression Atlas Experiment ID  http://edamontology.org/has_topic\n",
       "757       http://edamontology.org/data_3667                               Disease identifier  http://edamontology.org/has_topic\n",
       "758       http://edamontology.org/data_3668                                     Disease name  http://edamontology.org/has_topic\n",
       "759       http://edamontology.org/data_3669                                Training material  http://edamontology.org/has_topic\n",
       "760       http://edamontology.org/data_3670                                    Online course  http://edamontology.org/has_topic\n",
       "761       http://edamontology.org/data_3707                                Biodiversity data  http://edamontology.org/has_topic\n",
       "762       http://edamontology.org/data_3716                                 Biosafety report  http://edamontology.org/has_topic\n",
       "763       http://edamontology.org/data_3717                                 Isolation report  http://edamontology.org/has_topic\n",
       "764       http://edamontology.org/data_3718                             Pathogenicity report  http://edamontology.org/has_topic\n",
       "765       http://edamontology.org/data_3719                         Biosafety classification  http://edamontology.org/has_topic\n",
       "766       http://edamontology.org/data_3720                              Geographic location  http://edamontology.org/has_topic\n",
       "767       http://edamontology.org/data_3721                                 Isolation source  http://edamontology.org/has_topic\n",
       "768       http://edamontology.org/data_3722                             Physiology parameter  http://edamontology.org/has_topic\n",
       "769       http://edamontology.org/data_3723                             Morphology parameter  http://edamontology.org/has_topic\n",
       "770       http://edamontology.org/data_3724                            Cultivation parameter  http://edamontology.org/has_topic\n",
       "771       http://edamontology.org/data_3732                         Sequencing metadata name  http://edamontology.org/has_topic\n",
       "772       http://edamontology.org/data_3733                             Flow cell identifier  http://edamontology.org/has_topic\n",
       "773       http://edamontology.org/data_3734                                  Lane identifier  http://edamontology.org/has_topic\n",
       "774       http://edamontology.org/data_3735                                       Run number  http://edamontology.org/has_topic\n",
       "775       http://edamontology.org/data_3736                                  Ecological data  http://edamontology.org/has_topic\n",
       "776       http://edamontology.org/data_3737                             Alpha diversity data  http://edamontology.org/has_topic\n",
       "777       http://edamontology.org/data_3738                              Beta diversity data  http://edamontology.org/has_topic\n",
       "778       http://edamontology.org/data_3739                             Gamma diversity data  http://edamontology.org/has_topic\n",
       "779       http://edamontology.org/data_3743                                  Ordination plot  http://edamontology.org/has_topic\n",
       "780       http://edamontology.org/data_3753                         Over-representation data  http://edamontology.org/has_topic\n",
       "781       http://edamontology.org/data_3756                               Localisation score  http://edamontology.org/has_topic\n",
       "782       http://edamontology.org/data_3757                                        Unimod ID  http://edamontology.org/has_topic\n",
       "783       http://edamontology.org/data_3759                               ProteomeXchange ID  http://edamontology.org/has_topic\n",
       "784       http://edamontology.org/data_3769                       BRENDA ontology concept ID  http://edamontology.org/has_topic\n",
       "785       http://edamontology.org/data_3786                                     Query script  http://edamontology.org/has_topic\n",
       "786       http://edamontology.org/data_3856                                   RNA central ID  http://edamontology.org/has_topic\n",
       "787       http://edamontology.org/data_3861                         Electronic health record  http://edamontology.org/has_topic\n",
       "788       http://edamontology.org/data_3869                                       Simulation  http://edamontology.org/has_topic\n",
       "789       http://edamontology.org/data_3870                                  Trajectory data  http://edamontology.org/has_topic\n",
       "790       http://edamontology.org/data_3871                            Forcefield parameters  http://edamontology.org/has_topic\n",
       "791       http://edamontology.org/data_3872                                    Topology data  http://edamontology.org/has_topic\n",
       "792       http://edamontology.org/data_3905                                        Histogram  http://edamontology.org/has_topic\n",
       "793       http://edamontology.org/data_3914                           Quality control report  http://edamontology.org/has_topic\n",
       "794       http://edamontology.org/data_3917                                     Count matrix  http://edamontology.org/has_topic\n",
       "795       http://edamontology.org/data_3932                                          Q-value  http://edamontology.org/has_topic\n",
       "796       http://edamontology.org/data_3952                        Pathway ID (WikiPathways)  http://edamontology.org/has_topic\n",
       "797       http://edamontology.org/data_4022                                 ORCID Identifier  http://edamontology.org/has_topic\n",
       "798  http://edamontology.org/operation_0227                                         Indexing  http://edamontology.org/has_topic\n",
       "799  http://edamontology.org/operation_0244                              Simulation analysis  http://edamontology.org/has_topic\n",
       "800  http://edamontology.org/operation_0249                     Protein geometry calculation  http://edamontology.org/has_topic\n",
       "801  http://edamontology.org/operation_0250                     Protein property calculation  http://edamontology.org/has_topic\n",
       "802  http://edamontology.org/operation_0262                Nucleic acid property calculation  http://edamontology.org/has_topic\n",
       "803  http://edamontology.org/operation_0336                                Format validation  http://edamontology.org/has_topic\n",
       "804  http://edamontology.org/operation_0386                Protein dipole moment calculation  http://edamontology.org/has_topic\n",
       "805  http://edamontology.org/operation_0396                    Ramachandran plot calculation  http://edamontology.org/has_topic\n",
       "806  http://edamontology.org/operation_0399       Protein extinction coefficient calculation  http://edamontology.org/has_topic\n",
       "807  http://edamontology.org/operation_0455  Nucleic acid thermodynamic property calculation  http://edamontology.org/has_topic\n",
       "808  http://edamontology.org/operation_0461               Nucleic acid curvature calculation  http://edamontology.org/has_topic\n",
       "809  http://edamontology.org/operation_2238                          Statistical calculation  http://edamontology.org/has_topic\n",
       "810  http://edamontology.org/operation_2423                       Prediction and recognition  http://edamontology.org/has_topic\n",
       "811  http://edamontology.org/operation_2424                                       Comparison  http://edamontology.org/has_topic\n",
       "812  http://edamontology.org/operation_2425                      Optimisation and refinement  http://edamontology.org/has_topic\n",
       "813  http://edamontology.org/operation_2428                                       Validation  http://edamontology.org/has_topic\n",
       "814  http://edamontology.org/operation_2429                                          Mapping  http://edamontology.org/has_topic\n",
       "815  http://edamontology.org/operation_2430                                           Design  http://edamontology.org/has_topic\n",
       "816  http://edamontology.org/operation_2844                             Structure clustering  http://edamontology.org/has_topic\n",
       "817  http://edamontology.org/operation_2928                                        Alignment  http://edamontology.org/has_topic\n",
       "818  http://edamontology.org/operation_2930                      Protein property comparison  http://edamontology.org/has_topic\n",
       "819  http://edamontology.org/operation_2945                                         Analysis  http://edamontology.org/has_topic\n",
       "820  http://edamontology.org/operation_2990                                   Classification  http://edamontology.org/has_topic\n",
       "821  http://edamontology.org/operation_2997                               Protein comparison  http://edamontology.org/has_topic\n",
       "822  http://edamontology.org/operation_2998                          Nucleic acid comparison  http://edamontology.org/has_topic\n",
       "823  http://edamontology.org/operation_3196                                       Genotyping  http://edamontology.org/has_topic\n",
       "824  http://edamontology.org/operation_3197                       Genetic variation analysis  http://edamontology.org/has_topic\n",
       "825  http://edamontology.org/operation_3226                           Variant prioritisation  http://edamontology.org/has_topic\n",
       "826  http://edamontology.org/operation_3228                   Structural variation detection  http://edamontology.org/has_topic\n",
       "827  http://edamontology.org/operation_3233                           Copy number estimation  http://edamontology.org/has_topic\n",
       "828  http://edamontology.org/operation_3267                   Sequence coordinate conversion  http://edamontology.org/has_topic\n",
       "829  http://edamontology.org/operation_3282                                       ID mapping  http://edamontology.org/has_topic\n",
       "830  http://edamontology.org/operation_3349                          Bibliography generation  http://edamontology.org/has_topic\n",
       "831  http://edamontology.org/operation_3350          Protein quaternary structure prediction  http://edamontology.org/has_topic\n",
       "832  http://edamontology.org/operation_3352                              Ontology comparison  http://edamontology.org/has_topic\n",
       "833  http://edamontology.org/operation_3429                                       Generation  http://edamontology.org/has_topic\n",
       "834  http://edamontology.org/operation_3432                                       Clustering  http://edamontology.org/has_topic\n",
       "835  http://edamontology.org/operation_3434                                       Conversion  http://edamontology.org/has_topic\n",
       "836  http://edamontology.org/operation_3435                Standardisation and normalisation  http://edamontology.org/has_topic\n",
       "837  http://edamontology.org/operation_3438                                      Calculation  http://edamontology.org/has_topic\n",
       "838  http://edamontology.org/operation_3480                    Probabilistic data generation  http://edamontology.org/has_topic\n",
       "839  http://edamontology.org/operation_3501                              Enrichment analysis  http://edamontology.org/has_topic\n",
       "840  http://edamontology.org/operation_3504                         Variant pattern analysis  http://edamontology.org/has_topic\n",
       "841  http://edamontology.org/operation_3557                                       Imputation  http://edamontology.org/has_topic\n",
       "842  http://edamontology.org/operation_3561                              Database comparison  http://edamontology.org/has_topic\n",
       "843  http://edamontology.org/operation_3633                        Retention time prediction  http://edamontology.org/has_topic\n",
       "844  http://edamontology.org/operation_3658                            Statistical inference  http://edamontology.org/has_topic\n",
       "845  http://edamontology.org/operation_3659                              Regression analysis  http://edamontology.org/has_topic\n",
       "846  http://edamontology.org/operation_3664                            Statistical modelling  http://edamontology.org/has_topic\n",
       "847  http://edamontology.org/operation_3703                         Reference identification  http://edamontology.org/has_topic\n",
       "848  http://edamontology.org/operation_3731                                Sample comparison  http://edamontology.org/has_topic\n",
       "849  http://edamontology.org/operation_3760                               Service management  http://edamontology.org/has_topic\n",
       "850  http://edamontology.org/operation_3761                                Service discovery  http://edamontology.org/has_topic\n",
       "851  http://edamontology.org/operation_3762                              Service composition  http://edamontology.org/has_topic\n",
       "852  http://edamontology.org/operation_3763                               Service invocation  http://edamontology.org/has_topic\n",
       "853  http://edamontology.org/operation_3791                               Collapsing methods  http://edamontology.org/has_topic\n",
       "854  http://edamontology.org/operation_3797                                      Rarefaction  http://edamontology.org/has_topic\n",
       "855  http://edamontology.org/operation_3799                                   Quantification  http://edamontology.org/has_topic\n",
       "856  http://edamontology.org/operation_3800                           RNA-Seq quantification  http://edamontology.org/has_topic\n",
       "857  http://edamontology.org/operation_3840                       Multilocus sequence typing  http://edamontology.org/has_topic\n",
       "858  http://edamontology.org/operation_3960                     Principal component analysis  http://edamontology.org/has_topic\n",
       "859  http://edamontology.org/operation_3961                  Copy number variation detection  http://edamontology.org/has_topic\n",
       "860  http://edamontology.org/operation_3962                               Deletion detection  http://edamontology.org/has_topic\n",
       "861  http://edamontology.org/operation_3963                            Duplication detection  http://edamontology.org/has_topic\n",
       "862  http://edamontology.org/operation_3964                            Complex CNV detection  http://edamontology.org/has_topic\n",
       "863  http://edamontology.org/operation_3965                          Amplification detection  http://edamontology.org/has_topic\n",
       "864  http://edamontology.org/operation_4008                                   Protein design  http://edamontology.org/has_topic\n",
       "865  http://edamontology.org/operation_4009                            Small molecule design  http://edamontology.org/has_topic"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "print(f\"nb formats missing is_format_of property: {len(remoteQuery(query=q, endpoint=ep_edam))}\")\n",
    "remoteQuery(query=q, endpoint=ep_edam)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "281cf006-cc22-4343-b4f9-6113e5699ae1",
   "metadata": {
    "tags": []
   },
   "source": [
    "## query to get all *has_topic* property"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6fc49cdf-8c4e-4739-a58f-035b97183efc",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX obo: <http://purl.obolibrary.org/obo/>\n",
    "PREFIX owl: <http://www.w3.org/2002/07/owl#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>\n",
    "PREFIX edam:<http://edamontology.org/>\n",
    "PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>\n",
    "\n",
    "SELECT DISTINCT ?entity ?label ?topic ?tlabel\n",
    "WHERE {\n",
    "    ?entity rdfs:subClassOf ?restriction . \n",
    "    ?restriction rdf:type owl:Restriction ; \n",
    "            owl:onProperty edam:has_topic  ; \n",
    "            owl:someValuesFrom ?topic.\n",
    "    ?entity rdfs:label ?label .\n",
    "    ?topic rdfs:label ?tlabel\n",
    "    }\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "45077694-a648-43fb-a1ed-f958538edae4",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "nb data/operation with has_topic property: 1208\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>entity</th>\n",
       "      <th>label</th>\n",
       "      <th>topic</th>\n",
       "      <th>tlabel</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/data_0582</td>\n",
       "      <td>Ontology</td>\n",
       "      <td>http://edamontology.org/topic_0089</td>\n",
       "      <td>Ontology and terminology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>http://edamontology.org/data_0858</td>\n",
       "      <td>Sequence signature matches</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>http://edamontology.org/data_0860</td>\n",
       "      <td>Sequence signature data</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>http://edamontology.org/data_1353</td>\n",
       "      <td>Sequence motif</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>http://edamontology.org/data_1354</td>\n",
       "      <td>Sequence profile</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>http://edamontology.org/data_1361</td>\n",
       "      <td>Position frequency matrix</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>http://edamontology.org/data_2854</td>\n",
       "      <td>Position-specific scoring matrix</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>http://edamontology.org/data_1362</td>\n",
       "      <td>Position weight matrix</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>http://edamontology.org/data_1363</td>\n",
       "      <td>Information content matrix</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>http://edamontology.org/data_1365</td>\n",
       "      <td>Fingerprint</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>http://edamontology.org/data_2071</td>\n",
       "      <td>Sequence motif (protein)</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>http://edamontology.org/data_2070</td>\n",
       "      <td>Sequence motif (nucleic acid)</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>http://edamontology.org/data_3949</td>\n",
       "      <td>Profile HMM</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>http://edamontology.org/data_0863</td>\n",
       "      <td>Sequence alignment</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>http://edamontology.org/data_1381</td>\n",
       "      <td>Pair sequence alignment</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>http://edamontology.org/data_1383</td>\n",
       "      <td>Nucleic acid sequence alignment</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>http://edamontology.org/data_1384</td>\n",
       "      <td>Protein sequence alignment</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>http://edamontology.org/data_1385</td>\n",
       "      <td>Hybrid sequence alignment</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>http://edamontology.org/data_0872</td>\n",
       "      <td>Phylogenetic tree</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>http://edamontology.org/data_3271</td>\n",
       "      <td>Gene tree</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>http://edamontology.org/data_3272</td>\n",
       "      <td>Species tree</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>http://edamontology.org/data_0880</td>\n",
       "      <td>RNA secondary structure</td>\n",
       "      <td>http://edamontology.org/topic_0097</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>http://edamontology.org/data_0880</td>\n",
       "      <td>RNA secondary structure</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>http://edamontology.org/data_1465</td>\n",
       "      <td>RNA structure</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>http://edamontology.org/data_0883</td>\n",
       "      <td>Structure</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>http://edamontology.org/data_3806</td>\n",
       "      <td>3D EM Mask</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>http://edamontology.org/data_2851</td>\n",
       "      <td>Drug structure</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>http://edamontology.org/data_1467</td>\n",
       "      <td>Protein chain</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>http://edamontology.org/data_1468</td>\n",
       "      <td>Protein domain</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>http://edamontology.org/data_3805</td>\n",
       "      <td>3D EM Map</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>http://edamontology.org/data_1459</td>\n",
       "      <td>Nucleic acid structure</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>http://edamontology.org/data_1460</td>\n",
       "      <td>Protein structure</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>http://edamontology.org/data_1461</td>\n",
       "      <td>Protein-ligand complex</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>http://edamontology.org/data_1462</td>\n",
       "      <td>Carbohydrate structure</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>http://edamontology.org/data_1463</td>\n",
       "      <td>Small molecule structure</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>http://edamontology.org/data_1464</td>\n",
       "      <td>DNA structure</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>http://edamontology.org/data_1466</td>\n",
       "      <td>tRNA structure</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>http://edamontology.org/data_1470</td>\n",
       "      <td>C-alpha trace</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>http://edamontology.org/data_1755</td>\n",
       "      <td>Protein atom</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>http://edamontology.org/data_1756</td>\n",
       "      <td>Protein residue</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>http://edamontology.org/data_2852</td>\n",
       "      <td>Toxin structure</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>http://edamontology.org/data_2850</td>\n",
       "      <td>Lipid structure</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>http://edamontology.org/data_2877</td>\n",
       "      <td>Protein complex</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>http://edamontology.org/data_2878</td>\n",
       "      <td>Protein structural motif</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>http://edamontology.org/data_0886</td>\n",
       "      <td>Structure alignment</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>http://edamontology.org/data_1479</td>\n",
       "      <td>Structure alignment (pair)</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>http://edamontology.org/data_1481</td>\n",
       "      <td>Protein structure alignment</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>http://edamontology.org/data_1482</td>\n",
       "      <td>Nucleic acid structure alignment</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>http://edamontology.org/data_1493</td>\n",
       "      <td>RNA structure alignment</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>http://edamontology.org/data_3924</td>\n",
       "      <td>DNA structure alignment</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>http://edamontology.org/data_0889</td>\n",
       "      <td>Structural profile</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51</th>\n",
       "      <td>http://edamontology.org/data_0906</td>\n",
       "      <td>Protein interaction data</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>http://edamontology.org/data_1546</td>\n",
       "      <td>Protein distance matrix</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>53</th>\n",
       "      <td>http://edamontology.org/data_1547</td>\n",
       "      <td>Protein contact map</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>http://edamontology.org/data_1548</td>\n",
       "      <td>Protein residue 3D cluster</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>55</th>\n",
       "      <td>http://edamontology.org/data_1549</td>\n",
       "      <td>Protein hydrogen bonds</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>56</th>\n",
       "      <td>http://edamontology.org/data_1566</td>\n",
       "      <td>Protein-ligand interaction report</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>57</th>\n",
       "      <td>http://edamontology.org/data_0907</td>\n",
       "      <td>Protein family report</td>\n",
       "      <td>http://edamontology.org/topic_0623</td>\n",
       "      <td>Gene and protein families</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>58</th>\n",
       "      <td>http://edamontology.org/data_0914</td>\n",
       "      <td>Codon usage data</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>59</th>\n",
       "      <td>http://edamontology.org/data_1597</td>\n",
       "      <td>Codon usage table</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>60</th>\n",
       "      <td>http://edamontology.org/data_1598</td>\n",
       "      <td>Genetic code</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>61</th>\n",
       "      <td>http://edamontology.org/data_2865</td>\n",
       "      <td>Codon usage bias</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>62</th>\n",
       "      <td>http://edamontology.org/data_1600</td>\n",
       "      <td>Codon usage bias plot</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>63</th>\n",
       "      <td>http://edamontology.org/data_1602</td>\n",
       "      <td>Codon usage fraction difference</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>64</th>\n",
       "      <td>http://edamontology.org/data_0940</td>\n",
       "      <td>Volume map</td>\n",
       "      <td>http://edamontology.org/topic_1317</td>\n",
       "      <td>Structural biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>65</th>\n",
       "      <td>http://edamontology.org/data_0942</td>\n",
       "      <td>2D PAGE image</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66</th>\n",
       "      <td>http://edamontology.org/data_0943</td>\n",
       "      <td>Mass spectrum</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>67</th>\n",
       "      <td>http://edamontology.org/data_0944</td>\n",
       "      <td>Peptide mass fingerprint</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>68</th>\n",
       "      <td>http://edamontology.org/data_0945</td>\n",
       "      <td>Peptide identification</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69</th>\n",
       "      <td>http://edamontology.org/data_0950</td>\n",
       "      <td>Mathematical model</td>\n",
       "      <td>http://edamontology.org/topic_3307</td>\n",
       "      <td>Computational biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>70</th>\n",
       "      <td>http://edamontology.org/data_1364</td>\n",
       "      <td>Hidden Markov model</td>\n",
       "      <td>http://edamontology.org/topic_3307</td>\n",
       "      <td>Computational biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71</th>\n",
       "      <td>http://edamontology.org/data_1347</td>\n",
       "      <td>Dirichlet distribution</td>\n",
       "      <td>http://edamontology.org/topic_3307</td>\n",
       "      <td>Computational biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72</th>\n",
       "      <td>http://edamontology.org/data_1439</td>\n",
       "      <td>DNA substitution model</td>\n",
       "      <td>http://edamontology.org/topic_3307</td>\n",
       "      <td>Computational biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>73</th>\n",
       "      <td>http://edamontology.org/data_3241</td>\n",
       "      <td>Kinetic model</td>\n",
       "      <td>http://edamontology.org/topic_3307</td>\n",
       "      <td>Computational biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>74</th>\n",
       "      <td>http://edamontology.org/data_3949</td>\n",
       "      <td>Profile HMM</td>\n",
       "      <td>http://edamontology.org/topic_3307</td>\n",
       "      <td>Computational biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75</th>\n",
       "      <td>http://edamontology.org/data_0955</td>\n",
       "      <td>Data index</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76</th>\n",
       "      <td>http://edamontology.org/data_3210</td>\n",
       "      <td>Genome index</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>77</th>\n",
       "      <td>http://edamontology.org/data_0956</td>\n",
       "      <td>Data index report</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78</th>\n",
       "      <td>http://edamontology.org/data_0962</td>\n",
       "      <td>Small molecule report</td>\n",
       "      <td>http://edamontology.org/topic_0154</td>\n",
       "      <td>Small molecules</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>79</th>\n",
       "      <td>http://edamontology.org/data_1696</td>\n",
       "      <td>Drug report</td>\n",
       "      <td>http://edamontology.org/topic_0154</td>\n",
       "      <td>Small molecules</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80</th>\n",
       "      <td>http://edamontology.org/data_1235</td>\n",
       "      <td>Sequence cluster</td>\n",
       "      <td>http://edamontology.org/topic_0623</td>\n",
       "      <td>Gene and protein families</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>81</th>\n",
       "      <td>http://edamontology.org/data_1245</td>\n",
       "      <td>Sequence cluster (protein)</td>\n",
       "      <td>http://edamontology.org/topic_0623</td>\n",
       "      <td>Gene and protein families</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>82</th>\n",
       "      <td>http://edamontology.org/data_1246</td>\n",
       "      <td>Sequence cluster (nucleic acid)</td>\n",
       "      <td>http://edamontology.org/topic_0623</td>\n",
       "      <td>Gene and protein families</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>83</th>\n",
       "      <td>http://edamontology.org/data_1238</td>\n",
       "      <td>Proteolytic digest</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>84</th>\n",
       "      <td>http://edamontology.org/data_1274</td>\n",
       "      <td>Map</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>85</th>\n",
       "      <td>http://edamontology.org/data_1278</td>\n",
       "      <td>Genetic map</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>86</th>\n",
       "      <td>http://edamontology.org/data_1279</td>\n",
       "      <td>Sequence map</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>87</th>\n",
       "      <td>http://edamontology.org/data_1280</td>\n",
       "      <td>Physical map</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>88</th>\n",
       "      <td>http://edamontology.org/data_1283</td>\n",
       "      <td>Cytogenetic map</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>89</th>\n",
       "      <td>http://edamontology.org/data_1284</td>\n",
       "      <td>DNA transduction map</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>90</th>\n",
       "      <td>http://edamontology.org/data_1285</td>\n",
       "      <td>Gene map</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>91</th>\n",
       "      <td>http://edamontology.org/data_1286</td>\n",
       "      <td>Plasmid map</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>92</th>\n",
       "      <td>http://edamontology.org/data_1288</td>\n",
       "      <td>Genome map</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>93</th>\n",
       "      <td>http://edamontology.org/data_1289</td>\n",
       "      <td>Restriction map</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>94</th>\n",
       "      <td>http://edamontology.org/data_1859</td>\n",
       "      <td>Deletion map</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>95</th>\n",
       "      <td>http://edamontology.org/data_1860</td>\n",
       "      <td>QTL map</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>http://edamontology.org/data_1863</td>\n",
       "      <td>Haplotype map</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>97</th>\n",
       "      <td>http://edamontology.org/data_2870</td>\n",
       "      <td>Radiation hybrid map</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>98</th>\n",
       "      <td>http://edamontology.org/data_1429</td>\n",
       "      <td>Phylogenetic invariants</td>\n",
       "      <td>http://edamontology.org/topic_0199</td>\n",
       "      <td>Genetic variation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>99</th>\n",
       "      <td>http://edamontology.org/data_1465</td>\n",
       "      <td>RNA structure</td>\n",
       "      <td>http://edamontology.org/topic_0097</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>100</th>\n",
       "      <td>http://edamontology.org/data_1459</td>\n",
       "      <td>Nucleic acid structure</td>\n",
       "      <td>http://edamontology.org/topic_0097</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>101</th>\n",
       "      <td>http://edamontology.org/data_1464</td>\n",
       "      <td>DNA structure</td>\n",
       "      <td>http://edamontology.org/topic_0097</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>102</th>\n",
       "      <td>http://edamontology.org/data_1466</td>\n",
       "      <td>tRNA structure</td>\n",
       "      <td>http://edamontology.org/topic_0097</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>103</th>\n",
       "      <td>http://edamontology.org/data_1467</td>\n",
       "      <td>Protein chain</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>104</th>\n",
       "      <td>http://edamontology.org/data_1468</td>\n",
       "      <td>Protein domain</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>105</th>\n",
       "      <td>http://edamontology.org/data_1460</td>\n",
       "      <td>Protein structure</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>106</th>\n",
       "      <td>http://edamontology.org/data_1461</td>\n",
       "      <td>Protein-ligand complex</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>107</th>\n",
       "      <td>http://edamontology.org/data_1470</td>\n",
       "      <td>C-alpha trace</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>108</th>\n",
       "      <td>http://edamontology.org/data_1755</td>\n",
       "      <td>Protein atom</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>109</th>\n",
       "      <td>http://edamontology.org/data_1756</td>\n",
       "      <td>Protein residue</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>110</th>\n",
       "      <td>http://edamontology.org/data_2877</td>\n",
       "      <td>Protein complex</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>111</th>\n",
       "      <td>http://edamontology.org/data_2878</td>\n",
       "      <td>Protein structural motif</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>112</th>\n",
       "      <td>http://edamontology.org/data_1462</td>\n",
       "      <td>Carbohydrate structure</td>\n",
       "      <td>http://edamontology.org/topic_0152</td>\n",
       "      <td>Carbohydrates</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>113</th>\n",
       "      <td>http://edamontology.org/data_1462</td>\n",
       "      <td>Carbohydrate structure</td>\n",
       "      <td>http://edamontology.org/topic_0153</td>\n",
       "      <td>Lipids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>114</th>\n",
       "      <td>http://edamontology.org/data_2851</td>\n",
       "      <td>Drug structure</td>\n",
       "      <td>http://edamontology.org/topic_0154</td>\n",
       "      <td>Small molecules</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>115</th>\n",
       "      <td>http://edamontology.org/data_1463</td>\n",
       "      <td>Small molecule structure</td>\n",
       "      <td>http://edamontology.org/topic_0154</td>\n",
       "      <td>Small molecules</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>116</th>\n",
       "      <td>http://edamontology.org/data_2852</td>\n",
       "      <td>Toxin structure</td>\n",
       "      <td>http://edamontology.org/topic_0154</td>\n",
       "      <td>Small molecules</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>117</th>\n",
       "      <td>http://edamontology.org/data_1468</td>\n",
       "      <td>Protein domain</td>\n",
       "      <td>http://edamontology.org/topic_0736</td>\n",
       "      <td>Protein folds and structural domains</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>118</th>\n",
       "      <td>http://edamontology.org/data_1622</td>\n",
       "      <td>Disease report</td>\n",
       "      <td>http://edamontology.org/topic_0634</td>\n",
       "      <td>Pathology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>119</th>\n",
       "      <td>http://edamontology.org/data_1713</td>\n",
       "      <td>Fate map</td>\n",
       "      <td>http://edamontology.org/topic_3065</td>\n",
       "      <td>Embryology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>120</th>\n",
       "      <td>http://edamontology.org/data_2019</td>\n",
       "      <td>Map data</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>121</th>\n",
       "      <td>http://edamontology.org/data_2044</td>\n",
       "      <td>Sequence</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>122</th>\n",
       "      <td>http://edamontology.org/data_0849</td>\n",
       "      <td>Sequence record</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>123</th>\n",
       "      <td>http://edamontology.org/data_2976</td>\n",
       "      <td>Protein sequence</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>124</th>\n",
       "      <td>http://edamontology.org/data_2977</td>\n",
       "      <td>Nucleic acid sequence</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>125</th>\n",
       "      <td>http://edamontology.org/data_2886</td>\n",
       "      <td>Protein sequence record</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>126</th>\n",
       "      <td>http://edamontology.org/data_2887</td>\n",
       "      <td>Nucleic acid sequence record</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>127</th>\n",
       "      <td>http://edamontology.org/data_3494</td>\n",
       "      <td>DNA sequence</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>128</th>\n",
       "      <td>http://edamontology.org/data_3495</td>\n",
       "      <td>RNA sequence</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>129</th>\n",
       "      <td>http://edamontology.org/data_2223</td>\n",
       "      <td>Ontology metadata</td>\n",
       "      <td>http://edamontology.org/topic_0089</td>\n",
       "      <td>Ontology and terminology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>130</th>\n",
       "      <td>http://edamontology.org/data_2353</td>\n",
       "      <td>Ontology data</td>\n",
       "      <td>http://edamontology.org/topic_0089</td>\n",
       "      <td>Ontology and terminology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>131</th>\n",
       "      <td>http://edamontology.org/data_0966</td>\n",
       "      <td>Ontology term</td>\n",
       "      <td>http://edamontology.org/topic_0089</td>\n",
       "      <td>Ontology and terminology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>132</th>\n",
       "      <td>http://edamontology.org/data_0967</td>\n",
       "      <td>Ontology concept data</td>\n",
       "      <td>http://edamontology.org/topic_0089</td>\n",
       "      <td>Ontology and terminology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>133</th>\n",
       "      <td>http://edamontology.org/data_2858</td>\n",
       "      <td>Ontology concept</td>\n",
       "      <td>http://edamontology.org/topic_0089</td>\n",
       "      <td>Ontology and terminology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>134</th>\n",
       "      <td>http://edamontology.org/data_1731</td>\n",
       "      <td>Ontology concept definition</td>\n",
       "      <td>http://edamontology.org/topic_0089</td>\n",
       "      <td>Ontology and terminology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>135</th>\n",
       "      <td>http://edamontology.org/data_0970</td>\n",
       "      <td>Citation</td>\n",
       "      <td>http://edamontology.org/topic_3068</td>\n",
       "      <td>Literature and language</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>136</th>\n",
       "      <td>http://edamontology.org/data_2526</td>\n",
       "      <td>Text data</td>\n",
       "      <td>http://edamontology.org/topic_3068</td>\n",
       "      <td>Literature and language</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>137</th>\n",
       "      <td>http://edamontology.org/data_0971</td>\n",
       "      <td>Article</td>\n",
       "      <td>http://edamontology.org/topic_3068</td>\n",
       "      <td>Literature and language</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>138</th>\n",
       "      <td>http://edamontology.org/data_3671</td>\n",
       "      <td>Text</td>\n",
       "      <td>http://edamontology.org/topic_3068</td>\n",
       "      <td>Literature and language</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>139</th>\n",
       "      <td>http://edamontology.org/data_0972</td>\n",
       "      <td>Text mining report</td>\n",
       "      <td>http://edamontology.org/topic_3068</td>\n",
       "      <td>Literature and language</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>140</th>\n",
       "      <td>http://edamontology.org/data_2849</td>\n",
       "      <td>Abstract</td>\n",
       "      <td>http://edamontology.org/topic_3068</td>\n",
       "      <td>Literature and language</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>141</th>\n",
       "      <td>http://edamontology.org/data_3779</td>\n",
       "      <td>Annotated text</td>\n",
       "      <td>http://edamontology.org/topic_3068</td>\n",
       "      <td>Literature and language</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>142</th>\n",
       "      <td>http://edamontology.org/data_3505</td>\n",
       "      <td>Bibliography</td>\n",
       "      <td>http://edamontology.org/topic_3068</td>\n",
       "      <td>Literature and language</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>143</th>\n",
       "      <td>http://edamontology.org/data_2600</td>\n",
       "      <td>Pathway or network</td>\n",
       "      <td>http://edamontology.org/topic_0602</td>\n",
       "      <td>Molecular interactions, pathways and networks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>144</th>\n",
       "      <td>http://edamontology.org/data_0928</td>\n",
       "      <td>Gene expression profile</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>145</th>\n",
       "      <td>http://edamontology.org/data_2603</td>\n",
       "      <td>Expression data</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>146</th>\n",
       "      <td>http://edamontology.org/data_2717</td>\n",
       "      <td>Oligonucleotide probe annotation</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>147</th>\n",
       "      <td>http://edamontology.org/data_2535</td>\n",
       "      <td>Sequence tag profile</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>148</th>\n",
       "      <td>http://edamontology.org/data_3111</td>\n",
       "      <td>Processed microarray data</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>149</th>\n",
       "      <td>http://edamontology.org/data_1636</td>\n",
       "      <td>Heat map</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>150</th>\n",
       "      <td>http://edamontology.org/data_3768</td>\n",
       "      <td>Clustered expression profiles</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>151</th>\n",
       "      <td>http://edamontology.org/data_1714</td>\n",
       "      <td>Microarray spots image</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>152</th>\n",
       "      <td>http://edamontology.org/data_3115</td>\n",
       "      <td>Microarray metadata</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>153</th>\n",
       "      <td>http://edamontology.org/data_3110</td>\n",
       "      <td>Raw microarray data</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>154</th>\n",
       "      <td>http://edamontology.org/data_3117</td>\n",
       "      <td>Microarray hybridisation data</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>155</th>\n",
       "      <td>http://edamontology.org/data_3112</td>\n",
       "      <td>Gene expression matrix</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>156</th>\n",
       "      <td>http://edamontology.org/data_2717</td>\n",
       "      <td>Oligonucleotide probe annotation</td>\n",
       "      <td>http://edamontology.org/topic_0632</td>\n",
       "      <td>Probes and primers</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>157</th>\n",
       "      <td>http://edamontology.org/data_1355</td>\n",
       "      <td>Protein signature</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>158</th>\n",
       "      <td>http://edamontology.org/data_2762</td>\n",
       "      <td>Sequence signature report</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>159</th>\n",
       "      <td>http://edamontology.org/data_3492</td>\n",
       "      <td>Nucleic acid signature</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>160</th>\n",
       "      <td>http://edamontology.org/data_2984</td>\n",
       "      <td>Pathway or network report</td>\n",
       "      <td>http://edamontology.org/topic_0602</td>\n",
       "      <td>Molecular interactions, pathways and networks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>161</th>\n",
       "      <td>http://edamontology.org/data_3028</td>\n",
       "      <td>Taxonomy</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>162</th>\n",
       "      <td>http://edamontology.org/data_3442</td>\n",
       "      <td>MRI image</td>\n",
       "      <td>http://edamontology.org/topic_3384</td>\n",
       "      <td>Medical imaging</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>163</th>\n",
       "      <td>http://edamontology.org/data_3449</td>\n",
       "      <td>Cell migration track image</td>\n",
       "      <td>http://edamontology.org/topic_2229</td>\n",
       "      <td>Cell biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>164</th>\n",
       "      <td>http://edamontology.org/data_3488</td>\n",
       "      <td>NMR spectrum</td>\n",
       "      <td>http://edamontology.org/topic_1317</td>\n",
       "      <td>Structural biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>165</th>\n",
       "      <td>http://edamontology.org/data_3498</td>\n",
       "      <td>Sequence variations</td>\n",
       "      <td>http://edamontology.org/topic_0199</td>\n",
       "      <td>Genetic variation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>166</th>\n",
       "      <td>http://edamontology.org/data_3754</td>\n",
       "      <td>GO-term enrichment data</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>167</th>\n",
       "      <td>http://edamontology.org/data_3805</td>\n",
       "      <td>3D EM Map</td>\n",
       "      <td>http://edamontology.org/topic_1317</td>\n",
       "      <td>Structural biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>168</th>\n",
       "      <td>http://edamontology.org/data_3806</td>\n",
       "      <td>3D EM Mask</td>\n",
       "      <td>http://edamontology.org/topic_1317</td>\n",
       "      <td>Structural biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>169</th>\n",
       "      <td>http://edamontology.org/data_3807</td>\n",
       "      <td>EM Movie</td>\n",
       "      <td>http://edamontology.org/topic_1317</td>\n",
       "      <td>Structural biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>170</th>\n",
       "      <td>http://edamontology.org/data_3808</td>\n",
       "      <td>EM Micrograph</td>\n",
       "      <td>http://edamontology.org/topic_1317</td>\n",
       "      <td>Structural biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171</th>\n",
       "      <td>http://edamontology.org/data_3842</td>\n",
       "      <td>Molecular simulation data</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>172</th>\n",
       "      <td>http://edamontology.org/data_3953</td>\n",
       "      <td>Pathway overrepresentation data</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>173</th>\n",
       "      <td>http://edamontology.org/operation_0226</td>\n",
       "      <td>Annotation</td>\n",
       "      <td>http://edamontology.org/topic_0089</td>\n",
       "      <td>Ontology and terminology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>174</th>\n",
       "      <td>http://edamontology.org/operation_0361</td>\n",
       "      <td>Sequence annotation</td>\n",
       "      <td>http://edamontology.org/topic_0089</td>\n",
       "      <td>Ontology and terminology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>175</th>\n",
       "      <td>http://edamontology.org/operation_0362</td>\n",
       "      <td>Genome annotation</td>\n",
       "      <td>http://edamontology.org/topic_0089</td>\n",
       "      <td>Ontology and terminology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>176</th>\n",
       "      <td>http://edamontology.org/operation_0527</td>\n",
       "      <td>Sequence tag mapping</td>\n",
       "      <td>http://edamontology.org/topic_0089</td>\n",
       "      <td>Ontology and terminology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>177</th>\n",
       "      <td>http://edamontology.org/operation_0558</td>\n",
       "      <td>Phylogenetic tree annotation</td>\n",
       "      <td>http://edamontology.org/topic_0089</td>\n",
       "      <td>Ontology and terminology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>178</th>\n",
       "      <td>http://edamontology.org/operation_3778</td>\n",
       "      <td>Text annotation</td>\n",
       "      <td>http://edamontology.org/topic_0089</td>\n",
       "      <td>Ontology and terminology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>179</th>\n",
       "      <td>http://edamontology.org/operation_3553</td>\n",
       "      <td>Image annotation</td>\n",
       "      <td>http://edamontology.org/topic_0089</td>\n",
       "      <td>Ontology and terminology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>180</th>\n",
       "      <td>http://edamontology.org/operation_3661</td>\n",
       "      <td>SNP annotation</td>\n",
       "      <td>http://edamontology.org/topic_0089</td>\n",
       "      <td>Ontology and terminology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>181</th>\n",
       "      <td>http://edamontology.org/operation_3672</td>\n",
       "      <td>Gene functional annotation</td>\n",
       "      <td>http://edamontology.org/topic_0089</td>\n",
       "      <td>Ontology and terminology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>182</th>\n",
       "      <td>http://edamontology.org/operation_0234</td>\n",
       "      <td>Sequence complexity calculation</td>\n",
       "      <td>http://edamontology.org/topic_0157</td>\n",
       "      <td>Sequence composition, complexity and repeats</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>183</th>\n",
       "      <td>http://edamontology.org/operation_0235</td>\n",
       "      <td>Sequence ambiguity calculation</td>\n",
       "      <td>http://edamontology.org/topic_0157</td>\n",
       "      <td>Sequence composition, complexity and repeats</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>184</th>\n",
       "      <td>http://edamontology.org/operation_0236</td>\n",
       "      <td>Sequence composition calculation</td>\n",
       "      <td>http://edamontology.org/topic_0157</td>\n",
       "      <td>Sequence composition, complexity and repeats</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>185</th>\n",
       "      <td>http://edamontology.org/operation_0380</td>\n",
       "      <td>Repeat sequence organisation analysis</td>\n",
       "      <td>http://edamontology.org/topic_0157</td>\n",
       "      <td>Sequence composition, complexity and repeats</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>186</th>\n",
       "      <td>http://edamontology.org/operation_2284</td>\n",
       "      <td>Nucleic acid density plotting</td>\n",
       "      <td>http://edamontology.org/topic_0157</td>\n",
       "      <td>Sequence composition, complexity and repeats</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>187</th>\n",
       "      <td>http://edamontology.org/operation_3472</td>\n",
       "      <td>k-mer counting</td>\n",
       "      <td>http://edamontology.org/topic_0157</td>\n",
       "      <td>Sequence composition, complexity and repeats</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>188</th>\n",
       "      <td>http://edamontology.org/operation_0237</td>\n",
       "      <td>Repeat sequence analysis</td>\n",
       "      <td>http://edamontology.org/topic_0157</td>\n",
       "      <td>Sequence composition, complexity and repeats</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>189</th>\n",
       "      <td>http://edamontology.org/operation_0379</td>\n",
       "      <td>Repeat sequence detection</td>\n",
       "      <td>http://edamontology.org/topic_0157</td>\n",
       "      <td>Sequence composition, complexity and repeats</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>190</th>\n",
       "      <td>http://edamontology.org/operation_0238</td>\n",
       "      <td>Sequence motif discovery</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>191</th>\n",
       "      <td>http://edamontology.org/operation_0239</td>\n",
       "      <td>Sequence motif recognition</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>192</th>\n",
       "      <td>http://edamontology.org/operation_0240</td>\n",
       "      <td>Sequence motif comparison</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>193</th>\n",
       "      <td>http://edamontology.org/operation_0245</td>\n",
       "      <td>Structural motif discovery</td>\n",
       "      <td>http://edamontology.org/topic_0166</td>\n",
       "      <td>Protein structural motifs and surfaces</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>194</th>\n",
       "      <td>http://edamontology.org/operation_0246</td>\n",
       "      <td>Protein domain recognition</td>\n",
       "      <td>http://edamontology.org/topic_0736</td>\n",
       "      <td>Protein folds and structural domains</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>195</th>\n",
       "      <td>http://edamontology.org/operation_0390</td>\n",
       "      <td>Protein peeling</td>\n",
       "      <td>http://edamontology.org/topic_0736</td>\n",
       "      <td>Protein folds and structural domains</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>196</th>\n",
       "      <td>http://edamontology.org/operation_0248</td>\n",
       "      <td>Residue interaction calculation</td>\n",
       "      <td>http://edamontology.org/topic_0130</td>\n",
       "      <td>Protein folding, stability and design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>197</th>\n",
       "      <td>http://edamontology.org/operation_0393</td>\n",
       "      <td>Residue cluster calculation</td>\n",
       "      <td>http://edamontology.org/topic_0130</td>\n",
       "      <td>Protein folding, stability and design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>198</th>\n",
       "      <td>http://edamontology.org/operation_0391</td>\n",
       "      <td>Protein distance matrix calculation</td>\n",
       "      <td>http://edamontology.org/topic_0130</td>\n",
       "      <td>Protein folding, stability and design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>199</th>\n",
       "      <td>http://edamontology.org/operation_2950</td>\n",
       "      <td>Residue distance calculation</td>\n",
       "      <td>http://edamontology.org/topic_0130</td>\n",
       "      <td>Protein folding, stability and design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>200</th>\n",
       "      <td>http://edamontology.org/operation_0392</td>\n",
       "      <td>Contact map calculation</td>\n",
       "      <td>http://edamontology.org/topic_0130</td>\n",
       "      <td>Protein folding, stability and design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>201</th>\n",
       "      <td>http://edamontology.org/operation_0394</td>\n",
       "      <td>Hydrogen bond calculation</td>\n",
       "      <td>http://edamontology.org/topic_0130</td>\n",
       "      <td>Protein folding, stability and design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>202</th>\n",
       "      <td>http://edamontology.org/operation_1831</td>\n",
       "      <td>Metal-bound cysteine detection</td>\n",
       "      <td>http://edamontology.org/topic_0130</td>\n",
       "      <td>Protein folding, stability and design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>203</th>\n",
       "      <td>http://edamontology.org/operation_1834</td>\n",
       "      <td>Protein-metal contact calculation</td>\n",
       "      <td>http://edamontology.org/topic_0130</td>\n",
       "      <td>Protein folding, stability and design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>204</th>\n",
       "      <td>http://edamontology.org/operation_1839</td>\n",
       "      <td>Salt bridge calculation</td>\n",
       "      <td>http://edamontology.org/topic_0130</td>\n",
       "      <td>Protein folding, stability and design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>205</th>\n",
       "      <td>http://edamontology.org/operation_0252</td>\n",
       "      <td>Peptide immunogenicity prediction</td>\n",
       "      <td>http://edamontology.org/topic_0804</td>\n",
       "      <td>Immunology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>206</th>\n",
       "      <td>http://edamontology.org/operation_0253</td>\n",
       "      <td>Sequence feature detection</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>207</th>\n",
       "      <td>http://edamontology.org/operation_0256</td>\n",
       "      <td>Sequence feature comparison</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>208</th>\n",
       "      <td>http://edamontology.org/operation_3194</td>\n",
       "      <td>Genome feature comparison</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>209</th>\n",
       "      <td>http://edamontology.org/operation_0264</td>\n",
       "      <td>Alternative splicing prediction</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>210</th>\n",
       "      <td>http://edamontology.org/operation_0265</td>\n",
       "      <td>Frameshift detection</td>\n",
       "      <td>http://edamontology.org/topic_3168</td>\n",
       "      <td>Sequencing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>211</th>\n",
       "      <td>http://edamontology.org/operation_0269</td>\n",
       "      <td>Transmembrane protein prediction</td>\n",
       "      <td>http://edamontology.org/topic_0820</td>\n",
       "      <td>Membrane and lipoproteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>212</th>\n",
       "      <td>http://edamontology.org/operation_0270</td>\n",
       "      <td>Transmembrane protein analysis</td>\n",
       "      <td>http://edamontology.org/topic_0820</td>\n",
       "      <td>Membrane and lipoproteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>213</th>\n",
       "      <td>http://edamontology.org/operation_2241</td>\n",
       "      <td>Transmembrane protein visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0820</td>\n",
       "      <td>Membrane and lipoproteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>214</th>\n",
       "      <td>http://edamontology.org/operation_0272</td>\n",
       "      <td>Residue contact prediction</td>\n",
       "      <td>http://edamontology.org/topic_0130</td>\n",
       "      <td>Protein folding, stability and design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>215</th>\n",
       "      <td>http://edamontology.org/operation_0276</td>\n",
       "      <td>Protein interaction network analysis</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>216</th>\n",
       "      <td>http://edamontology.org/operation_0282</td>\n",
       "      <td>Genetic mapping</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>217</th>\n",
       "      <td>http://edamontology.org/operation_0283</td>\n",
       "      <td>Linkage analysis</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>218</th>\n",
       "      <td>http://edamontology.org/operation_0487</td>\n",
       "      <td>Haplotype mapping</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>219</th>\n",
       "      <td>http://edamontology.org/operation_0488</td>\n",
       "      <td>Linkage disequilibrium calculation</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>220</th>\n",
       "      <td>http://edamontology.org/operation_0284</td>\n",
       "      <td>Codon usage table generation</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>221</th>\n",
       "      <td>http://edamontology.org/operation_0286</td>\n",
       "      <td>Codon usage analysis</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>222</th>\n",
       "      <td>http://edamontology.org/operation_0285</td>\n",
       "      <td>Codon usage table comparison</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>223</th>\n",
       "      <td>http://edamontology.org/operation_0287</td>\n",
       "      <td>Base position variability plotting</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>224</th>\n",
       "      <td>http://edamontology.org/operation_0489</td>\n",
       "      <td>Genetic code prediction</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>225</th>\n",
       "      <td>http://edamontology.org/operation_0550</td>\n",
       "      <td>DNA substitution modelling</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>226</th>\n",
       "      <td>http://edamontology.org/operation_2962</td>\n",
       "      <td>Codon usage bias calculation</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>227</th>\n",
       "      <td>http://edamontology.org/operation_2964</td>\n",
       "      <td>Codon usage fraction calculation</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>228</th>\n",
       "      <td>http://edamontology.org/operation_3946</td>\n",
       "      <td>Ecological modelling</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>229</th>\n",
       "      <td>http://edamontology.org/operation_0287</td>\n",
       "      <td>Base position variability plotting</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>230</th>\n",
       "      <td>http://edamontology.org/operation_0289</td>\n",
       "      <td>Sequence distance matrix generation</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>231</th>\n",
       "      <td>http://edamontology.org/operation_0296</td>\n",
       "      <td>Sequence profile generation</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>232</th>\n",
       "      <td>http://edamontology.org/operation_0300</td>\n",
       "      <td>Sequence profile alignment</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>233</th>\n",
       "      <td>http://edamontology.org/operation_0305</td>\n",
       "      <td>Literature search</td>\n",
       "      <td>http://edamontology.org/topic_3068</td>\n",
       "      <td>Literature and language</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>234</th>\n",
       "      <td>http://edamontology.org/operation_2422</td>\n",
       "      <td>Data retrieval</td>\n",
       "      <td>http://edamontology.org/topic_0218</td>\n",
       "      <td>Natural language processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>235</th>\n",
       "      <td>http://edamontology.org/operation_0306</td>\n",
       "      <td>Text mining</td>\n",
       "      <td>http://edamontology.org/topic_0218</td>\n",
       "      <td>Natural language processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>236</th>\n",
       "      <td>http://edamontology.org/operation_3908</td>\n",
       "      <td>Information retrieval</td>\n",
       "      <td>http://edamontology.org/topic_0218</td>\n",
       "      <td>Natural language processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>237</th>\n",
       "      <td>http://edamontology.org/operation_3280</td>\n",
       "      <td>Named-entity and concept recognition</td>\n",
       "      <td>http://edamontology.org/topic_0218</td>\n",
       "      <td>Natural language processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>238</th>\n",
       "      <td>http://edamontology.org/operation_3907</td>\n",
       "      <td>Information extraction</td>\n",
       "      <td>http://edamontology.org/topic_0218</td>\n",
       "      <td>Natural language processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>239</th>\n",
       "      <td>http://edamontology.org/operation_3625</td>\n",
       "      <td>Relation extraction</td>\n",
       "      <td>http://edamontology.org/topic_0218</td>\n",
       "      <td>Natural language processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>240</th>\n",
       "      <td>http://edamontology.org/operation_0308</td>\n",
       "      <td>PCR primer design</td>\n",
       "      <td>http://edamontology.org/topic_0632</td>\n",
       "      <td>Probes and primers</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>241</th>\n",
       "      <td>http://edamontology.org/operation_0309</td>\n",
       "      <td>Microarray probe design</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>242</th>\n",
       "      <td>http://edamontology.org/operation_0309</td>\n",
       "      <td>Microarray probe design</td>\n",
       "      <td>http://edamontology.org/topic_0632</td>\n",
       "      <td>Probes and primers</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>243</th>\n",
       "      <td>http://edamontology.org/operation_0310</td>\n",
       "      <td>Sequence assembly</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>244</th>\n",
       "      <td>http://edamontology.org/operation_0523</td>\n",
       "      <td>Mapping assembly</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>245</th>\n",
       "      <td>http://edamontology.org/operation_0524</td>\n",
       "      <td>De-novo assembly</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>246</th>\n",
       "      <td>http://edamontology.org/operation_0525</td>\n",
       "      <td>Genome assembly</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>247</th>\n",
       "      <td>http://edamontology.org/operation_0526</td>\n",
       "      <td>EST assembly</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>248</th>\n",
       "      <td>http://edamontology.org/operation_3183</td>\n",
       "      <td>Localised reassembly</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>249</th>\n",
       "      <td>http://edamontology.org/operation_3216</td>\n",
       "      <td>Scaffolding</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>250</th>\n",
       "      <td>http://edamontology.org/operation_3217</td>\n",
       "      <td>Scaffold gap completion</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>251</th>\n",
       "      <td>http://edamontology.org/operation_3229</td>\n",
       "      <td>Exome assembly</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>252</th>\n",
       "      <td>http://edamontology.org/operation_3258</td>\n",
       "      <td>Transcriptome assembly</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>253</th>\n",
       "      <td>http://edamontology.org/operation_3730</td>\n",
       "      <td>Cross-assembly</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>254</th>\n",
       "      <td>http://edamontology.org/operation_0319</td>\n",
       "      <td>Protein secondary structure assignment</td>\n",
       "      <td>http://edamontology.org/topic_1317</td>\n",
       "      <td>Structural biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>255</th>\n",
       "      <td>http://edamontology.org/operation_0319</td>\n",
       "      <td>Protein secondary structure assignment</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>256</th>\n",
       "      <td>http://edamontology.org/operation_0320</td>\n",
       "      <td>Protein structure assignment</td>\n",
       "      <td>http://edamontology.org/topic_1317</td>\n",
       "      <td>Structural biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>257</th>\n",
       "      <td>http://edamontology.org/operation_1829</td>\n",
       "      <td>Cysteine bridge detection</td>\n",
       "      <td>http://edamontology.org/topic_1317</td>\n",
       "      <td>Structural biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>258</th>\n",
       "      <td>http://edamontology.org/operation_1850</td>\n",
       "      <td>Protein cysteine and disulfide bond assignment</td>\n",
       "      <td>http://edamontology.org/topic_1317</td>\n",
       "      <td>Structural biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>259</th>\n",
       "      <td>http://edamontology.org/operation_1830</td>\n",
       "      <td>Free cysteine detection</td>\n",
       "      <td>http://edamontology.org/topic_1317</td>\n",
       "      <td>Structural biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>260</th>\n",
       "      <td>http://edamontology.org/operation_1831</td>\n",
       "      <td>Metal-bound cysteine detection</td>\n",
       "      <td>http://edamontology.org/topic_1317</td>\n",
       "      <td>Structural biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>261</th>\n",
       "      <td>http://edamontology.org/operation_0321</td>\n",
       "      <td>Protein structure validation</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>262</th>\n",
       "      <td>http://edamontology.org/operation_1844</td>\n",
       "      <td>Protein geometry validation</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>263</th>\n",
       "      <td>http://edamontology.org/operation_1836</td>\n",
       "      <td>Residue bump detection</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>264</th>\n",
       "      <td>http://edamontology.org/operation_1843</td>\n",
       "      <td>Residue packing validation</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>265</th>\n",
       "      <td>http://edamontology.org/operation_3560</td>\n",
       "      <td>Maximum occurence analysis</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>266</th>\n",
       "      <td>http://edamontology.org/operation_0323</td>\n",
       "      <td>Phylogenetic inference</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>267</th>\n",
       "      <td>http://edamontology.org/operation_0327</td>\n",
       "      <td>Phylogenetic footprinting</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>268</th>\n",
       "      <td>http://edamontology.org/operation_0538</td>\n",
       "      <td>Phylogenetic inference (data centric)</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>269</th>\n",
       "      <td>http://edamontology.org/operation_0539</td>\n",
       "      <td>Phylogenetic inference (method centric)</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>270</th>\n",
       "      <td>http://edamontology.org/operation_0540</td>\n",
       "      <td>Phylogenetic inference (from molecular sequences)</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>271</th>\n",
       "      <td>http://edamontology.org/operation_0541</td>\n",
       "      <td>Phylogenetic inference (from continuous quantitative characters)</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>272</th>\n",
       "      <td>http://edamontology.org/operation_0542</td>\n",
       "      <td>Phylogenetic inference (from gene frequencies)</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>273</th>\n",
       "      <td>http://edamontology.org/operation_0543</td>\n",
       "      <td>Phylogenetic inference (from polymorphism data)</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>274</th>\n",
       "      <td>http://edamontology.org/operation_0544</td>\n",
       "      <td>Species tree construction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>275</th>\n",
       "      <td>http://edamontology.org/operation_0545</td>\n",
       "      <td>Phylogenetic inference (parsimony methods)</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>276</th>\n",
       "      <td>http://edamontology.org/operation_0546</td>\n",
       "      <td>Phylogenetic inference (minimum distance methods)</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>277</th>\n",
       "      <td>http://edamontology.org/operation_0547</td>\n",
       "      <td>Phylogenetic inference (maximum likelihood and Bayesian methods)</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>278</th>\n",
       "      <td>http://edamontology.org/operation_0548</td>\n",
       "      <td>Phylogenetic inference (quartet methods)</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>279</th>\n",
       "      <td>http://edamontology.org/operation_0549</td>\n",
       "      <td>Phylogenetic inference (AI methods)</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>280</th>\n",
       "      <td>http://edamontology.org/operation_0553</td>\n",
       "      <td>Gene tree construction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>281</th>\n",
       "      <td>http://edamontology.org/operation_0555</td>\n",
       "      <td>Consensus tree construction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>282</th>\n",
       "      <td>http://edamontology.org/operation_3478</td>\n",
       "      <td>Phylogenetic reconstruction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>283</th>\n",
       "      <td>http://edamontology.org/operation_0323</td>\n",
       "      <td>Phylogenetic inference</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>284</th>\n",
       "      <td>http://edamontology.org/operation_0324</td>\n",
       "      <td>Phylogenetic analysis</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>285</th>\n",
       "      <td>http://edamontology.org/operation_0325</td>\n",
       "      <td>Phylogenetic tree comparison</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>286</th>\n",
       "      <td>http://edamontology.org/operation_0326</td>\n",
       "      <td>Phylogenetic tree editing</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>287</th>\n",
       "      <td>http://edamontology.org/operation_0327</td>\n",
       "      <td>Phylogenetic footprinting</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>288</th>\n",
       "      <td>http://edamontology.org/operation_0538</td>\n",
       "      <td>Phylogenetic inference (data centric)</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>289</th>\n",
       "      <td>http://edamontology.org/operation_0539</td>\n",
       "      <td>Phylogenetic inference (method centric)</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>290</th>\n",
       "      <td>http://edamontology.org/operation_0540</td>\n",
       "      <td>Phylogenetic inference (from molecular sequences)</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>291</th>\n",
       "      <td>http://edamontology.org/operation_0541</td>\n",
       "      <td>Phylogenetic inference (from continuous quantitative characters)</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>292</th>\n",
       "      <td>http://edamontology.org/operation_0542</td>\n",
       "      <td>Phylogenetic inference (from gene frequencies)</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>293</th>\n",
       "      <td>http://edamontology.org/operation_0543</td>\n",
       "      <td>Phylogenetic inference (from polymorphism data)</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>294</th>\n",
       "      <td>http://edamontology.org/operation_0544</td>\n",
       "      <td>Species tree construction</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>295</th>\n",
       "      <td>http://edamontology.org/operation_0545</td>\n",
       "      <td>Phylogenetic inference (parsimony methods)</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>296</th>\n",
       "      <td>http://edamontology.org/operation_0546</td>\n",
       "      <td>Phylogenetic inference (minimum distance methods)</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>297</th>\n",
       "      <td>http://edamontology.org/operation_0547</td>\n",
       "      <td>Phylogenetic inference (maximum likelihood and Bayesian methods)</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>298</th>\n",
       "      <td>http://edamontology.org/operation_0548</td>\n",
       "      <td>Phylogenetic inference (quartet methods)</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>299</th>\n",
       "      <td>http://edamontology.org/operation_0549</td>\n",
       "      <td>Phylogenetic inference (AI methods)</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>300</th>\n",
       "      <td>http://edamontology.org/operation_0551</td>\n",
       "      <td>Phylogenetic tree topology analysis</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>301</th>\n",
       "      <td>http://edamontology.org/operation_0552</td>\n",
       "      <td>Phylogenetic tree bootstrapping</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>302</th>\n",
       "      <td>http://edamontology.org/operation_0553</td>\n",
       "      <td>Gene tree construction</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>303</th>\n",
       "      <td>http://edamontology.org/operation_0554</td>\n",
       "      <td>Allele frequency distribution analysis</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>304</th>\n",
       "      <td>http://edamontology.org/operation_0555</td>\n",
       "      <td>Consensus tree construction</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>305</th>\n",
       "      <td>http://edamontology.org/operation_0556</td>\n",
       "      <td>Phylogenetic sub/super tree construction</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>306</th>\n",
       "      <td>http://edamontology.org/operation_0557</td>\n",
       "      <td>Phylogenetic tree distances calculation</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>307</th>\n",
       "      <td>http://edamontology.org/operation_0567</td>\n",
       "      <td>Phylogenetic tree visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>308</th>\n",
       "      <td>http://edamontology.org/operation_3478</td>\n",
       "      <td>Phylogenetic reconstruction</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>309</th>\n",
       "      <td>http://edamontology.org/operation_3745</td>\n",
       "      <td>Ancestral reconstruction</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>310</th>\n",
       "      <td>http://edamontology.org/operation_3942</td>\n",
       "      <td>Tree dating</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>311</th>\n",
       "      <td>http://edamontology.org/operation_3947</td>\n",
       "      <td>Phylogenetic tree reconciliation</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>312</th>\n",
       "      <td>http://edamontology.org/operation_0327</td>\n",
       "      <td>Phylogenetic footprinting</td>\n",
       "      <td>http://edamontology.org/topic_0194</td>\n",
       "      <td>Phylogenomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>313</th>\n",
       "      <td>http://edamontology.org/operation_0331</td>\n",
       "      <td>Variant effect prediction</td>\n",
       "      <td>http://edamontology.org/topic_0130</td>\n",
       "      <td>Protein folding, stability and design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>314</th>\n",
       "      <td>http://edamontology.org/operation_0331</td>\n",
       "      <td>Variant effect prediction</td>\n",
       "      <td>http://edamontology.org/topic_0199</td>\n",
       "      <td>Genetic variation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>315</th>\n",
       "      <td>http://edamontology.org/operation_0334</td>\n",
       "      <td>Enzyme kinetics calculation</td>\n",
       "      <td>http://edamontology.org/topic_0821</td>\n",
       "      <td>Enzymes</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>316</th>\n",
       "      <td>http://edamontology.org/operation_0287</td>\n",
       "      <td>Base position variability plotting</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>317</th>\n",
       "      <td>http://edamontology.org/operation_0564</td>\n",
       "      <td>Sequence visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>318</th>\n",
       "      <td>http://edamontology.org/operation_0337</td>\n",
       "      <td>Visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>319</th>\n",
       "      <td>http://edamontology.org/operation_0402</td>\n",
       "      <td>Protein titration curve plotting</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>320</th>\n",
       "      <td>http://edamontology.org/operation_0407</td>\n",
       "      <td>Protein hydrophobic moment plotting</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>321</th>\n",
       "      <td>http://edamontology.org/operation_0575</td>\n",
       "      <td>Restriction map drawing</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>322</th>\n",
       "      <td>http://edamontology.org/operation_0456</td>\n",
       "      <td>Nucleic acid melting profile plotting</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>323</th>\n",
       "      <td>http://edamontology.org/operation_0457</td>\n",
       "      <td>Nucleic acid stitch profile plotting</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>324</th>\n",
       "      <td>http://edamontology.org/operation_0458</td>\n",
       "      <td>Nucleic acid melting curve plotting</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>325</th>\n",
       "      <td>http://edamontology.org/operation_0459</td>\n",
       "      <td>Nucleic acid probability profile plotting</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>326</th>\n",
       "      <td>http://edamontology.org/operation_0460</td>\n",
       "      <td>Nucleic acid temperature profile plotting</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>327</th>\n",
       "      <td>http://edamontology.org/operation_0490</td>\n",
       "      <td>Dot plot plotting</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>328</th>\n",
       "      <td>http://edamontology.org/operation_0531</td>\n",
       "      <td>Heat map generation</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>329</th>\n",
       "      <td>http://edamontology.org/operation_0571</td>\n",
       "      <td>Expression data visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>330</th>\n",
       "      <td>http://edamontology.org/operation_0566</td>\n",
       "      <td>Sequence cluster visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>331</th>\n",
       "      <td>http://edamontology.org/operation_0567</td>\n",
       "      <td>Phylogenetic tree visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>332</th>\n",
       "      <td>http://edamontology.org/operation_0570</td>\n",
       "      <td>Structure visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>333</th>\n",
       "      <td>http://edamontology.org/operation_3925</td>\n",
       "      <td>Network visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>334</th>\n",
       "      <td>http://edamontology.org/operation_0573</td>\n",
       "      <td>Map drawing</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>335</th>\n",
       "      <td>http://edamontology.org/operation_0578</td>\n",
       "      <td>Plasmid map drawing</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>336</th>\n",
       "      <td>http://edamontology.org/operation_0579</td>\n",
       "      <td>Operon drawing</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>337</th>\n",
       "      <td>http://edamontology.org/operation_1816</td>\n",
       "      <td>Surface rendering</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>338</th>\n",
       "      <td>http://edamontology.org/operation_2241</td>\n",
       "      <td>Transmembrane protein visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>339</th>\n",
       "      <td>http://edamontology.org/operation_2284</td>\n",
       "      <td>Nucleic acid density plotting</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>340</th>\n",
       "      <td>http://edamontology.org/operation_2485</td>\n",
       "      <td>Helical wheel drawing</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>341</th>\n",
       "      <td>http://edamontology.org/operation_2486</td>\n",
       "      <td>Topology diagram drawing</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>342</th>\n",
       "      <td>http://edamontology.org/operation_2938</td>\n",
       "      <td>Dendrogram visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>343</th>\n",
       "      <td>http://edamontology.org/operation_2935</td>\n",
       "      <td>Clustering profile plotting</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>344</th>\n",
       "      <td>http://edamontology.org/operation_2937</td>\n",
       "      <td>Proximity map plotting</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>345</th>\n",
       "      <td>http://edamontology.org/operation_2939</td>\n",
       "      <td>Principal component visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>346</th>\n",
       "      <td>http://edamontology.org/operation_2940</td>\n",
       "      <td>Scatter plot plotting</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>347</th>\n",
       "      <td>http://edamontology.org/operation_2942</td>\n",
       "      <td>Treemap visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>348</th>\n",
       "      <td>http://edamontology.org/operation_2943</td>\n",
       "      <td>Box-Whisker plot plotting</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>349</th>\n",
       "      <td>http://edamontology.org/operation_3926</td>\n",
       "      <td>Pathway visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>350</th>\n",
       "      <td>http://edamontology.org/operation_3184</td>\n",
       "      <td>Sequence assembly visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>351</th>\n",
       "      <td>http://edamontology.org/operation_3203</td>\n",
       "      <td>Chromatogram visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>352</th>\n",
       "      <td>http://edamontology.org/operation_3208</td>\n",
       "      <td>Genome visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>353</th>\n",
       "      <td>http://edamontology.org/operation_3503</td>\n",
       "      <td>Incident curve plotting</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>354</th>\n",
       "      <td>http://edamontology.org/operation_3552</td>\n",
       "      <td>Microscope image visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>355</th>\n",
       "      <td>http://edamontology.org/operation_3559</td>\n",
       "      <td>Ontology visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>356</th>\n",
       "      <td>http://edamontology.org/operation_3694</td>\n",
       "      <td>Mass spectrum visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>357</th>\n",
       "      <td>http://edamontology.org/operation_3744</td>\n",
       "      <td>Multiple sample visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>358</th>\n",
       "      <td>http://edamontology.org/operation_3890</td>\n",
       "      <td>Trajectory visualization</td>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>359</th>\n",
       "      <td>http://edamontology.org/operation_0339</td>\n",
       "      <td>Structure database search</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>360</th>\n",
       "      <td>http://edamontology.org/operation_0346</td>\n",
       "      <td>Sequence similarity search</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>361</th>\n",
       "      <td>http://edamontology.org/operation_0360</td>\n",
       "      <td>Structural similarity search</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>362</th>\n",
       "      <td>http://edamontology.org/operation_0366</td>\n",
       "      <td>Protein sequence cleavage</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>363</th>\n",
       "      <td>http://edamontology.org/operation_0371</td>\n",
       "      <td>DNA translation</td>\n",
       "      <td>http://edamontology.org/topic_0108</td>\n",
       "      <td>Protein expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>364</th>\n",
       "      <td>http://edamontology.org/operation_0372</td>\n",
       "      <td>DNA transcription</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>365</th>\n",
       "      <td>http://edamontology.org/operation_0389</td>\n",
       "      <td>Protein-nucleic acid interaction analysis</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>366</th>\n",
       "      <td>http://edamontology.org/operation_3900</td>\n",
       "      <td>DNA-binding protein prediction</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>367</th>\n",
       "      <td>http://edamontology.org/operation_3901</td>\n",
       "      <td>RNA-binding protein prediction</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>368</th>\n",
       "      <td>http://edamontology.org/operation_2929</td>\n",
       "      <td>Protein fragment weight comparison</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>369</th>\n",
       "      <td>http://edamontology.org/operation_0398</td>\n",
       "      <td>Protein molecular weight calculation</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>370</th>\n",
       "      <td>http://edamontology.org/operation_0400</td>\n",
       "      <td>Protein pKa calculation</td>\n",
       "      <td>http://edamontology.org/topic_0123</td>\n",
       "      <td>Protein properties</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>371</th>\n",
       "      <td>http://edamontology.org/operation_0402</td>\n",
       "      <td>Protein titration curve plotting</td>\n",
       "      <td>http://edamontology.org/topic_0123</td>\n",
       "      <td>Protein properties</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>372</th>\n",
       "      <td>http://edamontology.org/operation_0403</td>\n",
       "      <td>Protein isoelectric point calculation</td>\n",
       "      <td>http://edamontology.org/topic_0123</td>\n",
       "      <td>Protein properties</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>373</th>\n",
       "      <td>http://edamontology.org/operation_0404</td>\n",
       "      <td>Protein hydrogen exchange rate calculation</td>\n",
       "      <td>http://edamontology.org/topic_0123</td>\n",
       "      <td>Protein properties</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>374</th>\n",
       "      <td>http://edamontology.org/operation_0266</td>\n",
       "      <td>Vector sequence detection</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>375</th>\n",
       "      <td>http://edamontology.org/operation_0415</td>\n",
       "      <td>Nucleic acid feature detection</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>376</th>\n",
       "      <td>http://edamontology.org/operation_0278</td>\n",
       "      <td>RNA secondary structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>377</th>\n",
       "      <td>http://edamontology.org/operation_0379</td>\n",
       "      <td>Repeat sequence detection</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>378</th>\n",
       "      <td>http://edamontology.org/operation_0427</td>\n",
       "      <td>Transposon prediction</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>379</th>\n",
       "      <td>http://edamontology.org/operation_0428</td>\n",
       "      <td>PolyA signal detection</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>380</th>\n",
       "      <td>http://edamontology.org/operation_0429</td>\n",
       "      <td>Quadruplex formation site detection</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>381</th>\n",
       "      <td>http://edamontology.org/operation_0430</td>\n",
       "      <td>CpG island and isochore detection</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>382</th>\n",
       "      <td>http://edamontology.org/operation_0431</td>\n",
       "      <td>Restriction site recognition</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>383</th>\n",
       "      <td>http://edamontology.org/operation_0432</td>\n",
       "      <td>Nucleosome position prediction</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>384</th>\n",
       "      <td>http://edamontology.org/operation_3222</td>\n",
       "      <td>Peak calling</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>385</th>\n",
       "      <td>http://edamontology.org/operation_3677</td>\n",
       "      <td>Differential binding analysis</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>386</th>\n",
       "      <td>http://edamontology.org/operation_4032</td>\n",
       "      <td>DNA modification prediction</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>387</th>\n",
       "      <td>http://edamontology.org/operation_0416</td>\n",
       "      <td>Epitope mapping</td>\n",
       "      <td>http://edamontology.org/topic_0804</td>\n",
       "      <td>Immunology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>388</th>\n",
       "      <td>http://edamontology.org/operation_0417</td>\n",
       "      <td>Post-translational modification site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0601</td>\n",
       "      <td>Protein modifications</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>389</th>\n",
       "      <td>http://edamontology.org/operation_0418</td>\n",
       "      <td>Protein signal peptide detection</td>\n",
       "      <td>http://edamontology.org/topic_0140</td>\n",
       "      <td>Protein targeting and localisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>390</th>\n",
       "      <td>http://edamontology.org/operation_0422</td>\n",
       "      <td>Protein cleavage site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>391</th>\n",
       "      <td>http://edamontology.org/operation_0430</td>\n",
       "      <td>CpG island and isochore detection</td>\n",
       "      <td>http://edamontology.org/topic_0157</td>\n",
       "      <td>Sequence composition, complexity and repeats</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>392</th>\n",
       "      <td>http://edamontology.org/operation_0433</td>\n",
       "      <td>Splice site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>393</th>\n",
       "      <td>http://edamontology.org/operation_0438</td>\n",
       "      <td>Transcriptional regulatory element prediction</td>\n",
       "      <td>http://edamontology.org/topic_0749</td>\n",
       "      <td>Transcription factors and regulatory sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>394</th>\n",
       "      <td>http://edamontology.org/operation_0440</td>\n",
       "      <td>Promoter prediction</td>\n",
       "      <td>http://edamontology.org/topic_0749</td>\n",
       "      <td>Transcription factors and regulatory sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>395</th>\n",
       "      <td>http://edamontology.org/operation_0441</td>\n",
       "      <td>cis-regulatory element prediction</td>\n",
       "      <td>http://edamontology.org/topic_0749</td>\n",
       "      <td>Transcription factors and regulatory sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>396</th>\n",
       "      <td>http://edamontology.org/operation_0443</td>\n",
       "      <td>trans-regulatory element prediction</td>\n",
       "      <td>http://edamontology.org/topic_0749</td>\n",
       "      <td>Transcription factors and regulatory sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>397</th>\n",
       "      <td>http://edamontology.org/operation_0444</td>\n",
       "      <td>S/MAR prediction</td>\n",
       "      <td>http://edamontology.org/topic_0749</td>\n",
       "      <td>Transcription factors and regulatory sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>398</th>\n",
       "      <td>http://edamontology.org/operation_0445</td>\n",
       "      <td>Transcription factor binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0749</td>\n",
       "      <td>Transcription factors and regulatory sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>399</th>\n",
       "      <td>http://edamontology.org/operation_0446</td>\n",
       "      <td>Exonic splicing enhancer prediction</td>\n",
       "      <td>http://edamontology.org/topic_0749</td>\n",
       "      <td>Transcription factors and regulatory sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>400</th>\n",
       "      <td>http://edamontology.org/operation_0463</td>\n",
       "      <td>miRNA target prediction</td>\n",
       "      <td>http://edamontology.org/topic_0749</td>\n",
       "      <td>Transcription factors and regulatory sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>401</th>\n",
       "      <td>http://edamontology.org/operation_0465</td>\n",
       "      <td>siRNA binding specificity prediction</td>\n",
       "      <td>http://edamontology.org/topic_0749</td>\n",
       "      <td>Transcription factors and regulatory sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>402</th>\n",
       "      <td>http://edamontology.org/operation_2008</td>\n",
       "      <td>siRNA duplex prediction</td>\n",
       "      <td>http://edamontology.org/topic_0749</td>\n",
       "      <td>Transcription factors and regulatory sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>403</th>\n",
       "      <td>http://edamontology.org/operation_0439</td>\n",
       "      <td>Translation initiation site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0108</td>\n",
       "      <td>Protein expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>404</th>\n",
       "      <td>http://edamontology.org/operation_0443</td>\n",
       "      <td>trans-regulatory element prediction</td>\n",
       "      <td>http://edamontology.org/topic_0659</td>\n",
       "      <td>Functional, regulatory and non-coding RNA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>405</th>\n",
       "      <td>http://edamontology.org/operation_0463</td>\n",
       "      <td>miRNA target prediction</td>\n",
       "      <td>http://edamontology.org/topic_0659</td>\n",
       "      <td>Functional, regulatory and non-coding RNA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>406</th>\n",
       "      <td>http://edamontology.org/operation_0465</td>\n",
       "      <td>siRNA binding specificity prediction</td>\n",
       "      <td>http://edamontology.org/topic_0659</td>\n",
       "      <td>Functional, regulatory and non-coding RNA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>407</th>\n",
       "      <td>http://edamontology.org/operation_2008</td>\n",
       "      <td>siRNA duplex prediction</td>\n",
       "      <td>http://edamontology.org/topic_0659</td>\n",
       "      <td>Functional, regulatory and non-coding RNA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>408</th>\n",
       "      <td>http://edamontology.org/operation_0446</td>\n",
       "      <td>Exonic splicing enhancer prediction</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>409</th>\n",
       "      <td>http://edamontology.org/operation_0464</td>\n",
       "      <td>tRNA gene prediction</td>\n",
       "      <td>http://edamontology.org/topic_0659</td>\n",
       "      <td>Functional, regulatory and non-coding RNA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>410</th>\n",
       "      <td>http://edamontology.org/operation_0477</td>\n",
       "      <td>Protein modelling</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>411</th>\n",
       "      <td>http://edamontology.org/operation_0479</td>\n",
       "      <td>Backbone modelling</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>412</th>\n",
       "      <td>http://edamontology.org/operation_0480</td>\n",
       "      <td>Side chain modelling</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>413</th>\n",
       "      <td>http://edamontology.org/operation_0481</td>\n",
       "      <td>Loop modelling</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>414</th>\n",
       "      <td>http://edamontology.org/operation_0482</td>\n",
       "      <td>Protein-ligand docking</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>415</th>\n",
       "      <td>http://edamontology.org/operation_3938</td>\n",
       "      <td>Virtual screening</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>416</th>\n",
       "      <td>http://edamontology.org/operation_0499</td>\n",
       "      <td>Tree-based sequence alignment</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>417</th>\n",
       "      <td>http://edamontology.org/operation_0542</td>\n",
       "      <td>Phylogenetic inference (from gene frequencies)</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>418</th>\n",
       "      <td>http://edamontology.org/operation_0543</td>\n",
       "      <td>Phylogenetic inference (from polymorphism data)</td>\n",
       "      <td>http://edamontology.org/topic_0199</td>\n",
       "      <td>Genetic variation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>419</th>\n",
       "      <td>http://edamontology.org/operation_0550</td>\n",
       "      <td>DNA substitution modelling</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>420</th>\n",
       "      <td>http://edamontology.org/operation_0553</td>\n",
       "      <td>Gene tree construction</td>\n",
       "      <td>http://edamontology.org/topic_0194</td>\n",
       "      <td>Phylogenomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>421</th>\n",
       "      <td>http://edamontology.org/operation_0560</td>\n",
       "      <td>DNA vaccine design</td>\n",
       "      <td>http://edamontology.org/topic_0804</td>\n",
       "      <td>Immunology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>422</th>\n",
       "      <td>http://edamontology.org/operation_0579</td>\n",
       "      <td>Operon drawing</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>423</th>\n",
       "      <td>http://edamontology.org/operation_0252</td>\n",
       "      <td>Peptide immunogenicity prediction</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>424</th>\n",
       "      <td>http://edamontology.org/operation_1777</td>\n",
       "      <td>Protein function prediction</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>425</th>\n",
       "      <td>http://edamontology.org/operation_2949</td>\n",
       "      <td>Protein-protein interaction analysis</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>426</th>\n",
       "      <td>http://edamontology.org/operation_2464</td>\n",
       "      <td>Protein-protein binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>427</th>\n",
       "      <td>http://edamontology.org/operation_0276</td>\n",
       "      <td>Protein interaction network analysis</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>428</th>\n",
       "      <td>http://edamontology.org/operation_0420</td>\n",
       "      <td>Nucleic acids-binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>429</th>\n",
       "      <td>http://edamontology.org/operation_2575</td>\n",
       "      <td>Binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>430</th>\n",
       "      <td>http://edamontology.org/operation_0389</td>\n",
       "      <td>Protein-nucleic acid interaction analysis</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>431</th>\n",
       "      <td>http://edamontology.org/operation_0418</td>\n",
       "      <td>Protein signal peptide detection</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>432</th>\n",
       "      <td>http://edamontology.org/operation_0478</td>\n",
       "      <td>Molecular docking</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>433</th>\n",
       "      <td>http://edamontology.org/operation_0482</td>\n",
       "      <td>Protein-ligand docking</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>434</th>\n",
       "      <td>http://edamontology.org/operation_1778</td>\n",
       "      <td>Protein function comparison</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>435</th>\n",
       "      <td>http://edamontology.org/operation_2489</td>\n",
       "      <td>Subcellular localisation prediction</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>436</th>\n",
       "      <td>http://edamontology.org/operation_2492</td>\n",
       "      <td>Protein interaction prediction</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>437</th>\n",
       "      <td>http://edamontology.org/operation_3094</td>\n",
       "      <td>Protein interaction network prediction</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>438</th>\n",
       "      <td>http://edamontology.org/operation_3896</td>\n",
       "      <td>Active site prediction</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>439</th>\n",
       "      <td>http://edamontology.org/operation_3897</td>\n",
       "      <td>Ligand-binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>440</th>\n",
       "      <td>http://edamontology.org/operation_3898</td>\n",
       "      <td>Metal-binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>441</th>\n",
       "      <td>http://edamontology.org/operation_3899</td>\n",
       "      <td>Protein-protein docking</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>442</th>\n",
       "      <td>http://edamontology.org/operation_3900</td>\n",
       "      <td>DNA-binding protein prediction</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>443</th>\n",
       "      <td>http://edamontology.org/operation_3901</td>\n",
       "      <td>RNA-binding protein prediction</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>444</th>\n",
       "      <td>http://edamontology.org/operation_3902</td>\n",
       "      <td>RNA binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>445</th>\n",
       "      <td>http://edamontology.org/operation_3903</td>\n",
       "      <td>DNA binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>446</th>\n",
       "      <td>http://edamontology.org/operation_3938</td>\n",
       "      <td>Virtual screening</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>447</th>\n",
       "      <td>http://edamontology.org/operation_1781</td>\n",
       "      <td>Gene regulatory network analysis</td>\n",
       "      <td>http://edamontology.org/topic_0602</td>\n",
       "      <td>Molecular interactions, pathways and networks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>448</th>\n",
       "      <td>http://edamontology.org/operation_1829</td>\n",
       "      <td>Cysteine bridge detection</td>\n",
       "      <td>http://edamontology.org/topic_0130</td>\n",
       "      <td>Protein folding, stability and design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>449</th>\n",
       "      <td>http://edamontology.org/operation_1850</td>\n",
       "      <td>Protein cysteine and disulfide bond assignment</td>\n",
       "      <td>http://edamontology.org/topic_0130</td>\n",
       "      <td>Protein folding, stability and design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>450</th>\n",
       "      <td>http://edamontology.org/operation_1830</td>\n",
       "      <td>Free cysteine detection</td>\n",
       "      <td>http://edamontology.org/topic_0130</td>\n",
       "      <td>Protein folding, stability and design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>451</th>\n",
       "      <td>http://edamontology.org/operation_2239</td>\n",
       "      <td>3D-1D scoring matrix generation</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>452</th>\n",
       "      <td>http://edamontology.org/operation_0230</td>\n",
       "      <td>Sequence generation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>453</th>\n",
       "      <td>http://edamontology.org/operation_2403</td>\n",
       "      <td>Sequence analysis</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>454</th>\n",
       "      <td>http://edamontology.org/operation_0231</td>\n",
       "      <td>Sequence editing</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>455</th>\n",
       "      <td>http://edamontology.org/operation_0232</td>\n",
       "      <td>Sequence merging</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>456</th>\n",
       "      <td>http://edamontology.org/operation_0233</td>\n",
       "      <td>Sequence conversion</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>457</th>\n",
       "      <td>http://edamontology.org/operation_0234</td>\n",
       "      <td>Sequence complexity calculation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>458</th>\n",
       "      <td>http://edamontology.org/operation_0236</td>\n",
       "      <td>Sequence composition calculation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>459</th>\n",
       "      <td>http://edamontology.org/operation_0235</td>\n",
       "      <td>Sequence ambiguity calculation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>460</th>\n",
       "      <td>http://edamontology.org/operation_0237</td>\n",
       "      <td>Repeat sequence analysis</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>461</th>\n",
       "      <td>http://edamontology.org/operation_0238</td>\n",
       "      <td>Sequence motif discovery</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>462</th>\n",
       "      <td>http://edamontology.org/operation_0253</td>\n",
       "      <td>Sequence feature detection</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>463</th>\n",
       "      <td>http://edamontology.org/operation_2404</td>\n",
       "      <td>Sequence motif analysis</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>464</th>\n",
       "      <td>http://edamontology.org/operation_0239</td>\n",
       "      <td>Sequence motif recognition</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>465</th>\n",
       "      <td>http://edamontology.org/operation_0240</td>\n",
       "      <td>Sequence motif comparison</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>466</th>\n",
       "      <td>http://edamontology.org/operation_2451</td>\n",
       "      <td>Sequence comparison</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>467</th>\n",
       "      <td>http://edamontology.org/operation_0438</td>\n",
       "      <td>Transcriptional regulatory element prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>468</th>\n",
       "      <td>http://edamontology.org/operation_0245</td>\n",
       "      <td>Structural motif discovery</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>469</th>\n",
       "      <td>http://edamontology.org/operation_3092</td>\n",
       "      <td>Protein feature detection</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>470</th>\n",
       "      <td>http://edamontology.org/operation_0246</td>\n",
       "      <td>Protein domain recognition</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>471</th>\n",
       "      <td>http://edamontology.org/operation_0256</td>\n",
       "      <td>Sequence feature comparison</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>472</th>\n",
       "      <td>http://edamontology.org/operation_0258</td>\n",
       "      <td>Sequence alignment analysis</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>473</th>\n",
       "      <td>http://edamontology.org/operation_0259</td>\n",
       "      <td>Sequence alignment comparison</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>474</th>\n",
       "      <td>http://edamontology.org/operation_0264</td>\n",
       "      <td>Alternative splicing prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>475</th>\n",
       "      <td>http://edamontology.org/operation_2499</td>\n",
       "      <td>Splicing analysis</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>476</th>\n",
       "      <td>http://edamontology.org/operation_0265</td>\n",
       "      <td>Frameshift detection</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>477</th>\n",
       "      <td>http://edamontology.org/operation_3195</td>\n",
       "      <td>Sequencing error detection</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>478</th>\n",
       "      <td>http://edamontology.org/operation_3227</td>\n",
       "      <td>Variant calling</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>479</th>\n",
       "      <td>http://edamontology.org/operation_0266</td>\n",
       "      <td>Vector sequence detection</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>480</th>\n",
       "      <td>http://edamontology.org/operation_0415</td>\n",
       "      <td>Nucleic acid feature detection</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>481</th>\n",
       "      <td>http://edamontology.org/operation_0267</td>\n",
       "      <td>Protein secondary structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>482</th>\n",
       "      <td>http://edamontology.org/operation_2479</td>\n",
       "      <td>Protein sequence analysis</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>483</th>\n",
       "      <td>http://edamontology.org/operation_0268</td>\n",
       "      <td>Protein super-secondary structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>484</th>\n",
       "      <td>http://edamontology.org/operation_0269</td>\n",
       "      <td>Transmembrane protein prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>485</th>\n",
       "      <td>http://edamontology.org/operation_0474</td>\n",
       "      <td>Protein structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>486</th>\n",
       "      <td>http://edamontology.org/operation_0272</td>\n",
       "      <td>Residue contact prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>487</th>\n",
       "      <td>http://edamontology.org/operation_2464</td>\n",
       "      <td>Protein-protein binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>488</th>\n",
       "      <td>http://edamontology.org/operation_0278</td>\n",
       "      <td>RNA secondary structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>489</th>\n",
       "      <td>http://edamontology.org/operation_0282</td>\n",
       "      <td>Genetic mapping</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>490</th>\n",
       "      <td>http://edamontology.org/operation_0283</td>\n",
       "      <td>Linkage analysis</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>491</th>\n",
       "      <td>http://edamontology.org/operation_2478</td>\n",
       "      <td>Nucleic acid sequence analysis</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>492</th>\n",
       "      <td>http://edamontology.org/operation_0284</td>\n",
       "      <td>Codon usage table generation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>493</th>\n",
       "      <td>http://edamontology.org/operation_0286</td>\n",
       "      <td>Codon usage analysis</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>494</th>\n",
       "      <td>http://edamontology.org/operation_0285</td>\n",
       "      <td>Codon usage table comparison</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>495</th>\n",
       "      <td>http://edamontology.org/operation_0287</td>\n",
       "      <td>Base position variability plotting</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>496</th>\n",
       "      <td>http://edamontology.org/operation_0564</td>\n",
       "      <td>Sequence visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>497</th>\n",
       "      <td>http://edamontology.org/operation_0288</td>\n",
       "      <td>Sequence word comparison</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>498</th>\n",
       "      <td>http://edamontology.org/operation_0289</td>\n",
       "      <td>Sequence distance matrix generation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>499</th>\n",
       "      <td>http://edamontology.org/operation_0290</td>\n",
       "      <td>Sequence redundancy removal</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>500</th>\n",
       "      <td>http://edamontology.org/operation_0291</td>\n",
       "      <td>Sequence clustering</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>501</th>\n",
       "      <td>http://edamontology.org/operation_0292</td>\n",
       "      <td>Sequence alignment</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>502</th>\n",
       "      <td>http://edamontology.org/operation_0294</td>\n",
       "      <td>Structure-based sequence alignment</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>503</th>\n",
       "      <td>http://edamontology.org/operation_0296</td>\n",
       "      <td>Sequence profile generation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>504</th>\n",
       "      <td>http://edamontology.org/operation_0300</td>\n",
       "      <td>Sequence profile alignment</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>505</th>\n",
       "      <td>http://edamontology.org/operation_0303</td>\n",
       "      <td>Fold recognition</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>506</th>\n",
       "      <td>http://edamontology.org/operation_0302</td>\n",
       "      <td>Protein threading</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>507</th>\n",
       "      <td>http://edamontology.org/operation_0307</td>\n",
       "      <td>Virtual PCR</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>508</th>\n",
       "      <td>http://edamontology.org/operation_0308</td>\n",
       "      <td>PCR primer design</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>509</th>\n",
       "      <td>http://edamontology.org/operation_2419</td>\n",
       "      <td>Primer and probe design</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>510</th>\n",
       "      <td>http://edamontology.org/operation_0309</td>\n",
       "      <td>Microarray probe design</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>511</th>\n",
       "      <td>http://edamontology.org/operation_0310</td>\n",
       "      <td>Sequence assembly</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>512</th>\n",
       "      <td>http://edamontology.org/operation_0420</td>\n",
       "      <td>Nucleic acids-binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>513</th>\n",
       "      <td>http://edamontology.org/operation_0338</td>\n",
       "      <td>Sequence database search</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>514</th>\n",
       "      <td>http://edamontology.org/operation_0346</td>\n",
       "      <td>Sequence similarity search</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>515</th>\n",
       "      <td>http://edamontology.org/operation_0349</td>\n",
       "      <td>Sequence database search (by property)</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>516</th>\n",
       "      <td>http://edamontology.org/operation_0362</td>\n",
       "      <td>Genome annotation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>517</th>\n",
       "      <td>http://edamontology.org/operation_3918</td>\n",
       "      <td>Genome analysis</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>518</th>\n",
       "      <td>http://edamontology.org/operation_0363</td>\n",
       "      <td>Reverse complement</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>519</th>\n",
       "      <td>http://edamontology.org/operation_0364</td>\n",
       "      <td>Random sequence generation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>520</th>\n",
       "      <td>http://edamontology.org/operation_0365</td>\n",
       "      <td>Restriction digest</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>521</th>\n",
       "      <td>http://edamontology.org/operation_0366</td>\n",
       "      <td>Protein sequence cleavage</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>522</th>\n",
       "      <td>http://edamontology.org/operation_0367</td>\n",
       "      <td>Sequence mutation and randomisation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>523</th>\n",
       "      <td>http://edamontology.org/operation_0368</td>\n",
       "      <td>Sequence masking</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>524</th>\n",
       "      <td>http://edamontology.org/operation_0369</td>\n",
       "      <td>Sequence cutting</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>525</th>\n",
       "      <td>http://edamontology.org/operation_0370</td>\n",
       "      <td>Restriction site creation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>526</th>\n",
       "      <td>http://edamontology.org/operation_0371</td>\n",
       "      <td>DNA translation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>527</th>\n",
       "      <td>http://edamontology.org/operation_0372</td>\n",
       "      <td>DNA transcription</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>528</th>\n",
       "      <td>http://edamontology.org/operation_0379</td>\n",
       "      <td>Repeat sequence detection</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>529</th>\n",
       "      <td>http://edamontology.org/operation_0380</td>\n",
       "      <td>Repeat sequence organisation analysis</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>530</th>\n",
       "      <td>http://edamontology.org/operation_2575</td>\n",
       "      <td>Binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>531</th>\n",
       "      <td>http://edamontology.org/operation_0390</td>\n",
       "      <td>Protein peeling</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>532</th>\n",
       "      <td>http://edamontology.org/operation_0407</td>\n",
       "      <td>Protein hydrophobic moment plotting</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>533</th>\n",
       "      <td>http://edamontology.org/operation_0418</td>\n",
       "      <td>Protein signal peptide detection</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>534</th>\n",
       "      <td>http://edamontology.org/operation_0416</td>\n",
       "      <td>Epitope mapping</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>535</th>\n",
       "      <td>http://edamontology.org/operation_0417</td>\n",
       "      <td>Post-translational modification site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>536</th>\n",
       "      <td>http://edamontology.org/operation_0422</td>\n",
       "      <td>Protein cleavage site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>537</th>\n",
       "      <td>http://edamontology.org/operation_2454</td>\n",
       "      <td>Gene prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>538</th>\n",
       "      <td>http://edamontology.org/operation_0427</td>\n",
       "      <td>Transposon prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>539</th>\n",
       "      <td>http://edamontology.org/operation_0428</td>\n",
       "      <td>PolyA signal detection</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>540</th>\n",
       "      <td>http://edamontology.org/operation_0429</td>\n",
       "      <td>Quadruplex formation site detection</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>541</th>\n",
       "      <td>http://edamontology.org/operation_0430</td>\n",
       "      <td>CpG island and isochore detection</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>542</th>\n",
       "      <td>http://edamontology.org/operation_0431</td>\n",
       "      <td>Restriction site recognition</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>543</th>\n",
       "      <td>http://edamontology.org/operation_0432</td>\n",
       "      <td>Nucleosome position prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>544</th>\n",
       "      <td>http://edamontology.org/operation_0433</td>\n",
       "      <td>Splice site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>545</th>\n",
       "      <td>http://edamontology.org/operation_0435</td>\n",
       "      <td>Operon prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>546</th>\n",
       "      <td>http://edamontology.org/operation_0436</td>\n",
       "      <td>Coding region prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>547</th>\n",
       "      <td>http://edamontology.org/operation_0437</td>\n",
       "      <td>SECIS element prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>548</th>\n",
       "      <td>http://edamontology.org/operation_0439</td>\n",
       "      <td>Translation initiation site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>549</th>\n",
       "      <td>http://edamontology.org/operation_0440</td>\n",
       "      <td>Promoter prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>550</th>\n",
       "      <td>http://edamontology.org/operation_0441</td>\n",
       "      <td>cis-regulatory element prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>551</th>\n",
       "      <td>http://edamontology.org/operation_0443</td>\n",
       "      <td>trans-regulatory element prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>552</th>\n",
       "      <td>http://edamontology.org/operation_0444</td>\n",
       "      <td>S/MAR prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>553</th>\n",
       "      <td>http://edamontology.org/operation_0445</td>\n",
       "      <td>Transcription factor binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>554</th>\n",
       "      <td>http://edamontology.org/operation_0446</td>\n",
       "      <td>Exonic splicing enhancer prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>555</th>\n",
       "      <td>http://edamontology.org/operation_0447</td>\n",
       "      <td>Sequence alignment validation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>556</th>\n",
       "      <td>http://edamontology.org/operation_0448</td>\n",
       "      <td>Sequence alignment analysis (conservation)</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>557</th>\n",
       "      <td>http://edamontology.org/operation_0449</td>\n",
       "      <td>Sequence alignment analysis (site correlation)</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>558</th>\n",
       "      <td>http://edamontology.org/operation_0450</td>\n",
       "      <td>Chimera detection</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>559</th>\n",
       "      <td>http://edamontology.org/operation_0451</td>\n",
       "      <td>Recombination detection</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>560</th>\n",
       "      <td>http://edamontology.org/operation_0452</td>\n",
       "      <td>Indel detection</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>561</th>\n",
       "      <td>http://edamontology.org/operation_0463</td>\n",
       "      <td>miRNA target prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>562</th>\n",
       "      <td>http://edamontology.org/operation_0464</td>\n",
       "      <td>tRNA gene prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>563</th>\n",
       "      <td>http://edamontology.org/operation_0465</td>\n",
       "      <td>siRNA binding specificity prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>564</th>\n",
       "      <td>http://edamontology.org/operation_0468</td>\n",
       "      <td>Protein secondary structure prediction (helices)</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>565</th>\n",
       "      <td>http://edamontology.org/operation_0469</td>\n",
       "      <td>Protein secondary structure prediction (turns)</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>566</th>\n",
       "      <td>http://edamontology.org/operation_0470</td>\n",
       "      <td>Protein secondary structure prediction (coils)</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>567</th>\n",
       "      <td>http://edamontology.org/operation_0471</td>\n",
       "      <td>Disulfide bond prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>568</th>\n",
       "      <td>http://edamontology.org/operation_0476</td>\n",
       "      <td>Ab initio structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>569</th>\n",
       "      <td>http://edamontology.org/operation_0477</td>\n",
       "      <td>Protein modelling</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>570</th>\n",
       "      <td>http://edamontology.org/operation_0479</td>\n",
       "      <td>Backbone modelling</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>571</th>\n",
       "      <td>http://edamontology.org/operation_0480</td>\n",
       "      <td>Side chain modelling</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>572</th>\n",
       "      <td>http://edamontology.org/operation_0481</td>\n",
       "      <td>Loop modelling</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>573</th>\n",
       "      <td>http://edamontology.org/operation_0483</td>\n",
       "      <td>RNA inverse folding</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>574</th>\n",
       "      <td>http://edamontology.org/operation_3095</td>\n",
       "      <td>Nucleic acid design</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>575</th>\n",
       "      <td>http://edamontology.org/operation_0484</td>\n",
       "      <td>SNP detection</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>576</th>\n",
       "      <td>http://edamontology.org/operation_0487</td>\n",
       "      <td>Haplotype mapping</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>577</th>\n",
       "      <td>http://edamontology.org/operation_0488</td>\n",
       "      <td>Linkage disequilibrium calculation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>578</th>\n",
       "      <td>http://edamontology.org/operation_0489</td>\n",
       "      <td>Genetic code prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>579</th>\n",
       "      <td>http://edamontology.org/operation_0490</td>\n",
       "      <td>Dot plot plotting</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>580</th>\n",
       "      <td>http://edamontology.org/operation_0491</td>\n",
       "      <td>Pairwise sequence alignment</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>581</th>\n",
       "      <td>http://edamontology.org/operation_0492</td>\n",
       "      <td>Multiple sequence alignment</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>582</th>\n",
       "      <td>http://edamontology.org/operation_0495</td>\n",
       "      <td>Local alignment</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>583</th>\n",
       "      <td>http://edamontology.org/operation_0496</td>\n",
       "      <td>Global alignment</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>584</th>\n",
       "      <td>http://edamontology.org/operation_0499</td>\n",
       "      <td>Tree-based sequence alignment</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>585</th>\n",
       "      <td>http://edamontology.org/operation_0523</td>\n",
       "      <td>Mapping assembly</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>586</th>\n",
       "      <td>http://edamontology.org/operation_0524</td>\n",
       "      <td>De-novo assembly</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>587</th>\n",
       "      <td>http://edamontology.org/operation_0525</td>\n",
       "      <td>Genome assembly</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>588</th>\n",
       "      <td>http://edamontology.org/operation_0526</td>\n",
       "      <td>EST assembly</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>589</th>\n",
       "      <td>http://edamontology.org/operation_0550</td>\n",
       "      <td>DNA substitution modelling</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>590</th>\n",
       "      <td>http://edamontology.org/operation_0560</td>\n",
       "      <td>DNA vaccine design</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>591</th>\n",
       "      <td>http://edamontology.org/operation_2008</td>\n",
       "      <td>siRNA duplex prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>592</th>\n",
       "      <td>http://edamontology.org/operation_2089</td>\n",
       "      <td>Sequence alignment refinement</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>593</th>\n",
       "      <td>http://edamontology.org/operation_2121</td>\n",
       "      <td>Sequence file editing</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>594</th>\n",
       "      <td>http://edamontology.org/operation_2233</td>\n",
       "      <td>Representative sequence identification</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>595</th>\n",
       "      <td>http://edamontology.org/operation_2284</td>\n",
       "      <td>Nucleic acid density plotting</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>596</th>\n",
       "      <td>http://edamontology.org/operation_2995</td>\n",
       "      <td>Sequence classification</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>597</th>\n",
       "      <td>http://edamontology.org/operation_2510</td>\n",
       "      <td>DNA back-translation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>598</th>\n",
       "      <td>http://edamontology.org/operation_2962</td>\n",
       "      <td>Codon usage bias calculation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>599</th>\n",
       "      <td>http://edamontology.org/operation_2964</td>\n",
       "      <td>Codon usage fraction calculation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>600</th>\n",
       "      <td>http://edamontology.org/operation_3180</td>\n",
       "      <td>Sequence assembly validation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>601</th>\n",
       "      <td>http://edamontology.org/operation_3218</td>\n",
       "      <td>Sequencing quality control</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>602</th>\n",
       "      <td>http://edamontology.org/operation_3182</td>\n",
       "      <td>Genome alignment</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>603</th>\n",
       "      <td>http://edamontology.org/operation_3183</td>\n",
       "      <td>Localised reassembly</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>604</th>\n",
       "      <td>http://edamontology.org/operation_3184</td>\n",
       "      <td>Sequence assembly visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>605</th>\n",
       "      <td>http://edamontology.org/operation_3185</td>\n",
       "      <td>Base-calling</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>606</th>\n",
       "      <td>http://edamontology.org/operation_3186</td>\n",
       "      <td>Bisulfite mapping</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>607</th>\n",
       "      <td>http://edamontology.org/operation_3204</td>\n",
       "      <td>Methylation analysis</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>608</th>\n",
       "      <td>http://edamontology.org/operation_3187</td>\n",
       "      <td>Sequence contamination filtering</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>609</th>\n",
       "      <td>http://edamontology.org/operation_3192</td>\n",
       "      <td>Sequence trimming</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>610</th>\n",
       "      <td>http://edamontology.org/operation_3194</td>\n",
       "      <td>Genome feature comparison</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>611</th>\n",
       "      <td>http://edamontology.org/operation_3198</td>\n",
       "      <td>Read mapping</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>612</th>\n",
       "      <td>http://edamontology.org/operation_3921</td>\n",
       "      <td>Sequence read processing</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>613</th>\n",
       "      <td>http://edamontology.org/operation_3199</td>\n",
       "      <td>Split read mapping</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>614</th>\n",
       "      <td>http://edamontology.org/operation_3200</td>\n",
       "      <td>DNA barcoding</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>615</th>\n",
       "      <td>http://edamontology.org/operation_3206</td>\n",
       "      <td>Whole genome methylation analysis</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>616</th>\n",
       "      <td>http://edamontology.org/operation_3207</td>\n",
       "      <td>Gene methylation analysis</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>617</th>\n",
       "      <td>http://edamontology.org/operation_3208</td>\n",
       "      <td>Genome visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>618</th>\n",
       "      <td>http://edamontology.org/operation_3209</td>\n",
       "      <td>Genome comparison</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>619</th>\n",
       "      <td>http://edamontology.org/operation_3211</td>\n",
       "      <td>Genome indexing</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>620</th>\n",
       "      <td>http://edamontology.org/operation_3216</td>\n",
       "      <td>Scaffolding</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>621</th>\n",
       "      <td>http://edamontology.org/operation_3217</td>\n",
       "      <td>Scaffold gap completion</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>622</th>\n",
       "      <td>http://edamontology.org/operation_3219</td>\n",
       "      <td>Read pre-processing</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>623</th>\n",
       "      <td>http://edamontology.org/operation_3221</td>\n",
       "      <td>Species frequency estimation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>624</th>\n",
       "      <td>http://edamontology.org/operation_3222</td>\n",
       "      <td>Peak calling</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>625</th>\n",
       "      <td>http://edamontology.org/operation_3225</td>\n",
       "      <td>Variant classification</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>626</th>\n",
       "      <td>http://edamontology.org/operation_3229</td>\n",
       "      <td>Exome assembly</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>627</th>\n",
       "      <td>http://edamontology.org/operation_3230</td>\n",
       "      <td>Read depth analysis</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>628</th>\n",
       "      <td>http://edamontology.org/operation_3237</td>\n",
       "      <td>Primer removal</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>629</th>\n",
       "      <td>http://edamontology.org/operation_3258</td>\n",
       "      <td>Transcriptome assembly</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>630</th>\n",
       "      <td>http://edamontology.org/operation_3459</td>\n",
       "      <td>Functional clustering</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>631</th>\n",
       "      <td>http://edamontology.org/operation_3460</td>\n",
       "      <td>Taxonomic classification</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>632</th>\n",
       "      <td>http://edamontology.org/operation_3461</td>\n",
       "      <td>Virulence prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>633</th>\n",
       "      <td>http://edamontology.org/operation_3472</td>\n",
       "      <td>k-mer counting</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>634</th>\n",
       "      <td>http://edamontology.org/operation_3481</td>\n",
       "      <td>Probabilistic sequence generation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>635</th>\n",
       "      <td>http://edamontology.org/operation_3482</td>\n",
       "      <td>Antimicrobial resistance prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>636</th>\n",
       "      <td>http://edamontology.org/operation_3563</td>\n",
       "      <td>RNA-seq read count analysis</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>637</th>\n",
       "      <td>http://edamontology.org/operation_3680</td>\n",
       "      <td>RNA-Seq analysis</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>638</th>\n",
       "      <td>http://edamontology.org/operation_3565</td>\n",
       "      <td>RNA-seq time series data analysis</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>639</th>\n",
       "      <td>http://edamontology.org/operation_3644</td>\n",
       "      <td>de Novo sequencing</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>640</th>\n",
       "      <td>http://edamontology.org/operation_3662</td>\n",
       "      <td>Ab-initio gene prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>641</th>\n",
       "      <td>http://edamontology.org/operation_3663</td>\n",
       "      <td>Homology-based gene prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>642</th>\n",
       "      <td>http://edamontology.org/operation_3675</td>\n",
       "      <td>Variant filtering</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>643</th>\n",
       "      <td>http://edamontology.org/operation_3677</td>\n",
       "      <td>Differential binding analysis</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>644</th>\n",
       "      <td>http://edamontology.org/operation_3730</td>\n",
       "      <td>Cross-assembly</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>645</th>\n",
       "      <td>http://edamontology.org/operation_3793</td>\n",
       "      <td>Read summarisation</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>646</th>\n",
       "      <td>http://edamontology.org/operation_3795</td>\n",
       "      <td>In vitro selection</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>647</th>\n",
       "      <td>http://edamontology.org/operation_3798</td>\n",
       "      <td>Read binning</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>648</th>\n",
       "      <td>http://edamontology.org/operation_3809</td>\n",
       "      <td>DMR identification</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>649</th>\n",
       "      <td>http://edamontology.org/operation_3894</td>\n",
       "      <td>DNA profiling</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>650</th>\n",
       "      <td>http://edamontology.org/operation_3896</td>\n",
       "      <td>Active site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>651</th>\n",
       "      <td>http://edamontology.org/operation_3897</td>\n",
       "      <td>Ligand-binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>652</th>\n",
       "      <td>http://edamontology.org/operation_3898</td>\n",
       "      <td>Metal-binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>653</th>\n",
       "      <td>http://edamontology.org/operation_3902</td>\n",
       "      <td>RNA binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>654</th>\n",
       "      <td>http://edamontology.org/operation_3903</td>\n",
       "      <td>DNA binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>655</th>\n",
       "      <td>http://edamontology.org/operation_3904</td>\n",
       "      <td>Protein disorder prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>656</th>\n",
       "      <td>http://edamontology.org/operation_3919</td>\n",
       "      <td>Methylation calling</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>657</th>\n",
       "      <td>http://edamontology.org/operation_3920</td>\n",
       "      <td>DNA testing</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>658</th>\n",
       "      <td>http://edamontology.org/operation_3933</td>\n",
       "      <td>Demultiplexing</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>659</th>\n",
       "      <td>http://edamontology.org/operation_3946</td>\n",
       "      <td>Ecological modelling</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>660</th>\n",
       "      <td>http://edamontology.org/operation_3950</td>\n",
       "      <td>Selection detection</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>661</th>\n",
       "      <td>http://edamontology.org/operation_3968</td>\n",
       "      <td>Adhesin prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>662</th>\n",
       "      <td>http://edamontology.org/operation_4032</td>\n",
       "      <td>DNA modification prediction</td>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>663</th>\n",
       "      <td>http://edamontology.org/operation_2406</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>664</th>\n",
       "      <td>http://edamontology.org/operation_0245</td>\n",
       "      <td>Structural motif discovery</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>665</th>\n",
       "      <td>http://edamontology.org/operation_0246</td>\n",
       "      <td>Protein domain recognition</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>666</th>\n",
       "      <td>http://edamontology.org/operation_0247</td>\n",
       "      <td>Protein architecture analysis</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>667</th>\n",
       "      <td>http://edamontology.org/operation_0267</td>\n",
       "      <td>Protein secondary structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>668</th>\n",
       "      <td>http://edamontology.org/operation_2416</td>\n",
       "      <td>Protein secondary structure analysis</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>669</th>\n",
       "      <td>http://edamontology.org/operation_0268</td>\n",
       "      <td>Protein super-secondary structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>670</th>\n",
       "      <td>http://edamontology.org/operation_0269</td>\n",
       "      <td>Transmembrane protein prediction</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>671</th>\n",
       "      <td>http://edamontology.org/operation_0474</td>\n",
       "      <td>Protein structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>672</th>\n",
       "      <td>http://edamontology.org/operation_0303</td>\n",
       "      <td>Fold recognition</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>673</th>\n",
       "      <td>http://edamontology.org/operation_0302</td>\n",
       "      <td>Protein threading</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>674</th>\n",
       "      <td>http://edamontology.org/operation_0320</td>\n",
       "      <td>Protein structure assignment</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>675</th>\n",
       "      <td>http://edamontology.org/operation_0321</td>\n",
       "      <td>Protein structure validation</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>676</th>\n",
       "      <td>http://edamontology.org/operation_2415</td>\n",
       "      <td>Protein folding analysis</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>677</th>\n",
       "      <td>http://edamontology.org/operation_0390</td>\n",
       "      <td>Protein peeling</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>678</th>\n",
       "      <td>http://edamontology.org/operation_1844</td>\n",
       "      <td>Protein geometry validation</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>679</th>\n",
       "      <td>http://edamontology.org/operation_0468</td>\n",
       "      <td>Protein secondary structure prediction (helices)</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>680</th>\n",
       "      <td>http://edamontology.org/operation_0469</td>\n",
       "      <td>Protein secondary structure prediction (turns)</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>681</th>\n",
       "      <td>http://edamontology.org/operation_0470</td>\n",
       "      <td>Protein secondary structure prediction (coils)</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>682</th>\n",
       "      <td>http://edamontology.org/operation_0476</td>\n",
       "      <td>Ab initio structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>683</th>\n",
       "      <td>http://edamontology.org/operation_0477</td>\n",
       "      <td>Protein modelling</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>684</th>\n",
       "      <td>http://edamontology.org/operation_0479</td>\n",
       "      <td>Backbone modelling</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>685</th>\n",
       "      <td>http://edamontology.org/operation_0480</td>\n",
       "      <td>Side chain modelling</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>686</th>\n",
       "      <td>http://edamontology.org/operation_0481</td>\n",
       "      <td>Loop modelling</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>687</th>\n",
       "      <td>http://edamontology.org/operation_2488</td>\n",
       "      <td>Protein secondary structure comparison</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>688</th>\n",
       "      <td>http://edamontology.org/operation_1829</td>\n",
       "      <td>Cysteine bridge detection</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>689</th>\n",
       "      <td>http://edamontology.org/operation_1850</td>\n",
       "      <td>Protein cysteine and disulfide bond assignment</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>690</th>\n",
       "      <td>http://edamontology.org/operation_1830</td>\n",
       "      <td>Free cysteine detection</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>691</th>\n",
       "      <td>http://edamontology.org/operation_1831</td>\n",
       "      <td>Metal-bound cysteine detection</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>692</th>\n",
       "      <td>http://edamontology.org/operation_1836</td>\n",
       "      <td>Residue bump detection</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>693</th>\n",
       "      <td>http://edamontology.org/operation_1843</td>\n",
       "      <td>Residue packing validation</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>694</th>\n",
       "      <td>http://edamontology.org/operation_2474</td>\n",
       "      <td>Protein architecture comparison</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>695</th>\n",
       "      <td>http://edamontology.org/operation_2475</td>\n",
       "      <td>Protein architecture recognition</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>696</th>\n",
       "      <td>http://edamontology.org/operation_2487</td>\n",
       "      <td>Protein structure comparison</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>697</th>\n",
       "      <td>http://edamontology.org/operation_3560</td>\n",
       "      <td>Maximum occurence analysis</td>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>698</th>\n",
       "      <td>http://edamontology.org/operation_0224</td>\n",
       "      <td>Query and retrieval</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>699</th>\n",
       "      <td>http://edamontology.org/operation_2409</td>\n",
       "      <td>Data handling</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>700</th>\n",
       "      <td>http://edamontology.org/operation_2422</td>\n",
       "      <td>Data retrieval</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>701</th>\n",
       "      <td>http://edamontology.org/operation_0231</td>\n",
       "      <td>Sequence editing</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>702</th>\n",
       "      <td>http://edamontology.org/operation_3096</td>\n",
       "      <td>Editing</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>703</th>\n",
       "      <td>http://edamontology.org/operation_0232</td>\n",
       "      <td>Sequence merging</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>704</th>\n",
       "      <td>http://edamontology.org/operation_0233</td>\n",
       "      <td>Sequence conversion</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>705</th>\n",
       "      <td>http://edamontology.org/operation_0260</td>\n",
       "      <td>Sequence alignment conversion</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>706</th>\n",
       "      <td>http://edamontology.org/operation_3081</td>\n",
       "      <td>Sequence alignment editing</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>707</th>\n",
       "      <td>http://edamontology.org/operation_0305</td>\n",
       "      <td>Literature search</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>708</th>\n",
       "      <td>http://edamontology.org/operation_2421</td>\n",
       "      <td>Database search</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>709</th>\n",
       "      <td>http://edamontology.org/operation_0326</td>\n",
       "      <td>Phylogenetic tree editing</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>710</th>\n",
       "      <td>http://edamontology.org/operation_0335</td>\n",
       "      <td>Formatting</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>711</th>\n",
       "      <td>http://edamontology.org/operation_0338</td>\n",
       "      <td>Sequence database search</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>712</th>\n",
       "      <td>http://edamontology.org/operation_0339</td>\n",
       "      <td>Structure database search</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>713</th>\n",
       "      <td>http://edamontology.org/operation_0346</td>\n",
       "      <td>Sequence similarity search</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>714</th>\n",
       "      <td>http://edamontology.org/operation_0349</td>\n",
       "      <td>Sequence database search (by property)</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>715</th>\n",
       "      <td>http://edamontology.org/operation_0360</td>\n",
       "      <td>Structural similarity search</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>716</th>\n",
       "      <td>http://edamontology.org/operation_0367</td>\n",
       "      <td>Sequence mutation and randomisation</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>717</th>\n",
       "      <td>http://edamontology.org/operation_0368</td>\n",
       "      <td>Sequence masking</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>718</th>\n",
       "      <td>http://edamontology.org/operation_0369</td>\n",
       "      <td>Sequence cutting</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>719</th>\n",
       "      <td>http://edamontology.org/operation_0370</td>\n",
       "      <td>Restriction site creation</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>720</th>\n",
       "      <td>http://edamontology.org/operation_0371</td>\n",
       "      <td>DNA translation</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>721</th>\n",
       "      <td>http://edamontology.org/operation_0372</td>\n",
       "      <td>DNA transcription</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>722</th>\n",
       "      <td>http://edamontology.org/operation_3431</td>\n",
       "      <td>Deposition</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>723</th>\n",
       "      <td>http://edamontology.org/operation_1812</td>\n",
       "      <td>Parsing</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>724</th>\n",
       "      <td>http://edamontology.org/operation_2121</td>\n",
       "      <td>Sequence file editing</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>725</th>\n",
       "      <td>http://edamontology.org/operation_2510</td>\n",
       "      <td>DNA back-translation</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>726</th>\n",
       "      <td>http://edamontology.org/operation_3080</td>\n",
       "      <td>Structure editing</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>727</th>\n",
       "      <td>http://edamontology.org/operation_3192</td>\n",
       "      <td>Sequence trimming</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>728</th>\n",
       "      <td>http://edamontology.org/operation_3237</td>\n",
       "      <td>Primer removal</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>729</th>\n",
       "      <td>http://edamontology.org/operation_3283</td>\n",
       "      <td>Anonymisation</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>730</th>\n",
       "      <td>http://edamontology.org/operation_3357</td>\n",
       "      <td>Format detection</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>731</th>\n",
       "      <td>http://edamontology.org/operation_3359</td>\n",
       "      <td>Splitting</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>732</th>\n",
       "      <td>http://edamontology.org/operation_3436</td>\n",
       "      <td>Aggregation</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>733</th>\n",
       "      <td>http://edamontology.org/operation_3646</td>\n",
       "      <td>Peptide database search</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>734</th>\n",
       "      <td>http://edamontology.org/operation_3647</td>\n",
       "      <td>Blind peptide database search</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>735</th>\n",
       "      <td>http://edamontology.org/operation_3649</td>\n",
       "      <td>Target-Decoy</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>736</th>\n",
       "      <td>http://edamontology.org/operation_3695</td>\n",
       "      <td>Filtering</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>737</th>\n",
       "      <td>http://edamontology.org/operation_3802</td>\n",
       "      <td>Sorting</td>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>738</th>\n",
       "      <td>http://edamontology.org/operation_2415</td>\n",
       "      <td>Protein folding analysis</td>\n",
       "      <td>http://edamontology.org/topic_0130</td>\n",
       "      <td>Protein folding, stability and design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>739</th>\n",
       "      <td>http://edamontology.org/operation_2419</td>\n",
       "      <td>Primer and probe design</td>\n",
       "      <td>http://edamontology.org/topic_0632</td>\n",
       "      <td>Probes and primers</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>740</th>\n",
       "      <td>http://edamontology.org/operation_0264</td>\n",
       "      <td>Alternative splicing prediction</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>741</th>\n",
       "      <td>http://edamontology.org/operation_2499</td>\n",
       "      <td>Splicing analysis</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>742</th>\n",
       "      <td>http://edamontology.org/operation_0279</td>\n",
       "      <td>Nucleic acid folding analysis</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>743</th>\n",
       "      <td>http://edamontology.org/operation_2426</td>\n",
       "      <td>Modelling and simulation</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>744</th>\n",
       "      <td>http://edamontology.org/operation_2415</td>\n",
       "      <td>Protein folding analysis</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>745</th>\n",
       "      <td>http://edamontology.org/operation_0433</td>\n",
       "      <td>Splice site prediction</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>746</th>\n",
       "      <td>http://edamontology.org/operation_0550</td>\n",
       "      <td>DNA substitution modelling</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>747</th>\n",
       "      <td>http://edamontology.org/operation_2476</td>\n",
       "      <td>Molecular dynamics</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>748</th>\n",
       "      <td>http://edamontology.org/operation_3566</td>\n",
       "      <td>Simulated gene expression data generation</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>749</th>\n",
       "      <td>http://edamontology.org/operation_3893</td>\n",
       "      <td>Forcefield parameterisation</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>750</th>\n",
       "      <td>http://edamontology.org/operation_3946</td>\n",
       "      <td>Ecological modelling</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>751</th>\n",
       "      <td>http://edamontology.org/operation_4033</td>\n",
       "      <td>Disease transmission analysis</td>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>752</th>\n",
       "      <td>http://edamontology.org/operation_2436</td>\n",
       "      <td>Gene-set enrichment analysis</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>753</th>\n",
       "      <td>http://edamontology.org/operation_0278</td>\n",
       "      <td>RNA secondary structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0097</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>754</th>\n",
       "      <td>http://edamontology.org/operation_2439</td>\n",
       "      <td>RNA secondary structure analysis</td>\n",
       "      <td>http://edamontology.org/topic_0097</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>755</th>\n",
       "      <td>http://edamontology.org/operation_0502</td>\n",
       "      <td>RNA secondary structure alignment</td>\n",
       "      <td>http://edamontology.org/topic_0097</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>756</th>\n",
       "      <td>http://edamontology.org/operation_3469</td>\n",
       "      <td>RNA structure covariance model generation</td>\n",
       "      <td>http://edamontology.org/topic_0097</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>757</th>\n",
       "      <td>http://edamontology.org/operation_0438</td>\n",
       "      <td>Transcriptional regulatory element prediction</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>758</th>\n",
       "      <td>http://edamontology.org/operation_2499</td>\n",
       "      <td>Splicing analysis</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>759</th>\n",
       "      <td>http://edamontology.org/operation_2454</td>\n",
       "      <td>Gene prediction</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>760</th>\n",
       "      <td>http://edamontology.org/operation_0435</td>\n",
       "      <td>Operon prediction</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>761</th>\n",
       "      <td>http://edamontology.org/operation_0436</td>\n",
       "      <td>Coding region prediction</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>762</th>\n",
       "      <td>http://edamontology.org/operation_0437</td>\n",
       "      <td>SECIS element prediction</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>763</th>\n",
       "      <td>http://edamontology.org/operation_0439</td>\n",
       "      <td>Translation initiation site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>764</th>\n",
       "      <td>http://edamontology.org/operation_0440</td>\n",
       "      <td>Promoter prediction</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>765</th>\n",
       "      <td>http://edamontology.org/operation_0441</td>\n",
       "      <td>cis-regulatory element prediction</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>766</th>\n",
       "      <td>http://edamontology.org/operation_0443</td>\n",
       "      <td>trans-regulatory element prediction</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>767</th>\n",
       "      <td>http://edamontology.org/operation_0444</td>\n",
       "      <td>S/MAR prediction</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>768</th>\n",
       "      <td>http://edamontology.org/operation_0445</td>\n",
       "      <td>Transcription factor binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>769</th>\n",
       "      <td>http://edamontology.org/operation_0463</td>\n",
       "      <td>miRNA target prediction</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>770</th>\n",
       "      <td>http://edamontology.org/operation_0464</td>\n",
       "      <td>tRNA gene prediction</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>771</th>\n",
       "      <td>http://edamontology.org/operation_0465</td>\n",
       "      <td>siRNA binding specificity prediction</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>772</th>\n",
       "      <td>http://edamontology.org/operation_2008</td>\n",
       "      <td>siRNA duplex prediction</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>773</th>\n",
       "      <td>http://edamontology.org/operation_3662</td>\n",
       "      <td>Ab-initio gene prediction</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>774</th>\n",
       "      <td>http://edamontology.org/operation_3663</td>\n",
       "      <td>Homology-based gene prediction</td>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>775</th>\n",
       "      <td>http://edamontology.org/operation_2464</td>\n",
       "      <td>Protein-protein binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>776</th>\n",
       "      <td>http://edamontology.org/operation_2476</td>\n",
       "      <td>Molecular dynamics</td>\n",
       "      <td>http://edamontology.org/topic_0082</td>\n",
       "      <td>Structure prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>777</th>\n",
       "      <td>http://edamontology.org/operation_2476</td>\n",
       "      <td>Molecular dynamics</td>\n",
       "      <td>http://edamontology.org/topic_0176</td>\n",
       "      <td>Molecular dynamics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>778</th>\n",
       "      <td>http://edamontology.org/operation_0438</td>\n",
       "      <td>Transcriptional regulatory element prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>779</th>\n",
       "      <td>http://edamontology.org/operation_0264</td>\n",
       "      <td>Alternative splicing prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>780</th>\n",
       "      <td>http://edamontology.org/operation_2499</td>\n",
       "      <td>Splicing analysis</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>781</th>\n",
       "      <td>http://edamontology.org/operation_0265</td>\n",
       "      <td>Frameshift detection</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>782</th>\n",
       "      <td>http://edamontology.org/operation_3227</td>\n",
       "      <td>Variant calling</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>783</th>\n",
       "      <td>http://edamontology.org/operation_0266</td>\n",
       "      <td>Vector sequence detection</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>784</th>\n",
       "      <td>http://edamontology.org/operation_0415</td>\n",
       "      <td>Nucleic acid feature detection</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>785</th>\n",
       "      <td>http://edamontology.org/operation_0278</td>\n",
       "      <td>RNA secondary structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>786</th>\n",
       "      <td>http://edamontology.org/operation_0282</td>\n",
       "      <td>Genetic mapping</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>787</th>\n",
       "      <td>http://edamontology.org/operation_0283</td>\n",
       "      <td>Linkage analysis</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>788</th>\n",
       "      <td>http://edamontology.org/operation_2478</td>\n",
       "      <td>Nucleic acid sequence analysis</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>789</th>\n",
       "      <td>http://edamontology.org/operation_0284</td>\n",
       "      <td>Codon usage table generation</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>790</th>\n",
       "      <td>http://edamontology.org/operation_0286</td>\n",
       "      <td>Codon usage analysis</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>791</th>\n",
       "      <td>http://edamontology.org/operation_0285</td>\n",
       "      <td>Codon usage table comparison</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>792</th>\n",
       "      <td>http://edamontology.org/operation_0287</td>\n",
       "      <td>Base position variability plotting</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>793</th>\n",
       "      <td>http://edamontology.org/operation_0307</td>\n",
       "      <td>Virtual PCR</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>794</th>\n",
       "      <td>http://edamontology.org/operation_0308</td>\n",
       "      <td>PCR primer design</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>795</th>\n",
       "      <td>http://edamontology.org/operation_2419</td>\n",
       "      <td>Primer and probe design</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>796</th>\n",
       "      <td>http://edamontology.org/operation_0309</td>\n",
       "      <td>Microarray probe design</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>797</th>\n",
       "      <td>http://edamontology.org/operation_0310</td>\n",
       "      <td>Sequence assembly</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>798</th>\n",
       "      <td>http://edamontology.org/operation_0362</td>\n",
       "      <td>Genome annotation</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>799</th>\n",
       "      <td>http://edamontology.org/operation_3918</td>\n",
       "      <td>Genome analysis</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>800</th>\n",
       "      <td>http://edamontology.org/operation_0379</td>\n",
       "      <td>Repeat sequence detection</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>801</th>\n",
       "      <td>http://edamontology.org/operation_2454</td>\n",
       "      <td>Gene prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>802</th>\n",
       "      <td>http://edamontology.org/operation_0427</td>\n",
       "      <td>Transposon prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>803</th>\n",
       "      <td>http://edamontology.org/operation_0428</td>\n",
       "      <td>PolyA signal detection</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>804</th>\n",
       "      <td>http://edamontology.org/operation_0429</td>\n",
       "      <td>Quadruplex formation site detection</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>805</th>\n",
       "      <td>http://edamontology.org/operation_0430</td>\n",
       "      <td>CpG island and isochore detection</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>806</th>\n",
       "      <td>http://edamontology.org/operation_0431</td>\n",
       "      <td>Restriction site recognition</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>807</th>\n",
       "      <td>http://edamontology.org/operation_0432</td>\n",
       "      <td>Nucleosome position prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>808</th>\n",
       "      <td>http://edamontology.org/operation_0433</td>\n",
       "      <td>Splice site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>809</th>\n",
       "      <td>http://edamontology.org/operation_0435</td>\n",
       "      <td>Operon prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>810</th>\n",
       "      <td>http://edamontology.org/operation_0436</td>\n",
       "      <td>Coding region prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>811</th>\n",
       "      <td>http://edamontology.org/operation_0437</td>\n",
       "      <td>SECIS element prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>812</th>\n",
       "      <td>http://edamontology.org/operation_0439</td>\n",
       "      <td>Translation initiation site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>813</th>\n",
       "      <td>http://edamontology.org/operation_0440</td>\n",
       "      <td>Promoter prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>814</th>\n",
       "      <td>http://edamontology.org/operation_0441</td>\n",
       "      <td>cis-regulatory element prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>815</th>\n",
       "      <td>http://edamontology.org/operation_0443</td>\n",
       "      <td>trans-regulatory element prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>816</th>\n",
       "      <td>http://edamontology.org/operation_0444</td>\n",
       "      <td>S/MAR prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>817</th>\n",
       "      <td>http://edamontology.org/operation_0445</td>\n",
       "      <td>Transcription factor binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>818</th>\n",
       "      <td>http://edamontology.org/operation_0446</td>\n",
       "      <td>Exonic splicing enhancer prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>819</th>\n",
       "      <td>http://edamontology.org/operation_0450</td>\n",
       "      <td>Chimera detection</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>820</th>\n",
       "      <td>http://edamontology.org/operation_0451</td>\n",
       "      <td>Recombination detection</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>821</th>\n",
       "      <td>http://edamontology.org/operation_0452</td>\n",
       "      <td>Indel detection</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>822</th>\n",
       "      <td>http://edamontology.org/operation_0463</td>\n",
       "      <td>miRNA target prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>823</th>\n",
       "      <td>http://edamontology.org/operation_0464</td>\n",
       "      <td>tRNA gene prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>824</th>\n",
       "      <td>http://edamontology.org/operation_0465</td>\n",
       "      <td>siRNA binding specificity prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>825</th>\n",
       "      <td>http://edamontology.org/operation_0483</td>\n",
       "      <td>RNA inverse folding</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>826</th>\n",
       "      <td>http://edamontology.org/operation_3095</td>\n",
       "      <td>Nucleic acid design</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>827</th>\n",
       "      <td>http://edamontology.org/operation_0484</td>\n",
       "      <td>SNP detection</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>828</th>\n",
       "      <td>http://edamontology.org/operation_0487</td>\n",
       "      <td>Haplotype mapping</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>829</th>\n",
       "      <td>http://edamontology.org/operation_0488</td>\n",
       "      <td>Linkage disequilibrium calculation</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>830</th>\n",
       "      <td>http://edamontology.org/operation_0489</td>\n",
       "      <td>Genetic code prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>831</th>\n",
       "      <td>http://edamontology.org/operation_0523</td>\n",
       "      <td>Mapping assembly</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>832</th>\n",
       "      <td>http://edamontology.org/operation_0524</td>\n",
       "      <td>De-novo assembly</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>833</th>\n",
       "      <td>http://edamontology.org/operation_0525</td>\n",
       "      <td>Genome assembly</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>834</th>\n",
       "      <td>http://edamontology.org/operation_0526</td>\n",
       "      <td>EST assembly</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>835</th>\n",
       "      <td>http://edamontology.org/operation_0550</td>\n",
       "      <td>DNA substitution modelling</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>836</th>\n",
       "      <td>http://edamontology.org/operation_0560</td>\n",
       "      <td>DNA vaccine design</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>837</th>\n",
       "      <td>http://edamontology.org/operation_2008</td>\n",
       "      <td>siRNA duplex prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>838</th>\n",
       "      <td>http://edamontology.org/operation_2962</td>\n",
       "      <td>Codon usage bias calculation</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>839</th>\n",
       "      <td>http://edamontology.org/operation_2964</td>\n",
       "      <td>Codon usage fraction calculation</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>840</th>\n",
       "      <td>http://edamontology.org/operation_3180</td>\n",
       "      <td>Sequence assembly validation</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>841</th>\n",
       "      <td>http://edamontology.org/operation_3218</td>\n",
       "      <td>Sequencing quality control</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>842</th>\n",
       "      <td>http://edamontology.org/operation_3182</td>\n",
       "      <td>Genome alignment</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>843</th>\n",
       "      <td>http://edamontology.org/operation_3183</td>\n",
       "      <td>Localised reassembly</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>844</th>\n",
       "      <td>http://edamontology.org/operation_3186</td>\n",
       "      <td>Bisulfite mapping</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>845</th>\n",
       "      <td>http://edamontology.org/operation_3204</td>\n",
       "      <td>Methylation analysis</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>846</th>\n",
       "      <td>http://edamontology.org/operation_3187</td>\n",
       "      <td>Sequence contamination filtering</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>847</th>\n",
       "      <td>http://edamontology.org/operation_3194</td>\n",
       "      <td>Genome feature comparison</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>848</th>\n",
       "      <td>http://edamontology.org/operation_3198</td>\n",
       "      <td>Read mapping</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>849</th>\n",
       "      <td>http://edamontology.org/operation_3921</td>\n",
       "      <td>Sequence read processing</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>850</th>\n",
       "      <td>http://edamontology.org/operation_3199</td>\n",
       "      <td>Split read mapping</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>851</th>\n",
       "      <td>http://edamontology.org/operation_3200</td>\n",
       "      <td>DNA barcoding</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>852</th>\n",
       "      <td>http://edamontology.org/operation_3206</td>\n",
       "      <td>Whole genome methylation analysis</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>853</th>\n",
       "      <td>http://edamontology.org/operation_3207</td>\n",
       "      <td>Gene methylation analysis</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>854</th>\n",
       "      <td>http://edamontology.org/operation_3208</td>\n",
       "      <td>Genome visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>855</th>\n",
       "      <td>http://edamontology.org/operation_3209</td>\n",
       "      <td>Genome comparison</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>856</th>\n",
       "      <td>http://edamontology.org/operation_3211</td>\n",
       "      <td>Genome indexing</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>857</th>\n",
       "      <td>http://edamontology.org/operation_3216</td>\n",
       "      <td>Scaffolding</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>858</th>\n",
       "      <td>http://edamontology.org/operation_3217</td>\n",
       "      <td>Scaffold gap completion</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>859</th>\n",
       "      <td>http://edamontology.org/operation_3219</td>\n",
       "      <td>Read pre-processing</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>860</th>\n",
       "      <td>http://edamontology.org/operation_3221</td>\n",
       "      <td>Species frequency estimation</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>861</th>\n",
       "      <td>http://edamontology.org/operation_3222</td>\n",
       "      <td>Peak calling</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>862</th>\n",
       "      <td>http://edamontology.org/operation_3229</td>\n",
       "      <td>Exome assembly</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>863</th>\n",
       "      <td>http://edamontology.org/operation_3230</td>\n",
       "      <td>Read depth analysis</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>864</th>\n",
       "      <td>http://edamontology.org/operation_3258</td>\n",
       "      <td>Transcriptome assembly</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>865</th>\n",
       "      <td>http://edamontology.org/operation_3563</td>\n",
       "      <td>RNA-seq read count analysis</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>866</th>\n",
       "      <td>http://edamontology.org/operation_3680</td>\n",
       "      <td>RNA-Seq analysis</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>867</th>\n",
       "      <td>http://edamontology.org/operation_3565</td>\n",
       "      <td>RNA-seq time series data analysis</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>868</th>\n",
       "      <td>http://edamontology.org/operation_3662</td>\n",
       "      <td>Ab-initio gene prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>869</th>\n",
       "      <td>http://edamontology.org/operation_3663</td>\n",
       "      <td>Homology-based gene prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>870</th>\n",
       "      <td>http://edamontology.org/operation_3675</td>\n",
       "      <td>Variant filtering</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>871</th>\n",
       "      <td>http://edamontology.org/operation_3677</td>\n",
       "      <td>Differential binding analysis</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>872</th>\n",
       "      <td>http://edamontology.org/operation_3730</td>\n",
       "      <td>Cross-assembly</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>873</th>\n",
       "      <td>http://edamontology.org/operation_3793</td>\n",
       "      <td>Read summarisation</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>874</th>\n",
       "      <td>http://edamontology.org/operation_3795</td>\n",
       "      <td>In vitro selection</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>875</th>\n",
       "      <td>http://edamontology.org/operation_3798</td>\n",
       "      <td>Read binning</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>876</th>\n",
       "      <td>http://edamontology.org/operation_3809</td>\n",
       "      <td>DMR identification</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>877</th>\n",
       "      <td>http://edamontology.org/operation_3894</td>\n",
       "      <td>DNA profiling</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>878</th>\n",
       "      <td>http://edamontology.org/operation_3919</td>\n",
       "      <td>Methylation calling</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>879</th>\n",
       "      <td>http://edamontology.org/operation_3920</td>\n",
       "      <td>DNA testing</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>880</th>\n",
       "      <td>http://edamontology.org/operation_3933</td>\n",
       "      <td>Demultiplexing</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>881</th>\n",
       "      <td>http://edamontology.org/operation_3946</td>\n",
       "      <td>Ecological modelling</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>882</th>\n",
       "      <td>http://edamontology.org/operation_3950</td>\n",
       "      <td>Selection detection</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>883</th>\n",
       "      <td>http://edamontology.org/operation_4032</td>\n",
       "      <td>DNA modification prediction</td>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>884</th>\n",
       "      <td>http://edamontology.org/operation_0245</td>\n",
       "      <td>Structural motif discovery</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>885</th>\n",
       "      <td>http://edamontology.org/operation_3092</td>\n",
       "      <td>Protein feature detection</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>886</th>\n",
       "      <td>http://edamontology.org/operation_0246</td>\n",
       "      <td>Protein domain recognition</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>887</th>\n",
       "      <td>http://edamontology.org/operation_0267</td>\n",
       "      <td>Protein secondary structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>888</th>\n",
       "      <td>http://edamontology.org/operation_2479</td>\n",
       "      <td>Protein sequence analysis</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>889</th>\n",
       "      <td>http://edamontology.org/operation_0268</td>\n",
       "      <td>Protein super-secondary structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>890</th>\n",
       "      <td>http://edamontology.org/operation_0269</td>\n",
       "      <td>Transmembrane protein prediction</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>891</th>\n",
       "      <td>http://edamontology.org/operation_0474</td>\n",
       "      <td>Protein structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>892</th>\n",
       "      <td>http://edamontology.org/operation_0272</td>\n",
       "      <td>Residue contact prediction</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>893</th>\n",
       "      <td>http://edamontology.org/operation_2464</td>\n",
       "      <td>Protein-protein binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>894</th>\n",
       "      <td>http://edamontology.org/operation_0303</td>\n",
       "      <td>Fold recognition</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>895</th>\n",
       "      <td>http://edamontology.org/operation_0302</td>\n",
       "      <td>Protein threading</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>896</th>\n",
       "      <td>http://edamontology.org/operation_0420</td>\n",
       "      <td>Nucleic acids-binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>897</th>\n",
       "      <td>http://edamontology.org/operation_2575</td>\n",
       "      <td>Binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>898</th>\n",
       "      <td>http://edamontology.org/operation_0390</td>\n",
       "      <td>Protein peeling</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>899</th>\n",
       "      <td>http://edamontology.org/operation_0418</td>\n",
       "      <td>Protein signal peptide detection</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>900</th>\n",
       "      <td>http://edamontology.org/operation_0416</td>\n",
       "      <td>Epitope mapping</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>901</th>\n",
       "      <td>http://edamontology.org/operation_0417</td>\n",
       "      <td>Post-translational modification site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>902</th>\n",
       "      <td>http://edamontology.org/operation_0422</td>\n",
       "      <td>Protein cleavage site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>903</th>\n",
       "      <td>http://edamontology.org/operation_0468</td>\n",
       "      <td>Protein secondary structure prediction (helices)</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>904</th>\n",
       "      <td>http://edamontology.org/operation_0469</td>\n",
       "      <td>Protein secondary structure prediction (turns)</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>905</th>\n",
       "      <td>http://edamontology.org/operation_0470</td>\n",
       "      <td>Protein secondary structure prediction (coils)</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>906</th>\n",
       "      <td>http://edamontology.org/operation_0471</td>\n",
       "      <td>Disulfide bond prediction</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>907</th>\n",
       "      <td>http://edamontology.org/operation_0476</td>\n",
       "      <td>Ab initio structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>908</th>\n",
       "      <td>http://edamontology.org/operation_0477</td>\n",
       "      <td>Protein modelling</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>909</th>\n",
       "      <td>http://edamontology.org/operation_0479</td>\n",
       "      <td>Backbone modelling</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>910</th>\n",
       "      <td>http://edamontology.org/operation_0480</td>\n",
       "      <td>Side chain modelling</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>911</th>\n",
       "      <td>http://edamontology.org/operation_0481</td>\n",
       "      <td>Loop modelling</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>912</th>\n",
       "      <td>http://edamontology.org/operation_3896</td>\n",
       "      <td>Active site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>913</th>\n",
       "      <td>http://edamontology.org/operation_3897</td>\n",
       "      <td>Ligand-binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>914</th>\n",
       "      <td>http://edamontology.org/operation_3898</td>\n",
       "      <td>Metal-binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>915</th>\n",
       "      <td>http://edamontology.org/operation_3902</td>\n",
       "      <td>RNA binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>916</th>\n",
       "      <td>http://edamontology.org/operation_3903</td>\n",
       "      <td>DNA binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>917</th>\n",
       "      <td>http://edamontology.org/operation_3904</td>\n",
       "      <td>Protein disorder prediction</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>918</th>\n",
       "      <td>http://edamontology.org/operation_3968</td>\n",
       "      <td>Adhesin prediction</td>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>919</th>\n",
       "      <td>http://edamontology.org/operation_2406</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>920</th>\n",
       "      <td>http://edamontology.org/operation_0245</td>\n",
       "      <td>Structural motif discovery</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>921</th>\n",
       "      <td>http://edamontology.org/operation_0246</td>\n",
       "      <td>Protein domain recognition</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>922</th>\n",
       "      <td>http://edamontology.org/operation_0247</td>\n",
       "      <td>Protein architecture analysis</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>923</th>\n",
       "      <td>http://edamontology.org/operation_0267</td>\n",
       "      <td>Protein secondary structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>924</th>\n",
       "      <td>http://edamontology.org/operation_2416</td>\n",
       "      <td>Protein secondary structure analysis</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>925</th>\n",
       "      <td>http://edamontology.org/operation_0268</td>\n",
       "      <td>Protein super-secondary structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>926</th>\n",
       "      <td>http://edamontology.org/operation_0269</td>\n",
       "      <td>Transmembrane protein prediction</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>927</th>\n",
       "      <td>http://edamontology.org/operation_2480</td>\n",
       "      <td>Structure analysis</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>928</th>\n",
       "      <td>http://edamontology.org/operation_0474</td>\n",
       "      <td>Protein structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>929</th>\n",
       "      <td>http://edamontology.org/operation_0475</td>\n",
       "      <td>Nucleic acid structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>930</th>\n",
       "      <td>http://edamontology.org/operation_0278</td>\n",
       "      <td>RNA secondary structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>931</th>\n",
       "      <td>http://edamontology.org/operation_2439</td>\n",
       "      <td>RNA secondary structure analysis</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>932</th>\n",
       "      <td>http://edamontology.org/operation_0279</td>\n",
       "      <td>Nucleic acid folding analysis</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>933</th>\n",
       "      <td>http://edamontology.org/operation_2481</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>934</th>\n",
       "      <td>http://edamontology.org/operation_0295</td>\n",
       "      <td>Structure alignment</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>935</th>\n",
       "      <td>http://edamontology.org/operation_2483</td>\n",
       "      <td>Structure comparison</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>936</th>\n",
       "      <td>http://edamontology.org/operation_0297</td>\n",
       "      <td>3D profile generation</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>937</th>\n",
       "      <td>http://edamontology.org/operation_0303</td>\n",
       "      <td>Fold recognition</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>938</th>\n",
       "      <td>http://edamontology.org/operation_0302</td>\n",
       "      <td>Protein threading</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>939</th>\n",
       "      <td>http://edamontology.org/operation_0319</td>\n",
       "      <td>Protein secondary structure assignment</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>940</th>\n",
       "      <td>http://edamontology.org/operation_0320</td>\n",
       "      <td>Protein structure assignment</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>941</th>\n",
       "      <td>http://edamontology.org/operation_0321</td>\n",
       "      <td>Protein structure validation</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>942</th>\n",
       "      <td>http://edamontology.org/operation_0322</td>\n",
       "      <td>Molecular model refinement</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>943</th>\n",
       "      <td>http://edamontology.org/operation_2415</td>\n",
       "      <td>Protein folding analysis</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>944</th>\n",
       "      <td>http://edamontology.org/operation_0384</td>\n",
       "      <td>Accessible surface calculation</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>945</th>\n",
       "      <td>http://edamontology.org/operation_3351</td>\n",
       "      <td>Molecular surface analysis</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>946</th>\n",
       "      <td>http://edamontology.org/operation_0387</td>\n",
       "      <td>Molecular surface calculation</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>947</th>\n",
       "      <td>http://edamontology.org/operation_0390</td>\n",
       "      <td>Protein peeling</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>948</th>\n",
       "      <td>http://edamontology.org/operation_1844</td>\n",
       "      <td>Protein geometry validation</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>949</th>\n",
       "      <td>http://edamontology.org/operation_0468</td>\n",
       "      <td>Protein secondary structure prediction (helices)</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>950</th>\n",
       "      <td>http://edamontology.org/operation_0469</td>\n",
       "      <td>Protein secondary structure prediction (turns)</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>951</th>\n",
       "      <td>http://edamontology.org/operation_0470</td>\n",
       "      <td>Protein secondary structure prediction (coils)</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>952</th>\n",
       "      <td>http://edamontology.org/operation_0476</td>\n",
       "      <td>Ab initio structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>953</th>\n",
       "      <td>http://edamontology.org/operation_0477</td>\n",
       "      <td>Protein modelling</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>954</th>\n",
       "      <td>http://edamontology.org/operation_0478</td>\n",
       "      <td>Molecular docking</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>955</th>\n",
       "      <td>http://edamontology.org/operation_0479</td>\n",
       "      <td>Backbone modelling</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>956</th>\n",
       "      <td>http://edamontology.org/operation_0480</td>\n",
       "      <td>Side chain modelling</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>957</th>\n",
       "      <td>http://edamontology.org/operation_0481</td>\n",
       "      <td>Loop modelling</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>958</th>\n",
       "      <td>http://edamontology.org/operation_0482</td>\n",
       "      <td>Protein-ligand docking</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>959</th>\n",
       "      <td>http://edamontology.org/operation_2488</td>\n",
       "      <td>Protein secondary structure comparison</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>960</th>\n",
       "      <td>http://edamontology.org/operation_0502</td>\n",
       "      <td>RNA secondary structure alignment</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>961</th>\n",
       "      <td>http://edamontology.org/operation_0503</td>\n",
       "      <td>Pairwise structure alignment</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>962</th>\n",
       "      <td>http://edamontology.org/operation_0504</td>\n",
       "      <td>Multiple structure alignment</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>963</th>\n",
       "      <td>http://edamontology.org/operation_0509</td>\n",
       "      <td>Local structure alignment</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>964</th>\n",
       "      <td>http://edamontology.org/operation_0510</td>\n",
       "      <td>Global structure alignment</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>965</th>\n",
       "      <td>http://edamontology.org/operation_0570</td>\n",
       "      <td>Structure visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>966</th>\n",
       "      <td>http://edamontology.org/operation_1816</td>\n",
       "      <td>Surface rendering</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>967</th>\n",
       "      <td>http://edamontology.org/operation_1829</td>\n",
       "      <td>Cysteine bridge detection</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>968</th>\n",
       "      <td>http://edamontology.org/operation_1850</td>\n",
       "      <td>Protein cysteine and disulfide bond assignment</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>969</th>\n",
       "      <td>http://edamontology.org/operation_1830</td>\n",
       "      <td>Free cysteine detection</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>970</th>\n",
       "      <td>http://edamontology.org/operation_1831</td>\n",
       "      <td>Metal-bound cysteine detection</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>971</th>\n",
       "      <td>http://edamontology.org/operation_1836</td>\n",
       "      <td>Residue bump detection</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>972</th>\n",
       "      <td>http://edamontology.org/operation_1843</td>\n",
       "      <td>Residue packing validation</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>973</th>\n",
       "      <td>http://edamontology.org/operation_2241</td>\n",
       "      <td>Transmembrane protein visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>974</th>\n",
       "      <td>http://edamontology.org/operation_2441</td>\n",
       "      <td>RNA structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>975</th>\n",
       "      <td>http://edamontology.org/operation_2442</td>\n",
       "      <td>DNA structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>976</th>\n",
       "      <td>http://edamontology.org/operation_2474</td>\n",
       "      <td>Protein architecture comparison</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>977</th>\n",
       "      <td>http://edamontology.org/operation_2475</td>\n",
       "      <td>Protein architecture recognition</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>978</th>\n",
       "      <td>http://edamontology.org/operation_2996</td>\n",
       "      <td>Structure classification</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>979</th>\n",
       "      <td>http://edamontology.org/operation_2476</td>\n",
       "      <td>Molecular dynamics</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>980</th>\n",
       "      <td>http://edamontology.org/operation_2485</td>\n",
       "      <td>Helical wheel drawing</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>981</th>\n",
       "      <td>http://edamontology.org/operation_2486</td>\n",
       "      <td>Topology diagram drawing</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>982</th>\n",
       "      <td>http://edamontology.org/operation_2487</td>\n",
       "      <td>Protein structure comparison</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>983</th>\n",
       "      <td>http://edamontology.org/operation_2518</td>\n",
       "      <td>Nucleic acid structure comparison</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>984</th>\n",
       "      <td>http://edamontology.org/operation_3445</td>\n",
       "      <td>Diffraction data analysis</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>985</th>\n",
       "      <td>http://edamontology.org/operation_3447</td>\n",
       "      <td>Diffraction data reduction</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>986</th>\n",
       "      <td>http://edamontology.org/operation_3453</td>\n",
       "      <td>Diffraction data integration</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>987</th>\n",
       "      <td>http://edamontology.org/operation_3454</td>\n",
       "      <td>Phasing</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>988</th>\n",
       "      <td>http://edamontology.org/operation_3455</td>\n",
       "      <td>Molecular replacement</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>989</th>\n",
       "      <td>http://edamontology.org/operation_3456</td>\n",
       "      <td>Rigid body refinement</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>990</th>\n",
       "      <td>http://edamontology.org/operation_3457</td>\n",
       "      <td>Single particle analysis</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>991</th>\n",
       "      <td>http://edamontology.org/operation_3458</td>\n",
       "      <td>Single particle alignment and classification</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>992</th>\n",
       "      <td>http://edamontology.org/operation_3469</td>\n",
       "      <td>RNA structure covariance model generation</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>993</th>\n",
       "      <td>http://edamontology.org/operation_3560</td>\n",
       "      <td>Maximum occurence analysis</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>994</th>\n",
       "      <td>http://edamontology.org/operation_3564</td>\n",
       "      <td>Chemical redundancy removal</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>995</th>\n",
       "      <td>http://edamontology.org/operation_3666</td>\n",
       "      <td>Molecular surface comparison</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>996</th>\n",
       "      <td>http://edamontology.org/operation_3890</td>\n",
       "      <td>Trajectory visualization</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>997</th>\n",
       "      <td>http://edamontology.org/operation_3891</td>\n",
       "      <td>Essential dynamics</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>998</th>\n",
       "      <td>http://edamontology.org/operation_3899</td>\n",
       "      <td>Protein-protein docking</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>999</th>\n",
       "      <td>http://edamontology.org/operation_3938</td>\n",
       "      <td>Virtual screening</td>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1000</th>\n",
       "      <td>http://edamontology.org/operation_0475</td>\n",
       "      <td>Nucleic acid structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0097</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1001</th>\n",
       "      <td>http://edamontology.org/operation_0279</td>\n",
       "      <td>Nucleic acid folding analysis</td>\n",
       "      <td>http://edamontology.org/topic_0097</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1002</th>\n",
       "      <td>http://edamontology.org/operation_2481</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "      <td>http://edamontology.org/topic_0097</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1003</th>\n",
       "      <td>http://edamontology.org/operation_2441</td>\n",
       "      <td>RNA structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0097</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1004</th>\n",
       "      <td>http://edamontology.org/operation_2442</td>\n",
       "      <td>DNA structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0097</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1005</th>\n",
       "      <td>http://edamontology.org/operation_2518</td>\n",
       "      <td>Nucleic acid structure comparison</td>\n",
       "      <td>http://edamontology.org/topic_0097</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1006</th>\n",
       "      <td>http://edamontology.org/operation_3891</td>\n",
       "      <td>Essential dynamics</td>\n",
       "      <td>http://edamontology.org/topic_0097</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1007</th>\n",
       "      <td>http://edamontology.org/operation_2489</td>\n",
       "      <td>Subcellular localisation prediction</td>\n",
       "      <td>http://edamontology.org/topic_0140</td>\n",
       "      <td>Protein targeting and localisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1008</th>\n",
       "      <td>http://edamontology.org/operation_2492</td>\n",
       "      <td>Protein interaction prediction</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1009</th>\n",
       "      <td>http://edamontology.org/operation_2495</td>\n",
       "      <td>Expression analysis</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1010</th>\n",
       "      <td>http://edamontology.org/operation_0313</td>\n",
       "      <td>Expression profile clustering</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1011</th>\n",
       "      <td>http://edamontology.org/operation_0315</td>\n",
       "      <td>Expression profile comparison</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1012</th>\n",
       "      <td>http://edamontology.org/operation_0314</td>\n",
       "      <td>Gene expression profiling</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1013</th>\n",
       "      <td>http://edamontology.org/operation_0531</td>\n",
       "      <td>Heat map generation</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1014</th>\n",
       "      <td>http://edamontology.org/operation_0571</td>\n",
       "      <td>Expression data visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1015</th>\n",
       "      <td>http://edamontology.org/operation_0533</td>\n",
       "      <td>Expression profile pathway mapping</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1016</th>\n",
       "      <td>http://edamontology.org/operation_1781</td>\n",
       "      <td>Gene regulatory network analysis</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1017</th>\n",
       "      <td>http://edamontology.org/operation_2436</td>\n",
       "      <td>Gene-set enrichment analysis</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1018</th>\n",
       "      <td>http://edamontology.org/operation_2437</td>\n",
       "      <td>Gene regulatory network prediction</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1019</th>\n",
       "      <td>http://edamontology.org/operation_2938</td>\n",
       "      <td>Dendrogram visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1020</th>\n",
       "      <td>http://edamontology.org/operation_2935</td>\n",
       "      <td>Clustering profile plotting</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1021</th>\n",
       "      <td>http://edamontology.org/operation_2937</td>\n",
       "      <td>Proximity map plotting</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1022</th>\n",
       "      <td>http://edamontology.org/operation_2942</td>\n",
       "      <td>Treemap visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1023</th>\n",
       "      <td>http://edamontology.org/operation_3223</td>\n",
       "      <td>Differential gene expression profiling</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1024</th>\n",
       "      <td>http://edamontology.org/operation_3232</td>\n",
       "      <td>Gene expression QTL analysis</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1025</th>\n",
       "      <td>http://edamontology.org/operation_3463</td>\n",
       "      <td>Expression correlation analysis</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1026</th>\n",
       "      <td>http://edamontology.org/operation_3563</td>\n",
       "      <td>RNA-seq read count analysis</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1027</th>\n",
       "      <td>http://edamontology.org/operation_3680</td>\n",
       "      <td>RNA-Seq analysis</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1028</th>\n",
       "      <td>http://edamontology.org/operation_3565</td>\n",
       "      <td>RNA-seq time series data analysis</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1029</th>\n",
       "      <td>http://edamontology.org/operation_3741</td>\n",
       "      <td>Differential protein expression profiling</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1030</th>\n",
       "      <td>http://edamontology.org/operation_3792</td>\n",
       "      <td>miRNA expression analysis</td>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1031</th>\n",
       "      <td>http://edamontology.org/operation_2510</td>\n",
       "      <td>DNA back-translation</td>\n",
       "      <td>http://edamontology.org/topic_0108</td>\n",
       "      <td>Protein expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1032</th>\n",
       "      <td>http://edamontology.org/operation_2520</td>\n",
       "      <td>DNA mapping</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1033</th>\n",
       "      <td>http://edamontology.org/operation_0485</td>\n",
       "      <td>Radiation Hybrid Mapping</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1034</th>\n",
       "      <td>http://edamontology.org/operation_2944</td>\n",
       "      <td>Physical mapping</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1035</th>\n",
       "      <td>http://edamontology.org/operation_0527</td>\n",
       "      <td>Sequence tag mapping</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1036</th>\n",
       "      <td>http://edamontology.org/operation_2871</td>\n",
       "      <td>Sequence tagged site (STS) mapping</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1037</th>\n",
       "      <td>http://edamontology.org/operation_3186</td>\n",
       "      <td>Bisulfite mapping</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1038</th>\n",
       "      <td>http://edamontology.org/operation_3198</td>\n",
       "      <td>Read mapping</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1039</th>\n",
       "      <td>http://edamontology.org/operation_3199</td>\n",
       "      <td>Split read mapping</td>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1040</th>\n",
       "      <td>http://edamontology.org/operation_0282</td>\n",
       "      <td>Genetic mapping</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1041</th>\n",
       "      <td>http://edamontology.org/operation_2520</td>\n",
       "      <td>DNA mapping</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1042</th>\n",
       "      <td>http://edamontology.org/operation_0485</td>\n",
       "      <td>Radiation Hybrid Mapping</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1043</th>\n",
       "      <td>http://edamontology.org/operation_2944</td>\n",
       "      <td>Physical mapping</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1044</th>\n",
       "      <td>http://edamontology.org/operation_0487</td>\n",
       "      <td>Haplotype mapping</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1045</th>\n",
       "      <td>http://edamontology.org/operation_0527</td>\n",
       "      <td>Sequence tag mapping</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1046</th>\n",
       "      <td>http://edamontology.org/operation_2871</td>\n",
       "      <td>Sequence tagged site (STS) mapping</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1047</th>\n",
       "      <td>http://edamontology.org/operation_3186</td>\n",
       "      <td>Bisulfite mapping</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1048</th>\n",
       "      <td>http://edamontology.org/operation_3198</td>\n",
       "      <td>Read mapping</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1049</th>\n",
       "      <td>http://edamontology.org/operation_3199</td>\n",
       "      <td>Split read mapping</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1050</th>\n",
       "      <td>http://edamontology.org/operation_2574</td>\n",
       "      <td>Protein hydropathy calculation</td>\n",
       "      <td>http://edamontology.org/topic_0123</td>\n",
       "      <td>Protein properties</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1051</th>\n",
       "      <td>http://edamontology.org/operation_0405</td>\n",
       "      <td>Protein hydrophobic region calculation</td>\n",
       "      <td>http://edamontology.org/topic_0123</td>\n",
       "      <td>Protein properties</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1052</th>\n",
       "      <td>http://edamontology.org/operation_0406</td>\n",
       "      <td>Protein aliphatic index calculation</td>\n",
       "      <td>http://edamontology.org/topic_0123</td>\n",
       "      <td>Protein properties</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1053</th>\n",
       "      <td>http://edamontology.org/operation_0407</td>\n",
       "      <td>Protein hydrophobic moment plotting</td>\n",
       "      <td>http://edamontology.org/topic_0123</td>\n",
       "      <td>Protein properties</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1054</th>\n",
       "      <td>http://edamontology.org/operation_0408</td>\n",
       "      <td>Protein globularity prediction</td>\n",
       "      <td>http://edamontology.org/topic_0123</td>\n",
       "      <td>Protein properties</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1055</th>\n",
       "      <td>http://edamontology.org/operation_0409</td>\n",
       "      <td>Protein solubility prediction</td>\n",
       "      <td>http://edamontology.org/topic_0123</td>\n",
       "      <td>Protein properties</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1056</th>\n",
       "      <td>http://edamontology.org/operation_0410</td>\n",
       "      <td>Protein crystallizability prediction</td>\n",
       "      <td>http://edamontology.org/topic_0123</td>\n",
       "      <td>Protein properties</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1057</th>\n",
       "      <td>http://edamontology.org/operation_0420</td>\n",
       "      <td>Nucleic acids-binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1058</th>\n",
       "      <td>http://edamontology.org/operation_2575</td>\n",
       "      <td>Binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1059</th>\n",
       "      <td>http://edamontology.org/operation_3896</td>\n",
       "      <td>Active site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1060</th>\n",
       "      <td>http://edamontology.org/operation_3897</td>\n",
       "      <td>Ligand-binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1061</th>\n",
       "      <td>http://edamontology.org/operation_3898</td>\n",
       "      <td>Metal-binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1062</th>\n",
       "      <td>http://edamontology.org/operation_3902</td>\n",
       "      <td>RNA binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1063</th>\n",
       "      <td>http://edamontology.org/operation_3903</td>\n",
       "      <td>DNA binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1064</th>\n",
       "      <td>http://edamontology.org/operation_2949</td>\n",
       "      <td>Protein-protein interaction analysis</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1065</th>\n",
       "      <td>http://edamontology.org/operation_3094</td>\n",
       "      <td>Protein interaction network prediction</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1066</th>\n",
       "      <td>http://edamontology.org/operation_0245</td>\n",
       "      <td>Structural motif discovery</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1067</th>\n",
       "      <td>http://edamontology.org/operation_3092</td>\n",
       "      <td>Protein feature detection</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1068</th>\n",
       "      <td>http://edamontology.org/operation_0246</td>\n",
       "      <td>Protein domain recognition</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1069</th>\n",
       "      <td>http://edamontology.org/operation_0267</td>\n",
       "      <td>Protein secondary structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1070</th>\n",
       "      <td>http://edamontology.org/operation_0268</td>\n",
       "      <td>Protein super-secondary structure prediction</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1071</th>\n",
       "      <td>http://edamontology.org/operation_0269</td>\n",
       "      <td>Transmembrane protein prediction</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1072</th>\n",
       "      <td>http://edamontology.org/operation_2464</td>\n",
       "      <td>Protein-protein binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1073</th>\n",
       "      <td>http://edamontology.org/operation_0303</td>\n",
       "      <td>Fold recognition</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1074</th>\n",
       "      <td>http://edamontology.org/operation_0302</td>\n",
       "      <td>Protein threading</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1075</th>\n",
       "      <td>http://edamontology.org/operation_0420</td>\n",
       "      <td>Nucleic acids-binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1076</th>\n",
       "      <td>http://edamontology.org/operation_2575</td>\n",
       "      <td>Binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1077</th>\n",
       "      <td>http://edamontology.org/operation_0390</td>\n",
       "      <td>Protein peeling</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1078</th>\n",
       "      <td>http://edamontology.org/operation_0418</td>\n",
       "      <td>Protein signal peptide detection</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1079</th>\n",
       "      <td>http://edamontology.org/operation_0416</td>\n",
       "      <td>Epitope mapping</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1080</th>\n",
       "      <td>http://edamontology.org/operation_0417</td>\n",
       "      <td>Post-translational modification site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1081</th>\n",
       "      <td>http://edamontology.org/operation_0422</td>\n",
       "      <td>Protein cleavage site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1082</th>\n",
       "      <td>http://edamontology.org/operation_0468</td>\n",
       "      <td>Protein secondary structure prediction (helices)</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1083</th>\n",
       "      <td>http://edamontology.org/operation_0469</td>\n",
       "      <td>Protein secondary structure prediction (turns)</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1084</th>\n",
       "      <td>http://edamontology.org/operation_0470</td>\n",
       "      <td>Protein secondary structure prediction (coils)</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1085</th>\n",
       "      <td>http://edamontology.org/operation_0471</td>\n",
       "      <td>Disulfide bond prediction</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1086</th>\n",
       "      <td>http://edamontology.org/operation_3896</td>\n",
       "      <td>Active site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1087</th>\n",
       "      <td>http://edamontology.org/operation_3897</td>\n",
       "      <td>Ligand-binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1088</th>\n",
       "      <td>http://edamontology.org/operation_3898</td>\n",
       "      <td>Metal-binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1089</th>\n",
       "      <td>http://edamontology.org/operation_3902</td>\n",
       "      <td>RNA binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1090</th>\n",
       "      <td>http://edamontology.org/operation_3903</td>\n",
       "      <td>DNA binding site prediction</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1091</th>\n",
       "      <td>http://edamontology.org/operation_3904</td>\n",
       "      <td>Protein disorder prediction</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1092</th>\n",
       "      <td>http://edamontology.org/operation_3968</td>\n",
       "      <td>Adhesin prediction</td>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1093</th>\n",
       "      <td>http://edamontology.org/operation_3180</td>\n",
       "      <td>Sequence assembly validation</td>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1094</th>\n",
       "      <td>http://edamontology.org/operation_3185</td>\n",
       "      <td>Base-calling</td>\n",
       "      <td>http://edamontology.org/topic_3168</td>\n",
       "      <td>Sequencing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1095</th>\n",
       "      <td>http://edamontology.org/operation_3195</td>\n",
       "      <td>Sequencing error detection</td>\n",
       "      <td>http://edamontology.org/topic_3168</td>\n",
       "      <td>Sequencing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1096</th>\n",
       "      <td>http://edamontology.org/operation_3214</td>\n",
       "      <td>Spectral analysis</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1097</th>\n",
       "      <td>http://edamontology.org/operation_3215</td>\n",
       "      <td>Peak detection</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1098</th>\n",
       "      <td>http://edamontology.org/operation_3627</td>\n",
       "      <td>Mass spectra calibration</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1099</th>\n",
       "      <td>http://edamontology.org/operation_3628</td>\n",
       "      <td>Chromatographic alignment</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1100</th>\n",
       "      <td>http://edamontology.org/operation_3629</td>\n",
       "      <td>Deisotoping</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1101</th>\n",
       "      <td>http://edamontology.org/operation_3630</td>\n",
       "      <td>Protein quantification</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1102</th>\n",
       "      <td>http://edamontology.org/operation_3631</td>\n",
       "      <td>Peptide identification</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1103</th>\n",
       "      <td>http://edamontology.org/operation_3634</td>\n",
       "      <td>Label-free quantification</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1104</th>\n",
       "      <td>http://edamontology.org/operation_3635</td>\n",
       "      <td>Labeled quantification</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1105</th>\n",
       "      <td>http://edamontology.org/operation_3636</td>\n",
       "      <td>MRM/SRM</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1106</th>\n",
       "      <td>http://edamontology.org/operation_3637</td>\n",
       "      <td>Spectral counting</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1107</th>\n",
       "      <td>http://edamontology.org/operation_3638</td>\n",
       "      <td>SILAC</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1108</th>\n",
       "      <td>http://edamontology.org/operation_3639</td>\n",
       "      <td>iTRAQ</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1109</th>\n",
       "      <td>http://edamontology.org/operation_3640</td>\n",
       "      <td>18O labeling</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1110</th>\n",
       "      <td>http://edamontology.org/operation_3641</td>\n",
       "      <td>TMT-tag</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1111</th>\n",
       "      <td>http://edamontology.org/operation_3642</td>\n",
       "      <td>Dimethyl</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1112</th>\n",
       "      <td>http://edamontology.org/operation_3643</td>\n",
       "      <td>Tag-based peptide identification</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1113</th>\n",
       "      <td>http://edamontology.org/operation_3644</td>\n",
       "      <td>de Novo sequencing</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1114</th>\n",
       "      <td>http://edamontology.org/operation_3645</td>\n",
       "      <td>PTM identification</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1115</th>\n",
       "      <td>http://edamontology.org/operation_3646</td>\n",
       "      <td>Peptide database search</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1116</th>\n",
       "      <td>http://edamontology.org/operation_3647</td>\n",
       "      <td>Blind peptide database search</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1117</th>\n",
       "      <td>http://edamontology.org/operation_3649</td>\n",
       "      <td>Target-Decoy</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1118</th>\n",
       "      <td>http://edamontology.org/operation_3704</td>\n",
       "      <td>Ion counting</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1119</th>\n",
       "      <td>http://edamontology.org/operation_3705</td>\n",
       "      <td>Isotope-coded protein label</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1120</th>\n",
       "      <td>http://edamontology.org/operation_3715</td>\n",
       "      <td>Metabolic labeling</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1121</th>\n",
       "      <td>http://edamontology.org/operation_3755</td>\n",
       "      <td>PTM localisation</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1122</th>\n",
       "      <td>http://edamontology.org/operation_3767</td>\n",
       "      <td>Protein identification</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1123</th>\n",
       "      <td>http://edamontology.org/operation_3801</td>\n",
       "      <td>Spectral library search</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1124</th>\n",
       "      <td>http://edamontology.org/operation_3803</td>\n",
       "      <td>Natural product identification</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1125</th>\n",
       "      <td>http://edamontology.org/operation_3860</td>\n",
       "      <td>Spectrum calculation</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1126</th>\n",
       "      <td>http://edamontology.org/operation_3221</td>\n",
       "      <td>Species frequency estimation</td>\n",
       "      <td>http://edamontology.org/topic_3174</td>\n",
       "      <td>Metagenomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1127</th>\n",
       "      <td>http://edamontology.org/operation_3348</td>\n",
       "      <td>Sequence checksum generation</td>\n",
       "      <td>http://edamontology.org/topic_3077</td>\n",
       "      <td>Data acquisition</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1128</th>\n",
       "      <td>http://edamontology.org/operation_0384</td>\n",
       "      <td>Accessible surface calculation</td>\n",
       "      <td>http://edamontology.org/topic_0123</td>\n",
       "      <td>Protein properties</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1129</th>\n",
       "      <td>http://edamontology.org/operation_3351</td>\n",
       "      <td>Molecular surface analysis</td>\n",
       "      <td>http://edamontology.org/topic_0123</td>\n",
       "      <td>Protein properties</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1130</th>\n",
       "      <td>http://edamontology.org/operation_0387</td>\n",
       "      <td>Molecular surface calculation</td>\n",
       "      <td>http://edamontology.org/topic_0123</td>\n",
       "      <td>Protein properties</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1131</th>\n",
       "      <td>http://edamontology.org/operation_1816</td>\n",
       "      <td>Surface rendering</td>\n",
       "      <td>http://edamontology.org/topic_0123</td>\n",
       "      <td>Protein properties</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1132</th>\n",
       "      <td>http://edamontology.org/operation_3666</td>\n",
       "      <td>Molecular surface comparison</td>\n",
       "      <td>http://edamontology.org/topic_0123</td>\n",
       "      <td>Protein properties</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1133</th>\n",
       "      <td>http://edamontology.org/operation_0384</td>\n",
       "      <td>Accessible surface calculation</td>\n",
       "      <td>http://edamontology.org/topic_0166</td>\n",
       "      <td>Protein structural motifs and surfaces</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1134</th>\n",
       "      <td>http://edamontology.org/operation_3351</td>\n",
       "      <td>Molecular surface analysis</td>\n",
       "      <td>http://edamontology.org/topic_0166</td>\n",
       "      <td>Protein structural motifs and surfaces</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1135</th>\n",
       "      <td>http://edamontology.org/operation_0387</td>\n",
       "      <td>Molecular surface calculation</td>\n",
       "      <td>http://edamontology.org/topic_0166</td>\n",
       "      <td>Protein structural motifs and surfaces</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1136</th>\n",
       "      <td>http://edamontology.org/operation_1816</td>\n",
       "      <td>Surface rendering</td>\n",
       "      <td>http://edamontology.org/topic_0166</td>\n",
       "      <td>Protein structural motifs and surfaces</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1137</th>\n",
       "      <td>http://edamontology.org/operation_3666</td>\n",
       "      <td>Molecular surface comparison</td>\n",
       "      <td>http://edamontology.org/topic_0166</td>\n",
       "      <td>Protein structural motifs and surfaces</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1138</th>\n",
       "      <td>http://edamontology.org/operation_3278</td>\n",
       "      <td>Document similarity calculation</td>\n",
       "      <td>http://edamontology.org/topic_3068</td>\n",
       "      <td>Literature and language</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1139</th>\n",
       "      <td>http://edamontology.org/operation_3437</td>\n",
       "      <td>Article comparison</td>\n",
       "      <td>http://edamontology.org/topic_3068</td>\n",
       "      <td>Literature and language</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1140</th>\n",
       "      <td>http://edamontology.org/operation_3279</td>\n",
       "      <td>Document clustering</td>\n",
       "      <td>http://edamontology.org/topic_3068</td>\n",
       "      <td>Literature and language</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1141</th>\n",
       "      <td>http://edamontology.org/operation_3443</td>\n",
       "      <td>Image analysis</td>\n",
       "      <td>http://edamontology.org/topic_3382</td>\n",
       "      <td>Imaging</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1142</th>\n",
       "      <td>http://edamontology.org/operation_3446</td>\n",
       "      <td>Cell migration analysis</td>\n",
       "      <td>http://edamontology.org/topic_3382</td>\n",
       "      <td>Imaging</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1143</th>\n",
       "      <td>http://edamontology.org/operation_3450</td>\n",
       "      <td>Neurite measurement</td>\n",
       "      <td>http://edamontology.org/topic_3382</td>\n",
       "      <td>Imaging</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1144</th>\n",
       "      <td>http://edamontology.org/operation_3457</td>\n",
       "      <td>Single particle analysis</td>\n",
       "      <td>http://edamontology.org/topic_3382</td>\n",
       "      <td>Imaging</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1145</th>\n",
       "      <td>http://edamontology.org/operation_3458</td>\n",
       "      <td>Single particle alignment and classification</td>\n",
       "      <td>http://edamontology.org/topic_3382</td>\n",
       "      <td>Imaging</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1146</th>\n",
       "      <td>http://edamontology.org/operation_3446</td>\n",
       "      <td>Cell migration analysis</td>\n",
       "      <td>http://edamontology.org/topic_2229</td>\n",
       "      <td>Cell biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1147</th>\n",
       "      <td>http://edamontology.org/operation_3450</td>\n",
       "      <td>Neurite measurement</td>\n",
       "      <td>http://edamontology.org/topic_2229</td>\n",
       "      <td>Cell biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1148</th>\n",
       "      <td>http://edamontology.org/operation_3457</td>\n",
       "      <td>Single particle analysis</td>\n",
       "      <td>http://edamontology.org/topic_1317</td>\n",
       "      <td>Structural biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1149</th>\n",
       "      <td>http://edamontology.org/operation_3458</td>\n",
       "      <td>Single particle alignment and classification</td>\n",
       "      <td>http://edamontology.org/topic_1317</td>\n",
       "      <td>Structural biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1150</th>\n",
       "      <td>http://edamontology.org/operation_3459</td>\n",
       "      <td>Functional clustering</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1151</th>\n",
       "      <td>http://edamontology.org/operation_3461</td>\n",
       "      <td>Virulence prediction</td>\n",
       "      <td>http://edamontology.org/topic_3301</td>\n",
       "      <td>Microbiology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1152</th>\n",
       "      <td>http://edamontology.org/operation_0449</td>\n",
       "      <td>Sequence alignment analysis (site correlation)</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1153</th>\n",
       "      <td>http://edamontology.org/operation_3465</td>\n",
       "      <td>Correlation</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1154</th>\n",
       "      <td>http://edamontology.org/operation_3463</td>\n",
       "      <td>Expression correlation analysis</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1155</th>\n",
       "      <td>http://edamontology.org/operation_3482</td>\n",
       "      <td>Antimicrobial resistance prediction</td>\n",
       "      <td>http://edamontology.org/topic_3301</td>\n",
       "      <td>Microbiology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1156</th>\n",
       "      <td>http://edamontology.org/operation_3502</td>\n",
       "      <td>Chemical similarity enrichment</td>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1157</th>\n",
       "      <td>http://edamontology.org/operation_3552</td>\n",
       "      <td>Microscope image visualisation</td>\n",
       "      <td>http://edamontology.org/topic_3382</td>\n",
       "      <td>Imaging</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1158</th>\n",
       "      <td>http://edamontology.org/operation_3632</td>\n",
       "      <td>Isotopic distributions calculation</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1159</th>\n",
       "      <td>http://edamontology.org/operation_3660</td>\n",
       "      <td>Metabolic network modelling</td>\n",
       "      <td>http://edamontology.org/topic_2259</td>\n",
       "      <td>Systems biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1160</th>\n",
       "      <td>http://edamontology.org/operation_3766</td>\n",
       "      <td>Weighted correlation network analysis</td>\n",
       "      <td>http://edamontology.org/topic_0602</td>\n",
       "      <td>Molecular interactions, pathways and networks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1161</th>\n",
       "      <td>http://edamontology.org/operation_3778</td>\n",
       "      <td>Text annotation</td>\n",
       "      <td>http://edamontology.org/topic_3068</td>\n",
       "      <td>Literature and language</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1162</th>\n",
       "      <td>http://edamontology.org/operation_3893</td>\n",
       "      <td>Forcefield parameterisation</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1163</th>\n",
       "      <td>http://edamontology.org/operation_3899</td>\n",
       "      <td>Protein-protein docking</td>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1164</th>\n",
       "      <td>http://edamontology.org/operation_3904</td>\n",
       "      <td>Protein disorder prediction</td>\n",
       "      <td>http://edamontology.org/topic_3301</td>\n",
       "      <td>Microbiology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1165</th>\n",
       "      <td>http://edamontology.org/operation_0362</td>\n",
       "      <td>Genome annotation</td>\n",
       "      <td>http://edamontology.org/topic_0622</td>\n",
       "      <td>Genomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1166</th>\n",
       "      <td>http://edamontology.org/operation_3918</td>\n",
       "      <td>Genome analysis</td>\n",
       "      <td>http://edamontology.org/topic_0622</td>\n",
       "      <td>Genomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1167</th>\n",
       "      <td>http://edamontology.org/operation_0525</td>\n",
       "      <td>Genome assembly</td>\n",
       "      <td>http://edamontology.org/topic_0622</td>\n",
       "      <td>Genomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1168</th>\n",
       "      <td>http://edamontology.org/operation_3182</td>\n",
       "      <td>Genome alignment</td>\n",
       "      <td>http://edamontology.org/topic_0622</td>\n",
       "      <td>Genomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1169</th>\n",
       "      <td>http://edamontology.org/operation_3194</td>\n",
       "      <td>Genome feature comparison</td>\n",
       "      <td>http://edamontology.org/topic_0622</td>\n",
       "      <td>Genomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1170</th>\n",
       "      <td>http://edamontology.org/operation_3206</td>\n",
       "      <td>Whole genome methylation analysis</td>\n",
       "      <td>http://edamontology.org/topic_0622</td>\n",
       "      <td>Genomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1171</th>\n",
       "      <td>http://edamontology.org/operation_3208</td>\n",
       "      <td>Genome visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0622</td>\n",
       "      <td>Genomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1172</th>\n",
       "      <td>http://edamontology.org/operation_3209</td>\n",
       "      <td>Genome comparison</td>\n",
       "      <td>http://edamontology.org/topic_0622</td>\n",
       "      <td>Genomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1173</th>\n",
       "      <td>http://edamontology.org/operation_3211</td>\n",
       "      <td>Genome indexing</td>\n",
       "      <td>http://edamontology.org/topic_0622</td>\n",
       "      <td>Genomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1174</th>\n",
       "      <td>http://edamontology.org/operation_3920</td>\n",
       "      <td>DNA testing</td>\n",
       "      <td>http://edamontology.org/topic_0622</td>\n",
       "      <td>Genomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1175</th>\n",
       "      <td>http://edamontology.org/operation_0276</td>\n",
       "      <td>Protein interaction network analysis</td>\n",
       "      <td>http://edamontology.org/topic_0602</td>\n",
       "      <td>Molecular interactions, pathways and networks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1176</th>\n",
       "      <td>http://edamontology.org/operation_3927</td>\n",
       "      <td>Network analysis</td>\n",
       "      <td>http://edamontology.org/topic_0602</td>\n",
       "      <td>Molecular interactions, pathways and networks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1177</th>\n",
       "      <td>http://edamontology.org/operation_3925</td>\n",
       "      <td>Network visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0602</td>\n",
       "      <td>Molecular interactions, pathways and networks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1178</th>\n",
       "      <td>http://edamontology.org/operation_2437</td>\n",
       "      <td>Gene regulatory network prediction</td>\n",
       "      <td>http://edamontology.org/topic_0602</td>\n",
       "      <td>Molecular interactions, pathways and networks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1179</th>\n",
       "      <td>http://edamontology.org/operation_3094</td>\n",
       "      <td>Protein interaction network prediction</td>\n",
       "      <td>http://edamontology.org/topic_0602</td>\n",
       "      <td>Molecular interactions, pathways and networks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1180</th>\n",
       "      <td>http://edamontology.org/operation_3660</td>\n",
       "      <td>Metabolic network modelling</td>\n",
       "      <td>http://edamontology.org/topic_0602</td>\n",
       "      <td>Molecular interactions, pathways and networks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1181</th>\n",
       "      <td>http://edamontology.org/operation_0276</td>\n",
       "      <td>Protein interaction network analysis</td>\n",
       "      <td>http://edamontology.org/topic_2259</td>\n",
       "      <td>Systems biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1182</th>\n",
       "      <td>http://edamontology.org/operation_3927</td>\n",
       "      <td>Network analysis</td>\n",
       "      <td>http://edamontology.org/topic_2259</td>\n",
       "      <td>Systems biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1183</th>\n",
       "      <td>http://edamontology.org/operation_3925</td>\n",
       "      <td>Network visualisation</td>\n",
       "      <td>http://edamontology.org/topic_2259</td>\n",
       "      <td>Systems biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1184</th>\n",
       "      <td>http://edamontology.org/operation_1781</td>\n",
       "      <td>Gene regulatory network analysis</td>\n",
       "      <td>http://edamontology.org/topic_2259</td>\n",
       "      <td>Systems biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1185</th>\n",
       "      <td>http://edamontology.org/operation_2437</td>\n",
       "      <td>Gene regulatory network prediction</td>\n",
       "      <td>http://edamontology.org/topic_2259</td>\n",
       "      <td>Systems biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1186</th>\n",
       "      <td>http://edamontology.org/operation_3094</td>\n",
       "      <td>Protein interaction network prediction</td>\n",
       "      <td>http://edamontology.org/topic_2259</td>\n",
       "      <td>Systems biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1187</th>\n",
       "      <td>http://edamontology.org/operation_3766</td>\n",
       "      <td>Weighted correlation network analysis</td>\n",
       "      <td>http://edamontology.org/topic_2259</td>\n",
       "      <td>Systems biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1188</th>\n",
       "      <td>http://edamontology.org/operation_3928</td>\n",
       "      <td>Pathway analysis</td>\n",
       "      <td>http://edamontology.org/topic_0602</td>\n",
       "      <td>Molecular interactions, pathways and networks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1189</th>\n",
       "      <td>http://edamontology.org/operation_0533</td>\n",
       "      <td>Expression profile pathway mapping</td>\n",
       "      <td>http://edamontology.org/topic_0602</td>\n",
       "      <td>Molecular interactions, pathways and networks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1190</th>\n",
       "      <td>http://edamontology.org/operation_3926</td>\n",
       "      <td>Pathway visualisation</td>\n",
       "      <td>http://edamontology.org/topic_0602</td>\n",
       "      <td>Molecular interactions, pathways and networks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1191</th>\n",
       "      <td>http://edamontology.org/operation_3929</td>\n",
       "      <td>Metabolic pathway prediction</td>\n",
       "      <td>http://edamontology.org/topic_0602</td>\n",
       "      <td>Molecular interactions, pathways and networks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1192</th>\n",
       "      <td>http://edamontology.org/operation_3928</td>\n",
       "      <td>Pathway analysis</td>\n",
       "      <td>http://edamontology.org/topic_2259</td>\n",
       "      <td>Systems biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1193</th>\n",
       "      <td>http://edamontology.org/operation_0533</td>\n",
       "      <td>Expression profile pathway mapping</td>\n",
       "      <td>http://edamontology.org/topic_2259</td>\n",
       "      <td>Systems biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1194</th>\n",
       "      <td>http://edamontology.org/operation_3926</td>\n",
       "      <td>Pathway visualisation</td>\n",
       "      <td>http://edamontology.org/topic_2259</td>\n",
       "      <td>Systems biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1195</th>\n",
       "      <td>http://edamontology.org/operation_3929</td>\n",
       "      <td>Metabolic pathway prediction</td>\n",
       "      <td>http://edamontology.org/topic_2259</td>\n",
       "      <td>Systems biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1196</th>\n",
       "      <td>http://edamontology.org/operation_3935</td>\n",
       "      <td>Dimensionality reduction</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1197</th>\n",
       "      <td>http://edamontology.org/operation_3936</td>\n",
       "      <td>Feature selection</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1198</th>\n",
       "      <td>http://edamontology.org/operation_3937</td>\n",
       "      <td>Feature extraction</td>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1199</th>\n",
       "      <td>http://edamontology.org/operation_3946</td>\n",
       "      <td>Ecological modelling</td>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1200</th>\n",
       "      <td>http://edamontology.org/operation_3968</td>\n",
       "      <td>Adhesin prediction</td>\n",
       "      <td>http://edamontology.org/topic_0804</td>\n",
       "      <td>Immunology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1201</th>\n",
       "      <td>http://edamontology.org/operation_4031</td>\n",
       "      <td>Power test</td>\n",
       "      <td>http://edamontology.org/topic_3572</td>\n",
       "      <td>Data quality management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1202</th>\n",
       "      <td>http://edamontology.org/operation_4031</td>\n",
       "      <td>Power test</td>\n",
       "      <td>http://edamontology.org/topic_3365</td>\n",
       "      <td>Data architecture, analysis and design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1203</th>\n",
       "      <td>http://edamontology.org/operation_4031</td>\n",
       "      <td>Power test</td>\n",
       "      <td>http://edamontology.org/topic_2269</td>\n",
       "      <td>Statistics and probability</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1204</th>\n",
       "      <td>http://edamontology.org/operation_4032</td>\n",
       "      <td>DNA modification prediction</td>\n",
       "      <td>http://edamontology.org/topic_3295</td>\n",
       "      <td>Epigenetics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1205</th>\n",
       "      <td>http://edamontology.org/operation_4033</td>\n",
       "      <td>Disease transmission analysis</td>\n",
       "      <td>http://edamontology.org/topic_3324</td>\n",
       "      <td>Infectious disease</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1206</th>\n",
       "      <td>http://edamontology.org/operation_4033</td>\n",
       "      <td>Disease transmission analysis</td>\n",
       "      <td>http://edamontology.org/topic_3305</td>\n",
       "      <td>Public health and epidemiology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1207</th>\n",
       "      <td>http://edamontology.org/operation_4034</td>\n",
       "      <td>Multiple testing correction</td>\n",
       "      <td>http://edamontology.org/topic_2269</td>\n",
       "      <td>Statistics and probability</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                      entity                                                             label                               topic                                         tlabel\n",
       "0          http://edamontology.org/data_0582                                                          Ontology  http://edamontology.org/topic_0089                       Ontology and terminology\n",
       "1          http://edamontology.org/data_0858                                        Sequence signature matches  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "2          http://edamontology.org/data_0860                                           Sequence signature data  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "3          http://edamontology.org/data_1353                                                    Sequence motif  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "4          http://edamontology.org/data_1354                                                  Sequence profile  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "5          http://edamontology.org/data_1361                                         Position frequency matrix  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "6          http://edamontology.org/data_2854                                  Position-specific scoring matrix  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "7          http://edamontology.org/data_1362                                            Position weight matrix  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "8          http://edamontology.org/data_1363                                        Information content matrix  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "9          http://edamontology.org/data_1365                                                       Fingerprint  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "10         http://edamontology.org/data_2071                                          Sequence motif (protein)  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "11         http://edamontology.org/data_2070                                     Sequence motif (nucleic acid)  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "12         http://edamontology.org/data_3949                                                       Profile HMM  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "13         http://edamontology.org/data_0863                                                Sequence alignment  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "14         http://edamontology.org/data_1381                                           Pair sequence alignment  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "15         http://edamontology.org/data_1383                                   Nucleic acid sequence alignment  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "16         http://edamontology.org/data_1384                                        Protein sequence alignment  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "17         http://edamontology.org/data_1385                                         Hybrid sequence alignment  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "18         http://edamontology.org/data_0872                                                 Phylogenetic tree  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "19         http://edamontology.org/data_3271                                                         Gene tree  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "20         http://edamontology.org/data_3272                                                      Species tree  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "21         http://edamontology.org/data_0880                                           RNA secondary structure  http://edamontology.org/topic_0097                Nucleic acid structure analysis\n",
       "22         http://edamontology.org/data_0880                                           RNA secondary structure  http://edamontology.org/topic_0081                             Structure analysis\n",
       "23         http://edamontology.org/data_1465                                                     RNA structure  http://edamontology.org/topic_0081                             Structure analysis\n",
       "24         http://edamontology.org/data_0883                                                         Structure  http://edamontology.org/topic_0081                             Structure analysis\n",
       "25         http://edamontology.org/data_3806                                                        3D EM Mask  http://edamontology.org/topic_0081                             Structure analysis\n",
       "26         http://edamontology.org/data_2851                                                    Drug structure  http://edamontology.org/topic_0081                             Structure analysis\n",
       "27         http://edamontology.org/data_1467                                                     Protein chain  http://edamontology.org/topic_0081                             Structure analysis\n",
       "28         http://edamontology.org/data_1468                                                    Protein domain  http://edamontology.org/topic_0081                             Structure analysis\n",
       "29         http://edamontology.org/data_3805                                                         3D EM Map  http://edamontology.org/topic_0081                             Structure analysis\n",
       "30         http://edamontology.org/data_1459                                            Nucleic acid structure  http://edamontology.org/topic_0081                             Structure analysis\n",
       "31         http://edamontology.org/data_1460                                                 Protein structure  http://edamontology.org/topic_0081                             Structure analysis\n",
       "32         http://edamontology.org/data_1461                                            Protein-ligand complex  http://edamontology.org/topic_0081                             Structure analysis\n",
       "33         http://edamontology.org/data_1462                                            Carbohydrate structure  http://edamontology.org/topic_0081                             Structure analysis\n",
       "34         http://edamontology.org/data_1463                                          Small molecule structure  http://edamontology.org/topic_0081                             Structure analysis\n",
       "35         http://edamontology.org/data_1464                                                     DNA structure  http://edamontology.org/topic_0081                             Structure analysis\n",
       "36         http://edamontology.org/data_1466                                                    tRNA structure  http://edamontology.org/topic_0081                             Structure analysis\n",
       "37         http://edamontology.org/data_1470                                                     C-alpha trace  http://edamontology.org/topic_0081                             Structure analysis\n",
       "38         http://edamontology.org/data_1755                                                      Protein atom  http://edamontology.org/topic_0081                             Structure analysis\n",
       "39         http://edamontology.org/data_1756                                                   Protein residue  http://edamontology.org/topic_0081                             Structure analysis\n",
       "40         http://edamontology.org/data_2852                                                   Toxin structure  http://edamontology.org/topic_0081                             Structure analysis\n",
       "41         http://edamontology.org/data_2850                                                   Lipid structure  http://edamontology.org/topic_0081                             Structure analysis\n",
       "42         http://edamontology.org/data_2877                                                   Protein complex  http://edamontology.org/topic_0081                             Structure analysis\n",
       "43         http://edamontology.org/data_2878                                          Protein structural motif  http://edamontology.org/topic_0081                             Structure analysis\n",
       "44         http://edamontology.org/data_0886                                               Structure alignment  http://edamontology.org/topic_0081                             Structure analysis\n",
       "45         http://edamontology.org/data_1479                                        Structure alignment (pair)  http://edamontology.org/topic_0081                             Structure analysis\n",
       "46         http://edamontology.org/data_1481                                       Protein structure alignment  http://edamontology.org/topic_0081                             Structure analysis\n",
       "47         http://edamontology.org/data_1482                                  Nucleic acid structure alignment  http://edamontology.org/topic_0081                             Structure analysis\n",
       "48         http://edamontology.org/data_1493                                           RNA structure alignment  http://edamontology.org/topic_0081                             Structure analysis\n",
       "49         http://edamontology.org/data_3924                                           DNA structure alignment  http://edamontology.org/topic_0081                             Structure analysis\n",
       "50         http://edamontology.org/data_0889                                                Structural profile  http://edamontology.org/topic_0081                             Structure analysis\n",
       "51         http://edamontology.org/data_0906                                          Protein interaction data  http://edamontology.org/topic_0128                           Protein interactions\n",
       "52         http://edamontology.org/data_1546                                           Protein distance matrix  http://edamontology.org/topic_0128                           Protein interactions\n",
       "53         http://edamontology.org/data_1547                                               Protein contact map  http://edamontology.org/topic_0128                           Protein interactions\n",
       "54         http://edamontology.org/data_1548                                        Protein residue 3D cluster  http://edamontology.org/topic_0128                           Protein interactions\n",
       "55         http://edamontology.org/data_1549                                            Protein hydrogen bonds  http://edamontology.org/topic_0128                           Protein interactions\n",
       "56         http://edamontology.org/data_1566                                 Protein-ligand interaction report  http://edamontology.org/topic_0128                           Protein interactions\n",
       "57         http://edamontology.org/data_0907                                             Protein family report  http://edamontology.org/topic_0623                      Gene and protein families\n",
       "58         http://edamontology.org/data_0914                                                  Codon usage data  http://edamontology.org/topic_0203                                Gene expression\n",
       "59         http://edamontology.org/data_1597                                                 Codon usage table  http://edamontology.org/topic_0203                                Gene expression\n",
       "60         http://edamontology.org/data_1598                                                      Genetic code  http://edamontology.org/topic_0203                                Gene expression\n",
       "61         http://edamontology.org/data_2865                                                  Codon usage bias  http://edamontology.org/topic_0203                                Gene expression\n",
       "62         http://edamontology.org/data_1600                                             Codon usage bias plot  http://edamontology.org/topic_0203                                Gene expression\n",
       "63         http://edamontology.org/data_1602                                   Codon usage fraction difference  http://edamontology.org/topic_0203                                Gene expression\n",
       "64         http://edamontology.org/data_0940                                                        Volume map  http://edamontology.org/topic_1317                             Structural biology\n",
       "65         http://edamontology.org/data_0942                                                     2D PAGE image  http://edamontology.org/topic_0121                                     Proteomics\n",
       "66         http://edamontology.org/data_0943                                                     Mass spectrum  http://edamontology.org/topic_0121                                     Proteomics\n",
       "67         http://edamontology.org/data_0944                                          Peptide mass fingerprint  http://edamontology.org/topic_0121                                     Proteomics\n",
       "68         http://edamontology.org/data_0945                                            Peptide identification  http://edamontology.org/topic_0121                                     Proteomics\n",
       "69         http://edamontology.org/data_0950                                                Mathematical model  http://edamontology.org/topic_3307                          Computational biology\n",
       "70         http://edamontology.org/data_1364                                               Hidden Markov model  http://edamontology.org/topic_3307                          Computational biology\n",
       "71         http://edamontology.org/data_1347                                            Dirichlet distribution  http://edamontology.org/topic_3307                          Computational biology\n",
       "72         http://edamontology.org/data_1439                                            DNA substitution model  http://edamontology.org/topic_3307                          Computational biology\n",
       "73         http://edamontology.org/data_3241                                                     Kinetic model  http://edamontology.org/topic_3307                          Computational biology\n",
       "74         http://edamontology.org/data_3949                                                       Profile HMM  http://edamontology.org/topic_3307                          Computational biology\n",
       "75         http://edamontology.org/data_0955                                                        Data index  http://edamontology.org/topic_3489                            Database management\n",
       "76         http://edamontology.org/data_3210                                                      Genome index  http://edamontology.org/topic_3489                            Database management\n",
       "77         http://edamontology.org/data_0956                                                 Data index report  http://edamontology.org/topic_3489                            Database management\n",
       "78         http://edamontology.org/data_0962                                             Small molecule report  http://edamontology.org/topic_0154                                Small molecules\n",
       "79         http://edamontology.org/data_1696                                                       Drug report  http://edamontology.org/topic_0154                                Small molecules\n",
       "80         http://edamontology.org/data_1235                                                  Sequence cluster  http://edamontology.org/topic_0623                      Gene and protein families\n",
       "81         http://edamontology.org/data_1245                                        Sequence cluster (protein)  http://edamontology.org/topic_0623                      Gene and protein families\n",
       "82         http://edamontology.org/data_1246                                   Sequence cluster (nucleic acid)  http://edamontology.org/topic_0623                      Gene and protein families\n",
       "83         http://edamontology.org/data_1238                                                Proteolytic digest  http://edamontology.org/topic_0121                                     Proteomics\n",
       "84         http://edamontology.org/data_1274                                                               Map  http://edamontology.org/topic_0102                                        Mapping\n",
       "85         http://edamontology.org/data_1278                                                       Genetic map  http://edamontology.org/topic_0102                                        Mapping\n",
       "86         http://edamontology.org/data_1279                                                      Sequence map  http://edamontology.org/topic_0102                                        Mapping\n",
       "87         http://edamontology.org/data_1280                                                      Physical map  http://edamontology.org/topic_0102                                        Mapping\n",
       "88         http://edamontology.org/data_1283                                                   Cytogenetic map  http://edamontology.org/topic_0102                                        Mapping\n",
       "89         http://edamontology.org/data_1284                                              DNA transduction map  http://edamontology.org/topic_0102                                        Mapping\n",
       "90         http://edamontology.org/data_1285                                                          Gene map  http://edamontology.org/topic_0102                                        Mapping\n",
       "91         http://edamontology.org/data_1286                                                       Plasmid map  http://edamontology.org/topic_0102                                        Mapping\n",
       "92         http://edamontology.org/data_1288                                                        Genome map  http://edamontology.org/topic_0102                                        Mapping\n",
       "93         http://edamontology.org/data_1289                                                   Restriction map  http://edamontology.org/topic_0102                                        Mapping\n",
       "94         http://edamontology.org/data_1859                                                      Deletion map  http://edamontology.org/topic_0102                                        Mapping\n",
       "95         http://edamontology.org/data_1860                                                           QTL map  http://edamontology.org/topic_0102                                        Mapping\n",
       "96         http://edamontology.org/data_1863                                                     Haplotype map  http://edamontology.org/topic_0102                                        Mapping\n",
       "97         http://edamontology.org/data_2870                                              Radiation hybrid map  http://edamontology.org/topic_0102                                        Mapping\n",
       "98         http://edamontology.org/data_1429                                           Phylogenetic invariants  http://edamontology.org/topic_0199                              Genetic variation\n",
       "99         http://edamontology.org/data_1465                                                     RNA structure  http://edamontology.org/topic_0097                Nucleic acid structure analysis\n",
       "100        http://edamontology.org/data_1459                                            Nucleic acid structure  http://edamontology.org/topic_0097                Nucleic acid structure analysis\n",
       "101        http://edamontology.org/data_1464                                                     DNA structure  http://edamontology.org/topic_0097                Nucleic acid structure analysis\n",
       "102        http://edamontology.org/data_1466                                                    tRNA structure  http://edamontology.org/topic_0097                Nucleic acid structure analysis\n",
       "103        http://edamontology.org/data_1467                                                     Protein chain  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "104        http://edamontology.org/data_1468                                                    Protein domain  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "105        http://edamontology.org/data_1460                                                 Protein structure  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "106        http://edamontology.org/data_1461                                            Protein-ligand complex  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "107        http://edamontology.org/data_1470                                                     C-alpha trace  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "108        http://edamontology.org/data_1755                                                      Protein atom  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "109        http://edamontology.org/data_1756                                                   Protein residue  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "110        http://edamontology.org/data_2877                                                   Protein complex  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "111        http://edamontology.org/data_2878                                          Protein structural motif  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "112        http://edamontology.org/data_1462                                            Carbohydrate structure  http://edamontology.org/topic_0152                                  Carbohydrates\n",
       "113        http://edamontology.org/data_1462                                            Carbohydrate structure  http://edamontology.org/topic_0153                                         Lipids\n",
       "114        http://edamontology.org/data_2851                                                    Drug structure  http://edamontology.org/topic_0154                                Small molecules\n",
       "115        http://edamontology.org/data_1463                                          Small molecule structure  http://edamontology.org/topic_0154                                Small molecules\n",
       "116        http://edamontology.org/data_2852                                                   Toxin structure  http://edamontology.org/topic_0154                                Small molecules\n",
       "117        http://edamontology.org/data_1468                                                    Protein domain  http://edamontology.org/topic_0736           Protein folds and structural domains\n",
       "118        http://edamontology.org/data_1622                                                    Disease report  http://edamontology.org/topic_0634                                      Pathology\n",
       "119        http://edamontology.org/data_1713                                                          Fate map  http://edamontology.org/topic_3065                                     Embryology\n",
       "120        http://edamontology.org/data_2019                                                          Map data  http://edamontology.org/topic_0102                                        Mapping\n",
       "121        http://edamontology.org/data_2044                                                          Sequence  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "122        http://edamontology.org/data_0849                                                   Sequence record  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "123        http://edamontology.org/data_2976                                                  Protein sequence  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "124        http://edamontology.org/data_2977                                             Nucleic acid sequence  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "125        http://edamontology.org/data_2886                                           Protein sequence record  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "126        http://edamontology.org/data_2887                                      Nucleic acid sequence record  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "127        http://edamontology.org/data_3494                                                      DNA sequence  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "128        http://edamontology.org/data_3495                                                      RNA sequence  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "129        http://edamontology.org/data_2223                                                 Ontology metadata  http://edamontology.org/topic_0089                       Ontology and terminology\n",
       "130        http://edamontology.org/data_2353                                                     Ontology data  http://edamontology.org/topic_0089                       Ontology and terminology\n",
       "131        http://edamontology.org/data_0966                                                     Ontology term  http://edamontology.org/topic_0089                       Ontology and terminology\n",
       "132        http://edamontology.org/data_0967                                             Ontology concept data  http://edamontology.org/topic_0089                       Ontology and terminology\n",
       "133        http://edamontology.org/data_2858                                                  Ontology concept  http://edamontology.org/topic_0089                       Ontology and terminology\n",
       "134        http://edamontology.org/data_1731                                       Ontology concept definition  http://edamontology.org/topic_0089                       Ontology and terminology\n",
       "135        http://edamontology.org/data_0970                                                          Citation  http://edamontology.org/topic_3068                        Literature and language\n",
       "136        http://edamontology.org/data_2526                                                         Text data  http://edamontology.org/topic_3068                        Literature and language\n",
       "137        http://edamontology.org/data_0971                                                           Article  http://edamontology.org/topic_3068                        Literature and language\n",
       "138        http://edamontology.org/data_3671                                                              Text  http://edamontology.org/topic_3068                        Literature and language\n",
       "139        http://edamontology.org/data_0972                                                Text mining report  http://edamontology.org/topic_3068                        Literature and language\n",
       "140        http://edamontology.org/data_2849                                                          Abstract  http://edamontology.org/topic_3068                        Literature and language\n",
       "141        http://edamontology.org/data_3779                                                    Annotated text  http://edamontology.org/topic_3068                        Literature and language\n",
       "142        http://edamontology.org/data_3505                                                      Bibliography  http://edamontology.org/topic_3068                        Literature and language\n",
       "143        http://edamontology.org/data_2600                                                Pathway or network  http://edamontology.org/topic_0602  Molecular interactions, pathways and networks\n",
       "144        http://edamontology.org/data_0928                                           Gene expression profile  http://edamontology.org/topic_0203                                Gene expression\n",
       "145        http://edamontology.org/data_2603                                                   Expression data  http://edamontology.org/topic_0203                                Gene expression\n",
       "146        http://edamontology.org/data_2717                                  Oligonucleotide probe annotation  http://edamontology.org/topic_0203                                Gene expression\n",
       "147        http://edamontology.org/data_2535                                              Sequence tag profile  http://edamontology.org/topic_0203                                Gene expression\n",
       "148        http://edamontology.org/data_3111                                         Processed microarray data  http://edamontology.org/topic_0203                                Gene expression\n",
       "149        http://edamontology.org/data_1636                                                          Heat map  http://edamontology.org/topic_0203                                Gene expression\n",
       "150        http://edamontology.org/data_3768                                     Clustered expression profiles  http://edamontology.org/topic_0203                                Gene expression\n",
       "151        http://edamontology.org/data_1714                                            Microarray spots image  http://edamontology.org/topic_0203                                Gene expression\n",
       "152        http://edamontology.org/data_3115                                               Microarray metadata  http://edamontology.org/topic_0203                                Gene expression\n",
       "153        http://edamontology.org/data_3110                                               Raw microarray data  http://edamontology.org/topic_0203                                Gene expression\n",
       "154        http://edamontology.org/data_3117                                     Microarray hybridisation data  http://edamontology.org/topic_0203                                Gene expression\n",
       "155        http://edamontology.org/data_3112                                            Gene expression matrix  http://edamontology.org/topic_0203                                Gene expression\n",
       "156        http://edamontology.org/data_2717                                  Oligonucleotide probe annotation  http://edamontology.org/topic_0632                             Probes and primers\n",
       "157        http://edamontology.org/data_1355                                                 Protein signature  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "158        http://edamontology.org/data_2762                                         Sequence signature report  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "159        http://edamontology.org/data_3492                                            Nucleic acid signature  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "160        http://edamontology.org/data_2984                                         Pathway or network report  http://edamontology.org/topic_0602  Molecular interactions, pathways and networks\n",
       "161        http://edamontology.org/data_3028                                                          Taxonomy  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "162        http://edamontology.org/data_3442                                                         MRI image  http://edamontology.org/topic_3384                                Medical imaging\n",
       "163        http://edamontology.org/data_3449                                        Cell migration track image  http://edamontology.org/topic_2229                                   Cell biology\n",
       "164        http://edamontology.org/data_3488                                                      NMR spectrum  http://edamontology.org/topic_1317                             Structural biology\n",
       "165        http://edamontology.org/data_3498                                               Sequence variations  http://edamontology.org/topic_0199                              Genetic variation\n",
       "166        http://edamontology.org/data_3754                                           GO-term enrichment data  http://edamontology.org/topic_1775                              Function analysis\n",
       "167        http://edamontology.org/data_3805                                                         3D EM Map  http://edamontology.org/topic_1317                             Structural biology\n",
       "168        http://edamontology.org/data_3806                                                        3D EM Mask  http://edamontology.org/topic_1317                             Structural biology\n",
       "169        http://edamontology.org/data_3807                                                          EM Movie  http://edamontology.org/topic_1317                             Structural biology\n",
       "170        http://edamontology.org/data_3808                                                     EM Micrograph  http://edamontology.org/topic_1317                             Structural biology\n",
       "171        http://edamontology.org/data_3842                                         Molecular simulation data  http://edamontology.org/topic_0081                             Structure analysis\n",
       "172        http://edamontology.org/data_3953                                   Pathway overrepresentation data  http://edamontology.org/topic_1775                              Function analysis\n",
       "173   http://edamontology.org/operation_0226                                                        Annotation  http://edamontology.org/topic_0089                       Ontology and terminology\n",
       "174   http://edamontology.org/operation_0361                                               Sequence annotation  http://edamontology.org/topic_0089                       Ontology and terminology\n",
       "175   http://edamontology.org/operation_0362                                                 Genome annotation  http://edamontology.org/topic_0089                       Ontology and terminology\n",
       "176   http://edamontology.org/operation_0527                                              Sequence tag mapping  http://edamontology.org/topic_0089                       Ontology and terminology\n",
       "177   http://edamontology.org/operation_0558                                      Phylogenetic tree annotation  http://edamontology.org/topic_0089                       Ontology and terminology\n",
       "178   http://edamontology.org/operation_3778                                                   Text annotation  http://edamontology.org/topic_0089                       Ontology and terminology\n",
       "179   http://edamontology.org/operation_3553                                                  Image annotation  http://edamontology.org/topic_0089                       Ontology and terminology\n",
       "180   http://edamontology.org/operation_3661                                                    SNP annotation  http://edamontology.org/topic_0089                       Ontology and terminology\n",
       "181   http://edamontology.org/operation_3672                                        Gene functional annotation  http://edamontology.org/topic_0089                       Ontology and terminology\n",
       "182   http://edamontology.org/operation_0234                                   Sequence complexity calculation  http://edamontology.org/topic_0157   Sequence composition, complexity and repeats\n",
       "183   http://edamontology.org/operation_0235                                    Sequence ambiguity calculation  http://edamontology.org/topic_0157   Sequence composition, complexity and repeats\n",
       "184   http://edamontology.org/operation_0236                                  Sequence composition calculation  http://edamontology.org/topic_0157   Sequence composition, complexity and repeats\n",
       "185   http://edamontology.org/operation_0380                             Repeat sequence organisation analysis  http://edamontology.org/topic_0157   Sequence composition, complexity and repeats\n",
       "186   http://edamontology.org/operation_2284                                     Nucleic acid density plotting  http://edamontology.org/topic_0157   Sequence composition, complexity and repeats\n",
       "187   http://edamontology.org/operation_3472                                                    k-mer counting  http://edamontology.org/topic_0157   Sequence composition, complexity and repeats\n",
       "188   http://edamontology.org/operation_0237                                          Repeat sequence analysis  http://edamontology.org/topic_0157   Sequence composition, complexity and repeats\n",
       "189   http://edamontology.org/operation_0379                                         Repeat sequence detection  http://edamontology.org/topic_0157   Sequence composition, complexity and repeats\n",
       "190   http://edamontology.org/operation_0238                                          Sequence motif discovery  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "191   http://edamontology.org/operation_0239                                        Sequence motif recognition  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "192   http://edamontology.org/operation_0240                                         Sequence motif comparison  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "193   http://edamontology.org/operation_0245                                        Structural motif discovery  http://edamontology.org/topic_0166         Protein structural motifs and surfaces\n",
       "194   http://edamontology.org/operation_0246                                        Protein domain recognition  http://edamontology.org/topic_0736           Protein folds and structural domains\n",
       "195   http://edamontology.org/operation_0390                                                   Protein peeling  http://edamontology.org/topic_0736           Protein folds and structural domains\n",
       "196   http://edamontology.org/operation_0248                                   Residue interaction calculation  http://edamontology.org/topic_0130          Protein folding, stability and design\n",
       "197   http://edamontology.org/operation_0393                                       Residue cluster calculation  http://edamontology.org/topic_0130          Protein folding, stability and design\n",
       "198   http://edamontology.org/operation_0391                               Protein distance matrix calculation  http://edamontology.org/topic_0130          Protein folding, stability and design\n",
       "199   http://edamontology.org/operation_2950                                      Residue distance calculation  http://edamontology.org/topic_0130          Protein folding, stability and design\n",
       "200   http://edamontology.org/operation_0392                                           Contact map calculation  http://edamontology.org/topic_0130          Protein folding, stability and design\n",
       "201   http://edamontology.org/operation_0394                                         Hydrogen bond calculation  http://edamontology.org/topic_0130          Protein folding, stability and design\n",
       "202   http://edamontology.org/operation_1831                                    Metal-bound cysteine detection  http://edamontology.org/topic_0130          Protein folding, stability and design\n",
       "203   http://edamontology.org/operation_1834                                 Protein-metal contact calculation  http://edamontology.org/topic_0130          Protein folding, stability and design\n",
       "204   http://edamontology.org/operation_1839                                           Salt bridge calculation  http://edamontology.org/topic_0130          Protein folding, stability and design\n",
       "205   http://edamontology.org/operation_0252                                 Peptide immunogenicity prediction  http://edamontology.org/topic_0804                                     Immunology\n",
       "206   http://edamontology.org/operation_0253                                        Sequence feature detection  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "207   http://edamontology.org/operation_0256                                       Sequence feature comparison  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "208   http://edamontology.org/operation_3194                                         Genome feature comparison  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "209   http://edamontology.org/operation_0264                                   Alternative splicing prediction  http://edamontology.org/topic_0114                                 Gene structure\n",
       "210   http://edamontology.org/operation_0265                                              Frameshift detection  http://edamontology.org/topic_3168                                     Sequencing\n",
       "211   http://edamontology.org/operation_0269                                  Transmembrane protein prediction  http://edamontology.org/topic_0820                      Membrane and lipoproteins\n",
       "212   http://edamontology.org/operation_0270                                    Transmembrane protein analysis  http://edamontology.org/topic_0820                      Membrane and lipoproteins\n",
       "213   http://edamontology.org/operation_2241                               Transmembrane protein visualisation  http://edamontology.org/topic_0820                      Membrane and lipoproteins\n",
       "214   http://edamontology.org/operation_0272                                        Residue contact prediction  http://edamontology.org/topic_0130          Protein folding, stability and design\n",
       "215   http://edamontology.org/operation_0276                              Protein interaction network analysis  http://edamontology.org/topic_0128                           Protein interactions\n",
       "216   http://edamontology.org/operation_0282                                                   Genetic mapping  http://edamontology.org/topic_0102                                        Mapping\n",
       "217   http://edamontology.org/operation_0283                                                  Linkage analysis  http://edamontology.org/topic_0102                                        Mapping\n",
       "218   http://edamontology.org/operation_0487                                                 Haplotype mapping  http://edamontology.org/topic_0102                                        Mapping\n",
       "219   http://edamontology.org/operation_0488                                Linkage disequilibrium calculation  http://edamontology.org/topic_0102                                        Mapping\n",
       "220   http://edamontology.org/operation_0284                                      Codon usage table generation  http://edamontology.org/topic_0203                                Gene expression\n",
       "221   http://edamontology.org/operation_0286                                              Codon usage analysis  http://edamontology.org/topic_0203                                Gene expression\n",
       "222   http://edamontology.org/operation_0285                                      Codon usage table comparison  http://edamontology.org/topic_0203                                Gene expression\n",
       "223   http://edamontology.org/operation_0287                                Base position variability plotting  http://edamontology.org/topic_0203                                Gene expression\n",
       "224   http://edamontology.org/operation_0489                                           Genetic code prediction  http://edamontology.org/topic_0203                                Gene expression\n",
       "225   http://edamontology.org/operation_0550                                        DNA substitution modelling  http://edamontology.org/topic_0203                                Gene expression\n",
       "226   http://edamontology.org/operation_2962                                      Codon usage bias calculation  http://edamontology.org/topic_0203                                Gene expression\n",
       "227   http://edamontology.org/operation_2964                                  Codon usage fraction calculation  http://edamontology.org/topic_0203                                Gene expression\n",
       "228   http://edamontology.org/operation_3946                                              Ecological modelling  http://edamontology.org/topic_0203                                Gene expression\n",
       "229   http://edamontology.org/operation_0287                                Base position variability plotting  http://edamontology.org/topic_0114                                 Gene structure\n",
       "230   http://edamontology.org/operation_0289                               Sequence distance matrix generation  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "231   http://edamontology.org/operation_0296                                       Sequence profile generation  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "232   http://edamontology.org/operation_0300                                        Sequence profile alignment  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "233   http://edamontology.org/operation_0305                                                 Literature search  http://edamontology.org/topic_3068                        Literature and language\n",
       "234   http://edamontology.org/operation_2422                                                    Data retrieval  http://edamontology.org/topic_0218                    Natural language processing\n",
       "235   http://edamontology.org/operation_0306                                                       Text mining  http://edamontology.org/topic_0218                    Natural language processing\n",
       "236   http://edamontology.org/operation_3908                                             Information retrieval  http://edamontology.org/topic_0218                    Natural language processing\n",
       "237   http://edamontology.org/operation_3280                              Named-entity and concept recognition  http://edamontology.org/topic_0218                    Natural language processing\n",
       "238   http://edamontology.org/operation_3907                                            Information extraction  http://edamontology.org/topic_0218                    Natural language processing\n",
       "239   http://edamontology.org/operation_3625                                               Relation extraction  http://edamontology.org/topic_0218                    Natural language processing\n",
       "240   http://edamontology.org/operation_0308                                                 PCR primer design  http://edamontology.org/topic_0632                             Probes and primers\n",
       "241   http://edamontology.org/operation_0309                                           Microarray probe design  http://edamontology.org/topic_0203                                Gene expression\n",
       "242   http://edamontology.org/operation_0309                                           Microarray probe design  http://edamontology.org/topic_0632                             Probes and primers\n",
       "243   http://edamontology.org/operation_0310                                                 Sequence assembly  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "244   http://edamontology.org/operation_0523                                                  Mapping assembly  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "245   http://edamontology.org/operation_0524                                                  De-novo assembly  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "246   http://edamontology.org/operation_0525                                                   Genome assembly  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "247   http://edamontology.org/operation_0526                                                      EST assembly  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "248   http://edamontology.org/operation_3183                                              Localised reassembly  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "249   http://edamontology.org/operation_3216                                                       Scaffolding  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "250   http://edamontology.org/operation_3217                                           Scaffold gap completion  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "251   http://edamontology.org/operation_3229                                                    Exome assembly  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "252   http://edamontology.org/operation_3258                                            Transcriptome assembly  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "253   http://edamontology.org/operation_3730                                                    Cross-assembly  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "254   http://edamontology.org/operation_0319                            Protein secondary structure assignment  http://edamontology.org/topic_1317                             Structural biology\n",
       "255   http://edamontology.org/operation_0319                            Protein secondary structure assignment  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "256   http://edamontology.org/operation_0320                                      Protein structure assignment  http://edamontology.org/topic_1317                             Structural biology\n",
       "257   http://edamontology.org/operation_1829                                         Cysteine bridge detection  http://edamontology.org/topic_1317                             Structural biology\n",
       "258   http://edamontology.org/operation_1850                    Protein cysteine and disulfide bond assignment  http://edamontology.org/topic_1317                             Structural biology\n",
       "259   http://edamontology.org/operation_1830                                           Free cysteine detection  http://edamontology.org/topic_1317                             Structural biology\n",
       "260   http://edamontology.org/operation_1831                                    Metal-bound cysteine detection  http://edamontology.org/topic_1317                             Structural biology\n",
       "261   http://edamontology.org/operation_0321                                      Protein structure validation  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "262   http://edamontology.org/operation_1844                                       Protein geometry validation  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "263   http://edamontology.org/operation_1836                                            Residue bump detection  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "264   http://edamontology.org/operation_1843                                        Residue packing validation  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "265   http://edamontology.org/operation_3560                                        Maximum occurence analysis  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "266   http://edamontology.org/operation_0323                                            Phylogenetic inference  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "267   http://edamontology.org/operation_0327                                         Phylogenetic footprinting  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "268   http://edamontology.org/operation_0538                             Phylogenetic inference (data centric)  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "269   http://edamontology.org/operation_0539                           Phylogenetic inference (method centric)  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "270   http://edamontology.org/operation_0540                 Phylogenetic inference (from molecular sequences)  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "271   http://edamontology.org/operation_0541  Phylogenetic inference (from continuous quantitative characters)  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "272   http://edamontology.org/operation_0542                    Phylogenetic inference (from gene frequencies)  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "273   http://edamontology.org/operation_0543                   Phylogenetic inference (from polymorphism data)  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "274   http://edamontology.org/operation_0544                                         Species tree construction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "275   http://edamontology.org/operation_0545                        Phylogenetic inference (parsimony methods)  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "276   http://edamontology.org/operation_0546                 Phylogenetic inference (minimum distance methods)  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "277   http://edamontology.org/operation_0547  Phylogenetic inference (maximum likelihood and Bayesian methods)  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "278   http://edamontology.org/operation_0548                          Phylogenetic inference (quartet methods)  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "279   http://edamontology.org/operation_0549                               Phylogenetic inference (AI methods)  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "280   http://edamontology.org/operation_0553                                            Gene tree construction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "281   http://edamontology.org/operation_0555                                       Consensus tree construction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "282   http://edamontology.org/operation_3478                                       Phylogenetic reconstruction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "283   http://edamontology.org/operation_0323                                            Phylogenetic inference  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "284   http://edamontology.org/operation_0324                                             Phylogenetic analysis  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "285   http://edamontology.org/operation_0325                                      Phylogenetic tree comparison  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "286   http://edamontology.org/operation_0326                                         Phylogenetic tree editing  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "287   http://edamontology.org/operation_0327                                         Phylogenetic footprinting  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "288   http://edamontology.org/operation_0538                             Phylogenetic inference (data centric)  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "289   http://edamontology.org/operation_0539                           Phylogenetic inference (method centric)  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "290   http://edamontology.org/operation_0540                 Phylogenetic inference (from molecular sequences)  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "291   http://edamontology.org/operation_0541  Phylogenetic inference (from continuous quantitative characters)  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "292   http://edamontology.org/operation_0542                    Phylogenetic inference (from gene frequencies)  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "293   http://edamontology.org/operation_0543                   Phylogenetic inference (from polymorphism data)  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "294   http://edamontology.org/operation_0544                                         Species tree construction  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "295   http://edamontology.org/operation_0545                        Phylogenetic inference (parsimony methods)  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "296   http://edamontology.org/operation_0546                 Phylogenetic inference (minimum distance methods)  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "297   http://edamontology.org/operation_0547  Phylogenetic inference (maximum likelihood and Bayesian methods)  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "298   http://edamontology.org/operation_0548                          Phylogenetic inference (quartet methods)  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "299   http://edamontology.org/operation_0549                               Phylogenetic inference (AI methods)  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "300   http://edamontology.org/operation_0551                               Phylogenetic tree topology analysis  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "301   http://edamontology.org/operation_0552                                   Phylogenetic tree bootstrapping  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "302   http://edamontology.org/operation_0553                                            Gene tree construction  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "303   http://edamontology.org/operation_0554                            Allele frequency distribution analysis  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "304   http://edamontology.org/operation_0555                                       Consensus tree construction  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "305   http://edamontology.org/operation_0556                          Phylogenetic sub/super tree construction  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "306   http://edamontology.org/operation_0557                           Phylogenetic tree distances calculation  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "307   http://edamontology.org/operation_0567                                   Phylogenetic tree visualisation  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "308   http://edamontology.org/operation_3478                                       Phylogenetic reconstruction  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "309   http://edamontology.org/operation_3745                                          Ancestral reconstruction  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "310   http://edamontology.org/operation_3942                                                       Tree dating  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "311   http://edamontology.org/operation_3947                                  Phylogenetic tree reconciliation  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "312   http://edamontology.org/operation_0327                                         Phylogenetic footprinting  http://edamontology.org/topic_0194                                  Phylogenomics\n",
       "313   http://edamontology.org/operation_0331                                         Variant effect prediction  http://edamontology.org/topic_0130          Protein folding, stability and design\n",
       "314   http://edamontology.org/operation_0331                                         Variant effect prediction  http://edamontology.org/topic_0199                              Genetic variation\n",
       "315   http://edamontology.org/operation_0334                                       Enzyme kinetics calculation  http://edamontology.org/topic_0821                                        Enzymes\n",
       "316   http://edamontology.org/operation_0287                                Base position variability plotting  http://edamontology.org/topic_0092                             Data visualisation\n",
       "317   http://edamontology.org/operation_0564                                            Sequence visualisation  http://edamontology.org/topic_0092                             Data visualisation\n",
       "318   http://edamontology.org/operation_0337                                                     Visualisation  http://edamontology.org/topic_0092                             Data visualisation\n",
       "319   http://edamontology.org/operation_0402                                  Protein titration curve plotting  http://edamontology.org/topic_0092                             Data visualisation\n",
       "320   http://edamontology.org/operation_0407                               Protein hydrophobic moment plotting  http://edamontology.org/topic_0092                             Data visualisation\n",
       "321   http://edamontology.org/operation_0575                                           Restriction map drawing  http://edamontology.org/topic_0092                             Data visualisation\n",
       "322   http://edamontology.org/operation_0456                             Nucleic acid melting profile plotting  http://edamontology.org/topic_0092                             Data visualisation\n",
       "323   http://edamontology.org/operation_0457                              Nucleic acid stitch profile plotting  http://edamontology.org/topic_0092                             Data visualisation\n",
       "324   http://edamontology.org/operation_0458                               Nucleic acid melting curve plotting  http://edamontology.org/topic_0092                             Data visualisation\n",
       "325   http://edamontology.org/operation_0459                         Nucleic acid probability profile plotting  http://edamontology.org/topic_0092                             Data visualisation\n",
       "326   http://edamontology.org/operation_0460                         Nucleic acid temperature profile plotting  http://edamontology.org/topic_0092                             Data visualisation\n",
       "327   http://edamontology.org/operation_0490                                                 Dot plot plotting  http://edamontology.org/topic_0092                             Data visualisation\n",
       "328   http://edamontology.org/operation_0531                                               Heat map generation  http://edamontology.org/topic_0092                             Data visualisation\n",
       "329   http://edamontology.org/operation_0571                                     Expression data visualisation  http://edamontology.org/topic_0092                             Data visualisation\n",
       "330   http://edamontology.org/operation_0566                                    Sequence cluster visualisation  http://edamontology.org/topic_0092                             Data visualisation\n",
       "331   http://edamontology.org/operation_0567                                   Phylogenetic tree visualisation  http://edamontology.org/topic_0092                             Data visualisation\n",
       "332   http://edamontology.org/operation_0570                                           Structure visualisation  http://edamontology.org/topic_0092                             Data visualisation\n",
       "333   http://edamontology.org/operation_3925                                             Network visualisation  http://edamontology.org/topic_0092                             Data visualisation\n",
       "334   http://edamontology.org/operation_0573                                                       Map drawing  http://edamontology.org/topic_0092                             Data visualisation\n",
       "335   http://edamontology.org/operation_0578                                               Plasmid map drawing  http://edamontology.org/topic_0092                             Data visualisation\n",
       "336   http://edamontology.org/operation_0579                                                    Operon drawing  http://edamontology.org/topic_0092                             Data visualisation\n",
       "337   http://edamontology.org/operation_1816                                                 Surface rendering  http://edamontology.org/topic_0092                             Data visualisation\n",
       "338   http://edamontology.org/operation_2241                               Transmembrane protein visualisation  http://edamontology.org/topic_0092                             Data visualisation\n",
       "339   http://edamontology.org/operation_2284                                     Nucleic acid density plotting  http://edamontology.org/topic_0092                             Data visualisation\n",
       "340   http://edamontology.org/operation_2485                                             Helical wheel drawing  http://edamontology.org/topic_0092                             Data visualisation\n",
       "341   http://edamontology.org/operation_2486                                          Topology diagram drawing  http://edamontology.org/topic_0092                             Data visualisation\n",
       "342   http://edamontology.org/operation_2938                                          Dendrogram visualisation  http://edamontology.org/topic_0092                             Data visualisation\n",
       "343   http://edamontology.org/operation_2935                                       Clustering profile plotting  http://edamontology.org/topic_0092                             Data visualisation\n",
       "344   http://edamontology.org/operation_2937                                            Proximity map plotting  http://edamontology.org/topic_0092                             Data visualisation\n",
       "345   http://edamontology.org/operation_2939                                 Principal component visualisation  http://edamontology.org/topic_0092                             Data visualisation\n",
       "346   http://edamontology.org/operation_2940                                             Scatter plot plotting  http://edamontology.org/topic_0092                             Data visualisation\n",
       "347   http://edamontology.org/operation_2942                                             Treemap visualisation  http://edamontology.org/topic_0092                             Data visualisation\n",
       "348   http://edamontology.org/operation_2943                                         Box-Whisker plot plotting  http://edamontology.org/topic_0092                             Data visualisation\n",
       "349   http://edamontology.org/operation_3926                                             Pathway visualisation  http://edamontology.org/topic_0092                             Data visualisation\n",
       "350   http://edamontology.org/operation_3184                                   Sequence assembly visualisation  http://edamontology.org/topic_0092                             Data visualisation\n",
       "351   http://edamontology.org/operation_3203                                        Chromatogram visualisation  http://edamontology.org/topic_0092                             Data visualisation\n",
       "352   http://edamontology.org/operation_3208                                              Genome visualisation  http://edamontology.org/topic_0092                             Data visualisation\n",
       "353   http://edamontology.org/operation_3503                                           Incident curve plotting  http://edamontology.org/topic_0092                             Data visualisation\n",
       "354   http://edamontology.org/operation_3552                                    Microscope image visualisation  http://edamontology.org/topic_0092                             Data visualisation\n",
       "355   http://edamontology.org/operation_3559                                            Ontology visualisation  http://edamontology.org/topic_0092                             Data visualisation\n",
       "356   http://edamontology.org/operation_3694                                       Mass spectrum visualisation  http://edamontology.org/topic_0092                             Data visualisation\n",
       "357   http://edamontology.org/operation_3744                                     Multiple sample visualisation  http://edamontology.org/topic_0092                             Data visualisation\n",
       "358   http://edamontology.org/operation_3890                                          Trajectory visualization  http://edamontology.org/topic_0092                             Data visualisation\n",
       "359   http://edamontology.org/operation_0339                                         Structure database search  http://edamontology.org/topic_0081                             Structure analysis\n",
       "360   http://edamontology.org/operation_0346                                        Sequence similarity search  http://edamontology.org/topic_0081                             Structure analysis\n",
       "361   http://edamontology.org/operation_0360                                      Structural similarity search  http://edamontology.org/topic_0081                             Structure analysis\n",
       "362   http://edamontology.org/operation_0366                                         Protein sequence cleavage  http://edamontology.org/topic_0121                                     Proteomics\n",
       "363   http://edamontology.org/operation_0371                                                   DNA translation  http://edamontology.org/topic_0108                             Protein expression\n",
       "364   http://edamontology.org/operation_0372                                                 DNA transcription  http://edamontology.org/topic_0203                                Gene expression\n",
       "365   http://edamontology.org/operation_0389                         Protein-nucleic acid interaction analysis  http://edamontology.org/topic_0128                           Protein interactions\n",
       "366   http://edamontology.org/operation_3900                                    DNA-binding protein prediction  http://edamontology.org/topic_0128                           Protein interactions\n",
       "367   http://edamontology.org/operation_3901                                    RNA-binding protein prediction  http://edamontology.org/topic_0128                           Protein interactions\n",
       "368   http://edamontology.org/operation_2929                                Protein fragment weight comparison  http://edamontology.org/topic_0121                                     Proteomics\n",
       "369   http://edamontology.org/operation_0398                              Protein molecular weight calculation  http://edamontology.org/topic_0121                                     Proteomics\n",
       "370   http://edamontology.org/operation_0400                                           Protein pKa calculation  http://edamontology.org/topic_0123                             Protein properties\n",
       "371   http://edamontology.org/operation_0402                                  Protein titration curve plotting  http://edamontology.org/topic_0123                             Protein properties\n",
       "372   http://edamontology.org/operation_0403                             Protein isoelectric point calculation  http://edamontology.org/topic_0123                             Protein properties\n",
       "373   http://edamontology.org/operation_0404                        Protein hydrogen exchange rate calculation  http://edamontology.org/topic_0123                             Protein properties\n",
       "374   http://edamontology.org/operation_0266                                         Vector sequence detection  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "375   http://edamontology.org/operation_0415                                    Nucleic acid feature detection  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "376   http://edamontology.org/operation_0278                                RNA secondary structure prediction  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "377   http://edamontology.org/operation_0379                                         Repeat sequence detection  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "378   http://edamontology.org/operation_0427                                             Transposon prediction  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "379   http://edamontology.org/operation_0428                                            PolyA signal detection  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "380   http://edamontology.org/operation_0429                               Quadruplex formation site detection  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "381   http://edamontology.org/operation_0430                                 CpG island and isochore detection  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "382   http://edamontology.org/operation_0431                                      Restriction site recognition  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "383   http://edamontology.org/operation_0432                                    Nucleosome position prediction  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "384   http://edamontology.org/operation_3222                                                      Peak calling  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "385   http://edamontology.org/operation_3677                                     Differential binding analysis  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "386   http://edamontology.org/operation_4032                                       DNA modification prediction  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "387   http://edamontology.org/operation_0416                                                   Epitope mapping  http://edamontology.org/topic_0804                                     Immunology\n",
       "388   http://edamontology.org/operation_0417                   Post-translational modification site prediction  http://edamontology.org/topic_0601                          Protein modifications\n",
       "389   http://edamontology.org/operation_0418                                  Protein signal peptide detection  http://edamontology.org/topic_0140             Protein targeting and localisation\n",
       "390   http://edamontology.org/operation_0422                                  Protein cleavage site prediction  http://edamontology.org/topic_0121                                     Proteomics\n",
       "391   http://edamontology.org/operation_0430                                 CpG island and isochore detection  http://edamontology.org/topic_0157   Sequence composition, complexity and repeats\n",
       "392   http://edamontology.org/operation_0433                                            Splice site prediction  http://edamontology.org/topic_0114                                 Gene structure\n",
       "393   http://edamontology.org/operation_0438                     Transcriptional regulatory element prediction  http://edamontology.org/topic_0749     Transcription factors and regulatory sites\n",
       "394   http://edamontology.org/operation_0440                                               Promoter prediction  http://edamontology.org/topic_0749     Transcription factors and regulatory sites\n",
       "395   http://edamontology.org/operation_0441                                 cis-regulatory element prediction  http://edamontology.org/topic_0749     Transcription factors and regulatory sites\n",
       "396   http://edamontology.org/operation_0443                               trans-regulatory element prediction  http://edamontology.org/topic_0749     Transcription factors and regulatory sites\n",
       "397   http://edamontology.org/operation_0444                                                  S/MAR prediction  http://edamontology.org/topic_0749     Transcription factors and regulatory sites\n",
       "398   http://edamontology.org/operation_0445                      Transcription factor binding site prediction  http://edamontology.org/topic_0749     Transcription factors and regulatory sites\n",
       "399   http://edamontology.org/operation_0446                               Exonic splicing enhancer prediction  http://edamontology.org/topic_0749     Transcription factors and regulatory sites\n",
       "400   http://edamontology.org/operation_0463                                           miRNA target prediction  http://edamontology.org/topic_0749     Transcription factors and regulatory sites\n",
       "401   http://edamontology.org/operation_0465                              siRNA binding specificity prediction  http://edamontology.org/topic_0749     Transcription factors and regulatory sites\n",
       "402   http://edamontology.org/operation_2008                                           siRNA duplex prediction  http://edamontology.org/topic_0749     Transcription factors and regulatory sites\n",
       "403   http://edamontology.org/operation_0439                            Translation initiation site prediction  http://edamontology.org/topic_0108                             Protein expression\n",
       "404   http://edamontology.org/operation_0443                               trans-regulatory element prediction  http://edamontology.org/topic_0659      Functional, regulatory and non-coding RNA\n",
       "405   http://edamontology.org/operation_0463                                           miRNA target prediction  http://edamontology.org/topic_0659      Functional, regulatory and non-coding RNA\n",
       "406   http://edamontology.org/operation_0465                              siRNA binding specificity prediction  http://edamontology.org/topic_0659      Functional, regulatory and non-coding RNA\n",
       "407   http://edamontology.org/operation_2008                                           siRNA duplex prediction  http://edamontology.org/topic_0659      Functional, regulatory and non-coding RNA\n",
       "408   http://edamontology.org/operation_0446                               Exonic splicing enhancer prediction  http://edamontology.org/topic_0114                                 Gene structure\n",
       "409   http://edamontology.org/operation_0464                                              tRNA gene prediction  http://edamontology.org/topic_0659      Functional, regulatory and non-coding RNA\n",
       "410   http://edamontology.org/operation_0477                                                 Protein modelling  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "411   http://edamontology.org/operation_0479                                                Backbone modelling  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "412   http://edamontology.org/operation_0480                                              Side chain modelling  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "413   http://edamontology.org/operation_0481                                                    Loop modelling  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "414   http://edamontology.org/operation_0482                                            Protein-ligand docking  http://edamontology.org/topic_0128                           Protein interactions\n",
       "415   http://edamontology.org/operation_3938                                                 Virtual screening  http://edamontology.org/topic_0128                           Protein interactions\n",
       "416   http://edamontology.org/operation_0499                                     Tree-based sequence alignment  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "417   http://edamontology.org/operation_0542                    Phylogenetic inference (from gene frequencies)  http://edamontology.org/topic_0203                                Gene expression\n",
       "418   http://edamontology.org/operation_0543                   Phylogenetic inference (from polymorphism data)  http://edamontology.org/topic_0199                              Genetic variation\n",
       "419   http://edamontology.org/operation_0550                                        DNA substitution modelling  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "420   http://edamontology.org/operation_0553                                            Gene tree construction  http://edamontology.org/topic_0194                                  Phylogenomics\n",
       "421   http://edamontology.org/operation_0560                                                DNA vaccine design  http://edamontology.org/topic_0804                                     Immunology\n",
       "422   http://edamontology.org/operation_0579                                                    Operon drawing  http://edamontology.org/topic_0114                                 Gene structure\n",
       "423   http://edamontology.org/operation_0252                                 Peptide immunogenicity prediction  http://edamontology.org/topic_1775                              Function analysis\n",
       "424   http://edamontology.org/operation_1777                                       Protein function prediction  http://edamontology.org/topic_1775                              Function analysis\n",
       "425   http://edamontology.org/operation_2949                              Protein-protein interaction analysis  http://edamontology.org/topic_1775                              Function analysis\n",
       "426   http://edamontology.org/operation_2464                           Protein-protein binding site prediction  http://edamontology.org/topic_1775                              Function analysis\n",
       "427   http://edamontology.org/operation_0276                              Protein interaction network analysis  http://edamontology.org/topic_1775                              Function analysis\n",
       "428   http://edamontology.org/operation_0420                             Nucleic acids-binding site prediction  http://edamontology.org/topic_1775                              Function analysis\n",
       "429   http://edamontology.org/operation_2575                                           Binding site prediction  http://edamontology.org/topic_1775                              Function analysis\n",
       "430   http://edamontology.org/operation_0389                         Protein-nucleic acid interaction analysis  http://edamontology.org/topic_1775                              Function analysis\n",
       "431   http://edamontology.org/operation_0418                                  Protein signal peptide detection  http://edamontology.org/topic_1775                              Function analysis\n",
       "432   http://edamontology.org/operation_0478                                                 Molecular docking  http://edamontology.org/topic_1775                              Function analysis\n",
       "433   http://edamontology.org/operation_0482                                            Protein-ligand docking  http://edamontology.org/topic_1775                              Function analysis\n",
       "434   http://edamontology.org/operation_1778                                       Protein function comparison  http://edamontology.org/topic_1775                              Function analysis\n",
       "435   http://edamontology.org/operation_2489                               Subcellular localisation prediction  http://edamontology.org/topic_1775                              Function analysis\n",
       "436   http://edamontology.org/operation_2492                                    Protein interaction prediction  http://edamontology.org/topic_1775                              Function analysis\n",
       "437   http://edamontology.org/operation_3094                            Protein interaction network prediction  http://edamontology.org/topic_1775                              Function analysis\n",
       "438   http://edamontology.org/operation_3896                                            Active site prediction  http://edamontology.org/topic_1775                              Function analysis\n",
       "439   http://edamontology.org/operation_3897                                    Ligand-binding site prediction  http://edamontology.org/topic_1775                              Function analysis\n",
       "440   http://edamontology.org/operation_3898                                     Metal-binding site prediction  http://edamontology.org/topic_1775                              Function analysis\n",
       "441   http://edamontology.org/operation_3899                                           Protein-protein docking  http://edamontology.org/topic_1775                              Function analysis\n",
       "442   http://edamontology.org/operation_3900                                    DNA-binding protein prediction  http://edamontology.org/topic_1775                              Function analysis\n",
       "443   http://edamontology.org/operation_3901                                    RNA-binding protein prediction  http://edamontology.org/topic_1775                              Function analysis\n",
       "444   http://edamontology.org/operation_3902                                       RNA binding site prediction  http://edamontology.org/topic_1775                              Function analysis\n",
       "445   http://edamontology.org/operation_3903                                       DNA binding site prediction  http://edamontology.org/topic_1775                              Function analysis\n",
       "446   http://edamontology.org/operation_3938                                                 Virtual screening  http://edamontology.org/topic_1775                              Function analysis\n",
       "447   http://edamontology.org/operation_1781                                  Gene regulatory network analysis  http://edamontology.org/topic_0602  Molecular interactions, pathways and networks\n",
       "448   http://edamontology.org/operation_1829                                         Cysteine bridge detection  http://edamontology.org/topic_0130          Protein folding, stability and design\n",
       "449   http://edamontology.org/operation_1850                    Protein cysteine and disulfide bond assignment  http://edamontology.org/topic_0130          Protein folding, stability and design\n",
       "450   http://edamontology.org/operation_1830                                           Free cysteine detection  http://edamontology.org/topic_0130          Protein folding, stability and design\n",
       "451   http://edamontology.org/operation_2239                                   3D-1D scoring matrix generation  http://edamontology.org/topic_0081                             Structure analysis\n",
       "452   http://edamontology.org/operation_0230                                               Sequence generation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "453   http://edamontology.org/operation_2403                                                 Sequence analysis  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "454   http://edamontology.org/operation_0231                                                  Sequence editing  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "455   http://edamontology.org/operation_0232                                                  Sequence merging  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "456   http://edamontology.org/operation_0233                                               Sequence conversion  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "457   http://edamontology.org/operation_0234                                   Sequence complexity calculation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "458   http://edamontology.org/operation_0236                                  Sequence composition calculation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "459   http://edamontology.org/operation_0235                                    Sequence ambiguity calculation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "460   http://edamontology.org/operation_0237                                          Repeat sequence analysis  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "461   http://edamontology.org/operation_0238                                          Sequence motif discovery  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "462   http://edamontology.org/operation_0253                                        Sequence feature detection  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "463   http://edamontology.org/operation_2404                                           Sequence motif analysis  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "464   http://edamontology.org/operation_0239                                        Sequence motif recognition  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "465   http://edamontology.org/operation_0240                                         Sequence motif comparison  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "466   http://edamontology.org/operation_2451                                               Sequence comparison  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "467   http://edamontology.org/operation_0438                     Transcriptional regulatory element prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "468   http://edamontology.org/operation_0245                                        Structural motif discovery  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "469   http://edamontology.org/operation_3092                                         Protein feature detection  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "470   http://edamontology.org/operation_0246                                        Protein domain recognition  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "471   http://edamontology.org/operation_0256                                       Sequence feature comparison  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "472   http://edamontology.org/operation_0258                                       Sequence alignment analysis  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "473   http://edamontology.org/operation_0259                                     Sequence alignment comparison  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "474   http://edamontology.org/operation_0264                                   Alternative splicing prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "475   http://edamontology.org/operation_2499                                                 Splicing analysis  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "476   http://edamontology.org/operation_0265                                              Frameshift detection  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "477   http://edamontology.org/operation_3195                                        Sequencing error detection  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "478   http://edamontology.org/operation_3227                                                   Variant calling  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "479   http://edamontology.org/operation_0266                                         Vector sequence detection  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "480   http://edamontology.org/operation_0415                                    Nucleic acid feature detection  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "481   http://edamontology.org/operation_0267                            Protein secondary structure prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "482   http://edamontology.org/operation_2479                                         Protein sequence analysis  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "483   http://edamontology.org/operation_0268                      Protein super-secondary structure prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "484   http://edamontology.org/operation_0269                                  Transmembrane protein prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "485   http://edamontology.org/operation_0474                                      Protein structure prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "486   http://edamontology.org/operation_0272                                        Residue contact prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "487   http://edamontology.org/operation_2464                           Protein-protein binding site prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "488   http://edamontology.org/operation_0278                                RNA secondary structure prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "489   http://edamontology.org/operation_0282                                                   Genetic mapping  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "490   http://edamontology.org/operation_0283                                                  Linkage analysis  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "491   http://edamontology.org/operation_2478                                    Nucleic acid sequence analysis  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "492   http://edamontology.org/operation_0284                                      Codon usage table generation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "493   http://edamontology.org/operation_0286                                              Codon usage analysis  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "494   http://edamontology.org/operation_0285                                      Codon usage table comparison  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "495   http://edamontology.org/operation_0287                                Base position variability plotting  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "496   http://edamontology.org/operation_0564                                            Sequence visualisation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "497   http://edamontology.org/operation_0288                                          Sequence word comparison  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "498   http://edamontology.org/operation_0289                               Sequence distance matrix generation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "499   http://edamontology.org/operation_0290                                       Sequence redundancy removal  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "500   http://edamontology.org/operation_0291                                               Sequence clustering  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "501   http://edamontology.org/operation_0292                                                Sequence alignment  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "502   http://edamontology.org/operation_0294                                Structure-based sequence alignment  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "503   http://edamontology.org/operation_0296                                       Sequence profile generation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "504   http://edamontology.org/operation_0300                                        Sequence profile alignment  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "505   http://edamontology.org/operation_0303                                                  Fold recognition  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "506   http://edamontology.org/operation_0302                                                 Protein threading  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "507   http://edamontology.org/operation_0307                                                       Virtual PCR  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "508   http://edamontology.org/operation_0308                                                 PCR primer design  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "509   http://edamontology.org/operation_2419                                           Primer and probe design  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "510   http://edamontology.org/operation_0309                                           Microarray probe design  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "511   http://edamontology.org/operation_0310                                                 Sequence assembly  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "512   http://edamontology.org/operation_0420                             Nucleic acids-binding site prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "513   http://edamontology.org/operation_0338                                          Sequence database search  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "514   http://edamontology.org/operation_0346                                        Sequence similarity search  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "515   http://edamontology.org/operation_0349                            Sequence database search (by property)  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "516   http://edamontology.org/operation_0362                                                 Genome annotation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "517   http://edamontology.org/operation_3918                                                   Genome analysis  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "518   http://edamontology.org/operation_0363                                                Reverse complement  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "519   http://edamontology.org/operation_0364                                        Random sequence generation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "520   http://edamontology.org/operation_0365                                                Restriction digest  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "521   http://edamontology.org/operation_0366                                         Protein sequence cleavage  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "522   http://edamontology.org/operation_0367                               Sequence mutation and randomisation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "523   http://edamontology.org/operation_0368                                                  Sequence masking  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "524   http://edamontology.org/operation_0369                                                  Sequence cutting  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "525   http://edamontology.org/operation_0370                                         Restriction site creation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "526   http://edamontology.org/operation_0371                                                   DNA translation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "527   http://edamontology.org/operation_0372                                                 DNA transcription  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "528   http://edamontology.org/operation_0379                                         Repeat sequence detection  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "529   http://edamontology.org/operation_0380                             Repeat sequence organisation analysis  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "530   http://edamontology.org/operation_2575                                           Binding site prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "531   http://edamontology.org/operation_0390                                                   Protein peeling  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "532   http://edamontology.org/operation_0407                               Protein hydrophobic moment plotting  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "533   http://edamontology.org/operation_0418                                  Protein signal peptide detection  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "534   http://edamontology.org/operation_0416                                                   Epitope mapping  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "535   http://edamontology.org/operation_0417                   Post-translational modification site prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "536   http://edamontology.org/operation_0422                                  Protein cleavage site prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "537   http://edamontology.org/operation_2454                                                   Gene prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "538   http://edamontology.org/operation_0427                                             Transposon prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "539   http://edamontology.org/operation_0428                                            PolyA signal detection  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "540   http://edamontology.org/operation_0429                               Quadruplex formation site detection  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "541   http://edamontology.org/operation_0430                                 CpG island and isochore detection  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "542   http://edamontology.org/operation_0431                                      Restriction site recognition  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "543   http://edamontology.org/operation_0432                                    Nucleosome position prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "544   http://edamontology.org/operation_0433                                            Splice site prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "545   http://edamontology.org/operation_0435                                                 Operon prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "546   http://edamontology.org/operation_0436                                          Coding region prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "547   http://edamontology.org/operation_0437                                          SECIS element prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "548   http://edamontology.org/operation_0439                            Translation initiation site prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "549   http://edamontology.org/operation_0440                                               Promoter prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "550   http://edamontology.org/operation_0441                                 cis-regulatory element prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "551   http://edamontology.org/operation_0443                               trans-regulatory element prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "552   http://edamontology.org/operation_0444                                                  S/MAR prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "553   http://edamontology.org/operation_0445                      Transcription factor binding site prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "554   http://edamontology.org/operation_0446                               Exonic splicing enhancer prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "555   http://edamontology.org/operation_0447                                     Sequence alignment validation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "556   http://edamontology.org/operation_0448                        Sequence alignment analysis (conservation)  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "557   http://edamontology.org/operation_0449                    Sequence alignment analysis (site correlation)  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "558   http://edamontology.org/operation_0450                                                 Chimera detection  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "559   http://edamontology.org/operation_0451                                           Recombination detection  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "560   http://edamontology.org/operation_0452                                                   Indel detection  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "561   http://edamontology.org/operation_0463                                           miRNA target prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "562   http://edamontology.org/operation_0464                                              tRNA gene prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "563   http://edamontology.org/operation_0465                              siRNA binding specificity prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "564   http://edamontology.org/operation_0468                  Protein secondary structure prediction (helices)  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "565   http://edamontology.org/operation_0469                    Protein secondary structure prediction (turns)  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "566   http://edamontology.org/operation_0470                    Protein secondary structure prediction (coils)  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "567   http://edamontology.org/operation_0471                                         Disulfide bond prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "568   http://edamontology.org/operation_0476                                    Ab initio structure prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "569   http://edamontology.org/operation_0477                                                 Protein modelling  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "570   http://edamontology.org/operation_0479                                                Backbone modelling  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "571   http://edamontology.org/operation_0480                                              Side chain modelling  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "572   http://edamontology.org/operation_0481                                                    Loop modelling  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "573   http://edamontology.org/operation_0483                                               RNA inverse folding  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "574   http://edamontology.org/operation_3095                                               Nucleic acid design  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "575   http://edamontology.org/operation_0484                                                     SNP detection  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "576   http://edamontology.org/operation_0487                                                 Haplotype mapping  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "577   http://edamontology.org/operation_0488                                Linkage disequilibrium calculation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "578   http://edamontology.org/operation_0489                                           Genetic code prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "579   http://edamontology.org/operation_0490                                                 Dot plot plotting  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "580   http://edamontology.org/operation_0491                                       Pairwise sequence alignment  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "581   http://edamontology.org/operation_0492                                       Multiple sequence alignment  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "582   http://edamontology.org/operation_0495                                                   Local alignment  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "583   http://edamontology.org/operation_0496                                                  Global alignment  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "584   http://edamontology.org/operation_0499                                     Tree-based sequence alignment  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "585   http://edamontology.org/operation_0523                                                  Mapping assembly  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "586   http://edamontology.org/operation_0524                                                  De-novo assembly  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "587   http://edamontology.org/operation_0525                                                   Genome assembly  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "588   http://edamontology.org/operation_0526                                                      EST assembly  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "589   http://edamontology.org/operation_0550                                        DNA substitution modelling  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "590   http://edamontology.org/operation_0560                                                DNA vaccine design  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "591   http://edamontology.org/operation_2008                                           siRNA duplex prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "592   http://edamontology.org/operation_2089                                     Sequence alignment refinement  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "593   http://edamontology.org/operation_2121                                             Sequence file editing  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "594   http://edamontology.org/operation_2233                            Representative sequence identification  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "595   http://edamontology.org/operation_2284                                     Nucleic acid density plotting  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "596   http://edamontology.org/operation_2995                                           Sequence classification  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "597   http://edamontology.org/operation_2510                                              DNA back-translation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "598   http://edamontology.org/operation_2962                                      Codon usage bias calculation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "599   http://edamontology.org/operation_2964                                  Codon usage fraction calculation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "600   http://edamontology.org/operation_3180                                      Sequence assembly validation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "601   http://edamontology.org/operation_3218                                        Sequencing quality control  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "602   http://edamontology.org/operation_3182                                                  Genome alignment  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "603   http://edamontology.org/operation_3183                                              Localised reassembly  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "604   http://edamontology.org/operation_3184                                   Sequence assembly visualisation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "605   http://edamontology.org/operation_3185                                                      Base-calling  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "606   http://edamontology.org/operation_3186                                                 Bisulfite mapping  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "607   http://edamontology.org/operation_3204                                              Methylation analysis  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "608   http://edamontology.org/operation_3187                                  Sequence contamination filtering  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "609   http://edamontology.org/operation_3192                                                 Sequence trimming  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "610   http://edamontology.org/operation_3194                                         Genome feature comparison  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "611   http://edamontology.org/operation_3198                                                      Read mapping  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "612   http://edamontology.org/operation_3921                                          Sequence read processing  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "613   http://edamontology.org/operation_3199                                                Split read mapping  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "614   http://edamontology.org/operation_3200                                                     DNA barcoding  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "615   http://edamontology.org/operation_3206                                 Whole genome methylation analysis  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "616   http://edamontology.org/operation_3207                                         Gene methylation analysis  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "617   http://edamontology.org/operation_3208                                              Genome visualisation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "618   http://edamontology.org/operation_3209                                                 Genome comparison  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "619   http://edamontology.org/operation_3211                                                   Genome indexing  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "620   http://edamontology.org/operation_3216                                                       Scaffolding  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "621   http://edamontology.org/operation_3217                                           Scaffold gap completion  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "622   http://edamontology.org/operation_3219                                               Read pre-processing  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "623   http://edamontology.org/operation_3221                                      Species frequency estimation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "624   http://edamontology.org/operation_3222                                                      Peak calling  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "625   http://edamontology.org/operation_3225                                            Variant classification  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "626   http://edamontology.org/operation_3229                                                    Exome assembly  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "627   http://edamontology.org/operation_3230                                               Read depth analysis  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "628   http://edamontology.org/operation_3237                                                    Primer removal  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "629   http://edamontology.org/operation_3258                                            Transcriptome assembly  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "630   http://edamontology.org/operation_3459                                             Functional clustering  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "631   http://edamontology.org/operation_3460                                          Taxonomic classification  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "632   http://edamontology.org/operation_3461                                              Virulence prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "633   http://edamontology.org/operation_3472                                                    k-mer counting  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "634   http://edamontology.org/operation_3481                                 Probabilistic sequence generation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "635   http://edamontology.org/operation_3482                               Antimicrobial resistance prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "636   http://edamontology.org/operation_3563                                       RNA-seq read count analysis  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "637   http://edamontology.org/operation_3680                                                  RNA-Seq analysis  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "638   http://edamontology.org/operation_3565                                 RNA-seq time series data analysis  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "639   http://edamontology.org/operation_3644                                                de Novo sequencing  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "640   http://edamontology.org/operation_3662                                         Ab-initio gene prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "641   http://edamontology.org/operation_3663                                    Homology-based gene prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "642   http://edamontology.org/operation_3675                                                 Variant filtering  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "643   http://edamontology.org/operation_3677                                     Differential binding analysis  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "644   http://edamontology.org/operation_3730                                                    Cross-assembly  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "645   http://edamontology.org/operation_3793                                                Read summarisation  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "646   http://edamontology.org/operation_3795                                                In vitro selection  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "647   http://edamontology.org/operation_3798                                                      Read binning  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "648   http://edamontology.org/operation_3809                                                DMR identification  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "649   http://edamontology.org/operation_3894                                                     DNA profiling  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "650   http://edamontology.org/operation_3896                                            Active site prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "651   http://edamontology.org/operation_3897                                    Ligand-binding site prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "652   http://edamontology.org/operation_3898                                     Metal-binding site prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "653   http://edamontology.org/operation_3902                                       RNA binding site prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "654   http://edamontology.org/operation_3903                                       DNA binding site prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "655   http://edamontology.org/operation_3904                                       Protein disorder prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "656   http://edamontology.org/operation_3919                                               Methylation calling  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "657   http://edamontology.org/operation_3920                                                       DNA testing  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "658   http://edamontology.org/operation_3933                                                    Demultiplexing  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "659   http://edamontology.org/operation_3946                                              Ecological modelling  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "660   http://edamontology.org/operation_3950                                               Selection detection  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "661   http://edamontology.org/operation_3968                                                Adhesin prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "662   http://edamontology.org/operation_4032                                       DNA modification prediction  http://edamontology.org/topic_0080                              Sequence analysis\n",
       "663   http://edamontology.org/operation_2406                                        Protein structure analysis  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "664   http://edamontology.org/operation_0245                                        Structural motif discovery  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "665   http://edamontology.org/operation_0246                                        Protein domain recognition  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "666   http://edamontology.org/operation_0247                                     Protein architecture analysis  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "667   http://edamontology.org/operation_0267                            Protein secondary structure prediction  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "668   http://edamontology.org/operation_2416                              Protein secondary structure analysis  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "669   http://edamontology.org/operation_0268                      Protein super-secondary structure prediction  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "670   http://edamontology.org/operation_0269                                  Transmembrane protein prediction  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "671   http://edamontology.org/operation_0474                                      Protein structure prediction  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "672   http://edamontology.org/operation_0303                                                  Fold recognition  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "673   http://edamontology.org/operation_0302                                                 Protein threading  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "674   http://edamontology.org/operation_0320                                      Protein structure assignment  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "675   http://edamontology.org/operation_0321                                      Protein structure validation  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "676   http://edamontology.org/operation_2415                                          Protein folding analysis  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "677   http://edamontology.org/operation_0390                                                   Protein peeling  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "678   http://edamontology.org/operation_1844                                       Protein geometry validation  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "679   http://edamontology.org/operation_0468                  Protein secondary structure prediction (helices)  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "680   http://edamontology.org/operation_0469                    Protein secondary structure prediction (turns)  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "681   http://edamontology.org/operation_0470                    Protein secondary structure prediction (coils)  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "682   http://edamontology.org/operation_0476                                    Ab initio structure prediction  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "683   http://edamontology.org/operation_0477                                                 Protein modelling  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "684   http://edamontology.org/operation_0479                                                Backbone modelling  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "685   http://edamontology.org/operation_0480                                              Side chain modelling  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "686   http://edamontology.org/operation_0481                                                    Loop modelling  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "687   http://edamontology.org/operation_2488                            Protein secondary structure comparison  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "688   http://edamontology.org/operation_1829                                         Cysteine bridge detection  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "689   http://edamontology.org/operation_1850                    Protein cysteine and disulfide bond assignment  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "690   http://edamontology.org/operation_1830                                           Free cysteine detection  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "691   http://edamontology.org/operation_1831                                    Metal-bound cysteine detection  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "692   http://edamontology.org/operation_1836                                            Residue bump detection  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "693   http://edamontology.org/operation_1843                                        Residue packing validation  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "694   http://edamontology.org/operation_2474                                   Protein architecture comparison  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "695   http://edamontology.org/operation_2475                                  Protein architecture recognition  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "696   http://edamontology.org/operation_2487                                      Protein structure comparison  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "697   http://edamontology.org/operation_3560                                        Maximum occurence analysis  http://edamontology.org/topic_2814                     Protein structure analysis\n",
       "698   http://edamontology.org/operation_0224                                               Query and retrieval  http://edamontology.org/topic_3489                            Database management\n",
       "699   http://edamontology.org/operation_2409                                                     Data handling  http://edamontology.org/topic_3489                            Database management\n",
       "700   http://edamontology.org/operation_2422                                                    Data retrieval  http://edamontology.org/topic_3489                            Database management\n",
       "701   http://edamontology.org/operation_0231                                                  Sequence editing  http://edamontology.org/topic_3489                            Database management\n",
       "702   http://edamontology.org/operation_3096                                                           Editing  http://edamontology.org/topic_3489                            Database management\n",
       "703   http://edamontology.org/operation_0232                                                  Sequence merging  http://edamontology.org/topic_3489                            Database management\n",
       "704   http://edamontology.org/operation_0233                                               Sequence conversion  http://edamontology.org/topic_3489                            Database management\n",
       "705   http://edamontology.org/operation_0260                                     Sequence alignment conversion  http://edamontology.org/topic_3489                            Database management\n",
       "706   http://edamontology.org/operation_3081                                        Sequence alignment editing  http://edamontology.org/topic_3489                            Database management\n",
       "707   http://edamontology.org/operation_0305                                                 Literature search  http://edamontology.org/topic_3489                            Database management\n",
       "708   http://edamontology.org/operation_2421                                                   Database search  http://edamontology.org/topic_3489                            Database management\n",
       "709   http://edamontology.org/operation_0326                                         Phylogenetic tree editing  http://edamontology.org/topic_3489                            Database management\n",
       "710   http://edamontology.org/operation_0335                                                        Formatting  http://edamontology.org/topic_3489                            Database management\n",
       "711   http://edamontology.org/operation_0338                                          Sequence database search  http://edamontology.org/topic_3489                            Database management\n",
       "712   http://edamontology.org/operation_0339                                         Structure database search  http://edamontology.org/topic_3489                            Database management\n",
       "713   http://edamontology.org/operation_0346                                        Sequence similarity search  http://edamontology.org/topic_3489                            Database management\n",
       "714   http://edamontology.org/operation_0349                            Sequence database search (by property)  http://edamontology.org/topic_3489                            Database management\n",
       "715   http://edamontology.org/operation_0360                                      Structural similarity search  http://edamontology.org/topic_3489                            Database management\n",
       "716   http://edamontology.org/operation_0367                               Sequence mutation and randomisation  http://edamontology.org/topic_3489                            Database management\n",
       "717   http://edamontology.org/operation_0368                                                  Sequence masking  http://edamontology.org/topic_3489                            Database management\n",
       "718   http://edamontology.org/operation_0369                                                  Sequence cutting  http://edamontology.org/topic_3489                            Database management\n",
       "719   http://edamontology.org/operation_0370                                         Restriction site creation  http://edamontology.org/topic_3489                            Database management\n",
       "720   http://edamontology.org/operation_0371                                                   DNA translation  http://edamontology.org/topic_3489                            Database management\n",
       "721   http://edamontology.org/operation_0372                                                 DNA transcription  http://edamontology.org/topic_3489                            Database management\n",
       "722   http://edamontology.org/operation_3431                                                        Deposition  http://edamontology.org/topic_3489                            Database management\n",
       "723   http://edamontology.org/operation_1812                                                           Parsing  http://edamontology.org/topic_3489                            Database management\n",
       "724   http://edamontology.org/operation_2121                                             Sequence file editing  http://edamontology.org/topic_3489                            Database management\n",
       "725   http://edamontology.org/operation_2510                                              DNA back-translation  http://edamontology.org/topic_3489                            Database management\n",
       "726   http://edamontology.org/operation_3080                                                 Structure editing  http://edamontology.org/topic_3489                            Database management\n",
       "727   http://edamontology.org/operation_3192                                                 Sequence trimming  http://edamontology.org/topic_3489                            Database management\n",
       "728   http://edamontology.org/operation_3237                                                    Primer removal  http://edamontology.org/topic_3489                            Database management\n",
       "729   http://edamontology.org/operation_3283                                                     Anonymisation  http://edamontology.org/topic_3489                            Database management\n",
       "730   http://edamontology.org/operation_3357                                                  Format detection  http://edamontology.org/topic_3489                            Database management\n",
       "731   http://edamontology.org/operation_3359                                                         Splitting  http://edamontology.org/topic_3489                            Database management\n",
       "732   http://edamontology.org/operation_3436                                                       Aggregation  http://edamontology.org/topic_3489                            Database management\n",
       "733   http://edamontology.org/operation_3646                                           Peptide database search  http://edamontology.org/topic_3489                            Database management\n",
       "734   http://edamontology.org/operation_3647                                     Blind peptide database search  http://edamontology.org/topic_3489                            Database management\n",
       "735   http://edamontology.org/operation_3649                                                      Target-Decoy  http://edamontology.org/topic_3489                            Database management\n",
       "736   http://edamontology.org/operation_3695                                                         Filtering  http://edamontology.org/topic_3489                            Database management\n",
       "737   http://edamontology.org/operation_3802                                                           Sorting  http://edamontology.org/topic_3489                            Database management\n",
       "738   http://edamontology.org/operation_2415                                          Protein folding analysis  http://edamontology.org/topic_0130          Protein folding, stability and design\n",
       "739   http://edamontology.org/operation_2419                                           Primer and probe design  http://edamontology.org/topic_0632                             Probes and primers\n",
       "740   http://edamontology.org/operation_0264                                   Alternative splicing prediction  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "741   http://edamontology.org/operation_2499                                                 Splicing analysis  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "742   http://edamontology.org/operation_0279                                     Nucleic acid folding analysis  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "743   http://edamontology.org/operation_2426                                          Modelling and simulation  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "744   http://edamontology.org/operation_2415                                          Protein folding analysis  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "745   http://edamontology.org/operation_0433                                            Splice site prediction  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "746   http://edamontology.org/operation_0550                                        DNA substitution modelling  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "747   http://edamontology.org/operation_2476                                                Molecular dynamics  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "748   http://edamontology.org/operation_3566                         Simulated gene expression data generation  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "749   http://edamontology.org/operation_3893                                       Forcefield parameterisation  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "750   http://edamontology.org/operation_3946                                              Ecological modelling  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "751   http://edamontology.org/operation_4033                                     Disease transmission analysis  http://edamontology.org/topic_2275                            Molecular modelling\n",
       "752   http://edamontology.org/operation_2436                                      Gene-set enrichment analysis  http://edamontology.org/topic_1775                              Function analysis\n",
       "753   http://edamontology.org/operation_0278                                RNA secondary structure prediction  http://edamontology.org/topic_0097                Nucleic acid structure analysis\n",
       "754   http://edamontology.org/operation_2439                                  RNA secondary structure analysis  http://edamontology.org/topic_0097                Nucleic acid structure analysis\n",
       "755   http://edamontology.org/operation_0502                                 RNA secondary structure alignment  http://edamontology.org/topic_0097                Nucleic acid structure analysis\n",
       "756   http://edamontology.org/operation_3469                         RNA structure covariance model generation  http://edamontology.org/topic_0097                Nucleic acid structure analysis\n",
       "757   http://edamontology.org/operation_0438                     Transcriptional regulatory element prediction  http://edamontology.org/topic_0114                                 Gene structure\n",
       "758   http://edamontology.org/operation_2499                                                 Splicing analysis  http://edamontology.org/topic_0114                                 Gene structure\n",
       "759   http://edamontology.org/operation_2454                                                   Gene prediction  http://edamontology.org/topic_0114                                 Gene structure\n",
       "760   http://edamontology.org/operation_0435                                                 Operon prediction  http://edamontology.org/topic_0114                                 Gene structure\n",
       "761   http://edamontology.org/operation_0436                                          Coding region prediction  http://edamontology.org/topic_0114                                 Gene structure\n",
       "762   http://edamontology.org/operation_0437                                          SECIS element prediction  http://edamontology.org/topic_0114                                 Gene structure\n",
       "763   http://edamontology.org/operation_0439                            Translation initiation site prediction  http://edamontology.org/topic_0114                                 Gene structure\n",
       "764   http://edamontology.org/operation_0440                                               Promoter prediction  http://edamontology.org/topic_0114                                 Gene structure\n",
       "765   http://edamontology.org/operation_0441                                 cis-regulatory element prediction  http://edamontology.org/topic_0114                                 Gene structure\n",
       "766   http://edamontology.org/operation_0443                               trans-regulatory element prediction  http://edamontology.org/topic_0114                                 Gene structure\n",
       "767   http://edamontology.org/operation_0444                                                  S/MAR prediction  http://edamontology.org/topic_0114                                 Gene structure\n",
       "768   http://edamontology.org/operation_0445                      Transcription factor binding site prediction  http://edamontology.org/topic_0114                                 Gene structure\n",
       "769   http://edamontology.org/operation_0463                                           miRNA target prediction  http://edamontology.org/topic_0114                                 Gene structure\n",
       "770   http://edamontology.org/operation_0464                                              tRNA gene prediction  http://edamontology.org/topic_0114                                 Gene structure\n",
       "771   http://edamontology.org/operation_0465                              siRNA binding specificity prediction  http://edamontology.org/topic_0114                                 Gene structure\n",
       "772   http://edamontology.org/operation_2008                                           siRNA duplex prediction  http://edamontology.org/topic_0114                                 Gene structure\n",
       "773   http://edamontology.org/operation_3662                                         Ab-initio gene prediction  http://edamontology.org/topic_0114                                 Gene structure\n",
       "774   http://edamontology.org/operation_3663                                    Homology-based gene prediction  http://edamontology.org/topic_0114                                 Gene structure\n",
       "775   http://edamontology.org/operation_2464                           Protein-protein binding site prediction  http://edamontology.org/topic_0128                           Protein interactions\n",
       "776   http://edamontology.org/operation_2476                                                Molecular dynamics  http://edamontology.org/topic_0082                           Structure prediction\n",
       "777   http://edamontology.org/operation_2476                                                Molecular dynamics  http://edamontology.org/topic_0176                             Molecular dynamics\n",
       "778   http://edamontology.org/operation_0438                     Transcriptional regulatory element prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "779   http://edamontology.org/operation_0264                                   Alternative splicing prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "780   http://edamontology.org/operation_2499                                                 Splicing analysis  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "781   http://edamontology.org/operation_0265                                              Frameshift detection  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "782   http://edamontology.org/operation_3227                                                   Variant calling  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "783   http://edamontology.org/operation_0266                                         Vector sequence detection  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "784   http://edamontology.org/operation_0415                                    Nucleic acid feature detection  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "785   http://edamontology.org/operation_0278                                RNA secondary structure prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "786   http://edamontology.org/operation_0282                                                   Genetic mapping  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "787   http://edamontology.org/operation_0283                                                  Linkage analysis  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "788   http://edamontology.org/operation_2478                                    Nucleic acid sequence analysis  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "789   http://edamontology.org/operation_0284                                      Codon usage table generation  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "790   http://edamontology.org/operation_0286                                              Codon usage analysis  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "791   http://edamontology.org/operation_0285                                      Codon usage table comparison  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "792   http://edamontology.org/operation_0287                                Base position variability plotting  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "793   http://edamontology.org/operation_0307                                                       Virtual PCR  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "794   http://edamontology.org/operation_0308                                                 PCR primer design  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "795   http://edamontology.org/operation_2419                                           Primer and probe design  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "796   http://edamontology.org/operation_0309                                           Microarray probe design  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "797   http://edamontology.org/operation_0310                                                 Sequence assembly  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "798   http://edamontology.org/operation_0362                                                 Genome annotation  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "799   http://edamontology.org/operation_3918                                                   Genome analysis  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "800   http://edamontology.org/operation_0379                                         Repeat sequence detection  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "801   http://edamontology.org/operation_2454                                                   Gene prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "802   http://edamontology.org/operation_0427                                             Transposon prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "803   http://edamontology.org/operation_0428                                            PolyA signal detection  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "804   http://edamontology.org/operation_0429                               Quadruplex formation site detection  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "805   http://edamontology.org/operation_0430                                 CpG island and isochore detection  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "806   http://edamontology.org/operation_0431                                      Restriction site recognition  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "807   http://edamontology.org/operation_0432                                    Nucleosome position prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "808   http://edamontology.org/operation_0433                                            Splice site prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "809   http://edamontology.org/operation_0435                                                 Operon prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "810   http://edamontology.org/operation_0436                                          Coding region prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "811   http://edamontology.org/operation_0437                                          SECIS element prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "812   http://edamontology.org/operation_0439                            Translation initiation site prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "813   http://edamontology.org/operation_0440                                               Promoter prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "814   http://edamontology.org/operation_0441                                 cis-regulatory element prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "815   http://edamontology.org/operation_0443                               trans-regulatory element prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "816   http://edamontology.org/operation_0444                                                  S/MAR prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "817   http://edamontology.org/operation_0445                      Transcription factor binding site prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "818   http://edamontology.org/operation_0446                               Exonic splicing enhancer prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "819   http://edamontology.org/operation_0450                                                 Chimera detection  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "820   http://edamontology.org/operation_0451                                           Recombination detection  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "821   http://edamontology.org/operation_0452                                                   Indel detection  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "822   http://edamontology.org/operation_0463                                           miRNA target prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "823   http://edamontology.org/operation_0464                                              tRNA gene prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "824   http://edamontology.org/operation_0465                              siRNA binding specificity prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "825   http://edamontology.org/operation_0483                                               RNA inverse folding  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "826   http://edamontology.org/operation_3095                                               Nucleic acid design  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "827   http://edamontology.org/operation_0484                                                     SNP detection  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "828   http://edamontology.org/operation_0487                                                 Haplotype mapping  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "829   http://edamontology.org/operation_0488                                Linkage disequilibrium calculation  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "830   http://edamontology.org/operation_0489                                           Genetic code prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "831   http://edamontology.org/operation_0523                                                  Mapping assembly  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "832   http://edamontology.org/operation_0524                                                  De-novo assembly  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "833   http://edamontology.org/operation_0525                                                   Genome assembly  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "834   http://edamontology.org/operation_0526                                                      EST assembly  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "835   http://edamontology.org/operation_0550                                        DNA substitution modelling  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "836   http://edamontology.org/operation_0560                                                DNA vaccine design  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "837   http://edamontology.org/operation_2008                                           siRNA duplex prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "838   http://edamontology.org/operation_2962                                      Codon usage bias calculation  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "839   http://edamontology.org/operation_2964                                  Codon usage fraction calculation  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "840   http://edamontology.org/operation_3180                                      Sequence assembly validation  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "841   http://edamontology.org/operation_3218                                        Sequencing quality control  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "842   http://edamontology.org/operation_3182                                                  Genome alignment  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "843   http://edamontology.org/operation_3183                                              Localised reassembly  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "844   http://edamontology.org/operation_3186                                                 Bisulfite mapping  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "845   http://edamontology.org/operation_3204                                              Methylation analysis  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "846   http://edamontology.org/operation_3187                                  Sequence contamination filtering  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "847   http://edamontology.org/operation_3194                                         Genome feature comparison  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "848   http://edamontology.org/operation_3198                                                      Read mapping  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "849   http://edamontology.org/operation_3921                                          Sequence read processing  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "850   http://edamontology.org/operation_3199                                                Split read mapping  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "851   http://edamontology.org/operation_3200                                                     DNA barcoding  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "852   http://edamontology.org/operation_3206                                 Whole genome methylation analysis  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "853   http://edamontology.org/operation_3207                                         Gene methylation analysis  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "854   http://edamontology.org/operation_3208                                              Genome visualisation  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "855   http://edamontology.org/operation_3209                                                 Genome comparison  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "856   http://edamontology.org/operation_3211                                                   Genome indexing  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "857   http://edamontology.org/operation_3216                                                       Scaffolding  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "858   http://edamontology.org/operation_3217                                           Scaffold gap completion  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "859   http://edamontology.org/operation_3219                                               Read pre-processing  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "860   http://edamontology.org/operation_3221                                      Species frequency estimation  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "861   http://edamontology.org/operation_3222                                                      Peak calling  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "862   http://edamontology.org/operation_3229                                                    Exome assembly  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "863   http://edamontology.org/operation_3230                                               Read depth analysis  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "864   http://edamontology.org/operation_3258                                            Transcriptome assembly  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "865   http://edamontology.org/operation_3563                                       RNA-seq read count analysis  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "866   http://edamontology.org/operation_3680                                                  RNA-Seq analysis  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "867   http://edamontology.org/operation_3565                                 RNA-seq time series data analysis  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "868   http://edamontology.org/operation_3662                                         Ab-initio gene prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "869   http://edamontology.org/operation_3663                                    Homology-based gene prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "870   http://edamontology.org/operation_3675                                                 Variant filtering  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "871   http://edamontology.org/operation_3677                                     Differential binding analysis  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "872   http://edamontology.org/operation_3730                                                    Cross-assembly  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "873   http://edamontology.org/operation_3793                                                Read summarisation  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "874   http://edamontology.org/operation_3795                                                In vitro selection  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "875   http://edamontology.org/operation_3798                                                      Read binning  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "876   http://edamontology.org/operation_3809                                                DMR identification  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "877   http://edamontology.org/operation_3894                                                     DNA profiling  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "878   http://edamontology.org/operation_3919                                               Methylation calling  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "879   http://edamontology.org/operation_3920                                                       DNA testing  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "880   http://edamontology.org/operation_3933                                                    Demultiplexing  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "881   http://edamontology.org/operation_3946                                              Ecological modelling  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "882   http://edamontology.org/operation_3950                                               Selection detection  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "883   http://edamontology.org/operation_4032                                       DNA modification prediction  http://edamontology.org/topic_0077                                  Nucleic acids\n",
       "884   http://edamontology.org/operation_0245                                        Structural motif discovery  http://edamontology.org/topic_0078                                       Proteins\n",
       "885   http://edamontology.org/operation_3092                                         Protein feature detection  http://edamontology.org/topic_0078                                       Proteins\n",
       "886   http://edamontology.org/operation_0246                                        Protein domain recognition  http://edamontology.org/topic_0078                                       Proteins\n",
       "887   http://edamontology.org/operation_0267                            Protein secondary structure prediction  http://edamontology.org/topic_0078                                       Proteins\n",
       "888   http://edamontology.org/operation_2479                                         Protein sequence analysis  http://edamontology.org/topic_0078                                       Proteins\n",
       "889   http://edamontology.org/operation_0268                      Protein super-secondary structure prediction  http://edamontology.org/topic_0078                                       Proteins\n",
       "890   http://edamontology.org/operation_0269                                  Transmembrane protein prediction  http://edamontology.org/topic_0078                                       Proteins\n",
       "891   http://edamontology.org/operation_0474                                      Protein structure prediction  http://edamontology.org/topic_0078                                       Proteins\n",
       "892   http://edamontology.org/operation_0272                                        Residue contact prediction  http://edamontology.org/topic_0078                                       Proteins\n",
       "893   http://edamontology.org/operation_2464                           Protein-protein binding site prediction  http://edamontology.org/topic_0078                                       Proteins\n",
       "894   http://edamontology.org/operation_0303                                                  Fold recognition  http://edamontology.org/topic_0078                                       Proteins\n",
       "895   http://edamontology.org/operation_0302                                                 Protein threading  http://edamontology.org/topic_0078                                       Proteins\n",
       "896   http://edamontology.org/operation_0420                             Nucleic acids-binding site prediction  http://edamontology.org/topic_0078                                       Proteins\n",
       "897   http://edamontology.org/operation_2575                                           Binding site prediction  http://edamontology.org/topic_0078                                       Proteins\n",
       "898   http://edamontology.org/operation_0390                                                   Protein peeling  http://edamontology.org/topic_0078                                       Proteins\n",
       "899   http://edamontology.org/operation_0418                                  Protein signal peptide detection  http://edamontology.org/topic_0078                                       Proteins\n",
       "900   http://edamontology.org/operation_0416                                                   Epitope mapping  http://edamontology.org/topic_0078                                       Proteins\n",
       "901   http://edamontology.org/operation_0417                   Post-translational modification site prediction  http://edamontology.org/topic_0078                                       Proteins\n",
       "902   http://edamontology.org/operation_0422                                  Protein cleavage site prediction  http://edamontology.org/topic_0078                                       Proteins\n",
       "903   http://edamontology.org/operation_0468                  Protein secondary structure prediction (helices)  http://edamontology.org/topic_0078                                       Proteins\n",
       "904   http://edamontology.org/operation_0469                    Protein secondary structure prediction (turns)  http://edamontology.org/topic_0078                                       Proteins\n",
       "905   http://edamontology.org/operation_0470                    Protein secondary structure prediction (coils)  http://edamontology.org/topic_0078                                       Proteins\n",
       "906   http://edamontology.org/operation_0471                                         Disulfide bond prediction  http://edamontology.org/topic_0078                                       Proteins\n",
       "907   http://edamontology.org/operation_0476                                    Ab initio structure prediction  http://edamontology.org/topic_0078                                       Proteins\n",
       "908   http://edamontology.org/operation_0477                                                 Protein modelling  http://edamontology.org/topic_0078                                       Proteins\n",
       "909   http://edamontology.org/operation_0479                                                Backbone modelling  http://edamontology.org/topic_0078                                       Proteins\n",
       "910   http://edamontology.org/operation_0480                                              Side chain modelling  http://edamontology.org/topic_0078                                       Proteins\n",
       "911   http://edamontology.org/operation_0481                                                    Loop modelling  http://edamontology.org/topic_0078                                       Proteins\n",
       "912   http://edamontology.org/operation_3896                                            Active site prediction  http://edamontology.org/topic_0078                                       Proteins\n",
       "913   http://edamontology.org/operation_3897                                    Ligand-binding site prediction  http://edamontology.org/topic_0078                                       Proteins\n",
       "914   http://edamontology.org/operation_3898                                     Metal-binding site prediction  http://edamontology.org/topic_0078                                       Proteins\n",
       "915   http://edamontology.org/operation_3902                                       RNA binding site prediction  http://edamontology.org/topic_0078                                       Proteins\n",
       "916   http://edamontology.org/operation_3903                                       DNA binding site prediction  http://edamontology.org/topic_0078                                       Proteins\n",
       "917   http://edamontology.org/operation_3904                                       Protein disorder prediction  http://edamontology.org/topic_0078                                       Proteins\n",
       "918   http://edamontology.org/operation_3968                                                Adhesin prediction  http://edamontology.org/topic_0078                                       Proteins\n",
       "919   http://edamontology.org/operation_2406                                        Protein structure analysis  http://edamontology.org/topic_0081                             Structure analysis\n",
       "920   http://edamontology.org/operation_0245                                        Structural motif discovery  http://edamontology.org/topic_0081                             Structure analysis\n",
       "921   http://edamontology.org/operation_0246                                        Protein domain recognition  http://edamontology.org/topic_0081                             Structure analysis\n",
       "922   http://edamontology.org/operation_0247                                     Protein architecture analysis  http://edamontology.org/topic_0081                             Structure analysis\n",
       "923   http://edamontology.org/operation_0267                            Protein secondary structure prediction  http://edamontology.org/topic_0081                             Structure analysis\n",
       "924   http://edamontology.org/operation_2416                              Protein secondary structure analysis  http://edamontology.org/topic_0081                             Structure analysis\n",
       "925   http://edamontology.org/operation_0268                      Protein super-secondary structure prediction  http://edamontology.org/topic_0081                             Structure analysis\n",
       "926   http://edamontology.org/operation_0269                                  Transmembrane protein prediction  http://edamontology.org/topic_0081                             Structure analysis\n",
       "927   http://edamontology.org/operation_2480                                                Structure analysis  http://edamontology.org/topic_0081                             Structure analysis\n",
       "928   http://edamontology.org/operation_0474                                      Protein structure prediction  http://edamontology.org/topic_0081                             Structure analysis\n",
       "929   http://edamontology.org/operation_0475                                 Nucleic acid structure prediction  http://edamontology.org/topic_0081                             Structure analysis\n",
       "930   http://edamontology.org/operation_0278                                RNA secondary structure prediction  http://edamontology.org/topic_0081                             Structure analysis\n",
       "931   http://edamontology.org/operation_2439                                  RNA secondary structure analysis  http://edamontology.org/topic_0081                             Structure analysis\n",
       "932   http://edamontology.org/operation_0279                                     Nucleic acid folding analysis  http://edamontology.org/topic_0081                             Structure analysis\n",
       "933   http://edamontology.org/operation_2481                                   Nucleic acid structure analysis  http://edamontology.org/topic_0081                             Structure analysis\n",
       "934   http://edamontology.org/operation_0295                                               Structure alignment  http://edamontology.org/topic_0081                             Structure analysis\n",
       "935   http://edamontology.org/operation_2483                                              Structure comparison  http://edamontology.org/topic_0081                             Structure analysis\n",
       "936   http://edamontology.org/operation_0297                                             3D profile generation  http://edamontology.org/topic_0081                             Structure analysis\n",
       "937   http://edamontology.org/operation_0303                                                  Fold recognition  http://edamontology.org/topic_0081                             Structure analysis\n",
       "938   http://edamontology.org/operation_0302                                                 Protein threading  http://edamontology.org/topic_0081                             Structure analysis\n",
       "939   http://edamontology.org/operation_0319                            Protein secondary structure assignment  http://edamontology.org/topic_0081                             Structure analysis\n",
       "940   http://edamontology.org/operation_0320                                      Protein structure assignment  http://edamontology.org/topic_0081                             Structure analysis\n",
       "941   http://edamontology.org/operation_0321                                      Protein structure validation  http://edamontology.org/topic_0081                             Structure analysis\n",
       "942   http://edamontology.org/operation_0322                                        Molecular model refinement  http://edamontology.org/topic_0081                             Structure analysis\n",
       "943   http://edamontology.org/operation_2415                                          Protein folding analysis  http://edamontology.org/topic_0081                             Structure analysis\n",
       "944   http://edamontology.org/operation_0384                                    Accessible surface calculation  http://edamontology.org/topic_0081                             Structure analysis\n",
       "945   http://edamontology.org/operation_3351                                        Molecular surface analysis  http://edamontology.org/topic_0081                             Structure analysis\n",
       "946   http://edamontology.org/operation_0387                                     Molecular surface calculation  http://edamontology.org/topic_0081                             Structure analysis\n",
       "947   http://edamontology.org/operation_0390                                                   Protein peeling  http://edamontology.org/topic_0081                             Structure analysis\n",
       "948   http://edamontology.org/operation_1844                                       Protein geometry validation  http://edamontology.org/topic_0081                             Structure analysis\n",
       "949   http://edamontology.org/operation_0468                  Protein secondary structure prediction (helices)  http://edamontology.org/topic_0081                             Structure analysis\n",
       "950   http://edamontology.org/operation_0469                    Protein secondary structure prediction (turns)  http://edamontology.org/topic_0081                             Structure analysis\n",
       "951   http://edamontology.org/operation_0470                    Protein secondary structure prediction (coils)  http://edamontology.org/topic_0081                             Structure analysis\n",
       "952   http://edamontology.org/operation_0476                                    Ab initio structure prediction  http://edamontology.org/topic_0081                             Structure analysis\n",
       "953   http://edamontology.org/operation_0477                                                 Protein modelling  http://edamontology.org/topic_0081                             Structure analysis\n",
       "954   http://edamontology.org/operation_0478                                                 Molecular docking  http://edamontology.org/topic_0081                             Structure analysis\n",
       "955   http://edamontology.org/operation_0479                                                Backbone modelling  http://edamontology.org/topic_0081                             Structure analysis\n",
       "956   http://edamontology.org/operation_0480                                              Side chain modelling  http://edamontology.org/topic_0081                             Structure analysis\n",
       "957   http://edamontology.org/operation_0481                                                    Loop modelling  http://edamontology.org/topic_0081                             Structure analysis\n",
       "958   http://edamontology.org/operation_0482                                            Protein-ligand docking  http://edamontology.org/topic_0081                             Structure analysis\n",
       "959   http://edamontology.org/operation_2488                            Protein secondary structure comparison  http://edamontology.org/topic_0081                             Structure analysis\n",
       "960   http://edamontology.org/operation_0502                                 RNA secondary structure alignment  http://edamontology.org/topic_0081                             Structure analysis\n",
       "961   http://edamontology.org/operation_0503                                      Pairwise structure alignment  http://edamontology.org/topic_0081                             Structure analysis\n",
       "962   http://edamontology.org/operation_0504                                      Multiple structure alignment  http://edamontology.org/topic_0081                             Structure analysis\n",
       "963   http://edamontology.org/operation_0509                                         Local structure alignment  http://edamontology.org/topic_0081                             Structure analysis\n",
       "964   http://edamontology.org/operation_0510                                        Global structure alignment  http://edamontology.org/topic_0081                             Structure analysis\n",
       "965   http://edamontology.org/operation_0570                                           Structure visualisation  http://edamontology.org/topic_0081                             Structure analysis\n",
       "966   http://edamontology.org/operation_1816                                                 Surface rendering  http://edamontology.org/topic_0081                             Structure analysis\n",
       "967   http://edamontology.org/operation_1829                                         Cysteine bridge detection  http://edamontology.org/topic_0081                             Structure analysis\n",
       "968   http://edamontology.org/operation_1850                    Protein cysteine and disulfide bond assignment  http://edamontology.org/topic_0081                             Structure analysis\n",
       "969   http://edamontology.org/operation_1830                                           Free cysteine detection  http://edamontology.org/topic_0081                             Structure analysis\n",
       "970   http://edamontology.org/operation_1831                                    Metal-bound cysteine detection  http://edamontology.org/topic_0081                             Structure analysis\n",
       "971   http://edamontology.org/operation_1836                                            Residue bump detection  http://edamontology.org/topic_0081                             Structure analysis\n",
       "972   http://edamontology.org/operation_1843                                        Residue packing validation  http://edamontology.org/topic_0081                             Structure analysis\n",
       "973   http://edamontology.org/operation_2241                               Transmembrane protein visualisation  http://edamontology.org/topic_0081                             Structure analysis\n",
       "974   http://edamontology.org/operation_2441                                          RNA structure prediction  http://edamontology.org/topic_0081                             Structure analysis\n",
       "975   http://edamontology.org/operation_2442                                          DNA structure prediction  http://edamontology.org/topic_0081                             Structure analysis\n",
       "976   http://edamontology.org/operation_2474                                   Protein architecture comparison  http://edamontology.org/topic_0081                             Structure analysis\n",
       "977   http://edamontology.org/operation_2475                                  Protein architecture recognition  http://edamontology.org/topic_0081                             Structure analysis\n",
       "978   http://edamontology.org/operation_2996                                          Structure classification  http://edamontology.org/topic_0081                             Structure analysis\n",
       "979   http://edamontology.org/operation_2476                                                Molecular dynamics  http://edamontology.org/topic_0081                             Structure analysis\n",
       "980   http://edamontology.org/operation_2485                                             Helical wheel drawing  http://edamontology.org/topic_0081                             Structure analysis\n",
       "981   http://edamontology.org/operation_2486                                          Topology diagram drawing  http://edamontology.org/topic_0081                             Structure analysis\n",
       "982   http://edamontology.org/operation_2487                                      Protein structure comparison  http://edamontology.org/topic_0081                             Structure analysis\n",
       "983   http://edamontology.org/operation_2518                                 Nucleic acid structure comparison  http://edamontology.org/topic_0081                             Structure analysis\n",
       "984   http://edamontology.org/operation_3445                                         Diffraction data analysis  http://edamontology.org/topic_0081                             Structure analysis\n",
       "985   http://edamontology.org/operation_3447                                        Diffraction data reduction  http://edamontology.org/topic_0081                             Structure analysis\n",
       "986   http://edamontology.org/operation_3453                                      Diffraction data integration  http://edamontology.org/topic_0081                             Structure analysis\n",
       "987   http://edamontology.org/operation_3454                                                           Phasing  http://edamontology.org/topic_0081                             Structure analysis\n",
       "988   http://edamontology.org/operation_3455                                             Molecular replacement  http://edamontology.org/topic_0081                             Structure analysis\n",
       "989   http://edamontology.org/operation_3456                                             Rigid body refinement  http://edamontology.org/topic_0081                             Structure analysis\n",
       "990   http://edamontology.org/operation_3457                                          Single particle analysis  http://edamontology.org/topic_0081                             Structure analysis\n",
       "991   http://edamontology.org/operation_3458                      Single particle alignment and classification  http://edamontology.org/topic_0081                             Structure analysis\n",
       "992   http://edamontology.org/operation_3469                         RNA structure covariance model generation  http://edamontology.org/topic_0081                             Structure analysis\n",
       "993   http://edamontology.org/operation_3560                                        Maximum occurence analysis  http://edamontology.org/topic_0081                             Structure analysis\n",
       "994   http://edamontology.org/operation_3564                                       Chemical redundancy removal  http://edamontology.org/topic_0081                             Structure analysis\n",
       "995   http://edamontology.org/operation_3666                                      Molecular surface comparison  http://edamontology.org/topic_0081                             Structure analysis\n",
       "996   http://edamontology.org/operation_3890                                          Trajectory visualization  http://edamontology.org/topic_0081                             Structure analysis\n",
       "997   http://edamontology.org/operation_3891                                                Essential dynamics  http://edamontology.org/topic_0081                             Structure analysis\n",
       "998   http://edamontology.org/operation_3899                                           Protein-protein docking  http://edamontology.org/topic_0081                             Structure analysis\n",
       "999   http://edamontology.org/operation_3938                                                 Virtual screening  http://edamontology.org/topic_0081                             Structure analysis\n",
       "1000  http://edamontology.org/operation_0475                                 Nucleic acid structure prediction  http://edamontology.org/topic_0097                Nucleic acid structure analysis\n",
       "1001  http://edamontology.org/operation_0279                                     Nucleic acid folding analysis  http://edamontology.org/topic_0097                Nucleic acid structure analysis\n",
       "1002  http://edamontology.org/operation_2481                                   Nucleic acid structure analysis  http://edamontology.org/topic_0097                Nucleic acid structure analysis\n",
       "1003  http://edamontology.org/operation_2441                                          RNA structure prediction  http://edamontology.org/topic_0097                Nucleic acid structure analysis\n",
       "1004  http://edamontology.org/operation_2442                                          DNA structure prediction  http://edamontology.org/topic_0097                Nucleic acid structure analysis\n",
       "1005  http://edamontology.org/operation_2518                                 Nucleic acid structure comparison  http://edamontology.org/topic_0097                Nucleic acid structure analysis\n",
       "1006  http://edamontology.org/operation_3891                                                Essential dynamics  http://edamontology.org/topic_0097                Nucleic acid structure analysis\n",
       "1007  http://edamontology.org/operation_2489                               Subcellular localisation prediction  http://edamontology.org/topic_0140             Protein targeting and localisation\n",
       "1008  http://edamontology.org/operation_2492                                    Protein interaction prediction  http://edamontology.org/topic_0128                           Protein interactions\n",
       "1009  http://edamontology.org/operation_2495                                               Expression analysis  http://edamontology.org/topic_0203                                Gene expression\n",
       "1010  http://edamontology.org/operation_0313                                     Expression profile clustering  http://edamontology.org/topic_0203                                Gene expression\n",
       "1011  http://edamontology.org/operation_0315                                     Expression profile comparison  http://edamontology.org/topic_0203                                Gene expression\n",
       "1012  http://edamontology.org/operation_0314                                         Gene expression profiling  http://edamontology.org/topic_0203                                Gene expression\n",
       "1013  http://edamontology.org/operation_0531                                               Heat map generation  http://edamontology.org/topic_0203                                Gene expression\n",
       "1014  http://edamontology.org/operation_0571                                     Expression data visualisation  http://edamontology.org/topic_0203                                Gene expression\n",
       "1015  http://edamontology.org/operation_0533                                Expression profile pathway mapping  http://edamontology.org/topic_0203                                Gene expression\n",
       "1016  http://edamontology.org/operation_1781                                  Gene regulatory network analysis  http://edamontology.org/topic_0203                                Gene expression\n",
       "1017  http://edamontology.org/operation_2436                                      Gene-set enrichment analysis  http://edamontology.org/topic_0203                                Gene expression\n",
       "1018  http://edamontology.org/operation_2437                                Gene regulatory network prediction  http://edamontology.org/topic_0203                                Gene expression\n",
       "1019  http://edamontology.org/operation_2938                                          Dendrogram visualisation  http://edamontology.org/topic_0203                                Gene expression\n",
       "1020  http://edamontology.org/operation_2935                                       Clustering profile plotting  http://edamontology.org/topic_0203                                Gene expression\n",
       "1021  http://edamontology.org/operation_2937                                            Proximity map plotting  http://edamontology.org/topic_0203                                Gene expression\n",
       "1022  http://edamontology.org/operation_2942                                             Treemap visualisation  http://edamontology.org/topic_0203                                Gene expression\n",
       "1023  http://edamontology.org/operation_3223                            Differential gene expression profiling  http://edamontology.org/topic_0203                                Gene expression\n",
       "1024  http://edamontology.org/operation_3232                                      Gene expression QTL analysis  http://edamontology.org/topic_0203                                Gene expression\n",
       "1025  http://edamontology.org/operation_3463                                   Expression correlation analysis  http://edamontology.org/topic_0203                                Gene expression\n",
       "1026  http://edamontology.org/operation_3563                                       RNA-seq read count analysis  http://edamontology.org/topic_0203                                Gene expression\n",
       "1027  http://edamontology.org/operation_3680                                                  RNA-Seq analysis  http://edamontology.org/topic_0203                                Gene expression\n",
       "1028  http://edamontology.org/operation_3565                                 RNA-seq time series data analysis  http://edamontology.org/topic_0203                                Gene expression\n",
       "1029  http://edamontology.org/operation_3741                         Differential protein expression profiling  http://edamontology.org/topic_0203                                Gene expression\n",
       "1030  http://edamontology.org/operation_3792                                         miRNA expression analysis  http://edamontology.org/topic_0203                                Gene expression\n",
       "1031  http://edamontology.org/operation_2510                                              DNA back-translation  http://edamontology.org/topic_0108                             Protein expression\n",
       "1032  http://edamontology.org/operation_2520                                                       DNA mapping  http://edamontology.org/topic_0102                                        Mapping\n",
       "1033  http://edamontology.org/operation_0485                                          Radiation Hybrid Mapping  http://edamontology.org/topic_0102                                        Mapping\n",
       "1034  http://edamontology.org/operation_2944                                                  Physical mapping  http://edamontology.org/topic_0102                                        Mapping\n",
       "1035  http://edamontology.org/operation_0527                                              Sequence tag mapping  http://edamontology.org/topic_0102                                        Mapping\n",
       "1036  http://edamontology.org/operation_2871                                Sequence tagged site (STS) mapping  http://edamontology.org/topic_0102                                        Mapping\n",
       "1037  http://edamontology.org/operation_3186                                                 Bisulfite mapping  http://edamontology.org/topic_0102                                        Mapping\n",
       "1038  http://edamontology.org/operation_3198                                                      Read mapping  http://edamontology.org/topic_0102                                        Mapping\n",
       "1039  http://edamontology.org/operation_3199                                                Split read mapping  http://edamontology.org/topic_0102                                        Mapping\n",
       "1040  http://edamontology.org/operation_0282                                                   Genetic mapping  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "1041  http://edamontology.org/operation_2520                                                       DNA mapping  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "1042  http://edamontology.org/operation_0485                                          Radiation Hybrid Mapping  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "1043  http://edamontology.org/operation_2944                                                  Physical mapping  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "1044  http://edamontology.org/operation_0487                                                 Haplotype mapping  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "1045  http://edamontology.org/operation_0527                                              Sequence tag mapping  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "1046  http://edamontology.org/operation_2871                                Sequence tagged site (STS) mapping  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "1047  http://edamontology.org/operation_3186                                                 Bisulfite mapping  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "1048  http://edamontology.org/operation_3198                                                      Read mapping  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "1049  http://edamontology.org/operation_3199                                                Split read mapping  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "1050  http://edamontology.org/operation_2574                                    Protein hydropathy calculation  http://edamontology.org/topic_0123                             Protein properties\n",
       "1051  http://edamontology.org/operation_0405                            Protein hydrophobic region calculation  http://edamontology.org/topic_0123                             Protein properties\n",
       "1052  http://edamontology.org/operation_0406                               Protein aliphatic index calculation  http://edamontology.org/topic_0123                             Protein properties\n",
       "1053  http://edamontology.org/operation_0407                               Protein hydrophobic moment plotting  http://edamontology.org/topic_0123                             Protein properties\n",
       "1054  http://edamontology.org/operation_0408                                    Protein globularity prediction  http://edamontology.org/topic_0123                             Protein properties\n",
       "1055  http://edamontology.org/operation_0409                                     Protein solubility prediction  http://edamontology.org/topic_0123                             Protein properties\n",
       "1056  http://edamontology.org/operation_0410                              Protein crystallizability prediction  http://edamontology.org/topic_0123                             Protein properties\n",
       "1057  http://edamontology.org/operation_0420                             Nucleic acids-binding site prediction  http://edamontology.org/topic_0128                           Protein interactions\n",
       "1058  http://edamontology.org/operation_2575                                           Binding site prediction  http://edamontology.org/topic_0128                           Protein interactions\n",
       "1059  http://edamontology.org/operation_3896                                            Active site prediction  http://edamontology.org/topic_0128                           Protein interactions\n",
       "1060  http://edamontology.org/operation_3897                                    Ligand-binding site prediction  http://edamontology.org/topic_0128                           Protein interactions\n",
       "1061  http://edamontology.org/operation_3898                                     Metal-binding site prediction  http://edamontology.org/topic_0128                           Protein interactions\n",
       "1062  http://edamontology.org/operation_3902                                       RNA binding site prediction  http://edamontology.org/topic_0128                           Protein interactions\n",
       "1063  http://edamontology.org/operation_3903                                       DNA binding site prediction  http://edamontology.org/topic_0128                           Protein interactions\n",
       "1064  http://edamontology.org/operation_2949                              Protein-protein interaction analysis  http://edamontology.org/topic_0128                           Protein interactions\n",
       "1065  http://edamontology.org/operation_3094                            Protein interaction network prediction  http://edamontology.org/topic_0128                           Protein interactions\n",
       "1066  http://edamontology.org/operation_0245                                        Structural motif discovery  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1067  http://edamontology.org/operation_3092                                         Protein feature detection  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1068  http://edamontology.org/operation_0246                                        Protein domain recognition  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1069  http://edamontology.org/operation_0267                            Protein secondary structure prediction  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1070  http://edamontology.org/operation_0268                      Protein super-secondary structure prediction  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1071  http://edamontology.org/operation_0269                                  Transmembrane protein prediction  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1072  http://edamontology.org/operation_2464                           Protein-protein binding site prediction  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1073  http://edamontology.org/operation_0303                                                  Fold recognition  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1074  http://edamontology.org/operation_0302                                                 Protein threading  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1075  http://edamontology.org/operation_0420                             Nucleic acids-binding site prediction  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1076  http://edamontology.org/operation_2575                                           Binding site prediction  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1077  http://edamontology.org/operation_0390                                                   Protein peeling  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1078  http://edamontology.org/operation_0418                                  Protein signal peptide detection  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1079  http://edamontology.org/operation_0416                                                   Epitope mapping  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1080  http://edamontology.org/operation_0417                   Post-translational modification site prediction  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1081  http://edamontology.org/operation_0422                                  Protein cleavage site prediction  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1082  http://edamontology.org/operation_0468                  Protein secondary structure prediction (helices)  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1083  http://edamontology.org/operation_0469                    Protein secondary structure prediction (turns)  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1084  http://edamontology.org/operation_0470                    Protein secondary structure prediction (coils)  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1085  http://edamontology.org/operation_0471                                         Disulfide bond prediction  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1086  http://edamontology.org/operation_3896                                            Active site prediction  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1087  http://edamontology.org/operation_3897                                    Ligand-binding site prediction  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1088  http://edamontology.org/operation_3898                                     Metal-binding site prediction  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1089  http://edamontology.org/operation_3902                                       RNA binding site prediction  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1090  http://edamontology.org/operation_3903                                       DNA binding site prediction  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1091  http://edamontology.org/operation_3904                                       Protein disorder prediction  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1092  http://edamontology.org/operation_3968                                                Adhesin prediction  http://edamontology.org/topic_0160            Sequence sites, features and motifs\n",
       "1093  http://edamontology.org/operation_3180                                      Sequence assembly validation  http://edamontology.org/topic_0196                              Sequence assembly\n",
       "1094  http://edamontology.org/operation_3185                                                      Base-calling  http://edamontology.org/topic_3168                                     Sequencing\n",
       "1095  http://edamontology.org/operation_3195                                        Sequencing error detection  http://edamontology.org/topic_3168                                     Sequencing\n",
       "1096  http://edamontology.org/operation_3214                                                 Spectral analysis  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1097  http://edamontology.org/operation_3215                                                    Peak detection  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1098  http://edamontology.org/operation_3627                                          Mass spectra calibration  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1099  http://edamontology.org/operation_3628                                         Chromatographic alignment  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1100  http://edamontology.org/operation_3629                                                       Deisotoping  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1101  http://edamontology.org/operation_3630                                            Protein quantification  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1102  http://edamontology.org/operation_3631                                            Peptide identification  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1103  http://edamontology.org/operation_3634                                         Label-free quantification  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1104  http://edamontology.org/operation_3635                                            Labeled quantification  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1105  http://edamontology.org/operation_3636                                                           MRM/SRM  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1106  http://edamontology.org/operation_3637                                                 Spectral counting  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1107  http://edamontology.org/operation_3638                                                             SILAC  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1108  http://edamontology.org/operation_3639                                                             iTRAQ  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1109  http://edamontology.org/operation_3640                                                      18O labeling  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1110  http://edamontology.org/operation_3641                                                           TMT-tag  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1111  http://edamontology.org/operation_3642                                                          Dimethyl  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1112  http://edamontology.org/operation_3643                                  Tag-based peptide identification  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1113  http://edamontology.org/operation_3644                                                de Novo sequencing  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1114  http://edamontology.org/operation_3645                                                PTM identification  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1115  http://edamontology.org/operation_3646                                           Peptide database search  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1116  http://edamontology.org/operation_3647                                     Blind peptide database search  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1117  http://edamontology.org/operation_3649                                                      Target-Decoy  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1118  http://edamontology.org/operation_3704                                                      Ion counting  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1119  http://edamontology.org/operation_3705                                       Isotope-coded protein label  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1120  http://edamontology.org/operation_3715                                                Metabolic labeling  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1121  http://edamontology.org/operation_3755                                                  PTM localisation  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1122  http://edamontology.org/operation_3767                                            Protein identification  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1123  http://edamontology.org/operation_3801                                           Spectral library search  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1124  http://edamontology.org/operation_3803                                    Natural product identification  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1125  http://edamontology.org/operation_3860                                              Spectrum calculation  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1126  http://edamontology.org/operation_3221                                      Species frequency estimation  http://edamontology.org/topic_3174                                   Metagenomics\n",
       "1127  http://edamontology.org/operation_3348                                      Sequence checksum generation  http://edamontology.org/topic_3077                               Data acquisition\n",
       "1128  http://edamontology.org/operation_0384                                    Accessible surface calculation  http://edamontology.org/topic_0123                             Protein properties\n",
       "1129  http://edamontology.org/operation_3351                                        Molecular surface analysis  http://edamontology.org/topic_0123                             Protein properties\n",
       "1130  http://edamontology.org/operation_0387                                     Molecular surface calculation  http://edamontology.org/topic_0123                             Protein properties\n",
       "1131  http://edamontology.org/operation_1816                                                 Surface rendering  http://edamontology.org/topic_0123                             Protein properties\n",
       "1132  http://edamontology.org/operation_3666                                      Molecular surface comparison  http://edamontology.org/topic_0123                             Protein properties\n",
       "1133  http://edamontology.org/operation_0384                                    Accessible surface calculation  http://edamontology.org/topic_0166         Protein structural motifs and surfaces\n",
       "1134  http://edamontology.org/operation_3351                                        Molecular surface analysis  http://edamontology.org/topic_0166         Protein structural motifs and surfaces\n",
       "1135  http://edamontology.org/operation_0387                                     Molecular surface calculation  http://edamontology.org/topic_0166         Protein structural motifs and surfaces\n",
       "1136  http://edamontology.org/operation_1816                                                 Surface rendering  http://edamontology.org/topic_0166         Protein structural motifs and surfaces\n",
       "1137  http://edamontology.org/operation_3666                                      Molecular surface comparison  http://edamontology.org/topic_0166         Protein structural motifs and surfaces\n",
       "1138  http://edamontology.org/operation_3278                                   Document similarity calculation  http://edamontology.org/topic_3068                        Literature and language\n",
       "1139  http://edamontology.org/operation_3437                                                Article comparison  http://edamontology.org/topic_3068                        Literature and language\n",
       "1140  http://edamontology.org/operation_3279                                               Document clustering  http://edamontology.org/topic_3068                        Literature and language\n",
       "1141  http://edamontology.org/operation_3443                                                    Image analysis  http://edamontology.org/topic_3382                                        Imaging\n",
       "1142  http://edamontology.org/operation_3446                                           Cell migration analysis  http://edamontology.org/topic_3382                                        Imaging\n",
       "1143  http://edamontology.org/operation_3450                                               Neurite measurement  http://edamontology.org/topic_3382                                        Imaging\n",
       "1144  http://edamontology.org/operation_3457                                          Single particle analysis  http://edamontology.org/topic_3382                                        Imaging\n",
       "1145  http://edamontology.org/operation_3458                      Single particle alignment and classification  http://edamontology.org/topic_3382                                        Imaging\n",
       "1146  http://edamontology.org/operation_3446                                           Cell migration analysis  http://edamontology.org/topic_2229                                   Cell biology\n",
       "1147  http://edamontology.org/operation_3450                                               Neurite measurement  http://edamontology.org/topic_2229                                   Cell biology\n",
       "1148  http://edamontology.org/operation_3457                                          Single particle analysis  http://edamontology.org/topic_1317                             Structural biology\n",
       "1149  http://edamontology.org/operation_3458                      Single particle alignment and classification  http://edamontology.org/topic_1317                             Structural biology\n",
       "1150  http://edamontology.org/operation_3459                                             Functional clustering  http://edamontology.org/topic_1775                              Function analysis\n",
       "1151  http://edamontology.org/operation_3461                                              Virulence prediction  http://edamontology.org/topic_3301                                   Microbiology\n",
       "1152  http://edamontology.org/operation_0449                    Sequence alignment analysis (site correlation)  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1153  http://edamontology.org/operation_3465                                                       Correlation  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1154  http://edamontology.org/operation_3463                                   Expression correlation analysis  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1155  http://edamontology.org/operation_3482                               Antimicrobial resistance prediction  http://edamontology.org/topic_3301                                   Microbiology\n",
       "1156  http://edamontology.org/operation_3502                                    Chemical similarity enrichment  http://edamontology.org/topic_1775                              Function analysis\n",
       "1157  http://edamontology.org/operation_3552                                    Microscope image visualisation  http://edamontology.org/topic_3382                                        Imaging\n",
       "1158  http://edamontology.org/operation_3632                                Isotopic distributions calculation  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1159  http://edamontology.org/operation_3660                                       Metabolic network modelling  http://edamontology.org/topic_2259                                Systems biology\n",
       "1160  http://edamontology.org/operation_3766                             Weighted correlation network analysis  http://edamontology.org/topic_0602  Molecular interactions, pathways and networks\n",
       "1161  http://edamontology.org/operation_3778                                                   Text annotation  http://edamontology.org/topic_3068                        Literature and language\n",
       "1162  http://edamontology.org/operation_3893                                       Forcefield parameterisation  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "1163  http://edamontology.org/operation_3899                                           Protein-protein docking  http://edamontology.org/topic_0128                           Protein interactions\n",
       "1164  http://edamontology.org/operation_3904                                       Protein disorder prediction  http://edamontology.org/topic_3301                                   Microbiology\n",
       "1165  http://edamontology.org/operation_0362                                                 Genome annotation  http://edamontology.org/topic_0622                                       Genomics\n",
       "1166  http://edamontology.org/operation_3918                                                   Genome analysis  http://edamontology.org/topic_0622                                       Genomics\n",
       "1167  http://edamontology.org/operation_0525                                                   Genome assembly  http://edamontology.org/topic_0622                                       Genomics\n",
       "1168  http://edamontology.org/operation_3182                                                  Genome alignment  http://edamontology.org/topic_0622                                       Genomics\n",
       "1169  http://edamontology.org/operation_3194                                         Genome feature comparison  http://edamontology.org/topic_0622                                       Genomics\n",
       "1170  http://edamontology.org/operation_3206                                 Whole genome methylation analysis  http://edamontology.org/topic_0622                                       Genomics\n",
       "1171  http://edamontology.org/operation_3208                                              Genome visualisation  http://edamontology.org/topic_0622                                       Genomics\n",
       "1172  http://edamontology.org/operation_3209                                                 Genome comparison  http://edamontology.org/topic_0622                                       Genomics\n",
       "1173  http://edamontology.org/operation_3211                                                   Genome indexing  http://edamontology.org/topic_0622                                       Genomics\n",
       "1174  http://edamontology.org/operation_3920                                                       DNA testing  http://edamontology.org/topic_0622                                       Genomics\n",
       "1175  http://edamontology.org/operation_0276                              Protein interaction network analysis  http://edamontology.org/topic_0602  Molecular interactions, pathways and networks\n",
       "1176  http://edamontology.org/operation_3927                                                  Network analysis  http://edamontology.org/topic_0602  Molecular interactions, pathways and networks\n",
       "1177  http://edamontology.org/operation_3925                                             Network visualisation  http://edamontology.org/topic_0602  Molecular interactions, pathways and networks\n",
       "1178  http://edamontology.org/operation_2437                                Gene regulatory network prediction  http://edamontology.org/topic_0602  Molecular interactions, pathways and networks\n",
       "1179  http://edamontology.org/operation_3094                            Protein interaction network prediction  http://edamontology.org/topic_0602  Molecular interactions, pathways and networks\n",
       "1180  http://edamontology.org/operation_3660                                       Metabolic network modelling  http://edamontology.org/topic_0602  Molecular interactions, pathways and networks\n",
       "1181  http://edamontology.org/operation_0276                              Protein interaction network analysis  http://edamontology.org/topic_2259                                Systems biology\n",
       "1182  http://edamontology.org/operation_3927                                                  Network analysis  http://edamontology.org/topic_2259                                Systems biology\n",
       "1183  http://edamontology.org/operation_3925                                             Network visualisation  http://edamontology.org/topic_2259                                Systems biology\n",
       "1184  http://edamontology.org/operation_1781                                  Gene regulatory network analysis  http://edamontology.org/topic_2259                                Systems biology\n",
       "1185  http://edamontology.org/operation_2437                                Gene regulatory network prediction  http://edamontology.org/topic_2259                                Systems biology\n",
       "1186  http://edamontology.org/operation_3094                            Protein interaction network prediction  http://edamontology.org/topic_2259                                Systems biology\n",
       "1187  http://edamontology.org/operation_3766                             Weighted correlation network analysis  http://edamontology.org/topic_2259                                Systems biology\n",
       "1188  http://edamontology.org/operation_3928                                                  Pathway analysis  http://edamontology.org/topic_0602  Molecular interactions, pathways and networks\n",
       "1189  http://edamontology.org/operation_0533                                Expression profile pathway mapping  http://edamontology.org/topic_0602  Molecular interactions, pathways and networks\n",
       "1190  http://edamontology.org/operation_3926                                             Pathway visualisation  http://edamontology.org/topic_0602  Molecular interactions, pathways and networks\n",
       "1191  http://edamontology.org/operation_3929                                      Metabolic pathway prediction  http://edamontology.org/topic_0602  Molecular interactions, pathways and networks\n",
       "1192  http://edamontology.org/operation_3928                                                  Pathway analysis  http://edamontology.org/topic_2259                                Systems biology\n",
       "1193  http://edamontology.org/operation_0533                                Expression profile pathway mapping  http://edamontology.org/topic_2259                                Systems biology\n",
       "1194  http://edamontology.org/operation_3926                                             Pathway visualisation  http://edamontology.org/topic_2259                                Systems biology\n",
       "1195  http://edamontology.org/operation_3929                                      Metabolic pathway prediction  http://edamontology.org/topic_2259                                Systems biology\n",
       "1196  http://edamontology.org/operation_3935                                          Dimensionality reduction  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1197  http://edamontology.org/operation_3936                                                 Feature selection  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1198  http://edamontology.org/operation_3937                                                Feature extraction  http://edamontology.org/topic_0121                                     Proteomics\n",
       "1199  http://edamontology.org/operation_3946                                              Ecological modelling  http://edamontology.org/topic_0084                                      Phylogeny\n",
       "1200  http://edamontology.org/operation_3968                                                Adhesin prediction  http://edamontology.org/topic_0804                                     Immunology\n",
       "1201  http://edamontology.org/operation_4031                                                        Power test  http://edamontology.org/topic_3572                        Data quality management\n",
       "1202  http://edamontology.org/operation_4031                                                        Power test  http://edamontology.org/topic_3365         Data architecture, analysis and design\n",
       "1203  http://edamontology.org/operation_4031                                                        Power test  http://edamontology.org/topic_2269                     Statistics and probability\n",
       "1204  http://edamontology.org/operation_4032                                       DNA modification prediction  http://edamontology.org/topic_3295                                    Epigenetics\n",
       "1205  http://edamontology.org/operation_4033                                     Disease transmission analysis  http://edamontology.org/topic_3324                             Infectious disease\n",
       "1206  http://edamontology.org/operation_4033                                     Disease transmission analysis  http://edamontology.org/topic_3305                 Public health and epidemiology\n",
       "1207  http://edamontology.org/operation_4034                                       Multiple testing correction  http://edamontology.org/topic_2269                     Statistics and probability"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "print(f\"nb data/operation with has_topic property: {len(remoteQuery(query=q, endpoint=ep_edam))}\")\n",
    "remoteQuery(query=q, endpoint=ep_edam)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "3a9d11e7-0546-4e92-8dd8-616028fe8fd5",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3.9.5 64-bit",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.5"
  },
  "vscode": {
   "interpreter": {
    "hash": "31f2aee4e71d21fbe5cf8b01ff0e069b9275f58929596ceb00d14d90e3e16cd6"
   }
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
