{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Set up\n",
    "First you need to install GraphDB locally on you machine\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "from rdflib import ConjunctiveGraph\n",
    "from SPARQLWrapper import SPARQLWrapper, JSON\n",
    "import matplotlib.pyplot as plt\n",
    "from matplotlib_venn import venn2\n",
    "import numpy as np\n",
    "from IPython.display import display, HTML\n",
    "\n",
    "\n",
    "import pandas as pd\n",
    "\n",
    "def remoteQuery(query, endpoint):\n",
    "    endpoint.setQuery(query)\n",
    "    try:\n",
    "        result = endpoint.queryAndConvert()\n",
    "        pd.set_option(\"display.max_rows\",None,\"display.max_colwidth\",6000,\"display.width\",6000,)\n",
    "        df = pd.DataFrame(result['results']['bindings'])\n",
    "        df = df.applymap(lambda x: x['value'])\n",
    "        return df\n",
    "        #return (result['results']['bindings'])\n",
    "    except Exception as e:\n",
    "        print(e)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "ep_biotools = SPARQLWrapper(\"http://localhost:7200/repositories/Project25\")\n",
    "ep_biotools.setReturnFormat(JSON)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## List all EDAM formats with _is_format_of_ relations (transitive)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX biotools: <https://bio.tools/ontology/>\n",
    "PREFIX bsc: <http://bioschemas.org/>\n",
    "PREFIX bsct: <http://bioschemas.org/types/>\n",
    "PREFIX dct: <http://purl.org/dc/terms/>\n",
    "PREFIX edam: <http://edamontology.org/>\n",
    "PREFIX owl: <http://www.w3.org/2002/07/owl#>\n",
    "PREFIX sc: <http://schema.org/>\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>\n",
    "\n",
    "SELECT\n",
    "    ?format ?format_label\n",
    "    (COUNT(?is_format_of_data) as ?count)\n",
    "    #?is_format_of_data ?is_format_of_data_label\n",
    "WHERE\n",
    "{\n",
    "    ?format\n",
    "        rdfs:subClassOf [owl:onProperty edam:is_format_of ; owl:someValuesFrom ?is_format_of_data] ;\n",
    "        rdfs:label ?format_label .\n",
    "    ?is_format_of_data rdfs:label ?is_format_of_data_label .\n",
    "} GROUP BY ?format ?format_label\n",
    "ORDER BY DESC(?count) ASC(?format) ASC(?is_format_of_data)\n",
    "\"\"\"\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "nb of edam format with is_format_of relation: 533\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>format</th>\n",
       "      <th>format_label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/format_2352</td>\n",
       "      <td>BioXSD (XML)</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>http://edamontology.org/format_3772</td>\n",
       "      <td>BioJSON (BioXSD)</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>http://edamontology.org/format_3773</td>\n",
       "      <td>BioYAML</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>http://edamontology.org/format_2572</td>\n",
       "      <td>BAM</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>http://edamontology.org/format_2573</td>\n",
       "      <td>SAM</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>http://edamontology.org/format_3007</td>\n",
       "      <td>PSL</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>http://edamontology.org/format_3771</td>\n",
       "      <td>UniProtKB RDF</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>http://edamontology.org/format_3774</td>\n",
       "      <td>BioJSON (Jalview)</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>http://edamontology.org/format_3777</td>\n",
       "      <td>MCPD</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>http://edamontology.org/format_3826</td>\n",
       "      <td>proBAM</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>http://edamontology.org/format_3913</td>\n",
       "      <td>Loom</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>http://edamontology.org/format_3915</td>\n",
       "      <td>Zarr</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>http://edamontology.org/format_3916</td>\n",
       "      <td>MTX</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>http://edamontology.org/format_1927</td>\n",
       "      <td>EMBL format</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>http://edamontology.org/format_1936</td>\n",
       "      <td>GenBank format</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>http://edamontology.org/format_1938</td>\n",
       "      <td>GFF2-seq</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>http://edamontology.org/format_1939</td>\n",
       "      <td>GFF3-seq</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>http://edamontology.org/format_1950</td>\n",
       "      <td>pdbatom</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>http://edamontology.org/format_1951</td>\n",
       "      <td>pdbatomnuc</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>http://edamontology.org/format_1952</td>\n",
       "      <td>pdbseqresnuc</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>http://edamontology.org/format_1953</td>\n",
       "      <td>pdbseqres</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>http://edamontology.org/format_1963</td>\n",
       "      <td>UniProtKB format</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>http://edamontology.org/format_2187</td>\n",
       "      <td>UniProt-like (text)</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>http://edamontology.org/format_2547</td>\n",
       "      <td>uniprotkb-like format</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>http://edamontology.org/format_3008</td>\n",
       "      <td>MAF</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>http://edamontology.org/format_3019</td>\n",
       "      <td>GVF</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>http://edamontology.org/format_3157</td>\n",
       "      <td>EBI Application Result XML</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>http://edamontology.org/format_3164</td>\n",
       "      <td>GTrack</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>http://edamontology.org/format_3607</td>\n",
       "      <td>qual</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>http://edamontology.org/format_3608</td>\n",
       "      <td>qualsolexa</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>http://edamontology.org/format_3609</td>\n",
       "      <td>qualillumina</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>http://edamontology.org/format_3610</td>\n",
       "      <td>qualsolid</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>http://edamontology.org/format_3611</td>\n",
       "      <td>qual454</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>http://edamontology.org/format_3770</td>\n",
       "      <td>UniProtKB XML</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>http://edamontology.org/format_3776</td>\n",
       "      <td>BTrack</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>http://edamontology.org/format_3909</td>\n",
       "      <td>BpForms</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>http://edamontology.org/format_1351</td>\n",
       "      <td>HMMER emission and transition</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>http://edamontology.org/format_1369</td>\n",
       "      <td>MEME background Markov model</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>http://edamontology.org/format_1370</td>\n",
       "      <td>HMMER format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>http://edamontology.org/format_1391</td>\n",
       "      <td>HMMER-aln</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>http://edamontology.org/format_1430</td>\n",
       "      <td>Phylip continuous quantitative characters</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>http://edamontology.org/format_1432</td>\n",
       "      <td>Phylip character frequencies format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>http://edamontology.org/format_1433</td>\n",
       "      <td>Phylip discrete states format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>http://edamontology.org/format_1434</td>\n",
       "      <td>Phylip cliques format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>http://edamontology.org/format_1475</td>\n",
       "      <td>PDB database entry format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>http://edamontology.org/format_1476</td>\n",
       "      <td>PDB</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>http://edamontology.org/format_1477</td>\n",
       "      <td>mmCIF</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>http://edamontology.org/format_1478</td>\n",
       "      <td>PDBML</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>http://edamontology.org/format_1504</td>\n",
       "      <td>aaindex</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>http://edamontology.org/format_1628</td>\n",
       "      <td>ABI</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>http://edamontology.org/format_1629</td>\n",
       "      <td>mira</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51</th>\n",
       "      <td>http://edamontology.org/format_1632</td>\n",
       "      <td>SCF</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>http://edamontology.org/format_1633</td>\n",
       "      <td>PHD</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>53</th>\n",
       "      <td>http://edamontology.org/format_1637</td>\n",
       "      <td>dat</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>http://edamontology.org/format_1638</td>\n",
       "      <td>cel</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>55</th>\n",
       "      <td>http://edamontology.org/format_1644</td>\n",
       "      <td>CHP</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>56</th>\n",
       "      <td>http://edamontology.org/format_1734</td>\n",
       "      <td>PubMed citation</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>57</th>\n",
       "      <td>http://edamontology.org/format_1735</td>\n",
       "      <td>Medline Display Format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>58</th>\n",
       "      <td>http://edamontology.org/format_1736</td>\n",
       "      <td>CiteXplore-core</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>59</th>\n",
       "      <td>http://edamontology.org/format_1737</td>\n",
       "      <td>CiteXplore-all</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>60</th>\n",
       "      <td>http://edamontology.org/format_1740</td>\n",
       "      <td>iHOP format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>61</th>\n",
       "      <td>http://edamontology.org/format_1929</td>\n",
       "      <td>FASTA</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>62</th>\n",
       "      <td>http://edamontology.org/format_1930</td>\n",
       "      <td>FASTQ</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>63</th>\n",
       "      <td>http://edamontology.org/format_1931</td>\n",
       "      <td>FASTQ-illumina</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>64</th>\n",
       "      <td>http://edamontology.org/format_1932</td>\n",
       "      <td>FASTQ-sanger</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>65</th>\n",
       "      <td>http://edamontology.org/format_1933</td>\n",
       "      <td>FASTQ-solexa</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66</th>\n",
       "      <td>http://edamontology.org/format_1935</td>\n",
       "      <td>GCG</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>67</th>\n",
       "      <td>http://edamontology.org/format_1947</td>\n",
       "      <td>GCG MSF</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>68</th>\n",
       "      <td>http://edamontology.org/format_1949</td>\n",
       "      <td>nexus-seq</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69</th>\n",
       "      <td>http://edamontology.org/format_1973</td>\n",
       "      <td>nexusnon</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>70</th>\n",
       "      <td>http://edamontology.org/format_1974</td>\n",
       "      <td>GFF2</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71</th>\n",
       "      <td>http://edamontology.org/format_1975</td>\n",
       "      <td>GFF3</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72</th>\n",
       "      <td>http://edamontology.org/format_1978</td>\n",
       "      <td>DASGFF</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>73</th>\n",
       "      <td>http://edamontology.org/format_1984</td>\n",
       "      <td>FASTA-aln</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>74</th>\n",
       "      <td>http://edamontology.org/format_1991</td>\n",
       "      <td>mega</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75</th>\n",
       "      <td>http://edamontology.org/format_1992</td>\n",
       "      <td>meganon</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76</th>\n",
       "      <td>http://edamontology.org/format_1997</td>\n",
       "      <td>PHYLIP format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>77</th>\n",
       "      <td>http://edamontology.org/format_1998</td>\n",
       "      <td>PHYLIP sequential</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78</th>\n",
       "      <td>http://edamontology.org/format_2000</td>\n",
       "      <td>selex</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>79</th>\n",
       "      <td>http://edamontology.org/format_2003</td>\n",
       "      <td>srspair</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80</th>\n",
       "      <td>http://edamontology.org/format_2005</td>\n",
       "      <td>TreeCon-seq</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>81</th>\n",
       "      <td>http://edamontology.org/format_2017</td>\n",
       "      <td>Amino acid index format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>82</th>\n",
       "      <td>http://edamontology.org/format_2037</td>\n",
       "      <td>Phylogenetic continuous quantitative character format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>83</th>\n",
       "      <td>http://edamontology.org/format_2038</td>\n",
       "      <td>Phylogenetic discrete states format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>84</th>\n",
       "      <td>http://edamontology.org/format_2039</td>\n",
       "      <td>Phylogenetic tree report (cliques) format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>85</th>\n",
       "      <td>http://edamontology.org/format_2040</td>\n",
       "      <td>Phylogenetic tree report (invariants) format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>86</th>\n",
       "      <td>http://edamontology.org/format_2057</td>\n",
       "      <td>Sequence trace format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>87</th>\n",
       "      <td>http://edamontology.org/format_2064</td>\n",
       "      <td>3D-1D scoring matrix format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>88</th>\n",
       "      <td>http://edamontology.org/format_2072</td>\n",
       "      <td>Hidden Markov model format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>89</th>\n",
       "      <td>http://edamontology.org/format_2075</td>\n",
       "      <td>HMM emission and transition counts format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>90</th>\n",
       "      <td>http://edamontology.org/format_2182</td>\n",
       "      <td>FASTQ-like format (text)</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>91</th>\n",
       "      <td>http://edamontology.org/format_2206</td>\n",
       "      <td>Sequence feature table format (text)</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>92</th>\n",
       "      <td>http://edamontology.org/format_2305</td>\n",
       "      <td>GFF</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>93</th>\n",
       "      <td>http://edamontology.org/format_2306</td>\n",
       "      <td>GTF</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>94</th>\n",
       "      <td>http://edamontology.org/format_2310</td>\n",
       "      <td>FASTA-HTML</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>95</th>\n",
       "      <td>http://edamontology.org/format_2311</td>\n",
       "      <td>EMBL-HTML</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>http://edamontology.org/format_2532</td>\n",
       "      <td>GenBank-HTML</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>97</th>\n",
       "      <td>http://edamontology.org/format_2545</td>\n",
       "      <td>FASTQ-like format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>98</th>\n",
       "      <td>http://edamontology.org/format_2548</td>\n",
       "      <td>Sequence feature table format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>99</th>\n",
       "      <td>http://edamontology.org/format_2553</td>\n",
       "      <td>Sequence feature table format (XML)</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>100</th>\n",
       "      <td>http://edamontology.org/format_2848</td>\n",
       "      <td>Bibliographic reference format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>101</th>\n",
       "      <td>http://edamontology.org/format_2919</td>\n",
       "      <td>Sequence annotation track format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>102</th>\n",
       "      <td>http://edamontology.org/format_2920</td>\n",
       "      <td>Alignment format (pair only)</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>103</th>\n",
       "      <td>http://edamontology.org/format_2923</td>\n",
       "      <td>mega variant</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>104</th>\n",
       "      <td>http://edamontology.org/format_2924</td>\n",
       "      <td>Phylip format variant</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>105</th>\n",
       "      <td>http://edamontology.org/format_3000</td>\n",
       "      <td>AB1</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>106</th>\n",
       "      <td>http://edamontology.org/format_3003</td>\n",
       "      <td>BED</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>107</th>\n",
       "      <td>http://edamontology.org/format_3004</td>\n",
       "      <td>bigBed</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>108</th>\n",
       "      <td>http://edamontology.org/format_3005</td>\n",
       "      <td>WIG</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>109</th>\n",
       "      <td>http://edamontology.org/format_3006</td>\n",
       "      <td>bigWig</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>110</th>\n",
       "      <td>http://edamontology.org/format_3011</td>\n",
       "      <td>genePred</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>111</th>\n",
       "      <td>http://edamontology.org/format_3012</td>\n",
       "      <td>pgSnp</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>112</th>\n",
       "      <td>http://edamontology.org/format_3013</td>\n",
       "      <td>axt</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>113</th>\n",
       "      <td>http://edamontology.org/format_3014</td>\n",
       "      <td>LAV</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>114</th>\n",
       "      <td>http://edamontology.org/format_3015</td>\n",
       "      <td>Pileup</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>115</th>\n",
       "      <td>http://edamontology.org/format_3017</td>\n",
       "      <td>SRF</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>116</th>\n",
       "      <td>http://edamontology.org/format_3018</td>\n",
       "      <td>ZTR</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>117</th>\n",
       "      <td>http://edamontology.org/format_3161</td>\n",
       "      <td>MAGE-ML</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>118</th>\n",
       "      <td>http://edamontology.org/format_3162</td>\n",
       "      <td>MAGE-TAB</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>119</th>\n",
       "      <td>http://edamontology.org/format_3235</td>\n",
       "      <td>Cytoband format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>120</th>\n",
       "      <td>http://edamontology.org/format_3239</td>\n",
       "      <td>CopasiML</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>121</th>\n",
       "      <td>http://edamontology.org/format_3281</td>\n",
       "      <td>A2M</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>122</th>\n",
       "      <td>http://edamontology.org/format_3284</td>\n",
       "      <td>SFF</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>123</th>\n",
       "      <td>http://edamontology.org/format_3311</td>\n",
       "      <td>RNAML</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>124</th>\n",
       "      <td>http://edamontology.org/format_3327</td>\n",
       "      <td>BAI</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>125</th>\n",
       "      <td>http://edamontology.org/format_3328</td>\n",
       "      <td>HMMER2</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>126</th>\n",
       "      <td>http://edamontology.org/format_3329</td>\n",
       "      <td>HMMER3</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>127</th>\n",
       "      <td>http://edamontology.org/format_3462</td>\n",
       "      <td>CRAM</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>128</th>\n",
       "      <td>http://edamontology.org/format_3484</td>\n",
       "      <td>ebwt</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>129</th>\n",
       "      <td>http://edamontology.org/format_3485</td>\n",
       "      <td>RSF</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>130</th>\n",
       "      <td>http://edamontology.org/format_3486</td>\n",
       "      <td>GCG format variant</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>131</th>\n",
       "      <td>http://edamontology.org/format_3491</td>\n",
       "      <td>ebwtl</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>132</th>\n",
       "      <td>http://edamontology.org/format_3578</td>\n",
       "      <td>IDAT</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>133</th>\n",
       "      <td>http://edamontology.org/format_3583</td>\n",
       "      <td>bedgraph</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>134</th>\n",
       "      <td>http://edamontology.org/format_3584</td>\n",
       "      <td>bedstrict</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>135</th>\n",
       "      <td>http://edamontology.org/format_3585</td>\n",
       "      <td>bed6</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>136</th>\n",
       "      <td>http://edamontology.org/format_3586</td>\n",
       "      <td>bed12</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>137</th>\n",
       "      <td>http://edamontology.org/format_3587</td>\n",
       "      <td>chrominfo</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>138</th>\n",
       "      <td>http://edamontology.org/format_3588</td>\n",
       "      <td>customtrack</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>139</th>\n",
       "      <td>http://edamontology.org/format_3589</td>\n",
       "      <td>csfasta</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>140</th>\n",
       "      <td>http://edamontology.org/format_3612</td>\n",
       "      <td>ENCODE peak format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>141</th>\n",
       "      <td>http://edamontology.org/format_3613</td>\n",
       "      <td>ENCODE narrow peak format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>142</th>\n",
       "      <td>http://edamontology.org/format_3614</td>\n",
       "      <td>ENCODE broad peak format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>143</th>\n",
       "      <td>http://edamontology.org/format_3626</td>\n",
       "      <td>MAT</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>144</th>\n",
       "      <td>http://edamontology.org/format_3650</td>\n",
       "      <td>netCDF</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>145</th>\n",
       "      <td>http://edamontology.org/format_3683</td>\n",
       "      <td>qcML</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>146</th>\n",
       "      <td>http://edamontology.org/format_3684</td>\n",
       "      <td>PRIDE XML</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>147</th>\n",
       "      <td>http://edamontology.org/format_3686</td>\n",
       "      <td>COMBINE OMEX</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>148</th>\n",
       "      <td>http://edamontology.org/format_3687</td>\n",
       "      <td>ISA-TAB</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>149</th>\n",
       "      <td>http://edamontology.org/format_3700</td>\n",
       "      <td>Tabix index file format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>150</th>\n",
       "      <td>http://edamontology.org/format_3701</td>\n",
       "      <td>Sequin format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>151</th>\n",
       "      <td>http://edamontology.org/format_3708</td>\n",
       "      <td>ABCD format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>152</th>\n",
       "      <td>http://edamontology.org/format_3784</td>\n",
       "      <td>Open Annotation format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>153</th>\n",
       "      <td>http://edamontology.org/format_3811</td>\n",
       "      <td>XMFA</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>154</th>\n",
       "      <td>http://edamontology.org/format_3812</td>\n",
       "      <td>GEN</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>155</th>\n",
       "      <td>http://edamontology.org/format_3819</td>\n",
       "      <td>Relaxed PHYLIP Interleaved</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>156</th>\n",
       "      <td>http://edamontology.org/format_3820</td>\n",
       "      <td>Relaxed PHYLIP Sequential</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>157</th>\n",
       "      <td>http://edamontology.org/format_3827</td>\n",
       "      <td>proBED</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>158</th>\n",
       "      <td>http://edamontology.org/format_3848</td>\n",
       "      <td>PubMed XML</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>159</th>\n",
       "      <td>http://edamontology.org/format_3850</td>\n",
       "      <td>OrthoXML</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>160</th>\n",
       "      <td>http://edamontology.org/format_3864</td>\n",
       "      <td>mirGFF3</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>161</th>\n",
       "      <td>http://edamontology.org/format_3883</td>\n",
       "      <td>GROMACS itp</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>162</th>\n",
       "      <td>http://edamontology.org/format_3911</td>\n",
       "      <td>msh</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>163</th>\n",
       "      <td>http://edamontology.org/format_3951</td>\n",
       "      <td>BcForms</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>164</th>\n",
       "      <td>http://edamontology.org/format_3971</td>\n",
       "      <td>NeuroML</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>165</th>\n",
       "      <td>http://edamontology.org/format_3972</td>\n",
       "      <td>BNGL</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>166</th>\n",
       "      <td>http://edamontology.org/format_3982</td>\n",
       "      <td>CHAIN</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>167</th>\n",
       "      <td>http://edamontology.org/format_3991</td>\n",
       "      <td>TrackDB</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>168</th>\n",
       "      <td>http://edamontology.org/format_3992</td>\n",
       "      <td>CIGAR format</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>169</th>\n",
       "      <td>http://edamontology.org/format_4024</td>\n",
       "      <td>cif</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>170</th>\n",
       "      <td>http://edamontology.org/format_4035</td>\n",
       "      <td>PQR</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171</th>\n",
       "      <td>http://edamontology.org/format_4036</td>\n",
       "      <td>PDBQT</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>172</th>\n",
       "      <td>http://edamontology.org/format_1196</td>\n",
       "      <td>SMILES</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>173</th>\n",
       "      <td>http://edamontology.org/format_1197</td>\n",
       "      <td>InChI</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>174</th>\n",
       "      <td>http://edamontology.org/format_1198</td>\n",
       "      <td>mf</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>175</th>\n",
       "      <td>http://edamontology.org/format_1199</td>\n",
       "      <td>InChIKey</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>176</th>\n",
       "      <td>http://edamontology.org/format_1200</td>\n",
       "      <td>smarts</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>177</th>\n",
       "      <td>http://edamontology.org/format_1206</td>\n",
       "      <td>unambiguous pure</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>178</th>\n",
       "      <td>http://edamontology.org/format_1207</td>\n",
       "      <td>nucleotide</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>179</th>\n",
       "      <td>http://edamontology.org/format_1208</td>\n",
       "      <td>protein</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>180</th>\n",
       "      <td>http://edamontology.org/format_1209</td>\n",
       "      <td>consensus</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>181</th>\n",
       "      <td>http://edamontology.org/format_1210</td>\n",
       "      <td>pure nucleotide</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>182</th>\n",
       "      <td>http://edamontology.org/format_1211</td>\n",
       "      <td>unambiguous pure nucleotide</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>183</th>\n",
       "      <td>http://edamontology.org/format_1212</td>\n",
       "      <td>dna</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>184</th>\n",
       "      <td>http://edamontology.org/format_1213</td>\n",
       "      <td>rna</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>185</th>\n",
       "      <td>http://edamontology.org/format_1214</td>\n",
       "      <td>unambiguous pure dna</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>186</th>\n",
       "      <td>http://edamontology.org/format_1215</td>\n",
       "      <td>pure dna</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>187</th>\n",
       "      <td>http://edamontology.org/format_1216</td>\n",
       "      <td>unambiguous pure rna sequence</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>188</th>\n",
       "      <td>http://edamontology.org/format_1217</td>\n",
       "      <td>pure rna</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>189</th>\n",
       "      <td>http://edamontology.org/format_1218</td>\n",
       "      <td>unambiguous pure protein</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>190</th>\n",
       "      <td>http://edamontology.org/format_1219</td>\n",
       "      <td>pure protein</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>191</th>\n",
       "      <td>http://edamontology.org/format_1248</td>\n",
       "      <td>EMBL feature location</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>192</th>\n",
       "      <td>http://edamontology.org/format_1316</td>\n",
       "      <td>est2genome format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>193</th>\n",
       "      <td>http://edamontology.org/format_1332</td>\n",
       "      <td>FASTA search results format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>194</th>\n",
       "      <td>http://edamontology.org/format_1333</td>\n",
       "      <td>BLAST results</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>195</th>\n",
       "      <td>http://edamontology.org/format_1334</td>\n",
       "      <td>mspcrunch</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>196</th>\n",
       "      <td>http://edamontology.org/format_1335</td>\n",
       "      <td>Smith-Waterman format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>197</th>\n",
       "      <td>http://edamontology.org/format_1336</td>\n",
       "      <td>dhf</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>198</th>\n",
       "      <td>http://edamontology.org/format_1337</td>\n",
       "      <td>lhf</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>199</th>\n",
       "      <td>http://edamontology.org/format_1341</td>\n",
       "      <td>InterPro hits format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>200</th>\n",
       "      <td>http://edamontology.org/format_1342</td>\n",
       "      <td>InterPro protein view report format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>201</th>\n",
       "      <td>http://edamontology.org/format_1343</td>\n",
       "      <td>InterPro match table format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>202</th>\n",
       "      <td>http://edamontology.org/format_1349</td>\n",
       "      <td>HMMER Dirichlet prior</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>203</th>\n",
       "      <td>http://edamontology.org/format_1350</td>\n",
       "      <td>MEME Dirichlet prior</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>204</th>\n",
       "      <td>http://edamontology.org/format_1356</td>\n",
       "      <td>prosite-pattern</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>205</th>\n",
       "      <td>http://edamontology.org/format_1357</td>\n",
       "      <td>EMBOSS sequence pattern</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>206</th>\n",
       "      <td>http://edamontology.org/format_1360</td>\n",
       "      <td>meme-motif</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>207</th>\n",
       "      <td>http://edamontology.org/format_1366</td>\n",
       "      <td>prosite-profile</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>208</th>\n",
       "      <td>http://edamontology.org/format_1367</td>\n",
       "      <td>JASPAR format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>209</th>\n",
       "      <td>http://edamontology.org/format_1392</td>\n",
       "      <td>DIALIGN format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>210</th>\n",
       "      <td>http://edamontology.org/format_1393</td>\n",
       "      <td>daf</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>211</th>\n",
       "      <td>http://edamontology.org/format_1419</td>\n",
       "      <td>Sequence-MEME profile alignment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>212</th>\n",
       "      <td>http://edamontology.org/format_1421</td>\n",
       "      <td>HMMER profile alignment (sequences versus HMMs)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>213</th>\n",
       "      <td>http://edamontology.org/format_1422</td>\n",
       "      <td>HMMER profile alignment (HMM versus sequences)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>214</th>\n",
       "      <td>http://edamontology.org/format_1423</td>\n",
       "      <td>Phylip distance matrix</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>215</th>\n",
       "      <td>http://edamontology.org/format_1424</td>\n",
       "      <td>ClustalW dendrogram</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>216</th>\n",
       "      <td>http://edamontology.org/format_1425</td>\n",
       "      <td>Phylip tree raw</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>217</th>\n",
       "      <td>http://edamontology.org/format_1435</td>\n",
       "      <td>Phylip tree format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>218</th>\n",
       "      <td>http://edamontology.org/format_1436</td>\n",
       "      <td>TreeBASE format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>219</th>\n",
       "      <td>http://edamontology.org/format_1437</td>\n",
       "      <td>TreeFam format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>220</th>\n",
       "      <td>http://edamontology.org/format_1445</td>\n",
       "      <td>Phylip tree distance format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>221</th>\n",
       "      <td>http://edamontology.org/format_1457</td>\n",
       "      <td>Dot-bracket format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>222</th>\n",
       "      <td>http://edamontology.org/format_1458</td>\n",
       "      <td>Vienna local RNA secondary structure format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>223</th>\n",
       "      <td>http://edamontology.org/format_1551</td>\n",
       "      <td>Pcons report format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>224</th>\n",
       "      <td>http://edamontology.org/format_1552</td>\n",
       "      <td>ProQ report format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>225</th>\n",
       "      <td>http://edamontology.org/format_1582</td>\n",
       "      <td>findkm</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>226</th>\n",
       "      <td>http://edamontology.org/format_1630</td>\n",
       "      <td>CAF</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>227</th>\n",
       "      <td>http://edamontology.org/format_1631</td>\n",
       "      <td>EXP</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>228</th>\n",
       "      <td>http://edamontology.org/format_1639</td>\n",
       "      <td>affymetrix</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>229</th>\n",
       "      <td>http://edamontology.org/format_1641</td>\n",
       "      <td>affymetrix-exp</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>230</th>\n",
       "      <td>http://edamontology.org/format_1705</td>\n",
       "      <td>HET group dictionary entry format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>231</th>\n",
       "      <td>http://edamontology.org/format_1739</td>\n",
       "      <td>pmc</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>232</th>\n",
       "      <td>http://edamontology.org/format_1741</td>\n",
       "      <td>OSCAR format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>233</th>\n",
       "      <td>http://edamontology.org/format_1861</td>\n",
       "      <td>PlasMapper TextMap</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>234</th>\n",
       "      <td>http://edamontology.org/format_1910</td>\n",
       "      <td>newick</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>235</th>\n",
       "      <td>http://edamontology.org/format_1911</td>\n",
       "      <td>TreeCon format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>236</th>\n",
       "      <td>http://edamontology.org/format_1912</td>\n",
       "      <td>Nexus format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>237</th>\n",
       "      <td>http://edamontology.org/format_1919</td>\n",
       "      <td>Sequence record format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>238</th>\n",
       "      <td>http://edamontology.org/format_1920</td>\n",
       "      <td>Sequence feature annotation format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>239</th>\n",
       "      <td>http://edamontology.org/format_1921</td>\n",
       "      <td>Alignment format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>240</th>\n",
       "      <td>http://edamontology.org/format_1923</td>\n",
       "      <td>acedb</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>241</th>\n",
       "      <td>http://edamontology.org/format_1925</td>\n",
       "      <td>codata</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>242</th>\n",
       "      <td>http://edamontology.org/format_1926</td>\n",
       "      <td>dbid</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>243</th>\n",
       "      <td>http://edamontology.org/format_1928</td>\n",
       "      <td>Staden experiment format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>244</th>\n",
       "      <td>http://edamontology.org/format_1934</td>\n",
       "      <td>fitch program</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>245</th>\n",
       "      <td>http://edamontology.org/format_1937</td>\n",
       "      <td>genpept</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>246</th>\n",
       "      <td>http://edamontology.org/format_1940</td>\n",
       "      <td>giFASTA format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>247</th>\n",
       "      <td>http://edamontology.org/format_1941</td>\n",
       "      <td>hennig86</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>248</th>\n",
       "      <td>http://edamontology.org/format_1942</td>\n",
       "      <td>ig</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>249</th>\n",
       "      <td>http://edamontology.org/format_1943</td>\n",
       "      <td>igstrict</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>250</th>\n",
       "      <td>http://edamontology.org/format_1944</td>\n",
       "      <td>jackknifer</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>251</th>\n",
       "      <td>http://edamontology.org/format_1945</td>\n",
       "      <td>mase format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>252</th>\n",
       "      <td>http://edamontology.org/format_1946</td>\n",
       "      <td>mega-seq</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>253</th>\n",
       "      <td>http://edamontology.org/format_1954</td>\n",
       "      <td>Pearson format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>254</th>\n",
       "      <td>http://edamontology.org/format_1957</td>\n",
       "      <td>raw</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>255</th>\n",
       "      <td>http://edamontology.org/format_1958</td>\n",
       "      <td>refseqp</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>256</th>\n",
       "      <td>http://edamontology.org/format_1960</td>\n",
       "      <td>Staden format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>257</th>\n",
       "      <td>http://edamontology.org/format_1961</td>\n",
       "      <td>Stockholm format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>258</th>\n",
       "      <td>http://edamontology.org/format_1962</td>\n",
       "      <td>strider format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>259</th>\n",
       "      <td>http://edamontology.org/format_1966</td>\n",
       "      <td>ASN.1 sequence format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>260</th>\n",
       "      <td>http://edamontology.org/format_1967</td>\n",
       "      <td>DAS format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>261</th>\n",
       "      <td>http://edamontology.org/format_1968</td>\n",
       "      <td>dasdna</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>262</th>\n",
       "      <td>http://edamontology.org/format_1969</td>\n",
       "      <td>debug-seq</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>263</th>\n",
       "      <td>http://edamontology.org/format_1970</td>\n",
       "      <td>jackknifernon</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>264</th>\n",
       "      <td>http://edamontology.org/format_1972</td>\n",
       "      <td>NCBI format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>265</th>\n",
       "      <td>http://edamontology.org/format_1979</td>\n",
       "      <td>debug-feat</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>266</th>\n",
       "      <td>http://edamontology.org/format_1982</td>\n",
       "      <td>ClustalW format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>267</th>\n",
       "      <td>http://edamontology.org/format_1983</td>\n",
       "      <td>debug</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>268</th>\n",
       "      <td>http://edamontology.org/format_1985</td>\n",
       "      <td>markx0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>269</th>\n",
       "      <td>http://edamontology.org/format_1986</td>\n",
       "      <td>markx1</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>270</th>\n",
       "      <td>http://edamontology.org/format_1987</td>\n",
       "      <td>markx10</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>271</th>\n",
       "      <td>http://edamontology.org/format_1988</td>\n",
       "      <td>markx2</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>272</th>\n",
       "      <td>http://edamontology.org/format_1989</td>\n",
       "      <td>markx3</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>273</th>\n",
       "      <td>http://edamontology.org/format_1990</td>\n",
       "      <td>match</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>274</th>\n",
       "      <td>http://edamontology.org/format_1996</td>\n",
       "      <td>pair</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>275</th>\n",
       "      <td>http://edamontology.org/format_1999</td>\n",
       "      <td>scores format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>276</th>\n",
       "      <td>http://edamontology.org/format_2001</td>\n",
       "      <td>EMBOSS simple format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>277</th>\n",
       "      <td>http://edamontology.org/format_2002</td>\n",
       "      <td>srs format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>278</th>\n",
       "      <td>http://edamontology.org/format_2004</td>\n",
       "      <td>T-Coffee format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>279</th>\n",
       "      <td>http://edamontology.org/format_2006</td>\n",
       "      <td>Phylogenetic tree format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>280</th>\n",
       "      <td>http://edamontology.org/format_2013</td>\n",
       "      <td>Biological pathway or network format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>281</th>\n",
       "      <td>http://edamontology.org/format_2014</td>\n",
       "      <td>Sequence-profile alignment format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>282</th>\n",
       "      <td>http://edamontology.org/format_2020</td>\n",
       "      <td>Article format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>283</th>\n",
       "      <td>http://edamontology.org/format_2021</td>\n",
       "      <td>Text mining report format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>284</th>\n",
       "      <td>http://edamontology.org/format_2027</td>\n",
       "      <td>Enzyme kinetics report format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>285</th>\n",
       "      <td>http://edamontology.org/format_2030</td>\n",
       "      <td>Chemical data format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>286</th>\n",
       "      <td>http://edamontology.org/format_2031</td>\n",
       "      <td>Gene annotation format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>287</th>\n",
       "      <td>http://edamontology.org/format_2035</td>\n",
       "      <td>Chemical formula format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>288</th>\n",
       "      <td>http://edamontology.org/format_2036</td>\n",
       "      <td>Phylogenetic character data format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>289</th>\n",
       "      <td>http://edamontology.org/format_2049</td>\n",
       "      <td>Phylogenetic tree report (tree distances) format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>290</th>\n",
       "      <td>http://edamontology.org/format_2052</td>\n",
       "      <td>Protein family report format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>291</th>\n",
       "      <td>http://edamontology.org/format_2054</td>\n",
       "      <td>Protein interaction format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>292</th>\n",
       "      <td>http://edamontology.org/format_2055</td>\n",
       "      <td>Sequence assembly format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>293</th>\n",
       "      <td>http://edamontology.org/format_2056</td>\n",
       "      <td>Microarray experiment data format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>294</th>\n",
       "      <td>http://edamontology.org/format_2058</td>\n",
       "      <td>Gene expression report format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>295</th>\n",
       "      <td>http://edamontology.org/format_2060</td>\n",
       "      <td>Map format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>296</th>\n",
       "      <td>http://edamontology.org/format_2062</td>\n",
       "      <td>Protein report format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>297</th>\n",
       "      <td>http://edamontology.org/format_2065</td>\n",
       "      <td>Protein structure report (quality evaluation) format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>298</th>\n",
       "      <td>http://edamontology.org/format_2066</td>\n",
       "      <td>Database hits (sequence) format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>299</th>\n",
       "      <td>http://edamontology.org/format_2067</td>\n",
       "      <td>Sequence distance matrix format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>300</th>\n",
       "      <td>http://edamontology.org/format_2068</td>\n",
       "      <td>Sequence motif format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>301</th>\n",
       "      <td>http://edamontology.org/format_2069</td>\n",
       "      <td>Sequence profile format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>302</th>\n",
       "      <td>http://edamontology.org/format_2074</td>\n",
       "      <td>Dirichlet distribution format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>303</th>\n",
       "      <td>http://edamontology.org/format_2076</td>\n",
       "      <td>RNA secondary structure format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>304</th>\n",
       "      <td>http://edamontology.org/format_2078</td>\n",
       "      <td>Sequence range format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>305</th>\n",
       "      <td>http://edamontology.org/format_2094</td>\n",
       "      <td>pure</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>306</th>\n",
       "      <td>http://edamontology.org/format_2095</td>\n",
       "      <td>unpure</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>307</th>\n",
       "      <td>http://edamontology.org/format_2096</td>\n",
       "      <td>unambiguous sequence</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>308</th>\n",
       "      <td>http://edamontology.org/format_2097</td>\n",
       "      <td>ambiguous</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>309</th>\n",
       "      <td>http://edamontology.org/format_2170</td>\n",
       "      <td>Sequence cluster format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>310</th>\n",
       "      <td>http://edamontology.org/format_2171</td>\n",
       "      <td>Sequence cluster format (protein)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>311</th>\n",
       "      <td>http://edamontology.org/format_2172</td>\n",
       "      <td>Sequence cluster format (nucleic acid)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>312</th>\n",
       "      <td>http://edamontology.org/format_2181</td>\n",
       "      <td>EMBL-like (text)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>313</th>\n",
       "      <td>http://edamontology.org/format_2183</td>\n",
       "      <td>EMBLXML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>314</th>\n",
       "      <td>http://edamontology.org/format_2184</td>\n",
       "      <td>cdsxml</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>315</th>\n",
       "      <td>http://edamontology.org/format_2185</td>\n",
       "      <td>INSDSeq</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>316</th>\n",
       "      <td>http://edamontology.org/format_2186</td>\n",
       "      <td>geneseq</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>317</th>\n",
       "      <td>http://edamontology.org/format_2194</td>\n",
       "      <td>medline</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>318</th>\n",
       "      <td>http://edamontology.org/format_2195</td>\n",
       "      <td>Ontology format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>319</th>\n",
       "      <td>http://edamontology.org/format_2196</td>\n",
       "      <td>OBO format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>320</th>\n",
       "      <td>http://edamontology.org/format_2197</td>\n",
       "      <td>OWL format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>321</th>\n",
       "      <td>http://edamontology.org/format_2200</td>\n",
       "      <td>FASTA-like (text)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>322</th>\n",
       "      <td>http://edamontology.org/format_2204</td>\n",
       "      <td>EMBL format (XML)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>323</th>\n",
       "      <td>http://edamontology.org/format_2205</td>\n",
       "      <td>GenBank-like format (text)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>324</th>\n",
       "      <td>http://edamontology.org/format_2304</td>\n",
       "      <td>STRING entry format (XML)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>325</th>\n",
       "      <td>http://edamontology.org/format_2331</td>\n",
       "      <td>HTML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>326</th>\n",
       "      <td>http://edamontology.org/format_2376</td>\n",
       "      <td>RDF format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>327</th>\n",
       "      <td>http://edamontology.org/format_2543</td>\n",
       "      <td>EMBL-like format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>328</th>\n",
       "      <td>http://edamontology.org/format_2546</td>\n",
       "      <td>FASTA-like</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>329</th>\n",
       "      <td>http://edamontology.org/format_2549</td>\n",
       "      <td>OBO</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>330</th>\n",
       "      <td>http://edamontology.org/format_2550</td>\n",
       "      <td>OBO-XML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>331</th>\n",
       "      <td>http://edamontology.org/format_2551</td>\n",
       "      <td>Sequence record format (text)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>332</th>\n",
       "      <td>http://edamontology.org/format_2552</td>\n",
       "      <td>Sequence record format (XML)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>333</th>\n",
       "      <td>http://edamontology.org/format_2554</td>\n",
       "      <td>Alignment format (text)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>334</th>\n",
       "      <td>http://edamontology.org/format_2555</td>\n",
       "      <td>Alignment format (XML)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>335</th>\n",
       "      <td>http://edamontology.org/format_2556</td>\n",
       "      <td>Phylogenetic tree format (text)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>336</th>\n",
       "      <td>http://edamontology.org/format_2557</td>\n",
       "      <td>Phylogenetic tree format (XML)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>337</th>\n",
       "      <td>http://edamontology.org/format_2558</td>\n",
       "      <td>EMBL-like (XML)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>338</th>\n",
       "      <td>http://edamontology.org/format_2559</td>\n",
       "      <td>GenBank-like format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>339</th>\n",
       "      <td>http://edamontology.org/format_2561</td>\n",
       "      <td>Sequence assembly format (text)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>340</th>\n",
       "      <td>http://edamontology.org/format_2566</td>\n",
       "      <td>completely unambiguous</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>341</th>\n",
       "      <td>http://edamontology.org/format_2567</td>\n",
       "      <td>completely unambiguous pure</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>342</th>\n",
       "      <td>http://edamontology.org/format_2568</td>\n",
       "      <td>completely unambiguous pure nucleotide</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>343</th>\n",
       "      <td>http://edamontology.org/format_2569</td>\n",
       "      <td>completely unambiguous pure dna</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>344</th>\n",
       "      <td>http://edamontology.org/format_2570</td>\n",
       "      <td>completely unambiguous pure rna sequence</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>345</th>\n",
       "      <td>http://edamontology.org/format_2571</td>\n",
       "      <td>Raw sequence format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>346</th>\n",
       "      <td>http://edamontology.org/format_2585</td>\n",
       "      <td>SBML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>347</th>\n",
       "      <td>http://edamontology.org/format_2607</td>\n",
       "      <td>completely unambiguous pure protein</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>348</th>\n",
       "      <td>http://edamontology.org/format_2921</td>\n",
       "      <td>Sequence variation annotation format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>349</th>\n",
       "      <td>http://edamontology.org/format_2922</td>\n",
       "      <td>markx0 variant</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>350</th>\n",
       "      <td>http://edamontology.org/format_3001</td>\n",
       "      <td>ACE</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>351</th>\n",
       "      <td>http://edamontology.org/format_3009</td>\n",
       "      <td>2bit</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>352</th>\n",
       "      <td>http://edamontology.org/format_3010</td>\n",
       "      <td>.nib</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>353</th>\n",
       "      <td>http://edamontology.org/format_3016</td>\n",
       "      <td>VCF</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>354</th>\n",
       "      <td>http://edamontology.org/format_3020</td>\n",
       "      <td>BCF</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>355</th>\n",
       "      <td>http://edamontology.org/format_3033</td>\n",
       "      <td>Matrix format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>356</th>\n",
       "      <td>http://edamontology.org/format_3097</td>\n",
       "      <td>Protein domain classification format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>357</th>\n",
       "      <td>http://edamontology.org/format_3098</td>\n",
       "      <td>Raw SCOP domain classification format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>358</th>\n",
       "      <td>http://edamontology.org/format_3099</td>\n",
       "      <td>Raw CATH domain classification format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>359</th>\n",
       "      <td>http://edamontology.org/format_3100</td>\n",
       "      <td>CATH domain report format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>360</th>\n",
       "      <td>http://edamontology.org/format_3155</td>\n",
       "      <td>SBRML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>361</th>\n",
       "      <td>http://edamontology.org/format_3156</td>\n",
       "      <td>BioPAX</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>362</th>\n",
       "      <td>http://edamontology.org/format_3158</td>\n",
       "      <td>PSI MI XML (MIF)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>363</th>\n",
       "      <td>http://edamontology.org/format_3159</td>\n",
       "      <td>phyloXML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>364</th>\n",
       "      <td>http://edamontology.org/format_3160</td>\n",
       "      <td>NeXML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>365</th>\n",
       "      <td>http://edamontology.org/format_3163</td>\n",
       "      <td>GCDML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>366</th>\n",
       "      <td>http://edamontology.org/format_3166</td>\n",
       "      <td>Biological pathway or network report format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>367</th>\n",
       "      <td>http://edamontology.org/format_3167</td>\n",
       "      <td>Experiment annotation format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>368</th>\n",
       "      <td>http://edamontology.org/format_3240</td>\n",
       "      <td>CellML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>369</th>\n",
       "      <td>http://edamontology.org/format_3242</td>\n",
       "      <td>PSI MI TAB (MITAB)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>370</th>\n",
       "      <td>http://edamontology.org/format_3243</td>\n",
       "      <td>PSI-PAR</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>371</th>\n",
       "      <td>http://edamontology.org/format_3244</td>\n",
       "      <td>mzML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>372</th>\n",
       "      <td>http://edamontology.org/format_3245</td>\n",
       "      <td>Mass spectrometry data format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>373</th>\n",
       "      <td>http://edamontology.org/format_3246</td>\n",
       "      <td>TraML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>374</th>\n",
       "      <td>http://edamontology.org/format_3247</td>\n",
       "      <td>mzIdentML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>375</th>\n",
       "      <td>http://edamontology.org/format_3248</td>\n",
       "      <td>mzQuantML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>376</th>\n",
       "      <td>http://edamontology.org/format_3249</td>\n",
       "      <td>GelML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>377</th>\n",
       "      <td>http://edamontology.org/format_3250</td>\n",
       "      <td>spML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>378</th>\n",
       "      <td>http://edamontology.org/format_3252</td>\n",
       "      <td>OWL Functional Syntax</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>379</th>\n",
       "      <td>http://edamontology.org/format_3253</td>\n",
       "      <td>Manchester OWL Syntax</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>380</th>\n",
       "      <td>http://edamontology.org/format_3254</td>\n",
       "      <td>KRSS2 Syntax</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>381</th>\n",
       "      <td>http://edamontology.org/format_3255</td>\n",
       "      <td>Turtle</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>382</th>\n",
       "      <td>http://edamontology.org/format_3256</td>\n",
       "      <td>N-Triples</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>383</th>\n",
       "      <td>http://edamontology.org/format_3257</td>\n",
       "      <td>Notation3</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>384</th>\n",
       "      <td>http://edamontology.org/format_3261</td>\n",
       "      <td>RDF/XML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>385</th>\n",
       "      <td>http://edamontology.org/format_3262</td>\n",
       "      <td>OWL/XML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>386</th>\n",
       "      <td>http://edamontology.org/format_3309</td>\n",
       "      <td>CT</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>387</th>\n",
       "      <td>http://edamontology.org/format_3310</td>\n",
       "      <td>SS</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>388</th>\n",
       "      <td>http://edamontology.org/format_3312</td>\n",
       "      <td>GDE</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>389</th>\n",
       "      <td>http://edamontology.org/format_3313</td>\n",
       "      <td>BLC</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>390</th>\n",
       "      <td>http://edamontology.org/format_3326</td>\n",
       "      <td>Data index format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>391</th>\n",
       "      <td>http://edamontology.org/format_3330</td>\n",
       "      <td>PO</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>392</th>\n",
       "      <td>http://edamontology.org/format_3331</td>\n",
       "      <td>BLAST XML results format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>393</th>\n",
       "      <td>http://edamontology.org/format_3477</td>\n",
       "      <td>Cytoscape input file format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>394</th>\n",
       "      <td>http://edamontology.org/format_3487</td>\n",
       "      <td>BSML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>395</th>\n",
       "      <td>http://edamontology.org/format_3499</td>\n",
       "      <td>Ensembl variation file format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>396</th>\n",
       "      <td>http://edamontology.org/format_3547</td>\n",
       "      <td>Image format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>397</th>\n",
       "      <td>http://edamontology.org/format_3548</td>\n",
       "      <td>DICOM format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>398</th>\n",
       "      <td>http://edamontology.org/format_3549</td>\n",
       "      <td>nii</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>399</th>\n",
       "      <td>http://edamontology.org/format_3550</td>\n",
       "      <td>mhd</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>400</th>\n",
       "      <td>http://edamontology.org/format_3551</td>\n",
       "      <td>nrrd</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>401</th>\n",
       "      <td>http://edamontology.org/format_3556</td>\n",
       "      <td>MHTML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>402</th>\n",
       "      <td>http://edamontology.org/format_3579</td>\n",
       "      <td>JPG</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>403</th>\n",
       "      <td>http://edamontology.org/format_3580</td>\n",
       "      <td>rcc</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>404</th>\n",
       "      <td>http://edamontology.org/format_3582</td>\n",
       "      <td>afg</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>405</th>\n",
       "      <td>http://edamontology.org/format_3590</td>\n",
       "      <td>HDF5</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>406</th>\n",
       "      <td>http://edamontology.org/format_3591</td>\n",
       "      <td>TIFF</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>407</th>\n",
       "      <td>http://edamontology.org/format_3592</td>\n",
       "      <td>BMP</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>408</th>\n",
       "      <td>http://edamontology.org/format_3593</td>\n",
       "      <td>im</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>409</th>\n",
       "      <td>http://edamontology.org/format_3594</td>\n",
       "      <td>pcd</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>410</th>\n",
       "      <td>http://edamontology.org/format_3595</td>\n",
       "      <td>pcx</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>411</th>\n",
       "      <td>http://edamontology.org/format_3596</td>\n",
       "      <td>ppm</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>412</th>\n",
       "      <td>http://edamontology.org/format_3597</td>\n",
       "      <td>psd</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>413</th>\n",
       "      <td>http://edamontology.org/format_3598</td>\n",
       "      <td>xbm</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>414</th>\n",
       "      <td>http://edamontology.org/format_3599</td>\n",
       "      <td>xpm</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>415</th>\n",
       "      <td>http://edamontology.org/format_3600</td>\n",
       "      <td>rgb</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>416</th>\n",
       "      <td>http://edamontology.org/format_3601</td>\n",
       "      <td>pbm</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>417</th>\n",
       "      <td>http://edamontology.org/format_3602</td>\n",
       "      <td>pgm</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>418</th>\n",
       "      <td>http://edamontology.org/format_3603</td>\n",
       "      <td>PNG</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>419</th>\n",
       "      <td>http://edamontology.org/format_3604</td>\n",
       "      <td>SVG</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>420</th>\n",
       "      <td>http://edamontology.org/format_3605</td>\n",
       "      <td>rast</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>421</th>\n",
       "      <td>http://edamontology.org/format_3606</td>\n",
       "      <td>Sequence quality report format (text)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>422</th>\n",
       "      <td>http://edamontology.org/format_3615</td>\n",
       "      <td>bgzip</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>423</th>\n",
       "      <td>http://edamontology.org/format_3616</td>\n",
       "      <td>tabix</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>424</th>\n",
       "      <td>http://edamontology.org/format_3619</td>\n",
       "      <td>sif</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>425</th>\n",
       "      <td>http://edamontology.org/format_3622</td>\n",
       "      <td>Gemini SQLite format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>426</th>\n",
       "      <td>http://edamontology.org/format_3624</td>\n",
       "      <td>snpeffdb</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>427</th>\n",
       "      <td>http://edamontology.org/format_3651</td>\n",
       "      <td>MGF</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>428</th>\n",
       "      <td>http://edamontology.org/format_3652</td>\n",
       "      <td>dta</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>429</th>\n",
       "      <td>http://edamontology.org/format_3653</td>\n",
       "      <td>pkl</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>430</th>\n",
       "      <td>http://edamontology.org/format_3654</td>\n",
       "      <td>mzXML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>431</th>\n",
       "      <td>http://edamontology.org/format_3655</td>\n",
       "      <td>pepXML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>432</th>\n",
       "      <td>http://edamontology.org/format_3657</td>\n",
       "      <td>GPML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>433</th>\n",
       "      <td>http://edamontology.org/format_3681</td>\n",
       "      <td>mzTab</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>434</th>\n",
       "      <td>http://edamontology.org/format_3682</td>\n",
       "      <td>imzML metadata file</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>435</th>\n",
       "      <td>http://edamontology.org/format_3685</td>\n",
       "      <td>SED-ML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>436</th>\n",
       "      <td>http://edamontology.org/format_3688</td>\n",
       "      <td>SBtab</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>437</th>\n",
       "      <td>http://edamontology.org/format_3689</td>\n",
       "      <td>BCML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>438</th>\n",
       "      <td>http://edamontology.org/format_3692</td>\n",
       "      <td>SBGN-ML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>439</th>\n",
       "      <td>http://edamontology.org/format_3693</td>\n",
       "      <td>AGP</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>440</th>\n",
       "      <td>http://edamontology.org/format_3702</td>\n",
       "      <td>MSF</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>441</th>\n",
       "      <td>http://edamontology.org/format_3706</td>\n",
       "      <td>Biodiversity data format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>442</th>\n",
       "      <td>http://edamontology.org/format_3709</td>\n",
       "      <td>GCT/Res format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>443</th>\n",
       "      <td>http://edamontology.org/format_3710</td>\n",
       "      <td>WIFF format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>444</th>\n",
       "      <td>http://edamontology.org/format_3711</td>\n",
       "      <td>X!Tandem XML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>445</th>\n",
       "      <td>http://edamontology.org/format_3712</td>\n",
       "      <td>Thermo RAW</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>446</th>\n",
       "      <td>http://edamontology.org/format_3713</td>\n",
       "      <td>Mascot .dat file</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>447</th>\n",
       "      <td>http://edamontology.org/format_3714</td>\n",
       "      <td>MaxQuant APL peaklist format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>448</th>\n",
       "      <td>http://edamontology.org/format_3727</td>\n",
       "      <td>OME-TIFF</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>449</th>\n",
       "      <td>http://edamontology.org/format_3746</td>\n",
       "      <td>BIOM format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>450</th>\n",
       "      <td>http://edamontology.org/format_3747</td>\n",
       "      <td>protXML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>451</th>\n",
       "      <td>http://edamontology.org/format_3749</td>\n",
       "      <td>JSON-LD</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>452</th>\n",
       "      <td>http://edamontology.org/format_3758</td>\n",
       "      <td>SEQUEST .out file</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>453</th>\n",
       "      <td>http://edamontology.org/format_3764</td>\n",
       "      <td>idXML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>454</th>\n",
       "      <td>http://edamontology.org/format_3775</td>\n",
       "      <td>GSuite</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>455</th>\n",
       "      <td>http://edamontology.org/format_3780</td>\n",
       "      <td>Annotated text format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>456</th>\n",
       "      <td>http://edamontology.org/format_3781</td>\n",
       "      <td>PubAnnotation format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>457</th>\n",
       "      <td>http://edamontology.org/format_3782</td>\n",
       "      <td>BioC</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>458</th>\n",
       "      <td>http://edamontology.org/format_3783</td>\n",
       "      <td>PubTator format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>459</th>\n",
       "      <td>http://edamontology.org/format_3785</td>\n",
       "      <td>BioNLP Shared Task format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>460</th>\n",
       "      <td>http://edamontology.org/format_3787</td>\n",
       "      <td>Query language</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>461</th>\n",
       "      <td>http://edamontology.org/format_3814</td>\n",
       "      <td>SDF</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>462</th>\n",
       "      <td>http://edamontology.org/format_3815</td>\n",
       "      <td>Molfile</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>463</th>\n",
       "      <td>http://edamontology.org/format_3816</td>\n",
       "      <td>Mol2</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>464</th>\n",
       "      <td>http://edamontology.org/format_3818</td>\n",
       "      <td>ELAND format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>465</th>\n",
       "      <td>http://edamontology.org/format_3821</td>\n",
       "      <td>VisML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>466</th>\n",
       "      <td>http://edamontology.org/format_3822</td>\n",
       "      <td>GML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>467</th>\n",
       "      <td>http://edamontology.org/format_3823</td>\n",
       "      <td>FASTG</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>468</th>\n",
       "      <td>http://edamontology.org/format_3824</td>\n",
       "      <td>NMR data format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>469</th>\n",
       "      <td>http://edamontology.org/format_3825</td>\n",
       "      <td>nmrML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>470</th>\n",
       "      <td>http://edamontology.org/format_3828</td>\n",
       "      <td>Raw microarray data format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>471</th>\n",
       "      <td>http://edamontology.org/format_3829</td>\n",
       "      <td>GPR</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>472</th>\n",
       "      <td>http://edamontology.org/format_3830</td>\n",
       "      <td>ARB</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>473</th>\n",
       "      <td>http://edamontology.org/format_3832</td>\n",
       "      <td>consensusXML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>474</th>\n",
       "      <td>http://edamontology.org/format_3833</td>\n",
       "      <td>featureXML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>475</th>\n",
       "      <td>http://edamontology.org/format_3834</td>\n",
       "      <td>mzData</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>476</th>\n",
       "      <td>http://edamontology.org/format_3835</td>\n",
       "      <td>TIDE TXT</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>477</th>\n",
       "      <td>http://edamontology.org/format_3836</td>\n",
       "      <td>BLAST XML v2 results format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>478</th>\n",
       "      <td>http://edamontology.org/format_3838</td>\n",
       "      <td>pptx</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>479</th>\n",
       "      <td>http://edamontology.org/format_3839</td>\n",
       "      <td>ibd</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>480</th>\n",
       "      <td>http://edamontology.org/format_3843</td>\n",
       "      <td>BEAST</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>481</th>\n",
       "      <td>http://edamontology.org/format_3844</td>\n",
       "      <td>Chado-XML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>482</th>\n",
       "      <td>http://edamontology.org/format_3845</td>\n",
       "      <td>HSAML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>483</th>\n",
       "      <td>http://edamontology.org/format_3846</td>\n",
       "      <td>InterProScan XML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>484</th>\n",
       "      <td>http://edamontology.org/format_3847</td>\n",
       "      <td>KGML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>485</th>\n",
       "      <td>http://edamontology.org/format_3849</td>\n",
       "      <td>MSAML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>486</th>\n",
       "      <td>http://edamontology.org/format_3851</td>\n",
       "      <td>PSDML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>487</th>\n",
       "      <td>http://edamontology.org/format_3852</td>\n",
       "      <td>SeqXML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>488</th>\n",
       "      <td>http://edamontology.org/format_3853</td>\n",
       "      <td>UniParc XML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>489</th>\n",
       "      <td>http://edamontology.org/format_3854</td>\n",
       "      <td>UniRef XML</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>490</th>\n",
       "      <td>http://edamontology.org/format_3858</td>\n",
       "      <td>Waters RAW</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>491</th>\n",
       "      <td>http://edamontology.org/format_3859</td>\n",
       "      <td>JCAMP-DX</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>492</th>\n",
       "      <td>http://edamontology.org/format_3866</td>\n",
       "      <td>Trajectory format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>493</th>\n",
       "      <td>http://edamontology.org/format_3867</td>\n",
       "      <td>Trajectory format (binary)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>494</th>\n",
       "      <td>http://edamontology.org/format_3868</td>\n",
       "      <td>Trajectory format (text)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>495</th>\n",
       "      <td>http://edamontology.org/format_3873</td>\n",
       "      <td>HDF</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>496</th>\n",
       "      <td>http://edamontology.org/format_3874</td>\n",
       "      <td>PCAzip</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>497</th>\n",
       "      <td>http://edamontology.org/format_3875</td>\n",
       "      <td>XTC</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>498</th>\n",
       "      <td>http://edamontology.org/format_3876</td>\n",
       "      <td>TNG</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>499</th>\n",
       "      <td>http://edamontology.org/format_3877</td>\n",
       "      <td>XYZ</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>500</th>\n",
       "      <td>http://edamontology.org/format_3878</td>\n",
       "      <td>mdcrd</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>501</th>\n",
       "      <td>http://edamontology.org/format_3879</td>\n",
       "      <td>Topology format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>502</th>\n",
       "      <td>http://edamontology.org/format_3880</td>\n",
       "      <td>GROMACS top</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>503</th>\n",
       "      <td>http://edamontology.org/format_3881</td>\n",
       "      <td>AMBER top</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>504</th>\n",
       "      <td>http://edamontology.org/format_3882</td>\n",
       "      <td>PSF</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>505</th>\n",
       "      <td>http://edamontology.org/format_3884</td>\n",
       "      <td>FF parameter format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>506</th>\n",
       "      <td>http://edamontology.org/format_3885</td>\n",
       "      <td>BinPos</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>507</th>\n",
       "      <td>http://edamontology.org/format_3886</td>\n",
       "      <td>RST</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>508</th>\n",
       "      <td>http://edamontology.org/format_3887</td>\n",
       "      <td>CHARMM rtf</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>509</th>\n",
       "      <td>http://edamontology.org/format_3888</td>\n",
       "      <td>AMBER frcmod</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>510</th>\n",
       "      <td>http://edamontology.org/format_3889</td>\n",
       "      <td>AMBER off</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>511</th>\n",
       "      <td>http://edamontology.org/format_3906</td>\n",
       "      <td>NMReDATA</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>512</th>\n",
       "      <td>http://edamontology.org/format_3910</td>\n",
       "      <td>trr</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>513</th>\n",
       "      <td>http://edamontology.org/format_3956</td>\n",
       "      <td>N-Quads</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>514</th>\n",
       "      <td>http://edamontology.org/format_3969</td>\n",
       "      <td>Vega</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>515</th>\n",
       "      <td>http://edamontology.org/format_3970</td>\n",
       "      <td>Vega-lite</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>516</th>\n",
       "      <td>http://edamontology.org/format_3975</td>\n",
       "      <td>GFA 1</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>517</th>\n",
       "      <td>http://edamontology.org/format_3976</td>\n",
       "      <td>GFA 2</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>518</th>\n",
       "      <td>http://edamontology.org/format_3978</td>\n",
       "      <td>CONTIG</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>519</th>\n",
       "      <td>http://edamontology.org/format_3979</td>\n",
       "      <td>WEGO</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>520</th>\n",
       "      <td>http://edamontology.org/format_3980</td>\n",
       "      <td>RPKM</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>521</th>\n",
       "      <td>http://edamontology.org/format_3983</td>\n",
       "      <td>NET</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>522</th>\n",
       "      <td>http://edamontology.org/format_3984</td>\n",
       "      <td>QMAP</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>523</th>\n",
       "      <td>http://edamontology.org/format_3986</td>\n",
       "      <td>WMV</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>524</th>\n",
       "      <td>http://edamontology.org/format_3988</td>\n",
       "      <td>LSM</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>525</th>\n",
       "      <td>http://edamontology.org/format_3990</td>\n",
       "      <td>AVI</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>526</th>\n",
       "      <td>http://edamontology.org/format_3993</td>\n",
       "      <td>Stereolithography format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>527</th>\n",
       "      <td>http://edamontology.org/format_3994</td>\n",
       "      <td>U3D</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>528</th>\n",
       "      <td>http://edamontology.org/format_3995</td>\n",
       "      <td>Texture file format</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>529</th>\n",
       "      <td>http://edamontology.org/format_3997</td>\n",
       "      <td>MPEG-4</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>530</th>\n",
       "      <td>http://edamontology.org/format_4015</td>\n",
       "      <td>PEtab</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>531</th>\n",
       "      <td>http://edamontology.org/format_4018</td>\n",
       "      <td>gVCF</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>532</th>\n",
       "      <td>http://edamontology.org/format_4023</td>\n",
       "      <td>cml</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "results1=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "print(f\"nb of edam format with is_format_of relation: {len(results1)}\")\n",
    "\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results1.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## List all EDAM formats with has_topic relations (transitive)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX biotools: <https://bio.tools/ontology/>\n",
    "PREFIX bsc: <http://bioschemas.org/>\n",
    "PREFIX bsct: <http://bioschemas.org/types/>\n",
    "PREFIX dct: <http://purl.org/dc/terms/>\n",
    "PREFIX edam: <http://edamontology.org/>\n",
    "PREFIX owl: <http://www.w3.org/2002/07/owl#>\n",
    "PREFIX sc: <http://schema.org/>\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>\n",
    "\n",
    "SELECT\n",
    "    ?format ?format_label\n",
    "    (COUNT(?is_format_of_data) as ?count)\n",
    "    #?is_format_of_data ?is_format_of_data_label\n",
    "WHERE\n",
    "{\n",
    "    ?format\n",
    "        rdfs:subClassOf [owl:onProperty edam:has_topic ; owl:someValuesFrom ?is_format_of_data] ;\n",
    "        rdfs:label ?format_label .\n",
    "    ?is_format_of_data rdfs:label ?is_format_of_data_label .\n",
    "} GROUP BY ?format ?format_label\n",
    "ORDER BY DESC(?count) ASC(?format) ASC(?is_format_of_data)\n",
    "\"\"\"\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "nb of edam format with is_format_of relation: 618\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>format</th>\n",
       "      <th>format_label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/operation_0269</td>\n",
       "      <td>Transmembrane protein prediction</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>http://edamontology.org/operation_0245</td>\n",
       "      <td>Structural motif discovery</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>http://edamontology.org/operation_0246</td>\n",
       "      <td>Protein domain recognition</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>http://edamontology.org/operation_0267</td>\n",
       "      <td>Protein secondary structure prediction</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>http://edamontology.org/operation_0268</td>\n",
       "      <td>Protein super-secondary structure prediction</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>http://edamontology.org/operation_0390</td>\n",
       "      <td>Protein peeling</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>http://edamontology.org/operation_0468</td>\n",
       "      <td>Protein secondary structure prediction (helices)</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>http://edamontology.org/operation_0469</td>\n",
       "      <td>Protein secondary structure prediction (turns)</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>http://edamontology.org/operation_0470</td>\n",
       "      <td>Protein secondary structure prediction (coils)</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>http://edamontology.org/operation_2464</td>\n",
       "      <td>Protein-protein binding site prediction</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>http://edamontology.org/operation_0264</td>\n",
       "      <td>Alternative splicing prediction</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>http://edamontology.org/operation_0278</td>\n",
       "      <td>RNA secondary structure prediction</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>http://edamontology.org/operation_0302</td>\n",
       "      <td>Protein threading</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>http://edamontology.org/operation_0303</td>\n",
       "      <td>Fold recognition</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>http://edamontology.org/operation_0418</td>\n",
       "      <td>Protein signal peptide detection</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>http://edamontology.org/operation_0420</td>\n",
       "      <td>Nucleic acids-binding site prediction</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>http://edamontology.org/operation_0433</td>\n",
       "      <td>Splice site prediction</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>http://edamontology.org/operation_0465</td>\n",
       "      <td>siRNA binding specificity prediction</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>http://edamontology.org/operation_0477</td>\n",
       "      <td>Protein modelling</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>http://edamontology.org/operation_0479</td>\n",
       "      <td>Backbone modelling</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>http://edamontology.org/operation_0480</td>\n",
       "      <td>Side chain modelling</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>http://edamontology.org/operation_0481</td>\n",
       "      <td>Loop modelling</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>http://edamontology.org/operation_2008</td>\n",
       "      <td>siRNA duplex prediction</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>http://edamontology.org/operation_2575</td>\n",
       "      <td>Binding site prediction</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>http://edamontology.org/operation_3896</td>\n",
       "      <td>Active site prediction</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>http://edamontology.org/operation_3897</td>\n",
       "      <td>Ligand-binding site prediction</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>http://edamontology.org/operation_3898</td>\n",
       "      <td>Metal-binding site prediction</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>http://edamontology.org/operation_3902</td>\n",
       "      <td>RNA binding site prediction</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>http://edamontology.org/operation_3903</td>\n",
       "      <td>DNA binding site prediction</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>http://edamontology.org/operation_0276</td>\n",
       "      <td>Protein interaction network analysis</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>http://edamontology.org/operation_0282</td>\n",
       "      <td>Genetic mapping</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>http://edamontology.org/operation_0287</td>\n",
       "      <td>Base position variability plotting</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>http://edamontology.org/operation_0309</td>\n",
       "      <td>Microarray probe design</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>http://edamontology.org/operation_0319</td>\n",
       "      <td>Protein secondary structure assignment</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>http://edamontology.org/operation_0416</td>\n",
       "      <td>Epitope mapping</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>http://edamontology.org/operation_0417</td>\n",
       "      <td>Post-translational modification site prediction</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>http://edamontology.org/operation_0422</td>\n",
       "      <td>Protein cleavage site prediction</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>http://edamontology.org/operation_0443</td>\n",
       "      <td>trans-regulatory element prediction</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>http://edamontology.org/operation_0446</td>\n",
       "      <td>Exonic splicing enhancer prediction</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>http://edamontology.org/operation_0463</td>\n",
       "      <td>miRNA target prediction</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>http://edamontology.org/operation_0487</td>\n",
       "      <td>Haplotype mapping</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>http://edamontology.org/operation_0550</td>\n",
       "      <td>DNA substitution modelling</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>http://edamontology.org/operation_1831</td>\n",
       "      <td>Metal-bound cysteine detection</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>http://edamontology.org/operation_2499</td>\n",
       "      <td>Splicing analysis</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>http://edamontology.org/operation_3186</td>\n",
       "      <td>Bisulfite mapping</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>http://edamontology.org/operation_3198</td>\n",
       "      <td>Read mapping</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>http://edamontology.org/operation_3199</td>\n",
       "      <td>Split read mapping</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>http://edamontology.org/operation_3660</td>\n",
       "      <td>Metabolic network modelling</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>http://edamontology.org/operation_3904</td>\n",
       "      <td>Protein disorder prediction</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>http://edamontology.org/operation_3946</td>\n",
       "      <td>Ecological modelling</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>http://edamontology.org/operation_3968</td>\n",
       "      <td>Adhesin prediction</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51</th>\n",
       "      <td>http://edamontology.org/operation_4032</td>\n",
       "      <td>DNA modification prediction</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>http://edamontology.org/data_0880</td>\n",
       "      <td>RNA secondary structure</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>53</th>\n",
       "      <td>http://edamontology.org/operation_0265</td>\n",
       "      <td>Frameshift detection</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>http://edamontology.org/operation_0308</td>\n",
       "      <td>PCR primer design</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>55</th>\n",
       "      <td>http://edamontology.org/operation_0360</td>\n",
       "      <td>Structural similarity search</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>56</th>\n",
       "      <td>http://edamontology.org/operation_0362</td>\n",
       "      <td>Genome annotation</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>57</th>\n",
       "      <td>http://edamontology.org/operation_0379</td>\n",
       "      <td>Repeat sequence detection</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>58</th>\n",
       "      <td>http://edamontology.org/operation_0430</td>\n",
       "      <td>CpG island and isochore detection</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>59</th>\n",
       "      <td>http://edamontology.org/operation_0431</td>\n",
       "      <td>Restriction site recognition</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>60</th>\n",
       "      <td>http://edamontology.org/operation_0438</td>\n",
       "      <td>Transcriptional regulatory element prediction</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>61</th>\n",
       "      <td>http://edamontology.org/operation_0439</td>\n",
       "      <td>Translation initiation site prediction</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>62</th>\n",
       "      <td>http://edamontology.org/operation_0440</td>\n",
       "      <td>Promoter prediction</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>63</th>\n",
       "      <td>http://edamontology.org/operation_0441</td>\n",
       "      <td>cis-regulatory element prediction</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>64</th>\n",
       "      <td>http://edamontology.org/operation_0444</td>\n",
       "      <td>S/MAR prediction</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>65</th>\n",
       "      <td>http://edamontology.org/operation_0445</td>\n",
       "      <td>Transcription factor binding site prediction</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66</th>\n",
       "      <td>http://edamontology.org/operation_0464</td>\n",
       "      <td>tRNA gene prediction</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>67</th>\n",
       "      <td>http://edamontology.org/operation_0471</td>\n",
       "      <td>Disulfide bond prediction</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>68</th>\n",
       "      <td>http://edamontology.org/operation_0474</td>\n",
       "      <td>Protein structure prediction</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69</th>\n",
       "      <td>http://edamontology.org/operation_0476</td>\n",
       "      <td>Ab initio structure prediction</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>70</th>\n",
       "      <td>http://edamontology.org/operation_0525</td>\n",
       "      <td>Genome assembly</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71</th>\n",
       "      <td>http://edamontology.org/operation_1781</td>\n",
       "      <td>Gene regulatory network analysis</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72</th>\n",
       "      <td>http://edamontology.org/operation_1816</td>\n",
       "      <td>Surface rendering</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>73</th>\n",
       "      <td>http://edamontology.org/operation_1829</td>\n",
       "      <td>Cysteine bridge detection</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>74</th>\n",
       "      <td>http://edamontology.org/operation_1830</td>\n",
       "      <td>Free cysteine detection</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75</th>\n",
       "      <td>http://edamontology.org/operation_1850</td>\n",
       "      <td>Protein cysteine and disulfide bond assignment</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76</th>\n",
       "      <td>http://edamontology.org/operation_2415</td>\n",
       "      <td>Protein folding analysis</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>77</th>\n",
       "      <td>http://edamontology.org/operation_2476</td>\n",
       "      <td>Molecular dynamics</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78</th>\n",
       "      <td>http://edamontology.org/operation_3092</td>\n",
       "      <td>Protein feature detection</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>79</th>\n",
       "      <td>http://edamontology.org/operation_3094</td>\n",
       "      <td>Protein interaction network prediction</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80</th>\n",
       "      <td>http://edamontology.org/operation_3194</td>\n",
       "      <td>Genome feature comparison</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>81</th>\n",
       "      <td>http://edamontology.org/operation_3208</td>\n",
       "      <td>Genome visualisation</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>82</th>\n",
       "      <td>http://edamontology.org/operation_3216</td>\n",
       "      <td>Scaffolding</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>83</th>\n",
       "      <td>http://edamontology.org/operation_3217</td>\n",
       "      <td>Scaffold gap completion</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>84</th>\n",
       "      <td>http://edamontology.org/operation_3258</td>\n",
       "      <td>Transcriptome assembly</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>85</th>\n",
       "      <td>http://edamontology.org/operation_3469</td>\n",
       "      <td>RNA structure covariance model generation</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>86</th>\n",
       "      <td>http://edamontology.org/operation_3666</td>\n",
       "      <td>Molecular surface comparison</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>87</th>\n",
       "      <td>http://edamontology.org/operation_3938</td>\n",
       "      <td>Virtual screening</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>88</th>\n",
       "      <td>http://edamontology.org/data_1462</td>\n",
       "      <td>Carbohydrate structure</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>89</th>\n",
       "      <td>http://edamontology.org/data_1465</td>\n",
       "      <td>RNA structure</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>90</th>\n",
       "      <td>http://edamontology.org/data_1466</td>\n",
       "      <td>tRNA structure</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>91</th>\n",
       "      <td>http://edamontology.org/data_1468</td>\n",
       "      <td>Protein domain</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>92</th>\n",
       "      <td>http://edamontology.org/data_3949</td>\n",
       "      <td>Profile HMM</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>93</th>\n",
       "      <td>http://edamontology.org/operation_0234</td>\n",
       "      <td>Sequence complexity calculation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>94</th>\n",
       "      <td>http://edamontology.org/operation_0235</td>\n",
       "      <td>Sequence ambiguity calculation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>95</th>\n",
       "      <td>http://edamontology.org/operation_0238</td>\n",
       "      <td>Sequence motif discovery</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>http://edamontology.org/operation_0239</td>\n",
       "      <td>Sequence motif recognition</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>97</th>\n",
       "      <td>http://edamontology.org/operation_0266</td>\n",
       "      <td>Vector sequence detection</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>98</th>\n",
       "      <td>http://edamontology.org/operation_0272</td>\n",
       "      <td>Residue contact prediction</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>99</th>\n",
       "      <td>http://edamontology.org/operation_0279</td>\n",
       "      <td>Nucleic acid folding analysis</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>100</th>\n",
       "      <td>http://edamontology.org/operation_0283</td>\n",
       "      <td>Linkage analysis</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>101</th>\n",
       "      <td>http://edamontology.org/operation_0284</td>\n",
       "      <td>Codon usage table generation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>102</th>\n",
       "      <td>http://edamontology.org/operation_0285</td>\n",
       "      <td>Codon usage table comparison</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>103</th>\n",
       "      <td>http://edamontology.org/operation_0286</td>\n",
       "      <td>Codon usage analysis</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>104</th>\n",
       "      <td>http://edamontology.org/operation_0310</td>\n",
       "      <td>Sequence assembly</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>105</th>\n",
       "      <td>http://edamontology.org/operation_0320</td>\n",
       "      <td>Protein structure assignment</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>106</th>\n",
       "      <td>http://edamontology.org/operation_0321</td>\n",
       "      <td>Protein structure validation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>107</th>\n",
       "      <td>http://edamontology.org/operation_0327</td>\n",
       "      <td>Phylogenetic footprinting</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>108</th>\n",
       "      <td>http://edamontology.org/operation_0346</td>\n",
       "      <td>Sequence similarity search</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>109</th>\n",
       "      <td>http://edamontology.org/operation_0371</td>\n",
       "      <td>DNA translation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>110</th>\n",
       "      <td>http://edamontology.org/operation_0372</td>\n",
       "      <td>DNA transcription</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>111</th>\n",
       "      <td>http://edamontology.org/operation_0380</td>\n",
       "      <td>Repeat sequence organisation analysis</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>112</th>\n",
       "      <td>http://edamontology.org/operation_0384</td>\n",
       "      <td>Accessible surface calculation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>113</th>\n",
       "      <td>http://edamontology.org/operation_0387</td>\n",
       "      <td>Molecular surface calculation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>114</th>\n",
       "      <td>http://edamontology.org/operation_0407</td>\n",
       "      <td>Protein hydrophobic moment plotting</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>115</th>\n",
       "      <td>http://edamontology.org/operation_0415</td>\n",
       "      <td>Nucleic acid feature detection</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>116</th>\n",
       "      <td>http://edamontology.org/operation_0427</td>\n",
       "      <td>Transposon prediction</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>117</th>\n",
       "      <td>http://edamontology.org/operation_0428</td>\n",
       "      <td>PolyA signal detection</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>118</th>\n",
       "      <td>http://edamontology.org/operation_0429</td>\n",
       "      <td>Quadruplex formation site detection</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>119</th>\n",
       "      <td>http://edamontology.org/operation_0432</td>\n",
       "      <td>Nucleosome position prediction</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>120</th>\n",
       "      <td>http://edamontology.org/operation_0435</td>\n",
       "      <td>Operon prediction</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>121</th>\n",
       "      <td>http://edamontology.org/operation_0436</td>\n",
       "      <td>Coding region prediction</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>122</th>\n",
       "      <td>http://edamontology.org/operation_0437</td>\n",
       "      <td>SECIS element prediction</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>123</th>\n",
       "      <td>http://edamontology.org/operation_0482</td>\n",
       "      <td>Protein-ligand docking</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>124</th>\n",
       "      <td>http://edamontology.org/operation_0485</td>\n",
       "      <td>Radiation Hybrid Mapping</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>125</th>\n",
       "      <td>http://edamontology.org/operation_0488</td>\n",
       "      <td>Linkage disequilibrium calculation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>126</th>\n",
       "      <td>http://edamontology.org/operation_0489</td>\n",
       "      <td>Genetic code prediction</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>127</th>\n",
       "      <td>http://edamontology.org/operation_0502</td>\n",
       "      <td>RNA secondary structure alignment</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>128</th>\n",
       "      <td>http://edamontology.org/operation_0523</td>\n",
       "      <td>Mapping assembly</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>129</th>\n",
       "      <td>http://edamontology.org/operation_0524</td>\n",
       "      <td>De-novo assembly</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>130</th>\n",
       "      <td>http://edamontology.org/operation_0526</td>\n",
       "      <td>EST assembly</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>131</th>\n",
       "      <td>http://edamontology.org/operation_0527</td>\n",
       "      <td>Sequence tag mapping</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>132</th>\n",
       "      <td>http://edamontology.org/operation_0533</td>\n",
       "      <td>Expression profile pathway mapping</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>133</th>\n",
       "      <td>http://edamontology.org/operation_0540</td>\n",
       "      <td>Phylogenetic inference (from molecular sequences)</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>134</th>\n",
       "      <td>http://edamontology.org/operation_0542</td>\n",
       "      <td>Phylogenetic inference (from gene frequencies)</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>135</th>\n",
       "      <td>http://edamontology.org/operation_0543</td>\n",
       "      <td>Phylogenetic inference (from polymorphism data)</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>136</th>\n",
       "      <td>http://edamontology.org/operation_0553</td>\n",
       "      <td>Gene tree construction</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>137</th>\n",
       "      <td>http://edamontology.org/operation_0560</td>\n",
       "      <td>DNA vaccine design</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>138</th>\n",
       "      <td>http://edamontology.org/operation_1836</td>\n",
       "      <td>Residue bump detection</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>139</th>\n",
       "      <td>http://edamontology.org/operation_1843</td>\n",
       "      <td>Residue packing validation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>140</th>\n",
       "      <td>http://edamontology.org/operation_1844</td>\n",
       "      <td>Protein geometry validation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>141</th>\n",
       "      <td>http://edamontology.org/operation_2241</td>\n",
       "      <td>Transmembrane protein visualisation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>142</th>\n",
       "      <td>http://edamontology.org/operation_2284</td>\n",
       "      <td>Nucleic acid density plotting</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>143</th>\n",
       "      <td>http://edamontology.org/operation_2416</td>\n",
       "      <td>Protein secondary structure analysis</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>144</th>\n",
       "      <td>http://edamontology.org/operation_2419</td>\n",
       "      <td>Primer and probe design</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>145</th>\n",
       "      <td>http://edamontology.org/operation_2437</td>\n",
       "      <td>Gene regulatory network prediction</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>146</th>\n",
       "      <td>http://edamontology.org/operation_2439</td>\n",
       "      <td>RNA secondary structure analysis</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>147</th>\n",
       "      <td>http://edamontology.org/operation_2454</td>\n",
       "      <td>Gene prediction</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>148</th>\n",
       "      <td>http://edamontology.org/operation_2487</td>\n",
       "      <td>Protein structure comparison</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>149</th>\n",
       "      <td>http://edamontology.org/operation_2488</td>\n",
       "      <td>Protein secondary structure comparison</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>150</th>\n",
       "      <td>http://edamontology.org/operation_2492</td>\n",
       "      <td>Protein interaction prediction</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>151</th>\n",
       "      <td>http://edamontology.org/operation_2510</td>\n",
       "      <td>DNA back-translation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>152</th>\n",
       "      <td>http://edamontology.org/operation_2518</td>\n",
       "      <td>Nucleic acid structure comparison</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>153</th>\n",
       "      <td>http://edamontology.org/operation_2871</td>\n",
       "      <td>Sequence tagged site (STS) mapping</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>154</th>\n",
       "      <td>http://edamontology.org/operation_2944</td>\n",
       "      <td>Physical mapping</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>155</th>\n",
       "      <td>http://edamontology.org/operation_2962</td>\n",
       "      <td>Codon usage bias calculation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>156</th>\n",
       "      <td>http://edamontology.org/operation_2964</td>\n",
       "      <td>Codon usage fraction calculation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>157</th>\n",
       "      <td>http://edamontology.org/operation_3180</td>\n",
       "      <td>Sequence assembly validation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>158</th>\n",
       "      <td>http://edamontology.org/operation_3182</td>\n",
       "      <td>Genome alignment</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>159</th>\n",
       "      <td>http://edamontology.org/operation_3183</td>\n",
       "      <td>Localised reassembly</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>160</th>\n",
       "      <td>http://edamontology.org/operation_3206</td>\n",
       "      <td>Whole genome methylation analysis</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>161</th>\n",
       "      <td>http://edamontology.org/operation_3209</td>\n",
       "      <td>Genome comparison</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>162</th>\n",
       "      <td>http://edamontology.org/operation_3211</td>\n",
       "      <td>Genome indexing</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>163</th>\n",
       "      <td>http://edamontology.org/operation_3221</td>\n",
       "      <td>Species frequency estimation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>164</th>\n",
       "      <td>http://edamontology.org/operation_3222</td>\n",
       "      <td>Peak calling</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>165</th>\n",
       "      <td>http://edamontology.org/operation_3229</td>\n",
       "      <td>Exome assembly</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>166</th>\n",
       "      <td>http://edamontology.org/operation_3351</td>\n",
       "      <td>Molecular surface analysis</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>167</th>\n",
       "      <td>http://edamontology.org/operation_3457</td>\n",
       "      <td>Single particle analysis</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>168</th>\n",
       "      <td>http://edamontology.org/operation_3458</td>\n",
       "      <td>Single particle alignment and classification</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>169</th>\n",
       "      <td>http://edamontology.org/operation_3478</td>\n",
       "      <td>Phylogenetic reconstruction</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>170</th>\n",
       "      <td>http://edamontology.org/operation_3560</td>\n",
       "      <td>Maximum occurrence analysis</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171</th>\n",
       "      <td>http://edamontology.org/operation_3563</td>\n",
       "      <td>RNA-seq read count analysis</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>172</th>\n",
       "      <td>http://edamontology.org/operation_3565</td>\n",
       "      <td>RNA-seq time series data analysis</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>173</th>\n",
       "      <td>http://edamontology.org/operation_3662</td>\n",
       "      <td>Ab-initio gene prediction</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>174</th>\n",
       "      <td>http://edamontology.org/operation_3663</td>\n",
       "      <td>Homology-based gene prediction</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>175</th>\n",
       "      <td>http://edamontology.org/operation_3677</td>\n",
       "      <td>Differential binding analysis</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>176</th>\n",
       "      <td>http://edamontology.org/operation_3680</td>\n",
       "      <td>RNA-Seq analysis</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>177</th>\n",
       "      <td>http://edamontology.org/operation_3730</td>\n",
       "      <td>Cross-assembly</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>178</th>\n",
       "      <td>http://edamontology.org/operation_3766</td>\n",
       "      <td>Weighted correlation network analysis</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>179</th>\n",
       "      <td>http://edamontology.org/operation_3899</td>\n",
       "      <td>Protein-protein docking</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>180</th>\n",
       "      <td>http://edamontology.org/operation_3900</td>\n",
       "      <td>DNA-binding protein prediction</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>181</th>\n",
       "      <td>http://edamontology.org/operation_3901</td>\n",
       "      <td>RNA-binding protein prediction</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>182</th>\n",
       "      <td>http://edamontology.org/operation_3918</td>\n",
       "      <td>Genome analysis</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>183</th>\n",
       "      <td>http://edamontology.org/operation_3920</td>\n",
       "      <td>DNA testing</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>184</th>\n",
       "      <td>http://edamontology.org/operation_3925</td>\n",
       "      <td>Network visualisation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>185</th>\n",
       "      <td>http://edamontology.org/operation_3926</td>\n",
       "      <td>Pathway visualisation</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>186</th>\n",
       "      <td>http://edamontology.org/operation_4031</td>\n",
       "      <td>Power test</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>187</th>\n",
       "      <td>http://edamontology.org/operation_4033</td>\n",
       "      <td>Disease transmission analysis</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>188</th>\n",
       "      <td>http://edamontology.org/data_0582</td>\n",
       "      <td>Ontology</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>189</th>\n",
       "      <td>http://edamontology.org/data_1353</td>\n",
       "      <td>Sequence motif</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>190</th>\n",
       "      <td>http://edamontology.org/data_1354</td>\n",
       "      <td>Sequence profile</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>191</th>\n",
       "      <td>http://edamontology.org/data_1361</td>\n",
       "      <td>Position frequency matrix</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>192</th>\n",
       "      <td>http://edamontology.org/data_1362</td>\n",
       "      <td>Position weight matrix</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>193</th>\n",
       "      <td>http://edamontology.org/data_1363</td>\n",
       "      <td>Information content matrix</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>194</th>\n",
       "      <td>http://edamontology.org/data_1365</td>\n",
       "      <td>Fingerprint</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>195</th>\n",
       "      <td>http://edamontology.org/data_1459</td>\n",
       "      <td>Nucleic acid structure</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>196</th>\n",
       "      <td>http://edamontology.org/data_1460</td>\n",
       "      <td>Protein structure</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>197</th>\n",
       "      <td>http://edamontology.org/data_1461</td>\n",
       "      <td>Protein-ligand complex</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>198</th>\n",
       "      <td>http://edamontology.org/data_1463</td>\n",
       "      <td>Small molecule structure</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>199</th>\n",
       "      <td>http://edamontology.org/data_1464</td>\n",
       "      <td>DNA structure</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>200</th>\n",
       "      <td>http://edamontology.org/data_1467</td>\n",
       "      <td>Protein chain</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>201</th>\n",
       "      <td>http://edamontology.org/data_1470</td>\n",
       "      <td>C-alpha trace</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>202</th>\n",
       "      <td>http://edamontology.org/data_1597</td>\n",
       "      <td>Codon usage table</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>203</th>\n",
       "      <td>http://edamontology.org/data_1696</td>\n",
       "      <td>Drug report</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>204</th>\n",
       "      <td>http://edamontology.org/data_1755</td>\n",
       "      <td>Protein atom</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>205</th>\n",
       "      <td>http://edamontology.org/data_1756</td>\n",
       "      <td>Protein residue</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>206</th>\n",
       "      <td>http://edamontology.org/data_2070</td>\n",
       "      <td>Sequence motif (nucleic acid)</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>207</th>\n",
       "      <td>http://edamontology.org/data_2071</td>\n",
       "      <td>Sequence motif (protein)</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>208</th>\n",
       "      <td>http://edamontology.org/data_2717</td>\n",
       "      <td>Oligonucleotide probe annotation</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>209</th>\n",
       "      <td>http://edamontology.org/data_2851</td>\n",
       "      <td>Drug structure</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>210</th>\n",
       "      <td>http://edamontology.org/data_2852</td>\n",
       "      <td>Toxin structure</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>211</th>\n",
       "      <td>http://edamontology.org/data_2854</td>\n",
       "      <td>Position-specific scoring matrix</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>212</th>\n",
       "      <td>http://edamontology.org/data_2877</td>\n",
       "      <td>Protein complex</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>213</th>\n",
       "      <td>http://edamontology.org/data_2878</td>\n",
       "      <td>Protein structural motif</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>214</th>\n",
       "      <td>http://edamontology.org/data_3111</td>\n",
       "      <td>Processed microarray data</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>215</th>\n",
       "      <td>http://edamontology.org/data_3805</td>\n",
       "      <td>3D EM Map</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>216</th>\n",
       "      <td>http://edamontology.org/data_3806</td>\n",
       "      <td>3D EM Mask</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>217</th>\n",
       "      <td>http://edamontology.org/operation_0231</td>\n",
       "      <td>Sequence editing</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>218</th>\n",
       "      <td>http://edamontology.org/operation_0232</td>\n",
       "      <td>Sequence merging</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>219</th>\n",
       "      <td>http://edamontology.org/operation_0233</td>\n",
       "      <td>Sequence conversion</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>220</th>\n",
       "      <td>http://edamontology.org/operation_0236</td>\n",
       "      <td>Sequence composition calculation</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>221</th>\n",
       "      <td>http://edamontology.org/operation_0237</td>\n",
       "      <td>Repeat sequence analysis</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>222</th>\n",
       "      <td>http://edamontology.org/operation_0240</td>\n",
       "      <td>Sequence motif comparison</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>223</th>\n",
       "      <td>http://edamontology.org/operation_0247</td>\n",
       "      <td>Protein architecture analysis</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>224</th>\n",
       "      <td>http://edamontology.org/operation_0252</td>\n",
       "      <td>Peptide immunogenicity prediction</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>225</th>\n",
       "      <td>http://edamontology.org/operation_0253</td>\n",
       "      <td>Sequence feature detection</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>226</th>\n",
       "      <td>http://edamontology.org/operation_0256</td>\n",
       "      <td>Sequence feature comparison</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>227</th>\n",
       "      <td>http://edamontology.org/operation_0289</td>\n",
       "      <td>Sequence distance matrix generation</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>228</th>\n",
       "      <td>http://edamontology.org/operation_0295</td>\n",
       "      <td>Structure alignment</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>229</th>\n",
       "      <td>http://edamontology.org/operation_0296</td>\n",
       "      <td>Sequence profile generation</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>230</th>\n",
       "      <td>http://edamontology.org/operation_0300</td>\n",
       "      <td>Sequence profile alignment</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>231</th>\n",
       "      <td>http://edamontology.org/operation_0305</td>\n",
       "      <td>Literature search</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>232</th>\n",
       "      <td>http://edamontology.org/operation_0307</td>\n",
       "      <td>Virtual PCR</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>233</th>\n",
       "      <td>http://edamontology.org/operation_0323</td>\n",
       "      <td>Phylogenetic inference</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>234</th>\n",
       "      <td>http://edamontology.org/operation_0326</td>\n",
       "      <td>Phylogenetic tree editing</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>235</th>\n",
       "      <td>http://edamontology.org/operation_0331</td>\n",
       "      <td>Variant effect prediction</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>236</th>\n",
       "      <td>http://edamontology.org/operation_0338</td>\n",
       "      <td>Sequence database search</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>237</th>\n",
       "      <td>http://edamontology.org/operation_0339</td>\n",
       "      <td>Structure database search</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>238</th>\n",
       "      <td>http://edamontology.org/operation_0349</td>\n",
       "      <td>Sequence database search (by property)</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>239</th>\n",
       "      <td>http://edamontology.org/operation_0366</td>\n",
       "      <td>Protein sequence cleavage</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>240</th>\n",
       "      <td>http://edamontology.org/operation_0367</td>\n",
       "      <td>Sequence mutation and randomisation</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>241</th>\n",
       "      <td>http://edamontology.org/operation_0368</td>\n",
       "      <td>Sequence masking</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>242</th>\n",
       "      <td>http://edamontology.org/operation_0369</td>\n",
       "      <td>Sequence cutting</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>243</th>\n",
       "      <td>http://edamontology.org/operation_0370</td>\n",
       "      <td>Restriction site creation</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>244</th>\n",
       "      <td>http://edamontology.org/operation_0389</td>\n",
       "      <td>Protein-nucleic acid interaction analysis</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>245</th>\n",
       "      <td>http://edamontology.org/operation_0402</td>\n",
       "      <td>Protein titration curve plotting</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>246</th>\n",
       "      <td>http://edamontology.org/operation_0449</td>\n",
       "      <td>Sequence alignment analysis (site correlation)</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>247</th>\n",
       "      <td>http://edamontology.org/operation_0450</td>\n",
       "      <td>Chimera detection</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>248</th>\n",
       "      <td>http://edamontology.org/operation_0451</td>\n",
       "      <td>Recombination detection</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>249</th>\n",
       "      <td>http://edamontology.org/operation_0452</td>\n",
       "      <td>Indel detection</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>250</th>\n",
       "      <td>http://edamontology.org/operation_0475</td>\n",
       "      <td>Nucleic acid structure prediction</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>251</th>\n",
       "      <td>http://edamontology.org/operation_0478</td>\n",
       "      <td>Molecular docking</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>252</th>\n",
       "      <td>http://edamontology.org/operation_0483</td>\n",
       "      <td>RNA inverse folding</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>253</th>\n",
       "      <td>http://edamontology.org/operation_0484</td>\n",
       "      <td>SNP detection</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>254</th>\n",
       "      <td>http://edamontology.org/operation_0490</td>\n",
       "      <td>Dot plot plotting</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>255</th>\n",
       "      <td>http://edamontology.org/operation_0499</td>\n",
       "      <td>Tree-based sequence alignment</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>256</th>\n",
       "      <td>http://edamontology.org/operation_0503</td>\n",
       "      <td>Pairwise structure alignment</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>257</th>\n",
       "      <td>http://edamontology.org/operation_0504</td>\n",
       "      <td>Multiple structure alignment</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>258</th>\n",
       "      <td>http://edamontology.org/operation_0509</td>\n",
       "      <td>Local structure alignment</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>259</th>\n",
       "      <td>http://edamontology.org/operation_0510</td>\n",
       "      <td>Global structure alignment</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>260</th>\n",
       "      <td>http://edamontology.org/operation_0531</td>\n",
       "      <td>Heat map generation</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>261</th>\n",
       "      <td>http://edamontology.org/operation_0538</td>\n",
       "      <td>Phylogenetic inference (data centric)</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>262</th>\n",
       "      <td>http://edamontology.org/operation_0539</td>\n",
       "      <td>Phylogenetic inference (method centric)</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>263</th>\n",
       "      <td>http://edamontology.org/operation_0541</td>\n",
       "      <td>Phylogenetic inference (from continuous quantitative characters)</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>264</th>\n",
       "      <td>http://edamontology.org/operation_0544</td>\n",
       "      <td>Species tree construction</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>265</th>\n",
       "      <td>http://edamontology.org/operation_0545</td>\n",
       "      <td>Phylogenetic inference (parsimony methods)</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>266</th>\n",
       "      <td>http://edamontology.org/operation_0546</td>\n",
       "      <td>Phylogenetic inference (minimum distance methods)</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>267</th>\n",
       "      <td>http://edamontology.org/operation_0547</td>\n",
       "      <td>Phylogenetic inference (maximum likelihood and Bayesian methods)</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>268</th>\n",
       "      <td>http://edamontology.org/operation_0548</td>\n",
       "      <td>Phylogenetic inference (quartet methods)</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>269</th>\n",
       "      <td>http://edamontology.org/operation_0549</td>\n",
       "      <td>Phylogenetic inference (AI methods)</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>270</th>\n",
       "      <td>http://edamontology.org/operation_0555</td>\n",
       "      <td>Consensus tree construction</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>271</th>\n",
       "      <td>http://edamontology.org/operation_0564</td>\n",
       "      <td>Sequence visualisation</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>272</th>\n",
       "      <td>http://edamontology.org/operation_0567</td>\n",
       "      <td>Phylogenetic tree visualisation</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>273</th>\n",
       "      <td>http://edamontology.org/operation_0570</td>\n",
       "      <td>Structure visualisation</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>274</th>\n",
       "      <td>http://edamontology.org/operation_0571</td>\n",
       "      <td>Expression data visualisation</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>275</th>\n",
       "      <td>http://edamontology.org/operation_0579</td>\n",
       "      <td>Operon drawing</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>276</th>\n",
       "      <td>http://edamontology.org/operation_1778</td>\n",
       "      <td>Protein function comparison</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>277</th>\n",
       "      <td>http://edamontology.org/operation_2121</td>\n",
       "      <td>Sequence file editing</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>278</th>\n",
       "      <td>http://edamontology.org/operation_2406</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>279</th>\n",
       "      <td>http://edamontology.org/operation_2422</td>\n",
       "      <td>Data retrieval</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>280</th>\n",
       "      <td>http://edamontology.org/operation_2436</td>\n",
       "      <td>Gene-set enrichment analysis</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>281</th>\n",
       "      <td>http://edamontology.org/operation_2441</td>\n",
       "      <td>RNA structure prediction</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>282</th>\n",
       "      <td>http://edamontology.org/operation_2442</td>\n",
       "      <td>DNA structure prediction</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>283</th>\n",
       "      <td>http://edamontology.org/operation_2474</td>\n",
       "      <td>Protein architecture comparison</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>284</th>\n",
       "      <td>http://edamontology.org/operation_2475</td>\n",
       "      <td>Protein architecture recognition</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>285</th>\n",
       "      <td>http://edamontology.org/operation_2478</td>\n",
       "      <td>Nucleic acid sequence analysis</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>286</th>\n",
       "      <td>http://edamontology.org/operation_2479</td>\n",
       "      <td>Protein sequence analysis</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>287</th>\n",
       "      <td>http://edamontology.org/operation_2481</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>288</th>\n",
       "      <td>http://edamontology.org/operation_2483</td>\n",
       "      <td>Structure comparison</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>289</th>\n",
       "      <td>http://edamontology.org/operation_2485</td>\n",
       "      <td>Helical wheel drawing</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>290</th>\n",
       "      <td>http://edamontology.org/operation_2486</td>\n",
       "      <td>Topology diagram drawing</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>291</th>\n",
       "      <td>http://edamontology.org/operation_2489</td>\n",
       "      <td>Subcellular localisation prediction</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>292</th>\n",
       "      <td>http://edamontology.org/operation_2520</td>\n",
       "      <td>DNA mapping</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>293</th>\n",
       "      <td>http://edamontology.org/operation_2935</td>\n",
       "      <td>Clustering profile plotting</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>294</th>\n",
       "      <td>http://edamontology.org/operation_2937</td>\n",
       "      <td>Proximity map plotting</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>295</th>\n",
       "      <td>http://edamontology.org/operation_2938</td>\n",
       "      <td>Dendrogram visualisation</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>296</th>\n",
       "      <td>http://edamontology.org/operation_2942</td>\n",
       "      <td>Treemap visualisation</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>297</th>\n",
       "      <td>http://edamontology.org/operation_2949</td>\n",
       "      <td>Protein-protein interaction analysis</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>298</th>\n",
       "      <td>http://edamontology.org/operation_3095</td>\n",
       "      <td>Nucleic acid design</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>299</th>\n",
       "      <td>http://edamontology.org/operation_3184</td>\n",
       "      <td>Sequence assembly visualisation</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>300</th>\n",
       "      <td>http://edamontology.org/operation_3185</td>\n",
       "      <td>Base-calling</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>301</th>\n",
       "      <td>http://edamontology.org/operation_3187</td>\n",
       "      <td>Sequence contamination filtering</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>302</th>\n",
       "      <td>http://edamontology.org/operation_3192</td>\n",
       "      <td>Sequence trimming</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>303</th>\n",
       "      <td>http://edamontology.org/operation_3195</td>\n",
       "      <td>Sequencing error detection</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>304</th>\n",
       "      <td>http://edamontology.org/operation_3200</td>\n",
       "      <td>DNA barcoding</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>305</th>\n",
       "      <td>http://edamontology.org/operation_3204</td>\n",
       "      <td>Methylation analysis</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>306</th>\n",
       "      <td>http://edamontology.org/operation_3207</td>\n",
       "      <td>Gene methylation analysis</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>307</th>\n",
       "      <td>http://edamontology.org/operation_3218</td>\n",
       "      <td>Sequencing quality control</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>308</th>\n",
       "      <td>http://edamontology.org/operation_3219</td>\n",
       "      <td>Read pre-processing</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>309</th>\n",
       "      <td>http://edamontology.org/operation_3227</td>\n",
       "      <td>Variant calling</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>310</th>\n",
       "      <td>http://edamontology.org/operation_3230</td>\n",
       "      <td>Read depth analysis</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>311</th>\n",
       "      <td>http://edamontology.org/operation_3237</td>\n",
       "      <td>Primer removal</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>312</th>\n",
       "      <td>http://edamontology.org/operation_3446</td>\n",
       "      <td>Cell migration analysis</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>313</th>\n",
       "      <td>http://edamontology.org/operation_3450</td>\n",
       "      <td>Neurite measurement</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>314</th>\n",
       "      <td>http://edamontology.org/operation_3459</td>\n",
       "      <td>Functional clustering</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>315</th>\n",
       "      <td>http://edamontology.org/operation_3461</td>\n",
       "      <td>Virulence prediction</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>316</th>\n",
       "      <td>http://edamontology.org/operation_3463</td>\n",
       "      <td>Expression correlation analysis</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>317</th>\n",
       "      <td>http://edamontology.org/operation_3472</td>\n",
       "      <td>k-mer counting</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>318</th>\n",
       "      <td>http://edamontology.org/operation_3482</td>\n",
       "      <td>Antimicrobial resistance prediction</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>319</th>\n",
       "      <td>http://edamontology.org/operation_3552</td>\n",
       "      <td>Microscope image visualisation</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>320</th>\n",
       "      <td>http://edamontology.org/operation_3564</td>\n",
       "      <td>Chemical redundancy removal</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>321</th>\n",
       "      <td>http://edamontology.org/operation_3644</td>\n",
       "      <td>de Novo sequencing</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>322</th>\n",
       "      <td>http://edamontology.org/operation_3646</td>\n",
       "      <td>Peptide database search</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>323</th>\n",
       "      <td>http://edamontology.org/operation_3647</td>\n",
       "      <td>Blind peptide database search</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>324</th>\n",
       "      <td>http://edamontology.org/operation_3649</td>\n",
       "      <td>Target-Decoy</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>325</th>\n",
       "      <td>http://edamontology.org/operation_3675</td>\n",
       "      <td>Variant filtering</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>326</th>\n",
       "      <td>http://edamontology.org/operation_3778</td>\n",
       "      <td>Text annotation</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>327</th>\n",
       "      <td>http://edamontology.org/operation_3793</td>\n",
       "      <td>Read summarisation</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>328</th>\n",
       "      <td>http://edamontology.org/operation_3795</td>\n",
       "      <td>In vitro selection</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>329</th>\n",
       "      <td>http://edamontology.org/operation_3798</td>\n",
       "      <td>Read binning</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>330</th>\n",
       "      <td>http://edamontology.org/operation_3809</td>\n",
       "      <td>DMR identification</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>331</th>\n",
       "      <td>http://edamontology.org/operation_3890</td>\n",
       "      <td>Trajectory visualization</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>332</th>\n",
       "      <td>http://edamontology.org/operation_3891</td>\n",
       "      <td>Essential dynamics</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>333</th>\n",
       "      <td>http://edamontology.org/operation_3893</td>\n",
       "      <td>Forcefield parameterisation</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>334</th>\n",
       "      <td>http://edamontology.org/operation_3894</td>\n",
       "      <td>DNA profiling</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>335</th>\n",
       "      <td>http://edamontology.org/operation_3919</td>\n",
       "      <td>Methylation calling</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>336</th>\n",
       "      <td>http://edamontology.org/operation_3921</td>\n",
       "      <td>Sequence read processing</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>337</th>\n",
       "      <td>http://edamontology.org/operation_3927</td>\n",
       "      <td>Network analysis</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>338</th>\n",
       "      <td>http://edamontology.org/operation_3928</td>\n",
       "      <td>Pathway analysis</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>339</th>\n",
       "      <td>http://edamontology.org/operation_3929</td>\n",
       "      <td>Metabolic pathway prediction</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>340</th>\n",
       "      <td>http://edamontology.org/operation_3933</td>\n",
       "      <td>Demultiplexing</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>341</th>\n",
       "      <td>http://edamontology.org/operation_3936</td>\n",
       "      <td>Feature selection</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>342</th>\n",
       "      <td>http://edamontology.org/operation_3937</td>\n",
       "      <td>Feature extraction</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>343</th>\n",
       "      <td>http://edamontology.org/operation_3950</td>\n",
       "      <td>Selection detection</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>344</th>\n",
       "      <td>http://edamontology.org/data_0849</td>\n",
       "      <td>Sequence record</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>345</th>\n",
       "      <td>http://edamontology.org/data_0858</td>\n",
       "      <td>Sequence signature matches</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>346</th>\n",
       "      <td>http://edamontology.org/data_0860</td>\n",
       "      <td>Sequence signature data</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>347</th>\n",
       "      <td>http://edamontology.org/data_0863</td>\n",
       "      <td>Sequence alignment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>348</th>\n",
       "      <td>http://edamontology.org/data_0872</td>\n",
       "      <td>Phylogenetic tree</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>349</th>\n",
       "      <td>http://edamontology.org/data_0883</td>\n",
       "      <td>Structure</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>350</th>\n",
       "      <td>http://edamontology.org/data_0886</td>\n",
       "      <td>Structure alignment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>351</th>\n",
       "      <td>http://edamontology.org/data_0889</td>\n",
       "      <td>Structural profile</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>352</th>\n",
       "      <td>http://edamontology.org/data_0906</td>\n",
       "      <td>Protein interaction data</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>353</th>\n",
       "      <td>http://edamontology.org/data_0907</td>\n",
       "      <td>Protein family report</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>354</th>\n",
       "      <td>http://edamontology.org/data_0914</td>\n",
       "      <td>Codon usage data</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>355</th>\n",
       "      <td>http://edamontology.org/data_0928</td>\n",
       "      <td>Gene expression profile</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>356</th>\n",
       "      <td>http://edamontology.org/data_0940</td>\n",
       "      <td>Volume map</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>357</th>\n",
       "      <td>http://edamontology.org/data_0942</td>\n",
       "      <td>2D PAGE image</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>358</th>\n",
       "      <td>http://edamontology.org/data_0943</td>\n",
       "      <td>Mass spectrum</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>359</th>\n",
       "      <td>http://edamontology.org/data_0944</td>\n",
       "      <td>Peptide mass fingerprint</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>360</th>\n",
       "      <td>http://edamontology.org/data_0945</td>\n",
       "      <td>Peptide identification</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>361</th>\n",
       "      <td>http://edamontology.org/data_0950</td>\n",
       "      <td>Mathematical model</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>362</th>\n",
       "      <td>http://edamontology.org/data_0955</td>\n",
       "      <td>Data index</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>363</th>\n",
       "      <td>http://edamontology.org/data_0956</td>\n",
       "      <td>Data index report</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>364</th>\n",
       "      <td>http://edamontology.org/data_0962</td>\n",
       "      <td>Small molecule report</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>365</th>\n",
       "      <td>http://edamontology.org/data_0966</td>\n",
       "      <td>Ontology term</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>366</th>\n",
       "      <td>http://edamontology.org/data_0967</td>\n",
       "      <td>Ontology concept data</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>367</th>\n",
       "      <td>http://edamontology.org/data_0970</td>\n",
       "      <td>Citation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>368</th>\n",
       "      <td>http://edamontology.org/data_0971</td>\n",
       "      <td>Article</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>369</th>\n",
       "      <td>http://edamontology.org/data_0972</td>\n",
       "      <td>Text mining report</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>370</th>\n",
       "      <td>http://edamontology.org/data_1235</td>\n",
       "      <td>Sequence cluster</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>371</th>\n",
       "      <td>http://edamontology.org/data_1238</td>\n",
       "      <td>Proteolytic digest</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>372</th>\n",
       "      <td>http://edamontology.org/data_1245</td>\n",
       "      <td>Sequence cluster (protein)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>373</th>\n",
       "      <td>http://edamontology.org/data_1246</td>\n",
       "      <td>Sequence cluster (nucleic acid)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>374</th>\n",
       "      <td>http://edamontology.org/data_1274</td>\n",
       "      <td>Map</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>375</th>\n",
       "      <td>http://edamontology.org/data_1278</td>\n",
       "      <td>Genetic map</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>376</th>\n",
       "      <td>http://edamontology.org/data_1279</td>\n",
       "      <td>Sequence map</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>377</th>\n",
       "      <td>http://edamontology.org/data_1280</td>\n",
       "      <td>Physical map</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>378</th>\n",
       "      <td>http://edamontology.org/data_1283</td>\n",
       "      <td>Cytogenetic map</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>379</th>\n",
       "      <td>http://edamontology.org/data_1284</td>\n",
       "      <td>DNA transduction map</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>380</th>\n",
       "      <td>http://edamontology.org/data_1285</td>\n",
       "      <td>Gene map</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>381</th>\n",
       "      <td>http://edamontology.org/data_1286</td>\n",
       "      <td>Plasmid map</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>382</th>\n",
       "      <td>http://edamontology.org/data_1288</td>\n",
       "      <td>Genome map</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>383</th>\n",
       "      <td>http://edamontology.org/data_1289</td>\n",
       "      <td>Restriction map</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>384</th>\n",
       "      <td>http://edamontology.org/data_1347</td>\n",
       "      <td>Dirichlet distribution</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>385</th>\n",
       "      <td>http://edamontology.org/data_1355</td>\n",
       "      <td>Protein signature</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>386</th>\n",
       "      <td>http://edamontology.org/data_1364</td>\n",
       "      <td>Hidden Markov model</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>387</th>\n",
       "      <td>http://edamontology.org/data_1381</td>\n",
       "      <td>Pair sequence alignment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>388</th>\n",
       "      <td>http://edamontology.org/data_1383</td>\n",
       "      <td>Nucleic acid sequence alignment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>389</th>\n",
       "      <td>http://edamontology.org/data_1384</td>\n",
       "      <td>Protein sequence alignment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>390</th>\n",
       "      <td>http://edamontology.org/data_1385</td>\n",
       "      <td>Hybrid sequence alignment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>391</th>\n",
       "      <td>http://edamontology.org/data_1429</td>\n",
       "      <td>Phylogenetic invariants</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>392</th>\n",
       "      <td>http://edamontology.org/data_1439</td>\n",
       "      <td>DNA substitution model</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>393</th>\n",
       "      <td>http://edamontology.org/data_1479</td>\n",
       "      <td>Structure alignment (pair)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>394</th>\n",
       "      <td>http://edamontology.org/data_1481</td>\n",
       "      <td>Protein structure alignment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>395</th>\n",
       "      <td>http://edamontology.org/data_1482</td>\n",
       "      <td>Nucleic acid structure alignment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>396</th>\n",
       "      <td>http://edamontology.org/data_1493</td>\n",
       "      <td>RNA structure alignment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>397</th>\n",
       "      <td>http://edamontology.org/data_1546</td>\n",
       "      <td>Protein distance matrix</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>398</th>\n",
       "      <td>http://edamontology.org/data_1547</td>\n",
       "      <td>Protein contact map</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>399</th>\n",
       "      <td>http://edamontology.org/data_1548</td>\n",
       "      <td>Protein residue 3D cluster</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>400</th>\n",
       "      <td>http://edamontology.org/data_1549</td>\n",
       "      <td>Protein hydrogen bonds</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>401</th>\n",
       "      <td>http://edamontology.org/data_1566</td>\n",
       "      <td>Protein-ligand interaction report</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>402</th>\n",
       "      <td>http://edamontology.org/data_1598</td>\n",
       "      <td>Genetic code</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>403</th>\n",
       "      <td>http://edamontology.org/data_1600</td>\n",
       "      <td>Codon usage bias plot</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>404</th>\n",
       "      <td>http://edamontology.org/data_1602</td>\n",
       "      <td>Codon usage fraction difference</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>405</th>\n",
       "      <td>http://edamontology.org/data_1622</td>\n",
       "      <td>Disease report</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>406</th>\n",
       "      <td>http://edamontology.org/data_1636</td>\n",
       "      <td>Heat map</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>407</th>\n",
       "      <td>http://edamontology.org/data_1713</td>\n",
       "      <td>Fate map</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>408</th>\n",
       "      <td>http://edamontology.org/data_1714</td>\n",
       "      <td>Microarray spots image</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>409</th>\n",
       "      <td>http://edamontology.org/data_1731</td>\n",
       "      <td>Ontology concept definition</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>410</th>\n",
       "      <td>http://edamontology.org/data_1859</td>\n",
       "      <td>Deletion map</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>411</th>\n",
       "      <td>http://edamontology.org/data_1860</td>\n",
       "      <td>QTL map</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>412</th>\n",
       "      <td>http://edamontology.org/data_1863</td>\n",
       "      <td>Haplotype map</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>413</th>\n",
       "      <td>http://edamontology.org/data_2019</td>\n",
       "      <td>Map data</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>414</th>\n",
       "      <td>http://edamontology.org/data_2044</td>\n",
       "      <td>Sequence</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>415</th>\n",
       "      <td>http://edamontology.org/data_2223</td>\n",
       "      <td>Ontology metadata</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>416</th>\n",
       "      <td>http://edamontology.org/data_2353</td>\n",
       "      <td>Ontology data</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>417</th>\n",
       "      <td>http://edamontology.org/data_2526</td>\n",
       "      <td>Text data</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>418</th>\n",
       "      <td>http://edamontology.org/data_2535</td>\n",
       "      <td>Sequence tag profile</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>419</th>\n",
       "      <td>http://edamontology.org/data_2600</td>\n",
       "      <td>Pathway or network</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>420</th>\n",
       "      <td>http://edamontology.org/data_2603</td>\n",
       "      <td>Expression data</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>421</th>\n",
       "      <td>http://edamontology.org/data_2762</td>\n",
       "      <td>Sequence signature report</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>422</th>\n",
       "      <td>http://edamontology.org/data_2849</td>\n",
       "      <td>Abstract</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>423</th>\n",
       "      <td>http://edamontology.org/data_2850</td>\n",
       "      <td>Lipid structure</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>424</th>\n",
       "      <td>http://edamontology.org/data_2858</td>\n",
       "      <td>Ontology concept</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>425</th>\n",
       "      <td>http://edamontology.org/data_2865</td>\n",
       "      <td>Codon usage bias</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>426</th>\n",
       "      <td>http://edamontology.org/data_2870</td>\n",
       "      <td>Radiation hybrid map</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>427</th>\n",
       "      <td>http://edamontology.org/data_2886</td>\n",
       "      <td>Protein sequence record</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>428</th>\n",
       "      <td>http://edamontology.org/data_2887</td>\n",
       "      <td>Nucleic acid sequence record</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>429</th>\n",
       "      <td>http://edamontology.org/data_2976</td>\n",
       "      <td>Protein sequence</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>430</th>\n",
       "      <td>http://edamontology.org/data_2977</td>\n",
       "      <td>Nucleic acid sequence</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>431</th>\n",
       "      <td>http://edamontology.org/data_2984</td>\n",
       "      <td>Pathway or network report</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>432</th>\n",
       "      <td>http://edamontology.org/data_3028</td>\n",
       "      <td>Taxonomy</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>433</th>\n",
       "      <td>http://edamontology.org/data_3110</td>\n",
       "      <td>Raw microarray data</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>434</th>\n",
       "      <td>http://edamontology.org/data_3112</td>\n",
       "      <td>Gene expression matrix</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>435</th>\n",
       "      <td>http://edamontology.org/data_3115</td>\n",
       "      <td>Microarray metadata</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>436</th>\n",
       "      <td>http://edamontology.org/data_3117</td>\n",
       "      <td>Microarray hybridisation data</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>437</th>\n",
       "      <td>http://edamontology.org/data_3210</td>\n",
       "      <td>Genome index</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>438</th>\n",
       "      <td>http://edamontology.org/data_3241</td>\n",
       "      <td>Kinetic model</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>439</th>\n",
       "      <td>http://edamontology.org/data_3271</td>\n",
       "      <td>Gene tree</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>440</th>\n",
       "      <td>http://edamontology.org/data_3272</td>\n",
       "      <td>Species tree</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>441</th>\n",
       "      <td>http://edamontology.org/data_3442</td>\n",
       "      <td>MRI image</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>442</th>\n",
       "      <td>http://edamontology.org/data_3449</td>\n",
       "      <td>Cell migration track image</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>443</th>\n",
       "      <td>http://edamontology.org/data_3488</td>\n",
       "      <td>NMR spectrum</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>444</th>\n",
       "      <td>http://edamontology.org/data_3492</td>\n",
       "      <td>Nucleic acid signature</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>445</th>\n",
       "      <td>http://edamontology.org/data_3494</td>\n",
       "      <td>DNA sequence</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>446</th>\n",
       "      <td>http://edamontology.org/data_3495</td>\n",
       "      <td>RNA sequence</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>447</th>\n",
       "      <td>http://edamontology.org/data_3498</td>\n",
       "      <td>Sequence variations</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>448</th>\n",
       "      <td>http://edamontology.org/data_3505</td>\n",
       "      <td>Bibliography</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>449</th>\n",
       "      <td>http://edamontology.org/data_3671</td>\n",
       "      <td>Text</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>450</th>\n",
       "      <td>http://edamontology.org/data_3754</td>\n",
       "      <td>GO-term enrichment data</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>451</th>\n",
       "      <td>http://edamontology.org/data_3768</td>\n",
       "      <td>Clustered expression profiles</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>452</th>\n",
       "      <td>http://edamontology.org/data_3779</td>\n",
       "      <td>Annotated text</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>453</th>\n",
       "      <td>http://edamontology.org/data_3807</td>\n",
       "      <td>EM Movie</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>454</th>\n",
       "      <td>http://edamontology.org/data_3808</td>\n",
       "      <td>EM Micrograph</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>455</th>\n",
       "      <td>http://edamontology.org/data_3842</td>\n",
       "      <td>Molecular simulation data</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>456</th>\n",
       "      <td>http://edamontology.org/data_3924</td>\n",
       "      <td>DNA structure alignment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>457</th>\n",
       "      <td>http://edamontology.org/data_3953</td>\n",
       "      <td>Pathway overrepresentation data</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>458</th>\n",
       "      <td>http://edamontology.org/operation_0224</td>\n",
       "      <td>Query and retrieval</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>459</th>\n",
       "      <td>http://edamontology.org/operation_0226</td>\n",
       "      <td>Annotation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>460</th>\n",
       "      <td>http://edamontology.org/operation_0230</td>\n",
       "      <td>Sequence generation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>461</th>\n",
       "      <td>http://edamontology.org/operation_0248</td>\n",
       "      <td>Residue interaction calculation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>462</th>\n",
       "      <td>http://edamontology.org/operation_0258</td>\n",
       "      <td>Sequence alignment analysis</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>463</th>\n",
       "      <td>http://edamontology.org/operation_0259</td>\n",
       "      <td>Sequence alignment comparison</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>464</th>\n",
       "      <td>http://edamontology.org/operation_0260</td>\n",
       "      <td>Sequence alignment conversion</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>465</th>\n",
       "      <td>http://edamontology.org/operation_0270</td>\n",
       "      <td>Transmembrane protein analysis</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>466</th>\n",
       "      <td>http://edamontology.org/operation_0288</td>\n",
       "      <td>Sequence word comparison</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>467</th>\n",
       "      <td>http://edamontology.org/operation_0290</td>\n",
       "      <td>Sequence redundancy removal</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>468</th>\n",
       "      <td>http://edamontology.org/operation_0291</td>\n",
       "      <td>Sequence clustering</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>469</th>\n",
       "      <td>http://edamontology.org/operation_0292</td>\n",
       "      <td>Sequence alignment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>470</th>\n",
       "      <td>http://edamontology.org/operation_0294</td>\n",
       "      <td>Structure-based sequence alignment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>471</th>\n",
       "      <td>http://edamontology.org/operation_0297</td>\n",
       "      <td>3D profile generation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>472</th>\n",
       "      <td>http://edamontology.org/operation_0306</td>\n",
       "      <td>Text mining</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>473</th>\n",
       "      <td>http://edamontology.org/operation_0313</td>\n",
       "      <td>Expression profile clustering</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>474</th>\n",
       "      <td>http://edamontology.org/operation_0314</td>\n",
       "      <td>Gene expression profiling</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>475</th>\n",
       "      <td>http://edamontology.org/operation_0315</td>\n",
       "      <td>Expression profile comparison</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>476</th>\n",
       "      <td>http://edamontology.org/operation_0322</td>\n",
       "      <td>Molecular model refinement</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>477</th>\n",
       "      <td>http://edamontology.org/operation_0324</td>\n",
       "      <td>Phylogenetic analysis</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>478</th>\n",
       "      <td>http://edamontology.org/operation_0325</td>\n",
       "      <td>Phylogenetic tree comparison</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>479</th>\n",
       "      <td>http://edamontology.org/operation_0334</td>\n",
       "      <td>Enzyme kinetics calculation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>480</th>\n",
       "      <td>http://edamontology.org/operation_0335</td>\n",
       "      <td>Formatting</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>481</th>\n",
       "      <td>http://edamontology.org/operation_0337</td>\n",
       "      <td>Visualisation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>482</th>\n",
       "      <td>http://edamontology.org/operation_0361</td>\n",
       "      <td>Sequence annotation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>483</th>\n",
       "      <td>http://edamontology.org/operation_0363</td>\n",
       "      <td>Reverse complement</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>484</th>\n",
       "      <td>http://edamontology.org/operation_0364</td>\n",
       "      <td>Random sequence generation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>485</th>\n",
       "      <td>http://edamontology.org/operation_0365</td>\n",
       "      <td>Restriction digest</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>486</th>\n",
       "      <td>http://edamontology.org/operation_0391</td>\n",
       "      <td>Protein distance matrix calculation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>487</th>\n",
       "      <td>http://edamontology.org/operation_0392</td>\n",
       "      <td>Contact map calculation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>488</th>\n",
       "      <td>http://edamontology.org/operation_0393</td>\n",
       "      <td>Residue cluster calculation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>489</th>\n",
       "      <td>http://edamontology.org/operation_0394</td>\n",
       "      <td>Hydrogen bond calculation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>490</th>\n",
       "      <td>http://edamontology.org/operation_0398</td>\n",
       "      <td>Protein molecular weight calculation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>491</th>\n",
       "      <td>http://edamontology.org/operation_0400</td>\n",
       "      <td>Protein pKa calculation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>492</th>\n",
       "      <td>http://edamontology.org/operation_0403</td>\n",
       "      <td>Protein isoelectric point calculation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>493</th>\n",
       "      <td>http://edamontology.org/operation_0404</td>\n",
       "      <td>Protein hydrogen exchange rate calculation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>494</th>\n",
       "      <td>http://edamontology.org/operation_0405</td>\n",
       "      <td>Protein hydrophobic region calculation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>495</th>\n",
       "      <td>http://edamontology.org/operation_0406</td>\n",
       "      <td>Protein aliphatic index calculation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>496</th>\n",
       "      <td>http://edamontology.org/operation_0408</td>\n",
       "      <td>Protein globularity prediction</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>497</th>\n",
       "      <td>http://edamontology.org/operation_0409</td>\n",
       "      <td>Protein solubility prediction</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>498</th>\n",
       "      <td>http://edamontology.org/operation_0410</td>\n",
       "      <td>Protein crystallizability prediction</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>499</th>\n",
       "      <td>http://edamontology.org/operation_0447</td>\n",
       "      <td>Sequence alignment validation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>500</th>\n",
       "      <td>http://edamontology.org/operation_0448</td>\n",
       "      <td>Sequence alignment analysis (conservation)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>501</th>\n",
       "      <td>http://edamontology.org/operation_0456</td>\n",
       "      <td>Nucleic acid melting profile plotting</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>502</th>\n",
       "      <td>http://edamontology.org/operation_0457</td>\n",
       "      <td>Nucleic acid stitch profile plotting</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>503</th>\n",
       "      <td>http://edamontology.org/operation_0458</td>\n",
       "      <td>Nucleic acid melting curve plotting</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>504</th>\n",
       "      <td>http://edamontology.org/operation_0459</td>\n",
       "      <td>Nucleic acid probability profile plotting</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>505</th>\n",
       "      <td>http://edamontology.org/operation_0460</td>\n",
       "      <td>Nucleic acid temperature profile plotting</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>506</th>\n",
       "      <td>http://edamontology.org/operation_0491</td>\n",
       "      <td>Pairwise sequence alignment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>507</th>\n",
       "      <td>http://edamontology.org/operation_0492</td>\n",
       "      <td>Multiple sequence alignment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>508</th>\n",
       "      <td>http://edamontology.org/operation_0495</td>\n",
       "      <td>Local alignment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>509</th>\n",
       "      <td>http://edamontology.org/operation_0496</td>\n",
       "      <td>Global alignment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>510</th>\n",
       "      <td>http://edamontology.org/operation_0551</td>\n",
       "      <td>Phylogenetic tree topology analysis</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>511</th>\n",
       "      <td>http://edamontology.org/operation_0552</td>\n",
       "      <td>Phylogenetic tree bootstrapping</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>512</th>\n",
       "      <td>http://edamontology.org/operation_0554</td>\n",
       "      <td>Allele frequency distribution analysis</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>513</th>\n",
       "      <td>http://edamontology.org/operation_0556</td>\n",
       "      <td>Phylogenetic sub/super tree construction</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>514</th>\n",
       "      <td>http://edamontology.org/operation_0557</td>\n",
       "      <td>Phylogenetic tree distances calculation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>515</th>\n",
       "      <td>http://edamontology.org/operation_0558</td>\n",
       "      <td>Phylogenetic tree annotation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>516</th>\n",
       "      <td>http://edamontology.org/operation_0566</td>\n",
       "      <td>Sequence cluster visualisation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>517</th>\n",
       "      <td>http://edamontology.org/operation_0573</td>\n",
       "      <td>Map drawing</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>518</th>\n",
       "      <td>http://edamontology.org/operation_0575</td>\n",
       "      <td>Restriction map drawing</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>519</th>\n",
       "      <td>http://edamontology.org/operation_0578</td>\n",
       "      <td>Plasmid map drawing</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>520</th>\n",
       "      <td>http://edamontology.org/operation_1777</td>\n",
       "      <td>Protein function prediction</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>521</th>\n",
       "      <td>http://edamontology.org/operation_1812</td>\n",
       "      <td>Parsing</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>522</th>\n",
       "      <td>http://edamontology.org/operation_1834</td>\n",
       "      <td>Protein-metal contact calculation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>523</th>\n",
       "      <td>http://edamontology.org/operation_1839</td>\n",
       "      <td>Salt bridge calculation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>524</th>\n",
       "      <td>http://edamontology.org/operation_2089</td>\n",
       "      <td>Sequence alignment refinement</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>525</th>\n",
       "      <td>http://edamontology.org/operation_2233</td>\n",
       "      <td>Representative sequence identification</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>526</th>\n",
       "      <td>http://edamontology.org/operation_2239</td>\n",
       "      <td>3D-1D scoring matrix generation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>527</th>\n",
       "      <td>http://edamontology.org/operation_2403</td>\n",
       "      <td>Sequence analysis</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>528</th>\n",
       "      <td>http://edamontology.org/operation_2404</td>\n",
       "      <td>Sequence motif analysis</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>529</th>\n",
       "      <td>http://edamontology.org/operation_2409</td>\n",
       "      <td>Data handling</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>530</th>\n",
       "      <td>http://edamontology.org/operation_2421</td>\n",
       "      <td>Database search</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>531</th>\n",
       "      <td>http://edamontology.org/operation_2426</td>\n",
       "      <td>Modelling and simulation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>532</th>\n",
       "      <td>http://edamontology.org/operation_2451</td>\n",
       "      <td>Sequence comparison</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>533</th>\n",
       "      <td>http://edamontology.org/operation_2480</td>\n",
       "      <td>Structure analysis</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>534</th>\n",
       "      <td>http://edamontology.org/operation_2495</td>\n",
       "      <td>Expression analysis</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>535</th>\n",
       "      <td>http://edamontology.org/operation_2574</td>\n",
       "      <td>Protein hydropathy calculation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>536</th>\n",
       "      <td>http://edamontology.org/operation_2929</td>\n",
       "      <td>Protein fragment weight comparison</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>537</th>\n",
       "      <td>http://edamontology.org/operation_2939</td>\n",
       "      <td>Principal component visualisation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>538</th>\n",
       "      <td>http://edamontology.org/operation_2940</td>\n",
       "      <td>Scatter plot plotting</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>539</th>\n",
       "      <td>http://edamontology.org/operation_2943</td>\n",
       "      <td>Box-Whisker plot plotting</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>540</th>\n",
       "      <td>http://edamontology.org/operation_2950</td>\n",
       "      <td>Residue distance calculation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>541</th>\n",
       "      <td>http://edamontology.org/operation_2995</td>\n",
       "      <td>Sequence classification</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>542</th>\n",
       "      <td>http://edamontology.org/operation_2996</td>\n",
       "      <td>Structure classification</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>543</th>\n",
       "      <td>http://edamontology.org/operation_3080</td>\n",
       "      <td>Structure editing</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>544</th>\n",
       "      <td>http://edamontology.org/operation_3081</td>\n",
       "      <td>Sequence alignment editing</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>545</th>\n",
       "      <td>http://edamontology.org/operation_3096</td>\n",
       "      <td>Editing</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>546</th>\n",
       "      <td>http://edamontology.org/operation_3203</td>\n",
       "      <td>Chromatogram visualisation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>547</th>\n",
       "      <td>http://edamontology.org/operation_3214</td>\n",
       "      <td>Spectral analysis</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>548</th>\n",
       "      <td>http://edamontology.org/operation_3215</td>\n",
       "      <td>Peak detection</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>549</th>\n",
       "      <td>http://edamontology.org/operation_3223</td>\n",
       "      <td>Differential gene expression profiling</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>550</th>\n",
       "      <td>http://edamontology.org/operation_3225</td>\n",
       "      <td>Variant classification</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>551</th>\n",
       "      <td>http://edamontology.org/operation_3232</td>\n",
       "      <td>Gene expression QTL analysis</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>552</th>\n",
       "      <td>http://edamontology.org/operation_3278</td>\n",
       "      <td>Document similarity calculation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>553</th>\n",
       "      <td>http://edamontology.org/operation_3279</td>\n",
       "      <td>Document clustering</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>554</th>\n",
       "      <td>http://edamontology.org/operation_3280</td>\n",
       "      <td>Named-entity and concept recognition</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>555</th>\n",
       "      <td>http://edamontology.org/operation_3283</td>\n",
       "      <td>Anonymisation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>556</th>\n",
       "      <td>http://edamontology.org/operation_3348</td>\n",
       "      <td>Sequence checksum generation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>557</th>\n",
       "      <td>http://edamontology.org/operation_3357</td>\n",
       "      <td>Format detection</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>558</th>\n",
       "      <td>http://edamontology.org/operation_3359</td>\n",
       "      <td>Splitting</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>559</th>\n",
       "      <td>http://edamontology.org/operation_3431</td>\n",
       "      <td>Deposition</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>560</th>\n",
       "      <td>http://edamontology.org/operation_3436</td>\n",
       "      <td>Aggregation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>561</th>\n",
       "      <td>http://edamontology.org/operation_3437</td>\n",
       "      <td>Article comparison</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>562</th>\n",
       "      <td>http://edamontology.org/operation_3443</td>\n",
       "      <td>Image analysis</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>563</th>\n",
       "      <td>http://edamontology.org/operation_3445</td>\n",
       "      <td>Diffraction data analysis</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>564</th>\n",
       "      <td>http://edamontology.org/operation_3447</td>\n",
       "      <td>Diffraction data reduction</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>565</th>\n",
       "      <td>http://edamontology.org/operation_3453</td>\n",
       "      <td>Diffraction data integration</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>566</th>\n",
       "      <td>http://edamontology.org/operation_3454</td>\n",
       "      <td>Phasing</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>567</th>\n",
       "      <td>http://edamontology.org/operation_3455</td>\n",
       "      <td>Molecular replacement</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>568</th>\n",
       "      <td>http://edamontology.org/operation_3456</td>\n",
       "      <td>Rigid body refinement</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>569</th>\n",
       "      <td>http://edamontology.org/operation_3460</td>\n",
       "      <td>Taxonomic classification</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>570</th>\n",
       "      <td>http://edamontology.org/operation_3465</td>\n",
       "      <td>Correlation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>571</th>\n",
       "      <td>http://edamontology.org/operation_3481</td>\n",
       "      <td>Probabilistic sequence generation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>572</th>\n",
       "      <td>http://edamontology.org/operation_3502</td>\n",
       "      <td>Chemical similarity enrichment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>573</th>\n",
       "      <td>http://edamontology.org/operation_3503</td>\n",
       "      <td>Incident curve plotting</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>574</th>\n",
       "      <td>http://edamontology.org/operation_3553</td>\n",
       "      <td>Image annotation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>575</th>\n",
       "      <td>http://edamontology.org/operation_3559</td>\n",
       "      <td>Ontology visualisation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>576</th>\n",
       "      <td>http://edamontology.org/operation_3566</td>\n",
       "      <td>Simulated gene expression data generation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>577</th>\n",
       "      <td>http://edamontology.org/operation_3625</td>\n",
       "      <td>Relation extraction</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>578</th>\n",
       "      <td>http://edamontology.org/operation_3627</td>\n",
       "      <td>Mass spectra calibration</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>579</th>\n",
       "      <td>http://edamontology.org/operation_3628</td>\n",
       "      <td>Chromatographic alignment</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>580</th>\n",
       "      <td>http://edamontology.org/operation_3629</td>\n",
       "      <td>Deisotoping</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>581</th>\n",
       "      <td>http://edamontology.org/operation_3630</td>\n",
       "      <td>Protein quantification</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>582</th>\n",
       "      <td>http://edamontology.org/operation_3631</td>\n",
       "      <td>Peptide identification</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>583</th>\n",
       "      <td>http://edamontology.org/operation_3632</td>\n",
       "      <td>Isotopic distributions calculation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>584</th>\n",
       "      <td>http://edamontology.org/operation_3634</td>\n",
       "      <td>Label-free quantification</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>585</th>\n",
       "      <td>http://edamontology.org/operation_3635</td>\n",
       "      <td>Labeled quantification</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>586</th>\n",
       "      <td>http://edamontology.org/operation_3636</td>\n",
       "      <td>MRM/SRM</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>587</th>\n",
       "      <td>http://edamontology.org/operation_3637</td>\n",
       "      <td>Spectral counting</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>588</th>\n",
       "      <td>http://edamontology.org/operation_3638</td>\n",
       "      <td>SILAC</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>589</th>\n",
       "      <td>http://edamontology.org/operation_3639</td>\n",
       "      <td>iTRAQ</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>590</th>\n",
       "      <td>http://edamontology.org/operation_3640</td>\n",
       "      <td>18O labeling</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>591</th>\n",
       "      <td>http://edamontology.org/operation_3641</td>\n",
       "      <td>TMT-tag</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>592</th>\n",
       "      <td>http://edamontology.org/operation_3642</td>\n",
       "      <td>Dimethyl</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>593</th>\n",
       "      <td>http://edamontology.org/operation_3643</td>\n",
       "      <td>Tag-based peptide identification</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>594</th>\n",
       "      <td>http://edamontology.org/operation_3645</td>\n",
       "      <td>PTM identification</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>595</th>\n",
       "      <td>http://edamontology.org/operation_3661</td>\n",
       "      <td>SNP annotation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>596</th>\n",
       "      <td>http://edamontology.org/operation_3672</td>\n",
       "      <td>Gene functional annotation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>597</th>\n",
       "      <td>http://edamontology.org/operation_3694</td>\n",
       "      <td>Mass spectrum visualisation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>598</th>\n",
       "      <td>http://edamontology.org/operation_3695</td>\n",
       "      <td>Filtering</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>599</th>\n",
       "      <td>http://edamontology.org/operation_3704</td>\n",
       "      <td>Ion counting</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>600</th>\n",
       "      <td>http://edamontology.org/operation_3705</td>\n",
       "      <td>Isotope-coded protein label</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>601</th>\n",
       "      <td>http://edamontology.org/operation_3715</td>\n",
       "      <td>Metabolic labeling</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>602</th>\n",
       "      <td>http://edamontology.org/operation_3741</td>\n",
       "      <td>Differential protein expression profiling</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>603</th>\n",
       "      <td>http://edamontology.org/operation_3744</td>\n",
       "      <td>Multiple sample visualisation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>604</th>\n",
       "      <td>http://edamontology.org/operation_3745</td>\n",
       "      <td>Ancestral reconstruction</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>605</th>\n",
       "      <td>http://edamontology.org/operation_3755</td>\n",
       "      <td>PTM localisation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>606</th>\n",
       "      <td>http://edamontology.org/operation_3767</td>\n",
       "      <td>Protein identification</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>607</th>\n",
       "      <td>http://edamontology.org/operation_3792</td>\n",
       "      <td>miRNA expression analysis</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>608</th>\n",
       "      <td>http://edamontology.org/operation_3801</td>\n",
       "      <td>Spectral library search</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>609</th>\n",
       "      <td>http://edamontology.org/operation_3802</td>\n",
       "      <td>Sorting</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>610</th>\n",
       "      <td>http://edamontology.org/operation_3803</td>\n",
       "      <td>Natural product identification</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>611</th>\n",
       "      <td>http://edamontology.org/operation_3860</td>\n",
       "      <td>Spectrum calculation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>612</th>\n",
       "      <td>http://edamontology.org/operation_3907</td>\n",
       "      <td>Information extraction</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>613</th>\n",
       "      <td>http://edamontology.org/operation_3908</td>\n",
       "      <td>Information retrieval</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>614</th>\n",
       "      <td>http://edamontology.org/operation_3935</td>\n",
       "      <td>Dimensionality reduction</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>615</th>\n",
       "      <td>http://edamontology.org/operation_3942</td>\n",
       "      <td>Tree dating</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>616</th>\n",
       "      <td>http://edamontology.org/operation_3947</td>\n",
       "      <td>Phylogenetic tree reconciliation</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>617</th>\n",
       "      <td>http://edamontology.org/operation_4034</td>\n",
       "      <td>Multiple testing correction</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "results1=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "print(f\"nb of edam format with is_format_of relation: {len(results1)}\")\n",
    "\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results1.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3.9.5 64-bit",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.5"
  },
  "orig_nbformat": 4,
  "vscode": {
   "interpreter": {
    "hash": "31f2aee4e71d21fbe5cf8b01ff0e069b9275f58929596ceb00d14d90e3e16cd6"
   }
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
